
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 01 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 01 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="vtt1k6ph";>1</th>
      <td>Liu_W_2015</td>
      <td>State-of-the-art methods for inverse design of an enclosed environment</td>
      <td>Liu, Wei; Zhang, Tengfei; Xue, Yu; Zhai, Zhiqiang (John); Wang, Jihong; Wei, Yun; Chen, Qingyan</td>
      <td>2015</td>
      <td>2015-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127361" target="_blank">PMC7127361</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2015.02.041" target="_blank">10.1016/j.buildenv.2015.02.041</a></td>
      <td>vtt1k6ph</td>
      <td>0.993923</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="lpe49hcj";>2</th>
      <td>Wang_2014</td>
      <td>Identification of indoor contaminant source location by a single concentration sensor</td>
      <td>Wang, Lin-Lin; You, Xue-Yi</td>
      <td>2014</td>
      <td>2014-07-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089341" target="_blank">PMC7089341</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11869-014-0280-9" target="_blank">10.1007/s11869-014-0280-9</a></td>
      <td>lpe49hcj</td>
      <td>0.937389</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_01.html#t54c2si1">Bastani_2012</a></td>
    </tr>
    <tr>
      <th id="odupmsen";>3</th>
      <td>Krantz_2020</td>
      <td>True epidemic growth construction through harmonic analysis</td>
      <td>Krantz, Steven G.; Polyakov, Peter; Rao, Arni S.R. Srinivasa</td>
      <td>2020</td>
      <td>2020-06-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118632" target="_blank">PMC7118632</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32173304.0" target="_blank">32173304.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2020.110243" target="_blank">10.1016/j.jtbi.2020.110243</a></td>
      <td>odupmsen</td>
      <td>0.873733</td>
      <td><a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4ezebg45";>4</th>
      <td>Hui_E_2016</td>
      <td>Capture the abrupt changes in Asian residential property markets</td>
      <td>Hui, Eddie C.M.; Liang, Cong; Zhong, Jiawei; Ip, Wai-Cheung</td>
      <td>2016</td>
      <td>2016-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115552" target="_blank">PMC7115552</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.habitatint.2016.06.005" target="_blank">10.1016/j.habitatint.2016.06.005</a></td>
      <td>4ezebg45</td>
      <td>0.837537</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ufuin9r4";>5</th>
      <td>Ren_H_2017</td>
      <td>Anomaly detection based on a dynamic Markov model</td>
      <td>Ren, Huorong; Ye, Zhixing; Li, Zhiwu</td>
      <td>2017</td>
      <td>2017-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094635" target="_blank">PMC7094635</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ins.2017.05.021" target="_blank">10.1016/j.ins.2017.05.021</a></td>
      <td>ufuin9r4</td>
      <td>0.785870</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="lrcxwtx7";>6</th>
      <td>Peng_2018</td>
      <td>Algorithms for interval-valued fuzzy soft sets in emergency decision making based on WDBA and CODAS with new information measure</td>
      <td>Peng, Xindong; Garg, Harish</td>
      <td>2018</td>
      <td>2018-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7116931" target="_blank">PMC7116931</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cie.2018.04.001" target="_blank">10.1016/j.cie.2018.04.001</a></td>
      <td>lrcxwtx7</td>
      <td>0.765297</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_12.html#5q40uttg">Liu_Y_2014</a>, <a href="Topic_03.html#x1zq2i9h">Al-qaness_2020</a>, <a href="Topic_06.html#mxdlii77">Arji_2019</a></td>
    </tr>
    <tr>
      <th id="ycbyx6vu";>7</th>
      <td>McCrae_2008</td>
      <td>Synonym set extraction from the biomedical literature by lexical pattern discovery</td>
      <td>McCrae, John; Collier, Nigel</td>
      <td>2008</td>
      <td>2008-03-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2335115" target="_blank">PMC2335115</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18366721.0" target="_blank">18366721.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-159" target="_blank">10.1186/1471-2105-9-159</a></td>
      <td>ycbyx6vu</td>
      <td>0.757609</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_01.html#ak33vta6">Du_X_2019</a>, <a href="Topic_01.html#abrweljg">Yu_A_2011</a>, <a href="Topic_02.html#hgxhyg46">Greenbaum_2009</a>, <a href="Topic_02.html#uwbmox8a">Robson_2016</a></td>
    </tr>
    <tr>
      <th id="jsyuh368";>8</th>
      <td>Chu_H_2008</td>
      <td>A Delphi-based approach to developing expert systems with the cooperation of multiple experts</td>
      <td>Chu, Hui-Chun; Hwang, Gwo-Jen</td>
      <td>2008</td>
      <td>2008-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127119" target="_blank">PMC7127119</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2007.05.034" target="_blank">10.1016/j.eswa.2007.05.034</a></td>
      <td>jsyuh368</td>
      <td>0.752960</td>
      <td></td>
      <td><a href="Topic_09.html#e2coy5ms">Matsuzaw_2011</a></td>
    </tr>
    <tr>
      <th id="raycnjbl";>9</th>
      <td>Fontanini_2015</td>
      <td>Constructing Markov matrices for real-time transient contaminant transport analysis for indoor environments</td>
      <td>Fontanini, Anthony D.; Vaidya, Umesh; Ganapathysubramanian, Baskar</td>
      <td>2015</td>
      <td>2015-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125716" target="_blank">PMC7125716</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2015.07.020" target="_blank">10.1016/j.buildenv.2015.07.020</a></td>
      <td>raycnjbl</td>
      <td>0.733256</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="thtsulcy";>10</th>
      <td>Catanzaro_2013</td>
      <td>A mixed integer linear programming model to reconstruct phylogenies from single nucleotide polymorphism haplotypes under the maximum parsimony criterion</td>
      <td>Catanzaro, Daniele; Ravi, Ramamoorthi; Schwartz, Russell</td>
      <td>2013</td>
      <td>2013-01-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3599976" target="_blank">PMC3599976</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23343437.0" target="_blank">23343437.0</a></td>
      <td><a href="https://doi.org/10.1186/1748-7188-8-3" target="_blank">10.1186/1748-7188-8-3</a></td>
      <td>thtsulcy</td>
      <td>0.712286</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="1ul9zqhb";>11</th>
      <td>Mousavi_2012</td>
      <td>An enhanced beam search algorithm for the Shortest Common Supersequence Problem</td>
      <td>Mousavi, Sayyed Rasoul; Bahri, Fateme; Tabataba, Farzaneh Sadat</td>
      <td>2012</td>
      <td>2012-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127599" target="_blank">PMC7127599</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.engappai.2011.08.006" target="_blank">10.1016/j.engappai.2011.08.006</a></td>
      <td>1ul9zqhb</td>
      <td>0.708536</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_01.html#j51gaesp">Amiri_2018</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a>, <a href="Topic_10.html#jxgd1yc4">Kai_Z_2018</a>, <a href="Topic_01.html#z1xao2ic">Masoudi-Sobhanzadeh_2016</a></td>
    </tr>
    <tr>
      <th id="4mad9xws";>12</th>
      <td>Ming_2014</td>
      <td>Multistep-Ahead Air Passengers Traffic Prediction with Hybrid ARIMA-SVMs Models</td>
      <td>Ming, Wei; Bao, Yukun; Hu, Zhongyi; Xiong, Tao</td>
      <td>2014</td>
      <td>2014-02-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3958729" target="_blank">PMC3958729</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24723814.0" target="_blank">24723814.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/567246" target="_blank">10.1155/2014/567246</a></td>
      <td>4mad9xws</td>
      <td>0.687721</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="abrweljg";>13</th>
      <td>Yu_A_2011</td>
      <td>A comparison of two methods for retrieving ICD-9-CM data: The effect of using an ontology-based method for handling terminology changes</td>
      <td>Yu, Alexander C.; Cimino, James J.</td>
      <td>2011</td>
      <td>2011-04-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3440000" target="_blank">PMC3440000</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21262390.0" target="_blank">21262390.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbi.2011.01.005" target="_blank">10.1016/j.jbi.2011.01.005</a></td>
      <td>abrweljg</td>
      <td>0.671955</td>
      <td></td>
      <td><a href="Topic_01.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_01.html#ak33vta6">Du_X_2019</a>, <a href="Topic_02.html#uwbmox8a">Robson_2016</a></td>
    </tr>
    <tr>
      <th id="nexylnv4";>14</th>
      <td>Abdulrahman_2020</td>
      <td>SimCOVID: An Open-Source Simulink-Based Program for Simulating the COVID-19 Epidemic</td>
      <td>Ismael Khorshed Abdulrahman</td>
      <td>2020</td>
      <td>2020-04-17</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.13.20063354" target="_blank">10.1101/2020.04.13.20063354</a></td>
      <td>nexylnv4</td>
      <td>0.666028</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="jn3aqdck";>15</th>
      <td>Rizzi_2010</td>
      <td>Hierarchical Clustering Using the Arithmetic-Harmonic Cut: Complexity and Experiments</td>
      <td>Rizzi, Romeo; Mahata, Pritha; Mathieson, Luke; Moscato, Pablo</td>
      <td>2010</td>
      <td>2010-12-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2997101" target="_blank">PMC2997101</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21151943.0" target="_blank">21151943.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0014067" target="_blank">10.1371/journal.pone.0014067</a></td>
      <td>jn3aqdck</td>
      <td>0.659443</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_05.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_10.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_05.html#7u5frg8w">Zhao_2011</a>, <a href="Topic_04.html#fsh3gsvd">Xiao_2005</a></td>
    </tr>
    <tr>
      <th id="gi8c1omv";>16</th>
      <td>Capotorti_2006</td>
      <td>Management of uncertainty orderings through ASP</td>
      <td>Capotorti, Andrea; Formisano, Andrea</td>
      <td>2006</td>
      <td>2006-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7151761" target="_blank">PMC7151761</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-044452075-3/50012-1" target="_blank">10.1016/b978-044452075-3/50012-1</a></td>
      <td>gi8c1omv</td>
      <td>0.657845</td>
      <td></td>
      <td><a href="Topic_09.html#elf16dno">Jiang_2009</a>, <a href="Topic_09.html#6dcghdxf">Jiang_2009</a>, <a href="Topic_01.html#jsyuh368">Chu_H_2008</a></td>
    </tr>
    <tr>
      <th id="vxg6b6d0";>17</th>
      <td>Obadia_2012</td>
      <td>The R0 package: a toolbox to estimate reproduction numbers for epidemic outbreaks</td>
      <td>Obadia, Thomas; Haneef, Romana; Boëlle, Pierre-Yves</td>
      <td>2012</td>
      <td>2012-12-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3582628" target="_blank">PMC3582628</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23249562.0" target="_blank">23249562.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6947-12-147" target="_blank">10.1186/1472-6947-12-147</a></td>
      <td>vxg6b6d0</td>
      <td>0.653589</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="it5pj8ub";>18</th>
      <td>Feng_2017</td>
      <td>Warehouse capacity sharing via transshipment for an integrated two-echelon supply chain</td>
      <td>Feng, Xuehao; Moon, Ilkyeong; Ryu, Kwangyeol</td>
      <td>2017</td>
      <td>2017-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.tre.2017.04.014" target="_blank">10.1016/j.tre.2017.04.014</a></td>
      <td>it5pj8ub</td>
      <td>0.648964</td>
      <td></td>
      <td><a href="Topic_01.html#j51gaesp">Amiri_2018</a></td>
    </tr>
    <tr>
      <th id="vgq2nt3q";>19</th>
      <td>Fontanini_2016</td>
      <td>A methodology for optimal placement of sensors in enclosed environments: A dynamical systems approach</td>
      <td>Fontanini, Anthony D.; Vaidya, Umesh; Ganapathysubramanian, Baskar</td>
      <td>2016</td>
      <td>2016-05-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126557" target="_blank">PMC7126557</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2016.02.003" target="_blank">10.1016/j.buildenv.2016.02.003</a></td>
      <td>vgq2nt3q</td>
      <td>0.631748</td>
      <td></td>
      <td><a href="Topic_01.html#t54c2si1">Bastani_2012</a></td>
    </tr>
    <tr>
      <th id="t54c2si1";>20</th>
      <td>Bastani_2012</td>
      <td>Contaminant source identification within a building: Toward design of immune buildings</td>
      <td>Bastani, Arash; Haghighat, Fariborz; Kozinski, Janusz A.</td>
      <td>2012</td>
      <td>2012-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127283" target="_blank">PMC7127283</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2011.12.002" target="_blank">10.1016/j.buildenv.2011.12.002</a></td>
      <td>t54c2si1</td>
      <td>0.630203</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="u2mt3xzc";>21</th>
      <td>smith_2019</td>
      <td>Kin and group selection are both flawed but useful data analysis tools</td>
      <td>jeff smith; R. Fredrik Inglis</td>
      <td>2019</td>
      <td>2019-08-29</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/742122" target="_blank">10.1101/742122</a></td>
      <td>u2mt3xzc</td>
      <td>0.629966</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="c674xxbo";>22</th>
      <td>Farina_2014</td>
      <td>Chapter 22 Integrated Sensor Systems and Data Fusion for Homeland Protection</td>
      <td>Farina, Alfonso; Ortenzi, Luciana; Ristic, Branko; Skvortsov, Alex</td>
      <td>2014</td>
      <td>2014-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7150256" target="_blank">PMC7150256</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-396500-4.00022-3" target="_blank">10.1016/b978-0-12-396500-4.00022-3</a></td>
      <td>c674xxbo</td>
      <td>0.626475</td>
      <td></td>
      <td><a href="Topic_01.html#vgq2nt3q">Fontanini_2016</a></td>
    </tr>
    <tr>
      <th id="z1xao2ic";>23</th>
      <td>Masoudi-Sobhanzadeh_2016</td>
      <td>World Competitive Contests (WCC) algorithm: A novel intelligent optimization algorithm for biological and non-biological problems</td>
      <td>Masoudi-Sobhanzadeh, Yosef; Motieghader, Habib</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.imu.2016.06.002" target="_blank">10.1016/j.imu.2016.06.002</a></td>
      <td>z1xao2ic</td>
      <td>0.619170</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_01.html#1ul9zqhb">Mousavi_2012</a>, <a href="Topic_01.html#j51gaesp">Amiri_2018</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="ip0fxvhp";>24</th>
      <td>Schuck_2009</td>
      <td>On computational approaches for size-and-shape distributions from sedimentation velocity analytical ultracentrifugation</td>
      <td>Schuck, Peter</td>
      <td>2009</td>
      <td>2009-10-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892069" target="_blank">PMC2892069</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19806353.0" target="_blank">19806353.0</a></td>
      <td><a href="https://doi.org/10.1007/s00249-009-0545-7" target="_blank">10.1007/s00249-009-0545-7</a></td>
      <td>ip0fxvhp</td>
      <td>0.606090</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_07.html#cw2opick">Chou_2011</a>, <a href="Topic_14.html#wvw6q7px">Jonić_2016</a></td>
    </tr>
    <tr>
      <th id="ak33vta6";>25</th>
      <td>Du_X_2019</td>
      <td>Language model-based automatic prefix abbreviation expansion method for biomedical big data analysis</td>
      <td>Du, Xiaokun; Zhu, Rongbo; Li, Yanhong; Anjum, Ashiq</td>
      <td>2019</td>
      <td>2019-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103364" target="_blank">PMC7103364</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.future.2019.01.016" target="_blank">10.1016/j.future.2019.01.016</a></td>
      <td>ak33vta6</td>
      <td>0.604116</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_01.html#abrweljg">Yu_A_2011</a></td>
    </tr>
    <tr>
      <th id="j3jskkmw";>26</th>
      <td>Shao_2017</td>
      <td>Long-term prediction of dynamic distribution of passive contaminant in complex recirculating ventilation system</td>
      <td>Shao, Xiaoliang; Li, Xianting; Ma, Xiaojun; Zhu, Fenfei</td>
      <td>2017</td>
      <td>2017-08-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126526" target="_blank">PMC7126526</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2017.05.012" target="_blank">10.1016/j.buildenv.2017.05.012</a></td>
      <td>j3jskkmw</td>
      <td>0.602193</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="gmw88elf";>27</th>
      <td>Vella_2017</td>
      <td>From protein-protein interactions to protein co-expression networks: a new perspective to evaluate large-scale proteomic data</td>
      <td>Vella, Danila; Zoppis, Italo; Mauri, Giancarlo; Mauri, Pierluigi; Di Silvestre, Dario</td>
      <td>2017</td>
      <td>2017-03-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5359264" target="_blank">PMC5359264</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28477207.0" target="_blank">28477207.0</a></td>
      <td><a href="https://doi.org/10.1186/s13637-017-0059-z" target="_blank">10.1186/s13637-017-0059-z</a></td>
      <td>gmw88elf</td>
      <td>0.590526</td>
      <td></td>
      <td><a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_13.html#g55witln">Gibbs_2013</a></td>
    </tr>
    <tr>
      <th id="397hp1yt";>28</th>
      <td>Coelho_2008</td>
      <td>Epigrass: a tool to study disease spread in complex networks</td>
      <td>Coelho, Flávio C; Cruz, Oswaldo G; Codeço, Cláudia T</td>
      <td>2008</td>
      <td>2008-02-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2275240" target="_blank">PMC2275240</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18302744.0" target="_blank">18302744.0</a></td>
      <td><a href="https://doi.org/10.1186/1751-0473-3-3" target="_blank">10.1186/1751-0473-3-3</a></td>
      <td>397hp1yt</td>
      <td>0.590016</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="in8mbp7c";>29</th>
      <td>Li_W_2011</td>
      <td>Cross-Crystal Averaging with Search Models to Improve Molecular Replacement Phases</td>
      <td>Li, Weikai; Li, Fang</td>
      <td>2011</td>
      <td>2011-02-09</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037595" target="_blank">PMC3037595</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21300285.0" target="_blank">21300285.0</a></td>
      <td><a href="https://doi.org/10.1016/j.str.2010.12.007" target="_blank">10.1016/j.str.2010.12.007</a></td>
      <td>in8mbp7c</td>
      <td>0.586243</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_15.html#qu4enz7m">Frenz_2019</a></td>
    </tr>
    <tr>
      <th id="ygr1kv9r";>30</th>
      <td>Hu_P_2007</td>
      <td>A Web-Based System for Infectious Disease Data Integration and Sharing: Evaluating Outcome, Task Performance Efficiency, User Information Satisfaction, and Usability</td>
      <td>Hu, Paul Jen-Hwa; Zeng, Daniel; Chen, Hsinchun; Larson, Catherine A.; Tseng, Chunju</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122142" target="_blank">PMC7122142</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72608-1_13" target="_blank">10.1007/978-3-540-72608-1_13</a></td>
      <td>ygr1kv9r</td>
      <td>0.583084</td>
      <td></td>
      <td><a href="Topic_06.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_06.html#jx8wulcg">Seo_D_2017</a></td>
    </tr>
    <tr>
      <th id="ejaditj5";>31</th>
      <td>Maday_2020</td>
      <td>An adaptive parareal algorithm</td>
      <td>Maday, Y.; Mula, O.</td>
      <td>2020</td>
      <td>2020-04-09</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cam.2020.112915" target="_blank">10.1016/j.cam.2020.112915</a></td>
      <td>ejaditj5</td>
      <td>0.578949</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_14.html#3u81ntle">Acar_2016</a></td>
    </tr>
    <tr>
      <th id="pdf3i2tt";>32</th>
      <td>Wu_Y_2020</td>
      <td>NetNCSP: Nonoverlapping closed sequential pattern mining</td>
      <td>Wu, Youxi; Zhu, Changrui; Li, Yan; Guo, Lei; Wu, Xindong</td>
      <td>2020</td>
      <td>2020-03-31</td>
      <td>PMC</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2020.105812" target="_blank">10.1016/j.knosys.2020.105812</a></td>
      <td>pdf3i2tt</td>
      <td>0.575645</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_08.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_01.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="qrnhp1ek";>33</th>
      <td>Kirillova_2009</td>
      <td>Protein Domain Boundary Predictions: A Structural Biology Perspective</td>
      <td>Kirillova, Svetlana; Kumar, Suresh; Carugo, Oliviero</td>
      <td>2009</td>
      <td>2009-01-21</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2669640" target="_blank">PMC2669640</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19401756.0" target="_blank">19401756.0</a></td>
      <td><a href="https://doi.org/10.2174/1874091x00903010001" target="_blank">10.2174/1874091x00903010001</a></td>
      <td>qrnhp1ek</td>
      <td>0.571921</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="7zgmiwbd";>34</th>
      <td>Lam_B_2016</td>
      <td>Jump point detection using empirical mode decomposition</td>
      <td>Lam, Benson S.Y.; Yu, Carisa K.W.; Choy, Siu-Kai; Leung, Jacky K.T.</td>
      <td>2016</td>
      <td>2016-12-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115718" target="_blank">PMC7115718</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.landusepol.2016.07.006" target="_blank">10.1016/j.landusepol.2016.07.006</a></td>
      <td>7zgmiwbd</td>
      <td>0.569993</td>
      <td></td>
      <td><a href="Topic_01.html#4ezebg45">Hui_E_2016</a></td>
    </tr>
    <tr>
      <th id="5wtrij3m";>35</th>
      <td>Zhou_2018</td>
      <td>Early detection of dynamic harmful cascades in large-scale networks</td>
      <td>Zhou, Chuan; Lu, Wei-Xue; Zhang, Jingzun; Li, Lei; Hu, Yue; Guo, Li</td>
      <td>2018</td>
      <td>2018-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102699" target="_blank">PMC7102699</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jocs.2017.10.014" target="_blank">10.1016/j.jocs.2017.10.014</a></td>
      <td>5wtrij3m</td>
      <td>0.568671</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="uv7g3liw";>36</th>
      <td>Heine_2011</td>
      <td>Statistical learning techniques applied to epidemiology: a simulated case-control comparison study with logistic regression</td>
      <td>Heine, John J; Land, Walker H; Egan, Kathleen M</td>
      <td>2011</td>
      <td>2011-01-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045299" target="_blank">PMC3045299</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21272346.0" target="_blank">21272346.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-37" target="_blank">10.1186/1471-2105-12-37</a></td>
      <td>uv7g3liw</td>
      <td>0.568198</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#auasueaw">Kenah_2015</a></td>
    </tr>
    <tr>
      <th id="cxzlmfst";>37</th>
      <td>Malanoski_2006</td>
      <td>Automated identification of multiple micro-organisms from resequencing DNA microarrays</td>
      <td>Malanoski, Anthony P.; Lin, Baochuan; Wang, Zheng; Schnur, Joel M.; Stenger, David A.</td>
      <td>2006</td>
      <td>2006-09-29</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1636417" target="_blank">PMC1636417</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17012284.0" target="_blank">17012284.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl565" target="_blank">10.1093/nar/gkl565</a></td>
      <td>cxzlmfst</td>
      <td>0.567529</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4qgjglx3";>38</th>
      <td>Tian_2007</td>
      <td>Epidemic Alert & Response Framework and Technology Based on Spreading Dynamics Simulation</td>
      <td>Tian, Chunhua; Zhang, Xun; Ding, Wei; Cao, Rongzeng</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120482" target="_blank">PMC7120482</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72588-6_166" target="_blank">10.1007/978-3-540-72588-6_166</a></td>
      <td>4qgjglx3</td>
      <td>0.562627</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="zr0bo9el";>39</th>
      <td>Pfannschmidt_2016</td>
      <td>Evaluating Tests in Medical Diagnosis: Combining Machine Learning with Game-Theoretical Concepts</td>
      <td>Pfannschmidt, Karlson; Hüllermeier, Eyke; Held, Susanne; Neiger, Reto</td>
      <td>2016</td>
      <td>2016-05-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121901" target="_blank">PMC7121901</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-40596-4_38" target="_blank">10.1007/978-3-319-40596-4_38</a></td>
      <td>zr0bo9el</td>
      <td>0.557871</td>
      <td></td>
      <td><a href="Topic_03.html#1r4gm2d4">Liu_Y_2020</a>, <a href="Topic_07.html#oeokiay7">Fusani_2018</a></td>
    </tr>
    <tr>
      <th id="61va8q92";>40</th>
      <td>Niethammer_2013</td>
      <td>Segmentation with area constraints</td>
      <td>Niethammer, Marc; Zach, Christopher</td>
      <td>2013</td>
      <td>2013-01-31</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3656501" target="_blank">PMC3656501</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23084504.0" target="_blank">23084504.0</a></td>
      <td><a href="https://doi.org/10.1016/j.media.2012.09.002" target="_blank">10.1016/j.media.2012.09.002</a></td>
      <td>61va8q92</td>
      <td>0.555461</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_14.html#3u81ntle">Acar_2016</a>, <a href="Topic_15.html#bqwmx5dy">Punjani_2019</a>, <a href="Topic_01.html#89xnnvuv">Saad_2005</a>, <a href="Topic_01.html#yxo8oanc">Punjani_2020</a></td>
    </tr>
    <tr>
      <th id="j51gaesp";>41</th>
      <td>Amiri_2018</td>
      <td>Time-constrained maximal covering routing problem</td>
      <td>Amiri, Afsaneh; Salari, Majid</td>
      <td>2018</td>
      <td>2018-12-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079968" target="_blank">PMC7079968</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s00291-018-0541-3" target="_blank">10.1007/s00291-018-0541-3</a></td>
      <td>j51gaesp</td>
      <td>0.552414</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#1ul9zqhb">Mousavi_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_01.html#it5pj8ub">Feng_2017</a>, <a href="Topic_01.html#1ul9zqhb">Mousavi_2012</a></td>
    </tr>
    <tr>
      <th id="qy4kirx1";>42</th>
      <td>Mendizabal-Ruiz_2018</td>
      <td>Genomic signal processing for DNA sequence clustering</td>
      <td>Mendizabal-Ruiz, Gerardo; Román-Godínez, Israel; Torres-Ramos, Sulema; Salido-Ruiz, Ricardo A.; Vélez-Pérez, Hugo; Morales, J. Alejandro</td>
      <td>2018</td>
      <td>2018-01-24</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5786891" target="_blank">PMC5786891</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29379686.0" target="_blank">29379686.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.4264" target="_blank">10.7717/peerj.4264</a></td>
      <td>qy4kirx1</td>
      <td>0.544988</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_05.html#7u5frg8w">Zhao_2011</a>, <a href="Topic_05.html#nw3tybw9">Hoang_2015</a>, <a href="Topic_05.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_04.html#ut9z9eo7">Mu_Z_2019</a></td>
    </tr>
    <tr>
      <th id="gc9hevy5";>43</th>
      <td>Cho_Y_2005</td>
      <td>Cleavage Site Analysis Using Rule Extraction from Neural Networks</td>
      <td>Cho, Yeun-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114972" target="_blank">PMC7114972</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11539087_132" target="_blank">10.1007/11539087_132</a></td>
      <td>gc9hevy5</td>
      <td>0.544313</td>
      <td><a href="Topic_01.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_01.html#my2l85bn">Cho_Y_2005</a></td>
    </tr>
    <tr>
      <th id="m56agj4z";>44</th>
      <td>Reddy_2007</td>
      <td>Positional clustering improves computational binding site detection and identifies novel cis-regulatory sites in mammalian GABA(A) receptor subunit genes</td>
      <td>Reddy, Timothy E.; Shakhnovich, Boris E.; Roberts, Daniel S.; Russek, Shelley J.; DeLisi, Charles</td>
      <td>2007</td>
      <td>2007-01-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1807961" target="_blank">PMC1807961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17204484.0" target="_blank">17204484.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl1062" target="_blank">10.1093/nar/gkl1062</a></td>
      <td>m56agj4z</td>
      <td>0.540214</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="89xnnvuv";>45</th>
      <td>Saad_2005</td>
      <td>Orientation determination by wavelets matching for 3D reconstruction of very noisy electron microscopic virus images</td>
      <td>Saad, Ali Samir</td>
      <td>2005</td>
      <td>2005-03-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC555555" target="_blank">PMC555555</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15743519.0" target="_blank">15743519.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6807-5-5" target="_blank">10.1186/1472-6807-5-5</a></td>
      <td>89xnnvuv</td>
      <td>0.527225</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_14.html#3u81ntle">Acar_2016</a></td>
    </tr>
    <tr>
      <th id="my2l85bn";>46</th>
      <td>Cho_Y_2005</td>
      <td>Rule Generation Using NN and GA for SARS-CoV Cleavage Site Prediction</td>
      <td>Cho, Yeon-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122303" target="_blank">PMC7122303</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11553939_111" target="_blank">10.1007/11553939_111</a></td>
      <td>my2l85bn</td>
      <td>0.526724</td>
      <td><a href="Topic_01.html#gc9hevy5">Cho_Y_2005</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ear9cyri";>47</th>
      <td>Bakker_2018</td>
      <td>Dynamic graphs, community detection, and Riemannian geometry</td>
      <td>Bakker, Craig; Halappanavar, Mahantesh; Visweswara Sathanur, Arun</td>
      <td>2018</td>
      <td>2018-03-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6214282" target="_blank">PMC6214282</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30839776.0" target="_blank">30839776.0</a></td>
      <td><a href="https://doi.org/10.1007/s41109-018-0059-2" target="_blank">10.1007/s41109-018-0059-2</a></td>
      <td>ear9cyri</td>
      <td>0.525961</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_01.html#397hp1yt">Coelho_2008</a></td>
    </tr>
    <tr>
      <th id="nu0pn2q8";>48</th>
      <td>Ardabili_2020</td>
      <td>COVID-19 Outbreak Prediction with Machine Learning</td>
      <td>Sina F. Ardabili; Amir MOSAVI; Pedram Ghamisi; Filip Ferdinand; Annamaria R. Varkonyi-Koczy; Uwe Reuter; Timon Rabczuk; Peter M. Atkinson</td>
      <td>2020</td>
      <td>2020-04-22</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.17.20070094" target="_blank">10.1101/2020.04.17.20070094</a></td>
      <td>nu0pn2q8</td>
      <td>0.524670</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="b13p3mca";>49</th>
      <td>Prosperi_2011</td>
      <td>A novel methodology for large-scale phylogeny partition</td>
      <td>Prosperi, Mattia C.F.; Ciccozzi, Massimo; Fanti, luri; Saladini, Francesco; Pecorari, Monica; Borghi, Vanni; Di Giambenedetto, Simona; Bruzzone, Bianca; Capetti, Amedeo; Vivarelli, Angela; Rusconi, Stefano; Carla Re, Maria; Gismondo, Maria Rita; Sighinolfi, Laura; Gray, Rebecca R.; Salemi, Marco; Zazzi, Maurizio; De Luca, Andrea</td>
      <td>2011</td>
      <td>2011-05-24</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6045912" target="_blank">PMC6045912</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21610724.0" target="_blank">21610724.0</a></td>
      <td><a href="https://doi.org/10.1038/ncomms1325" target="_blank">10.1038/ncomms1325</a></td>
      <td>b13p3mca</td>
      <td>0.524345</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4y5mnok8";>50</th>
      <td>Ehrt_2018</td>
      <td>A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs)</td>
      <td>Ehrt, Christiane; Brinkjost, Tobias; Koch, Oliver</td>
      <td>2018</td>
      <td>2018-11-08</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224041" target="_blank">PMC6224041</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30408032.0" target="_blank">30408032.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1006483" target="_blank">10.1371/journal.pcbi.1006483</a></td>
      <td>4y5mnok8</td>
      <td>0.520162</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#d66rz6dc">Webb_2014</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="j22q89l9";>51</th>
      <td>Vanderwende_2007</td>
      <td>Beyond SumBasic: Task-focused summarization with sentence simplification and lexical expansion</td>
      <td>Vanderwende, Lucy; Suzuki, Hisami; Brockett, Chris; Nenkova, Ani</td>
      <td>2007</td>
      <td>2007-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115785" target="_blank">PMC7115785</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ipm.2007.01.023" target="_blank">10.1016/j.ipm.2007.01.023</a></td>
      <td>j22q89l9</td>
      <td>0.519358</td>
      <td></td>
      <td><a href="Topic_01.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_02.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="geiq55mq";>52</th>
      <td>Xiong_2016</td>
      <td>Weighted multifractal cross-correlation analysis based on Shannon entropy</td>
      <td>Xiong, Hui; Shang, Pengjian</td>
      <td>2016</td>
      <td>2016-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128505" target="_blank">PMC7128505</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cnsns.2015.06.029" target="_blank">10.1016/j.cnsns.2015.06.029</a></td>
      <td>geiq55mq</td>
      <td>0.519358</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_13.html#a2m46j0z">Tian_2016</a></td>
    </tr>
    <tr>
      <th id="7ou1uqis";>53</th>
      <td>Bonham_2009</td>
      <td>Modeling tourism: A fully identified VECM approach</td>
      <td>Bonham, Carl; Gangnes, Byron; Zhou, Ting</td>
      <td>2009</td>
      <td>2009-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114862" target="_blank">PMC7114862</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ijforecast.2008.11.014" target="_blank">10.1016/j.ijforecast.2008.11.014</a></td>
      <td>7ou1uqis</td>
      <td>0.518602</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="4wnoa4l7";>54</th>
      <td>Cai_Y_2019</td>
      <td>Temporal event searches based on event maps and relationships</td>
      <td>Cai, Yi; Xie, Haoran; Lau, Raymond Y.K.; Li, Qing; Wong, Tak-Lam; Wang, Fu Lee</td>
      <td>2019</td>
      <td>2019-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7105191" target="_blank">PMC7105191</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.asoc.2019.105750" target="_blank">10.1016/j.asoc.2019.105750</a></td>
      <td>4wnoa4l7</td>
      <td>0.517195</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_01.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_09.html#y0s4cf6g">Zou_Z_2013</a>, <a href="Topic_06.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_08.html#a0bbw3er">Waagmeester_2020</a></td>
    </tr>
    <tr>
      <th id="a8w5cs6k";>55</th>
      <td>Lau_M_2017</td>
      <td>A mechanistic spatio-temporal framework for modelling individual-to-individual transmission—With an application to the 2014-2015 West Africa Ebola outbreak</td>
      <td>Lau, Max S. Y.; Gibson, Gavin J.; Adrakey, Hola; McClelland, Amanda; Riley, Steven; Zelner, Jon; Streftaris, George; Funk, Sebastian; Metcalf, Jessica; Dalziel, Benjamin D.; Grenfell, Bryan T.</td>
      <td>2017</td>
      <td>2017-10-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5679647" target="_blank">PMC5679647</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29084216.0" target="_blank">29084216.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1005798" target="_blank">10.1371/journal.pcbi.1005798</a></td>
      <td>a8w5cs6k</td>
      <td>0.514727</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="hmx2wy1p";>56</th>
      <td>Cope_2020</td>
      <td>Identification of the relative timing of infectiousness and symptom onset for outbreak control</td>
      <td>Cope, Robert C.; Ross, Joshua V.</td>
      <td>2020</td>
      <td>2020-02-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094159" target="_blank">PMC7094159</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31734243.0" target="_blank">31734243.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2019.110079" target="_blank">10.1016/j.jtbi.2019.110079</a></td>
      <td>hmx2wy1p</td>
      <td>0.512947</td>
      <td><a href="Topic_06.html#8r0vfzeu">Cope_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_06.html#8r0vfzeu">Cope_2019</a></td>
    </tr>
    <tr>
      <th id="uh5rkjwx";>57</th>
      <td>González-Díaz_2014</td>
      <td>Model for Vaccine Design by Prediction of B-Epitopes of IEDB Given Perturbations in Peptide Sequence, In Vivo Process, Experimental Techniques, and Source or Host Organisms</td>
      <td>González-Díaz, Humberto; Pérez-Montoto, Lázaro G.; Ubeira, Florencio M.</td>
      <td>2014</td>
      <td>2014-01-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3987976" target="_blank">PMC3987976</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24741624.0" target="_blank">24741624.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/768515" target="_blank">10.1155/2014/768515</a></td>
      <td>uh5rkjwx</td>
      <td>0.511981</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="j1p4nmsa";>58</th>
      <td>Fenga_2020</td>
      <td>CoViD19 Meta heuristic optimization based forecast method on time dependent bootstrapped data</td>
      <td>Livio Fenga; Carlo Del Castello</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.02.20050153" target="_blank">10.1101/2020.04.02.20050153</a></td>
      <td>j1p4nmsa</td>
      <td>0.495896</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="s96d3ust";>59</th>
      <td>Hu_Y_2006</td>
      <td>Mining association rules with multiple minimum supports: a new mining algorithm and a support tuning mechanism</td>
      <td>Hu, Ya-Han; Chen, Yen-Liang</td>
      <td>2006</td>
      <td>2006-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127670" target="_blank">PMC7127670</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.dss.2004.09.007" target="_blank">10.1016/j.dss.2004.09.007</a></td>
      <td>s96d3ust</td>
      <td>0.493618</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_08.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_01.html#pdf3i2tt">Wu_Y_2020</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="ym9j0ei0";>60</th>
      <td>Tao_J_2005</td>
      <td>Probabilistic Reasoning for Closed-Room People Monitoring</td>
      <td>Tao, Ji; Tan, Yap-Peng</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123562" target="_blank">PMC7123562</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/3-540-32367-8_15" target="_blank">10.1007/3-540-32367-8_15</a></td>
      <td>ym9j0ei0</td>
      <td>0.490157</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_09.html#dslwklo4">Tao_J_2004</a>, <a href="Topic_14.html#flgc5l04">Budzan_2013</a>, <a href="Topic_01.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_01.html#gm1e9j80">Thi_T_2014</a></td>
    </tr>
    <tr>
      <th id="q3vrzlz1";>61</th>
      <td>Weber_2017</td>
      <td>Novel algorithm for accelerated electroanatomic mapping and prediction of earliest activation of focal cardiac arrhythmias using mathematical optimization</td>
      <td>Weber, Tobias; Katus, Hugo A.; Sager, Sebastian; Scholz, Eberhard P.</td>
      <td>2017</td>
      <td>2017-06-23</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5446561" target="_blank">PMC5446561</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28279745.0" target="_blank">28279745.0</a></td>
      <td><a href="https://doi.org/10.1016/j.hrthm.2017.03.001" target="_blank">10.1016/j.hrthm.2017.03.001</a></td>
      <td>q3vrzlz1</td>
      <td>0.489290</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="xailjga5";>62</th>
      <td>Wang_2010</td>
      <td>Comparing early outbreak detection algorithms based on their optimized parameter values</td>
      <td>Wang, Xiaoli; Zeng, Daniel; Seale, Holly; Li, Su; Cheng, He; Luan, Rongsheng; He, Xiong; Pang, Xinghuo; Dou, Xiangfeng; Wang, Quanyi</td>
      <td>2010</td>
      <td>2010-02-28</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jbi.2009.08.003" target="_blank">10.1016/j.jbi.2009.08.003</a></td>
      <td>xailjga5</td>
      <td>0.487908</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="wvw6q7px";>63</th>
      <td>Jonić_2016</td>
      <td>Versatility of Approximating Single-Particle Electron Microscopy Density Maps Using Pseudoatoms and Approximation-Accuracy Control</td>
      <td>Jonić, Slavica; Sorzano, Carlos Oscar S.</td>
      <td>2016</td>
      <td>2016-12-21</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5209604" target="_blank">PMC5209604</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28097146.0" target="_blank">28097146.0</a></td>
      <td><a href="https://doi.org/10.1155/2016/7060348" target="_blank">10.1155/2016/7060348</a></td>
      <td>wvw6q7px</td>
      <td>0.485288</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_14.html#wsgb5s6z">Quintana_1998</a>, <a href="Topic_15.html#bqwmx5dy">Punjani_2019</a>, <a href="Topic_01.html#oohx14kl">Zhong_2020</a></td>
    </tr>
    <tr>
      <th id="lqviqh3m";>64</th>
      <td>Höhl_2007</td>
      <td>Is Multiple-Sequence Alignment Required for Accurate Inference of Phylogeny?</td>
      <td>Höhl, Michael; Ragan, Mark A.</td>
      <td>2007</td>
      <td>2007-04-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107264" target="_blank">PMC7107264</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17454975.0" target="_blank">17454975.0</a></td>
      <td><a href="https://doi.org/10.1080/10635150701294741" target="_blank">10.1080/10635150701294741</a></td>
      <td>lqviqh3m</td>
      <td>0.484053</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="gm1e9j80";>65</th>
      <td>Thi_T_2014</td>
      <td>Recognizing flu-like symptoms from videos</td>
      <td>Thi, Tuan Hue; Wang, Li; Ye, Ning; Zhang, Jian; Maurer-Stroh, Sebastian; Cheng, Li</td>
      <td>2014</td>
      <td>2014-09-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4180141" target="_blank">PMC4180141</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25217118.0" target="_blank">25217118.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-300" target="_blank">10.1186/1471-2105-15-300</a></td>
      <td>gm1e9j80</td>
      <td>0.482615</td>
      <td></td>
      <td><a href="Topic_03.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_01.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_03.html#ilc2bzkx">Zhou_2020</a></td>
    </tr>
    <tr>
      <th id="bkfkmtld";>66</th>
      <td>Redelings_2007</td>
      <td>Incorporating indel information into phylogeny estimation for rapidly emerging pathogens</td>
      <td>Redelings, Benjamin D; Suchard, Marc A</td>
      <td>2007</td>
      <td>2007-03-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1853084" target="_blank">PMC1853084</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17359539.0" target="_blank">17359539.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2148-7-40" target="_blank">10.1186/1471-2148-7-40</a></td>
      <td>bkfkmtld</td>
      <td>0.480826</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="wfn50oqq";>67</th>
      <td>Adoteye_2015</td>
      <td>Optimal design of non-equilibrium experiments for genetic network interrogation</td>
      <td>Adoteye, Kaska; Banks, H.T.; Flores, Kevin B.</td>
      <td>2015</td>
      <td>2015-02-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4281269" target="_blank">PMC4281269</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25558126.0" target="_blank">25558126.0</a></td>
      <td><a href="https://doi.org/10.1016/j.aml.2014.09.013" target="_blank">10.1016/j.aml.2014.09.013</a></td>
      <td>wfn50oqq</td>
      <td>0.479901</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="vhvwxvwq";>68</th>
      <td>Shikder_2019</td>
      <td>An OpenMP-based tool for finding longest common subsequence in bioinformatics</td>
      <td>Shikder, Rayhan; Thulasiraman, Parimala; Irani, Pourang; Hu, Pingzhao</td>
      <td>2019</td>
      <td>2019-04-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6458724" target="_blank">PMC6458724</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30971295.0" target="_blank">30971295.0</a></td>
      <td><a href="https://doi.org/10.1186/s13104-019-4256-6" target="_blank">10.1186/s13104-019-4256-6</a></td>
      <td>vhvwxvwq</td>
      <td>0.477959</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_10.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_05.html#buc4dd5y">Dong_2019</a></td>
    </tr>
    <tr>
      <th id="ehrww6uu";>69</th>
      <td>Bender_2006</td>
      <td>Chapter 9 Molecular Similarity: Advances in Methods, Applications and Validations in Virtual Screening and QSAR</td>
      <td>Bender, Andreas; Jenkins, Jeremy L.; Li, Qingliang; Adams, Sam E.; Cannon, Edward O.; Glen, Robert C.</td>
      <td>2006</td>
      <td>2006-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/s1574-1400(06)02009-3" target="_blank">10.1016/s1574-1400(06)02009-3</a></td>
      <td>ehrww6uu</td>
      <td>0.474780</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_07.html#qah877pk">Ekins_2007</a></td>
    </tr>
    <tr>
      <th id="48jx3de2";>70</th>
      <td>Mohammadzadeh_2017</td>
      <td>A data mining approach for modeling churn behavior via RFM model in specialized clinics Case study: A public sector hospital in Tehran</td>
      <td>Mohammadzadeh, Mehdi; Hoseini, Zeinab Zare; Derafshi, Hamid</td>
      <td>2017</td>
      <td>2017-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2017.11.206" target="_blank">10.1016/j.procs.2017.11.206</a></td>
      <td>48jx3de2</td>
      <td>0.472189</td>
      <td></td>
      <td><a href="Topic_03.html#fridlqhv">AlMoammar_2018</a></td>
    </tr>
    <tr>
      <th id="8r0vfzeu";>71</th>
      <td>Cope_2019</td>
      <td>Identification of the relative timing of infectiousness and symptom onset for outbreak control</td>
      <td>Robert C. Cope; Joshua V. Ross</td>
      <td>2019</td>
      <td>2019-03-08</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/571547" target="_blank">10.1101/571547</a></td>
      <td>8r0vfzeu</td>
      <td>0.471681</td>
      <td><a href="Topic_01.html#hmx2wy1p">Cope_2020</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_01.html#hmx2wy1p">Cope_2020</a></td>
    </tr>
    <tr>
      <th id="735m0znd";>72</th>
      <td>Tien_2008</td>
      <td>Methods for simultaneously identifying coherent local clusters with smooth global patterns in gene expression profiles</td>
      <td>Tien, Yin-Jing; Lee, Yun-Shien; Wu, Han-Ming; Chen, Chun-Houh</td>
      <td>2008</td>
      <td>2008-03-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2322988" target="_blank">PMC2322988</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18366693.0" target="_blank">18366693.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-155" target="_blank">10.1186/1471-2105-9-155</a></td>
      <td>735m0znd</td>
      <td>0.470202</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_05.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_05.html#qi630tpo">Ragonnet-Cronin_2013</a>, <a href="Topic_01.html#hltgn9aj">Gatherer_2007</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="jzpt9ldx";>73</th>
      <td>Ren_R_2016</td>
      <td>The attribute reductions of three-way concept lattices</td>
      <td>Ren, Ruisi; Wei, Ling</td>
      <td>2016</td>
      <td>2016-05-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126604" target="_blank">PMC7126604</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2016.01.045" target="_blank">10.1016/j.knosys.2016.01.045</a></td>
      <td>jzpt9ldx</td>
      <td>0.469499</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_09.html#6zjf6qzz">Li_J_2015</a>, <a href="Topic_13.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_01.html#j22q89l9">Vanderwende_2007</a>, <a href="Topic_01.html#4wnoa4l7">Cai_Y_2019</a></td>
    </tr>
    <tr>
      <th id="3u81ntle";>74</th>
      <td>Acar_2016</td>
      <td>Adaptive multiresolution method for MAP reconstruction in electron tomography</td>
      <td>Acar, Erman; Peltonen, Sari; Ruotsalainen, Ulla</td>
      <td>2016</td>
      <td>2016-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115799" target="_blank">PMC7115799</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27522477.0" target="_blank">27522477.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ultramic.2016.08.002" target="_blank">10.1016/j.ultramic.2016.08.002</a></td>
      <td>3u81ntle</td>
      <td>0.469322</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#89xnnvuv">Saad_2005</a>, <a href="Topic_15.html#bqwmx5dy">Punjani_2019</a></td>
    </tr>
    <tr>
      <th id="ovuofl9q";>75</th>
      <td>Chen_2018</td>
      <td>Probability Sampling by Connecting Space with Households Using GIS/GPS Technologies</td>
      <td>Chen, Xinguang; Hu, Hui; Xu, Xiaohui; Gong, Jie; Yan, Yaqiong; Li, Fang</td>
      <td>2018</td>
      <td>2018-01-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107540" target="_blank">PMC7107540</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1093/jssam/smx032" target="_blank">10.1093/jssam/smx032</a></td>
      <td>ovuofl9q</td>
      <td>0.468388</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="hltgn9aj";>76</th>
      <td>Gatherer_2007</td>
      <td>Genome Signatures, Self-Organizing Maps and Higher Order Phylogenies: A Parametric Analysis</td>
      <td>Gatherer, Derek</td>
      <td>2007</td>
      <td>2007-09-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2684143" target="_blank">PMC2684143</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19468314.0" target="_blank">19468314.0</a></td>
      <td></td>
      <td>hltgn9aj</td>
      <td>0.468271</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_01.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="oohx14kl";>77</th>
      <td>Zhong_2020</td>
      <td>CryoDRGN: Reconstruction of heterogeneous structures from cryo-electron micrographs using neural networks</td>
      <td>Ellen D. Zhong; Tristan Bepler; Bonnie Berger; Joseph H Davis</td>
      <td>2020</td>
      <td>2020-03-29</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.27.003871" target="_blank">10.1101/2020.03.27.003871</a></td>
      <td>oohx14kl</td>
      <td>0.464329</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_01.html#yxo8oanc">Punjani_2020</a>, <a href="Topic_04.html#er6lz09f">Pfab_2020</a></td>
    </tr>
    <tr>
      <th id="635lbedk";>78</th>
      <td>Zhao_2020</td>
      <td>Spreading predictability in complex networks</td>
      <td>Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen</td>
      <td>2020</td>
      <td>2020-01-28</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.28.922757" target="_blank">10.1101/2020.01.28.922757</a></td>
      <td>635lbedk</td>
      <td>0.463344</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="jxgd1yc4";>79</th>
      <td>Kai_Z_2018</td>
      <td>A Novel Efficient Simulated Annealing Algorithm for the RNA Secondary Structure Predicting with Pseudoknots</td>
      <td>Kai, Zhang; Yulin, Lv</td>
      <td>2018</td>
      <td>2018-07-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121944" target="_blank">PMC7121944</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-95933-7_44" target="_blank">10.1007/978-3-319-95933-7_44</a></td>
      <td>jxgd1yc4</td>
      <td>0.462183</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="ff6kor0c";>80</th>
      <td>Holmes_2017</td>
      <td>Solving the master equation for Indels</td>
      <td>Holmes, Ian H.</td>
      <td>2017</td>
      <td>2017-05-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5427538" target="_blank">PMC5427538</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28494756.0" target="_blank">28494756.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-017-1665-1" target="_blank">10.1186/s12859-017-1665-1</a></td>
      <td>ff6kor0c</td>
      <td>0.460455</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_13.html#3atcqejc">Kinene_2016</a>, <a href="Topic_01.html#bkfkmtld">Redelings_2007</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_01.html#bkfkmtld">Redelings_2007</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="486j76e1";>81</th>
      <td>Swindell_2008</td>
      <td>HOW LONG WILL MY MOUSE LIVE? MACHINE LEARNING APPROACHES FOR PREDICTION OF MOUSE LIFESPAN</td>
      <td>Swindell, William R.; Harper, James M.; Miller, Richard A.</td>
      <td>2008</td>
      <td>2008-09-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2693389" target="_blank">PMC2693389</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18840793.0" target="_blank">18840793.0</a></td>
      <td></td>
      <td>486j76e1</td>
      <td>0.459580</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="w8luu86h";>82</th>
      <td>Liou_2013</td>
      <td>Structural Complexity of DNA Sequence</td>
      <td>Liou, Cheng-Yuan; Tseng, Shen-Han; Cheng, Wei-Chen; Tsai, Huai-Ying</td>
      <td>2013</td>
      <td>2013-04-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3638703" target="_blank">PMC3638703</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23662161.0" target="_blank">23662161.0</a></td>
      <td><a href="https://doi.org/10.1155/2013/628036" target="_blank">10.1155/2013/628036</a></td>
      <td>w8luu86h</td>
      <td>0.459060</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_05.html#le7v89mp">Chang_2011</a>, <a href="Topic_10.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="zult69f2";>83</th>
      <td>Nguyen_2019</td>
      <td>GraphDTA: Predicting drug–target binding affinity with graph neural networks</td>
      <td>Thin Nguyen; Hang Le; Thomas P. Quinn; Thuc Le; Svetha Venkatesh</td>
      <td>2019</td>
      <td>2019-06-27</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/684662" target="_blank">10.1101/684662</a></td>
      <td>zult69f2</td>
      <td>0.458233</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="b265mbwp";>84</th>
      <td>Kao_H_2005</td>
      <td>A diagnostic reasoning and optimal treatment model for bacterial infections with fuzzy information</td>
      <td>Kao, Han-Ying; Li, Han-Lin</td>
      <td>2005</td>
      <td>2005-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125802" target="_blank">PMC7125802</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15639707.0" target="_blank">15639707.0</a></td>
      <td><a href="https://doi.org/10.1016/j.cmpb.2004.08.003" target="_blank">10.1016/j.cmpb.2004.08.003</a></td>
      <td>b265mbwp</td>
      <td>0.454551</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_06.html#mxdlii77">Arji_2019</a></td>
    </tr>
    <tr>
      <th id="auasueaw";>85</th>
      <td>Kenah_2015</td>
      <td>Semiparametric Relative-risk Regression for Infectious Disease Transmission Data</td>
      <td>Kenah, Eben</td>
      <td>2015</td>
      <td>2015-01-02</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4489164" target="_blank">PMC4489164</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26146425.0" target="_blank">26146425.0</a></td>
      <td><a href="https://doi.org/10.1080/01621459.2014.896807" target="_blank">10.1080/01621459.2014.896807</a></td>
      <td>auasueaw</td>
      <td>0.450698</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="qzrhihoz";>86</th>
      <td>Brahmbhatt_2016</td>
      <td>Epidemiology-based Task Assignment Algorithm for Distributed Systems</td>
      <td>Brahmbhatt, Parth; Camorlinga, Sergio G.</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2016.09.356" target="_blank">10.1016/j.procs.2016.09.356</a></td>
      <td>qzrhihoz</td>
      <td>0.450154</td>
      <td></td>
      <td><a href="Topic_01.html#397hp1yt">Coelho_2008</a></td>
    </tr>
    <tr>
      <th id="abn1i3ir";>87</th>
      <td>Modak_2019</td>
      <td>Application of Support Vector Machines in Viral Biology</td>
      <td>Modak, Sonal; Mehta, Swati; Sehgal, Deepak; Valadi, Jayaraman</td>
      <td>2019</td>
      <td>2019-11-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114997" target="_blank">PMC7114997</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-29022-1_12" target="_blank">10.1007/978-3-030-29022-1_12</a></td>
      <td>abn1i3ir</td>
      <td>0.449631</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="y7697hx8";>88</th>
      <td>Dai_Q_2013</td>
      <td>Study of LZ-word distribution and its application for sequence comparison</td>
      <td>Dai, Qi; Yan, Zhaofang; Shi, Zhuoxing; Liu, Xiaoqing; Yao, Yuhua; He, Pingan</td>
      <td>2013</td>
      <td>2013-11-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094135" target="_blank">PMC7094135</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23876763.0" target="_blank">23876763.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2013.07.008" target="_blank">10.1016/j.jtbi.2013.07.008</a></td>
      <td>y7697hx8</td>
      <td>0.446900</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_05.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_05.html#d6wcgkln">Wang_2008</a>, <a href="Topic_10.html#p1i7bx94">Zhang_2013</a></td>
    </tr>
    <tr>
      <th id="9z8z03rp";>89</th>
      <td>Kai_Z_2019</td>
      <td>An efficient simulated annealing algorithm for the RNA secondary structure prediction with Pseudoknots</td>
      <td>Kai, Zhang; Yuting, Wang; Yulin, Lv; Jun, Liu; Juanjuan, He</td>
      <td>2019</td>
      <td>2019-12-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6933665" target="_blank">PMC6933665</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31881969.0" target="_blank">31881969.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-019-6300-2" target="_blank">10.1186/s12864-019-6300-2</a></td>
      <td>9z8z03rp</td>
      <td>0.446860</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#jxgd1yc4">Kai_Z_2018</a>, <a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="zbz88pqx";>90</th>
      <td>Lopes_2017</td>
      <td>Computational Comparison and Visualization of Viruses in the Perspective of Clinical Information</td>
      <td>Lopes, António M.; Machado, J. A. Tenreiro; Galhano, Alexandra M.</td>
      <td>2017</td>
      <td>2017-04-08</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090701" target="_blank">PMC7090701</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28391493.0" target="_blank">28391493.0</a></td>
      <td><a href="https://doi.org/10.1007/s12539-017-0229-4" target="_blank">10.1007/s12539-017-0229-4</a></td>
      <td>zbz88pqx</td>
      <td>0.445050</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_03.html#r7tg0s45">John_2019</a></td>
    </tr>
    <tr>
      <th id="oim1ifw0";>91</th>
      <td>Hou_W_2017</td>
      <td>A new method to analyze protein sequence similarity using Dynamic Time Warping</td>
      <td>Hou, Wenbing; Pan, Qiuhui; Peng, Qianying; He, Mingfeng</td>
      <td>2017</td>
      <td>2017-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125777" target="_blank">PMC7125777</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27974244.0" target="_blank">27974244.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ygeno.2016.12.002" target="_blank">10.1016/j.ygeno.2016.12.002</a></td>
      <td>oim1ifw0</td>
      <td>0.444967</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_05.html#41atidai">Abo-Elkhier_2019</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a>, <a href="Topic_05.html#nw3tybw9">Hoang_2015</a></td>
    </tr>
    <tr>
      <th id="7ch054mt";>92</th>
      <td>Lin_C_2009</td>
      <td>Developing a hierarchy relation with an expert decision analysis process for selecting the optimal resort type for a Taiwanese international resort park</td>
      <td>Lin, Chin-Tsai; Juan, Pin-Ju</td>
      <td>2009</td>
      <td>2009-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125690" target="_blank">PMC7125690</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2007.11.044" target="_blank">10.1016/j.eswa.2007.11.044</a></td>
      <td>7ch054mt</td>
      <td>0.442401</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="pd0cmefo";>93</th>
      <td>Shang_2020</td>
      <td>CHEER: hierarCHical taxonomic classification for viral mEtagEnomic data via deep leaRning</td>
      <td>Jiayu Shang; Yanni Sun</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.26.009001" target="_blank">10.1101/2020.03.26.009001</a></td>
      <td>pd0cmefo</td>
      <td>0.440104</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_08.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a></td>
    </tr>
    <tr>
      <th id="w45h121w";>94</th>
      <td>Tikhomirov_2018</td>
      <td>News Timeline Generation: Accounting for Structural Aspects and Temporal Nature of News Stream</td>
      <td>Tikhomirov, Mikhail; Dobrov, Boris</td>
      <td>2018</td>
      <td>2018-06-22</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120637" target="_blank">PMC7120637</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-96553-6_19" target="_blank">10.1007/978-3-319-96553-6_19</a></td>
      <td>w45h121w</td>
      <td>0.438198</td>
      <td></td>
      <td><a href="Topic_01.html#j22q89l9">Vanderwende_2007</a>, <a href="Topic_06.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_01.html#4wnoa4l7">Cai_Y_2019</a></td>
    </tr>
    <tr>
      <th id="fn929z9y";>95</th>
      <td>Carrillo-de-Gea_2016</td>
      <td>A Computer-Aided Detection System for Digital Chest Radiographs</td>
      <td>Carrillo-de-Gea, Juan Manuel; García-Mateos, Ginés; Fernández-Alemán, José Luis; Hernández-Hernández, José Luis</td>
      <td>2016</td>
      <td>2016-05-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5058572" target="_blank">PMC5058572</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27372536.0" target="_blank">27372536.0</a></td>
      <td><a href="https://doi.org/10.1155/2016/8208923" target="_blank">10.1155/2016/8208923</a></td>
      <td>fn929z9y</td>
      <td>0.433407</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_12.html#yzqcet2l">Javadi_2006</a>, <a href="Topic_03.html#45dpoepu">Hassanien_2020</a></td>
    </tr>
    <tr>
      <th id="f6lfykot";>96</th>
      <td>Wu_C_2018</td>
      <td>A novel model for protein sequence similarity analysis based on spectral radius</td>
      <td>Wu, Chuanyan; Gao, Rui; De Marinis, Yang; Zhang, Yusen</td>
      <td>2018</td>
      <td>2018-06-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094169" target="_blank">PMC7094169</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29524440.0" target="_blank">29524440.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2018.03.001" target="_blank">10.1016/j.jtbi.2018.03.001</a></td>
      <td>f6lfykot</td>
      <td>0.430972</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_04.html#u1imic5l">Li_C_2018</a>, <a href="Topic_05.html#41atidai">Abo-Elkhier_2019</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_05.html#9t3kneoo">Abd_Elwahaab_2019</a></td>
    </tr>
    <tr>
      <th id="f4q1r3vu";>97</th>
      <td>Keller_2009</td>
      <td>Automated vocabulary discovery for geo-parsing online epidemic intelligence</td>
      <td>Keller, Mikaela; Freifeld, Clark C; Brownstein, John S</td>
      <td>2009</td>
      <td>2009-11-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2787530" target="_blank">PMC2787530</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19930702.0" target="_blank">19930702.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-385" target="_blank">10.1186/1471-2105-10-385</a></td>
      <td>f4q1r3vu</td>
      <td>0.426998</td>
      <td></td>
      <td><a href="Topic_06.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_06.html#4524vg4x">Collier_2007</a>, <a href="Topic_06.html#r0lduvs1">Zhang_2009</a></td>
    </tr>
    <tr>
      <th id="y5j07feh";>98</th>
      <td>Li_B_2008</td>
      <td>Flow-injection chemiluminescence determination of chrysin and baicalein assisted by theoretical prediction of chemiluminescence behavior of chrysin and baicalein</td>
      <td>Li, Baoxin; Guo, Lili; Xu, Chunli; Ma, Limei</td>
      <td>2008</td>
      <td>2008-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.saa.2008.02.007" target="_blank">10.1016/j.saa.2008.02.007</a></td>
      <td>y5j07feh</td>
      <td>0.425011</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="dslwklo4";>99</th>
      <td>Tao_J_2004</td>
      <td>A probabilistic approach to incorporating domain knowledge for closed-room people monitoring</td>
      <td>Tao, Ji; Tan, Yap-Peng</td>
      <td>2004</td>
      <td>2004-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126007" target="_blank">PMC7126007</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.image.2004.07.001" target="_blank">10.1016/j.image.2004.07.001</a></td>
      <td>dslwklo4</td>
      <td>0.424393</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_01.html#ym9j0ei0">Tao_J_2005</a>, <a href="Topic_01.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_01.html#gm1e9j80">Thi_T_2014</a>, <a href="Topic_01.html#gi8c1omv">Capotorti_2006</a></td>
    </tr>
    <tr>
      <th id="flgc5l04";>100</th>
      <td>Budzan_2013</td>
      <td>Face and eyes localization algorithm in thermal images for temperature measurement of the inner canthus of the eyes</td>
      <td>Budzan, Sebastian; Wyżgolik, Roman</td>
      <td>2013</td>
      <td>2013-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110491" target="_blank">PMC7110491</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.infrared.2013.05.007" target="_blank">10.1016/j.infrared.2013.05.007</a></td>
      <td>flgc5l04</td>
      <td>0.424000</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_14.html#uqvjx98j">Dwith_Chenna_2018</a></td>
    </tr>
    <tr>
      <th id="058r9486";>101</th>
      <td>Morfopoulou_2014</td>
      <td>Bayesian mixture analysis for metagenomic community profiling.</td>
      <td>Sofia Morfopoulou; Vincent Plagnol</td>
      <td>2014</td>
      <td>2014-07-25</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/007476" target="_blank">10.1101/007476</a></td>
      <td>058r9486</td>
      <td>0.421888</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_08.html#z29xsj1o">Morfopoulou_2015</a></td>
    </tr>
    <tr>
      <th id="e17qi1v2";>102</th>
      <td>Raj_A_2011</td>
      <td>Identifying Hosts of Families of Viruses: A Machine Learning Approach</td>
      <td>Raj, Anil; Dewar, Michael; Palacios, Gustavo; Rabadan, Raul; Wiggins, Christopher H.</td>
      <td>2011</td>
      <td>2011-12-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3235098" target="_blank">PMC3235098</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22174744.0" target="_blank">22174744.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0027631" target="_blank">10.1371/journal.pone.0027631</a></td>
      <td>e17qi1v2</td>
      <td>0.419242</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_08.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="abvpy8dg";>103</th>
      <td>Shaban-Nejad_2012</td>
      <td>HAIKU: A Semantic Framework for Surveillance of Healthcare-Associated Infections</td>
      <td>Shaban-Nejad, Arash; Riazanov, Alexandre; Charland, Katia M.; Rose, Gregory W.; Baker, Christopher J.O.; Tamblyn, Robyn; Forster, Alan J.; Buckeridge, David L.</td>
      <td>2012</td>
      <td>2012-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2012.06.151" target="_blank">10.1016/j.procs.2012.06.151</a></td>
      <td>abvpy8dg</td>
      <td>0.418294</td>
      <td></td>
      <td><a href="Topic_06.html#4524vg4x">Collier_2007</a></td>
    </tr>
    <tr>
      <th id="nfxcqsc2";>104</th>
      <td>Kim_K_2007</td>
      <td>A LAD-based method for selecting short oligo probes for genotyping applications</td>
      <td>Kim, Kwangsoo; Ryoo, Hong Seo</td>
      <td>2007</td>
      <td>2007-06-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7080176" target="_blank">PMC7080176</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s00291-007-0089-0" target="_blank">10.1007/s00291-007-0089-0</a></td>
      <td>nfxcqsc2</td>
      <td>0.412898</td>
      <td><a href="Topic_05.html#9rx4jlw8">Kim_K_2007</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_05.html#9rx4jlw8">Kim_K_2007</a></td>
    </tr>
    <tr>
      <th id="xn2wulnp";>105</th>
      <td>Fonseca_2019</td>
      <td>A computational method for the identification of Dengue, Zika and Chikungunya virus species and genotypes</td>
      <td>Fonseca, Vagner; Libin, Pieter J. K.; Theys, Kristof; Faria, Nuno R.; Nunes, Marcio R. T.; Restovic, Maria I.; Freire, Murilo; Giovanetti, Marta; Cuypers, Lize; Nowé, Ann; Abecasis, Ana; Deforche, Koen; Santiago, Gilberto A.; de Siqueira, Isadora C.; San, Emmanuel J.; Machado, Kaliane C. B.; Azevedo, Vasco; Filippis, Ana Maria Bispo-de; da Cunha, Rivaldo Venâncio; Pybus, Oliver G.; Vandamme, Anne-Mieke; Alcantara, Luiz C. J.; de Oliveira, Tulio</td>
      <td>2019</td>
      <td>2019-05-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6527240" target="_blank">PMC6527240</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31067235.0" target="_blank">31067235.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pntd.0007231" target="_blank">10.1371/journal.pntd.0007231</a></td>
      <td>xn2wulnp</td>
      <td>0.412196</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="whoydkd5";>106</th>
      <td>Cao_Y_2005</td>
      <td>Information theory-based algorithm for in silico prediction of PCR products with whole genomic sequences as templates</td>
      <td>Cao, Youfang; Wang, Lianjie; Xu, Kexue; Kou, Chunhai; Zhang, Yulei; Wei, Guifang; He, Junjian; Wang, Yunfang; Zhao, Liping</td>
      <td>2005</td>
      <td>2005-07-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1183192" target="_blank">PMC1183192</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16042814.0" target="_blank">16042814.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-6-190" target="_blank">10.1186/1471-2105-6-190</a></td>
      <td>whoydkd5</td>
      <td>0.404793</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ki0nnptv";>107</th>
      <td>Guo_S_2020</td>
      <td>Droplet-Transmitted Infection Risk Ranking Based on Close Proximity Interaction</td>
      <td>Guo, Shihui; Yu, Jubo; Shi, Xinyu; Wang, Hongran; Xie, Feibin; Gao, Xing; Jiang, Min</td>
      <td>2020</td>
      <td>2020-01-21</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6985151" target="_blank">PMC6985151</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32038220.0" target="_blank">32038220.0</a></td>
      <td><a href="https://doi.org/10.3389/fnbot.2019.00113" target="_blank">10.3389/fnbot.2019.00113</a></td>
      <td>ki0nnptv</td>
      <td>0.403490</td>
      <td></td>
      <td><a href="Topic_08.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="hs30wwiq";>108</th>
      <td>Gibbs_2014</td>
      <td>Multi-omic network signatures of disease</td>
      <td>Gibbs, David L.; Gralinski, Lisa; Baric, Ralph S.; McWeeney, Shannon K.</td>
      <td>2014</td>
      <td>2014-01-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882664" target="_blank">PMC3882664</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24432028.0" target="_blank">24432028.0</a></td>
      <td><a href="https://doi.org/10.3389/fgene.2013.00309" target="_blank">10.3389/fgene.2013.00309</a></td>
      <td>hs30wwiq</td>
      <td>0.400983</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#m0q52a2t">Kurc_2006</a></td>
      <td><a href="Topic_13.html#4grjr0h3">McClure_2019</a></td>
    </tr>
    <tr>
      <th id="eydkfqi2";>109</th>
      <td>Feng_2019</td>
      <td>Relative space-based GIS data model to analyze the group dynamics of moving objects</td>
      <td>Feng, Mingxiang; Shaw, Shih-Lung; Fang, Zhixiang; Cheng, Hao</td>
      <td>2019</td>
      <td>2019-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111340" target="_blank">PMC7111340</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.isprsjprs.2019.05.002" target="_blank">10.1016/j.isprsjprs.2019.05.002</a></td>
      <td>eydkfqi2</td>
      <td>0.400934</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_13.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_12.html#aebmrvea">Davis_2007</a>, <a href="Topic_09.html#y0s4cf6g">Zou_Z_2013</a></td>
    </tr>
    <tr>
      <th id="g55witln";>110</th>
      <td>Gibbs_2013</td>
      <td>Protein co-expression network analysis (ProCoNA)</td>
      <td>Gibbs, David L; Baratt, Arie; Baric, Ralph S; Kawaoka, Yoshihiro; Smith, Richard D; Orwoll, Eric S; Katze, Michael G; McWeeney, Shannon K</td>
      <td>2013</td>
      <td>2013-06-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3695838" target="_blank">PMC3695838</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23724967.0" target="_blank">23724967.0</a></td>
      <td><a href="https://doi.org/10.1186/2043-9113-3-11" target="_blank">10.1186/2043-9113-3-11</a></td>
      <td>g55witln</td>
      <td>0.399903</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_01.html#gmw88elf">Vella_2017</a></td>
    </tr>
    <tr>
      <th id="iomwwsud";>111</th>
      <td>Sadreyev_2003</td>
      <td>Profile–profile comparisons by COMPASS predict intricate homologies between protein families</td>
      <td>Sadreyev, Ruslan I.; Baker, David; Grishin, Nick V.</td>
      <td>2003</td>
      <td>2003-10-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2366929" target="_blank">PMC2366929</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/14500884.0" target="_blank">14500884.0</a></td>
      <td></td>
      <td>iomwwsud</td>
      <td>0.399847</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#s7kzp795">Zemla_2011</a></td>
    </tr>
    <tr>
      <th id="nz4qpumr";>112</th>
      <td>Sahli_2012</td>
      <td>Arapan-S: a fast and highly accurate whole-genome assembly software for viruses and small genomes</td>
      <td>Sahli, Mohammed; Shibuya, Tetsuo</td>
      <td>2012</td>
      <td>2012-05-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441218" target="_blank">PMC3441218</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22591859.0" target="_blank">22591859.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-5-243" target="_blank">10.1186/1756-0500-5-243</a></td>
      <td>nz4qpumr</td>
      <td>0.397365</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="qgit5qqo";>113</th>
      <td>Reeder_2006</td>
      <td>Beyond Mfold: Recent advances in RNA bioinformatics</td>
      <td>Reeder, Jens; Höchsmann, Matthias; Rehmsmeier, Marc; Voss, Björn; Giegerich, Robert</td>
      <td>2006</td>
      <td>2006-06-25</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125698" target="_blank">PMC7125698</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16530285.0" target="_blank">16530285.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbiotec.2006.01.034" target="_blank">10.1016/j.jbiotec.2006.01.034</a></td>
      <td>qgit5qqo</td>
      <td>0.395652</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="1r4gm2d4";>114</th>
      <td>Liu_Y_2020</td>
      <td>Deep Learning-Based Method of Diagnosing Hyperlipidemia and Providing Diagnostic Markers Automatically</td>
      <td>Liu, Yuliang; Zhang, Quan; Zhao, Geng; Liu, Guohua; Liu, Zhiang</td>
      <td>2020</td>
      <td>2020-03-11</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7073442" target="_blank">PMC7073442</a></td>
      <td></td>
      <td><a href="https://doi.org/10.2147/dmso.s242585" target="_blank">10.2147/dmso.s242585</a></td>
      <td>1r4gm2d4</td>
      <td>0.394199</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="ddu6yl8r";>115</th>
      <td>Adetiba_2018</td>
      <td>Alignment-Free Z-Curve Genomic Cepstral Coefficients and Machine Learning for Classification of Viruses</td>
      <td>Adetiba, Emmanuel; Olugbara, Oludayo O.; Taiwo, Tunmike B.; Adebiyi, Marion O.; Badejo, Joke A.; Akanle, Matthew B.; Matthews, Victor O.</td>
      <td>2018</td>
      <td>2018-03-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120486" target="_blank">PMC7120486</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-78723-7_25" target="_blank">10.1007/978-3-319-78723-7_25</a></td>
      <td>ddu6yl8r</td>
      <td>0.389991</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_08.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_03.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_03.html#2l1zw19o">Ozturk_2020</a></td>
    </tr>
    <tr>
      <th id="ne9d0siv";>116</th>
      <td>Banks_2006</td>
      <td>Dissecting the ancient rapid radiation of microgastrine wasp genera using additional nuclear genes</td>
      <td>Banks, Jonathan C.; Whitfield, James B.</td>
      <td>2006</td>
      <td>2006-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129091" target="_blank">PMC7129091</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16854601.0" target="_blank">16854601.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ympev.2006.06.001" target="_blank">10.1016/j.ympev.2006.06.001</a></td>
      <td>ne9d0siv</td>
      <td>0.388363</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_13.html#3atcqejc">Kinene_2016</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="fbl3szsu";>117</th>
      <td>Pfeiffer_2015</td>
      <td>Spatial and temporal epidemiological analysis in the Big Data era</td>
      <td>Pfeiffer, Dirk U.; Stevens, Kim B.</td>
      <td>2015</td>
      <td>2015-11-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114113" target="_blank">PMC7114113</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26092722.0" target="_blank">26092722.0</a></td>
      <td><a href="https://doi.org/10.1016/j.prevetmed.2015.05.012" target="_blank">10.1016/j.prevetmed.2015.05.012</a></td>
      <td>fbl3szsu</td>
      <td>0.387111</td>
      <td></td>
      <td><a href="Topic_01.html#397hp1yt">Coelho_2008</a></td>
    </tr>
    <tr>
      <th id="c3ewuebx";>118</th>
      <td>Jia_J_2016</td>
      <td>iPPBS-Opt: A Sequence-Based Ensemble Classifier for Identifying Protein-Protein Binding Sites by Optimizing Imbalanced Training Datasets</td>
      <td>Jia, Jianhua; Liu, Zi; Xiao, Xuan; Liu, Bingxiang; Chou, Kuo-Chen</td>
      <td>2016</td>
      <td>2016-01-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6274413" target="_blank">PMC6274413</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26797600.0" target="_blank">26797600.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules21010095" target="_blank">10.3390/molecules21010095</a></td>
      <td>c3ewuebx</td>
      <td>0.386395</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="zuh78iul";>119</th>
      <td>Pessia_2015</td>
      <td>K-Pax2: Bayesian identification of cluster-defining amino acid positions in large sequence datasets</td>
      <td>Pessia, Alberto; Grad, Yonatan; Cobey, Sarah; Puranen, Juha Santeri; Corander, Jukka</td>
      <td>2015</td>
      <td>2015-07-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320600" target="_blank">PMC5320600</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28348810.0" target="_blank">28348810.0</a></td>
      <td><a href="https://doi.org/10.1099/mgen.0.000025" target="_blank">10.1099/mgen.0.000025</a></td>
      <td>zuh78iul</td>
      <td>0.386006</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="fk8vz0g0";>120</th>
      <td>Zeng_2007</td>
      <td>Mass spectral profiling: An effective tool for quality control of herbal medicines</td>
      <td>Zeng, Zhong-Da; Liang, Yi-Zeng; Chau, Foo-Tim; Chen, Shuo; Daniel, Mok Kam-Wah; Chan, Chi-On</td>
      <td>2007</td>
      <td>2007-12-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7119444" target="_blank">PMC7119444</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17996528.0" target="_blank">17996528.0</a></td>
      <td><a href="https://doi.org/10.1016/j.aca.2007.09.057" target="_blank">10.1016/j.aca.2007.09.057</a></td>
      <td>fk8vz0g0</td>
      <td>0.385092</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="eg63bjl5";>121</th>
      <td>Jacques_2010</td>
      <td>Small-angle scattering for structural biology—Expanding the frontier while avoiding the pitfalls</td>
      <td>Jacques, David A; Trewhella, Jill</td>
      <td>2010</td>
      <td>2010-01-29</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2867006" target="_blank">PMC2867006</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20120026.0" target="_blank">20120026.0</a></td>
      <td><a href="https://doi.org/10.1002/pro.351" target="_blank">10.1002/pro.351</a></td>
      <td>eg63bjl5</td>
      <td>0.383811</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_07.html#cw2opick">Chou_2011</a></td>
    </tr>
    <tr>
      <th id="hphn7cs6";>122</th>
      <td>Cacciabue_2019</td>
      <td>A beginner’s guide for FMDV quasispecies analysis: sub-consensus variant detection and haplotype reconstruction using next-generation sequencing</td>
      <td>Cacciabue, Marco; Currá, Anabella; Carrillo, Elisa; König, Guido; Gismondi, María Inés</td>
      <td>2019</td>
      <td>2019-11-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110011" target="_blank">PMC7110011</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31697321.0" target="_blank">31697321.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bbz086" target="_blank">10.1093/bib/bbz086</a></td>
      <td>hphn7cs6</td>
      <td>0.383670</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="buc4dd5y";>123</th>
      <td>Dong_2019</td>
      <td>A Novel Approach to Clustering Genome Sequences Using Inter-nucleotide Covariance</td>
      <td>Dong, Rui; He, Lily; He, Rong Lucy; Yau, Stephen S.-T.</td>
      <td>2019</td>
      <td>2019-04-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6465635" target="_blank">PMC6465635</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31024610.0" target="_blank">31024610.0</a></td>
      <td><a href="https://doi.org/10.3389/fgene.2019.00234" target="_blank">10.3389/fgene.2019.00234</a></td>
      <td>buc4dd5y</td>
      <td>0.382135</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_05.html#gf718lbl">Yu_Z_2010</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="mrbi0a7r";>124</th>
      <td>Izakian_2012</td>
      <td>A new PSO-optimized geometry of spatial and spatio-temporal scan statistics for disease outbreak detection</td>
      <td>Izakian, Hesam; Pedrycz, Witold</td>
      <td>2012</td>
      <td>2012-06-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104009" target="_blank">PMC7104009</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.swevo.2012.02.001" target="_blank">10.1016/j.swevo.2012.02.001</a></td>
      <td>mrbi0a7r</td>
      <td>0.382135</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_06.html#fpiew1jy">Takahashi_2008</a></td>
    </tr>
    <tr>
      <th id="43igjn6x";>125</th>
      <td>Kylberg_2009</td>
      <td>Towards Automated TEM for Virus Diagnostics: Segmentation of Grid Squares and Detection of Regions of Interest</td>
      <td>Kylberg, Gustaf; Sintorn, Ida-Maria; Borgefors, Gunilla</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120801" target="_blank">PMC7120801</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-642-02230-2_18" target="_blank">10.1007/978-3-642-02230-2_18</a></td>
      <td>43igjn6x</td>
      <td>0.381719</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="bqpsihm6";>126</th>
      <td>González-Díaz_2009</td>
      <td>Generalized lattice graphs for 2D-visualization of biological information</td>
      <td>González-Díaz, H.; Pérez-Montoto, L.G.; Duardo-Sanchez, A.; Paniagua, E.; Vázquez-Prieto, S.; Vilas, R.; Dea-Ayuela, M.A.; Bolas-Fernández, F.; Munteanu, C.R.; Dorado, J.; Costas, J.; Ubeira, F.M.</td>
      <td>2009</td>
      <td>2009-11-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094121" target="_blank">PMC7094121</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19646452.0" target="_blank">19646452.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2009.07.029" target="_blank">10.1016/j.jtbi.2009.07.029</a></td>
      <td>bqpsihm6</td>
      <td>0.381031</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_13.html#g4k6pc82">Munteanu_2008</a>, <a href="Topic_10.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="wyrvxocw";>127</th>
      <td>Porter_2014</td>
      <td>A multivariate CAR model for mismatched lattices</td>
      <td>Porter, Aaron T.; Oleson, Jacob J.</td>
      <td>2014</td>
      <td>2014-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25457598.0" target="_blank">25457598.0</a></td>
      <td><a href="https://doi.org/10.1016/j.sste.2014.08.001" target="_blank">10.1016/j.sste.2014.08.001</a></td>
      <td>wyrvxocw</td>
      <td>0.380456</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="l3f469ht";>128</th>
      <td>He_X_2020</td>
      <td>Sample-Efficient Deep Learning for COVID-19 Diagnosis Based on CT Scans</td>
      <td>Xuehai He; Xingyi Yang; Shanghang Zhang; Jinyu Zhao; Yichen Zhang; Eric Xing; Pengtao Xie</td>
      <td>2020</td>
      <td>2020-04-17</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.13.20063941" target="_blank">10.1101/2020.04.13.20063941</a></td>
      <td>l3f469ht</td>
      <td>0.379946</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="9wrw6dv1";>129</th>
      <td>Huang_2016</td>
      <td>Artificial infectious disease optimization: A SEIQR epidemic dynamic model-based function optimization algorithm</td>
      <td>Huang, Guangqiu</td>
      <td>2016</td>
      <td>2016-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104270" target="_blank">PMC7104270</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.swevo.2015.09.007" target="_blank">10.1016/j.swevo.2015.09.007</a></td>
      <td>9wrw6dv1</td>
      <td>0.379751</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="dc63kane";>130</th>
      <td>Andreatta_2011</td>
      <td>NNAlign: A Web-Based Prediction Method Allowing Non-Expert End-User Discovery of Sequence Motifs in Quantitative Peptide Data</td>
      <td>Andreatta, Massimo; Schafer-Nielsen, Claus; Lund, Ole; Buus, Søren; Nielsen, Morten</td>
      <td>2011</td>
      <td>2011-11-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3206854" target="_blank">PMC3206854</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22073191.0" target="_blank">22073191.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0026781" target="_blank">10.1371/journal.pone.0026781</a></td>
      <td>dc63kane</td>
      <td>0.379147</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#6oca1mrm">Shen_2018</a></td>
    </tr>
    <tr>
      <th id="uo1d8xka";>131</th>
      <td>Singh_2008</td>
      <td>Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment</td>
      <td>Singh, Dadabhai T; Trehan, Rahul; Schmidt, Bertil; Bretschneider, Timo</td>
      <td>2008</td>
      <td>2008-02-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2259424" target="_blank">PMC2259424</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18315855.0" target="_blank">18315855.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-s1-s23" target="_blank">10.1186/1471-2105-9-s1-s23</a></td>
      <td>uo1d8xka</td>
      <td>0.378187</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="qw5ly6rl";>132</th>
      <td>Khobahi_2020</td>
      <td>CoroNet: A Deep Network Architecture for Semi-Supervised Task-Based Identification of COVID-19 from Chest X-ray Images</td>
      <td>Shahin Khobahi; Chirag Agarwal; Mojtaba Soltanalian</td>
      <td>2020</td>
      <td>2020-04-17</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.14.20065722" target="_blank">10.1101/2020.04.14.20065722</a></td>
      <td>qw5ly6rl</td>
      <td>0.377661</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="t3zampk7";>133</th>
      <td>Chen_2012</td>
      <td>PepMapper: A Collaborative Web Tool for Mapping Epitopes from Affinity-Selected Peptides</td>
      <td>Chen, Wenhan; Guo, William W.; Huang, Yanxin; Ma, Zhiqiang</td>
      <td>2012</td>
      <td>2012-05-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3360666" target="_blank">PMC3360666</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22701536.0" target="_blank">22701536.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0037869" target="_blank">10.1371/journal.pone.0037869</a></td>
      <td>t3zampk7</td>
      <td>0.377514</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_11.html#7rs4jtfh">Mayrose_2007</a>, <a href="Topic_11.html#qk1pr335">Sun_P_2015</a></td>
    </tr>
    <tr>
      <th id="jbmj21c5";>134</th>
      <td>Du_Q_2017</td>
      <td>2L-PCA: a two-level principal component analyzer for quantitative drug design and its applications</td>
      <td>Du, Qi-Shi; Wang, Shu-Qing; Xie, Neng-Zhong; Wang, Qing-Yan; Huang, Ri-Bo; Chou, Kuo-Chen</td>
      <td>2017</td>
      <td>2017-08-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5642577" target="_blank">PMC5642577</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29050302.0" target="_blank">29050302.0</a></td>
      <td><a href="https://doi.org/10.18632/oncotarget.19757" target="_blank">10.18632/oncotarget.19757</a></td>
      <td>jbmj21c5</td>
      <td>0.375404</td>
      <td><a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_07.html#bs17igjs">Zhou_2010</a>, <a href="Topic_04.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_04.html#ni6iyzdn">He_Z_2010</a>, <a href="Topic_07.html#fewy8y4a">Wang_2018</a></td>
    </tr>
    <tr>
      <th id="a2m46j0z";>135</th>
      <td>Tian_2016</td>
      <td>The similarity analysis of financial stocks based on information clustering</td>
      <td>Tian, Qiang; Shang, Pengjian; Feng, Guochen</td>
      <td>2016</td>
      <td>2016-05-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088863" target="_blank">PMC7088863</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11071-016-2851-9" target="_blank">10.1007/s11071-016-2851-9</a></td>
      <td>a2m46j0z</td>
      <td>0.375154</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_13.html#e4shf7qq">Tian_2014</a>, <a href="Topic_01.html#4ezebg45">Hui_E_2016</a></td>
    </tr>
    <tr>
      <th id="2l1zw19o";>136</th>
      <td>Ozturk_2020</td>
      <td>Classification of Coronavirus Images using Shrunken Features</td>
      <td>Saban Ozturk; Umut Ozkaya; Mucahid Barstugan</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.03.20048868" target="_blank">10.1101/2020.04.03.20048868</a></td>
      <td>2l1zw19o</td>
      <td>0.375145</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_03.html#zqsy6r4i">Song_2020</a></td>
    </tr>
    <tr>
      <th id="v6on5gv9";>137</th>
      <td>Asteris_2020</td>
      <td>A Novel Heuristic Global Algorithm to Predict the COVID-19 Pandemic Trend</td>
      <td>Panagiotis G. Asteris; Maria Douvika; Christina Karamani; Athanasia Skentou; Tryfon Daras; Liborio Cavaleri; Danial Jahed Armaghani; Katerina Chlichlia; Theoklis E. Zaoutis</td>
      <td>2020</td>
      <td>2020-04-22</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.16.20068445" target="_blank">10.1101/2020.04.16.20068445</a></td>
      <td>v6on5gv9</td>
      <td>0.373194</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="y5xsf3ij";>138</th>
      <td>Tapinos_2019</td>
      <td>The Utility of Data Transformation for Alignment, De Novo Assembly and Classification of Short Read Virus Sequences</td>
      <td>Tapinos, Avraam; Constantinides, Bede; Phan, My V. T.; Kouchaki, Samaneh; Cotten, Matthew; Robertson, David L.</td>
      <td>2019</td>
      <td>2019-04-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563281" target="_blank">PMC6563281</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31035503.0" target="_blank">31035503.0</a></td>
      <td><a href="https://doi.org/10.3390/v11050394" target="_blank">10.3390/v11050394</a></td>
      <td>y5xsf3ij</td>
      <td>0.373073</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="ljli6a2z";>139</th>
      <td>Balasubramanian_2020</td>
      <td>Automatic Identification of SARS Coronavirus using Compression-Complexity Measures</td>
      <td>Karthi Balasubramanian; Nithin Nagaraj</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.24.006007" target="_blank">10.1101/2020.03.24.006007</a></td>
      <td>ljli6a2z</td>
      <td>0.372916</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a></td>
    </tr>
    <tr>
      <th id="bmcku2yb";>140</th>
      <td>Wilm_2012</td>
      <td>LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets</td>
      <td>Wilm, Andreas; Aw, Pauline Poh Kim; Bertrand, Denis; Yeo, Grace Hui Ting; Ong, Swee Hoe; Wong, Chang Hua; Khor, Chiea Chuen; Petric, Rosemary; Hibberd, Martin Lloyd; Nagarajan, Niranjan</td>
      <td>2012</td>
      <td>2012-10-12</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" target="_blank">PMC3526318</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23066108.0" target="_blank">23066108.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gks918" target="_blank">10.1093/nar/gks918</a></td>
      <td>bmcku2yb</td>
      <td>0.371663</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="hef1tfjx";>141</th>
      <td>Spencer_2004</td>
      <td>Do Newton’s rules of reasoning guarantee truth … must they?</td>
      <td>Spencer, Quayshawn</td>
      <td>2004</td>
      <td>2004-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134605" target="_blank">PMC7134605</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.shpsa.2004.02.001" target="_blank">10.1016/j.shpsa.2004.02.001</a></td>
      <td>hef1tfjx</td>
      <td>0.371371</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="qi630tpo";>142</th>
      <td>Ragonnet-Cronin_2013</td>
      <td>Automated analysis of phylogenetic clusters</td>
      <td>Ragonnet-Cronin, Manon; Hodcroft, Emma; Hué, Stéphane; Fearnhill, Esther; Delpech, Valerie; Brown, Andrew J Leigh; Lycett, Samantha</td>
      <td>2013</td>
      <td>2013-11-06</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228337" target="_blank">PMC4228337</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24191891.0" target="_blank">24191891.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-14-317" target="_blank">10.1186/1471-2105-14-317</a></td>
      <td>qi630tpo</td>
      <td>0.370287</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="a0pzw26l";>143</th>
      <td>Yu_Z_2010</td>
      <td>Proper Distance Metrics for Phylogenetic Analysis Using Complete Genomes without Sequence Alignment</td>
      <td>Yu, Zu-Guo; Zhan, Xiao-Wen; Han, Guo-Sheng; Wang, Roger W.; Anh, Vo; Chu, Ka Hou</td>
      <td>2010</td>
      <td>2010-03-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2869232" target="_blank">PMC2869232</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20480005.0" target="_blank">20480005.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms11031141" target="_blank">10.3390/ijms11031141</a></td>
      <td>a0pzw26l</td>
      <td>0.370227</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_05.html#gf718lbl">Yu_Z_2010</a></td>
    </tr>
    <tr>
      <th id="dk7eij1l";>144</th>
      <td>Araújo_2011</td>
      <td>A rigorous approach to facilitate and guarantee the correctness of the genetic testing management in human genome information systems</td>
      <td>Araújo, Luciano V; Malkowski, Simon; Braghetto, Kelly R; Passos-Bueno, Maria R; Zatz, Mayana; Pu, Calton; Ferreira, João E</td>
      <td>2011</td>
      <td>2011-12-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3287582" target="_blank">PMC3287582</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22369688.0" target="_blank">22369688.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2164-12-s4-s13" target="_blank">10.1186/1471-2164-12-s4-s13</a></td>
      <td>dk7eij1l</td>
      <td>0.366840</td>
      <td><a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="wtvfow2f";>145</th>
      <td>Seemann_2008</td>
      <td>Unifying evolutionary and thermodynamic information for RNA folding of multiple alignments</td>
      <td>Seemann, Stefan E.; Gorodkin, Jan; Backofen, Rolf</td>
      <td>2008</td>
      <td>2008-10-04</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2582601" target="_blank">PMC2582601</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18836192.0" target="_blank">18836192.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkn544" target="_blank">10.1093/nar/gkn544</a></td>
      <td>wtvfow2f</td>
      <td>0.366357</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="t1e0tr4w";>146</th>
      <td>Lee_C_2017</td>
      <td>A Pairwise Alignment Algorithm for Long Sequences of High Similarity</td>
      <td>Lee, Chien-Tai; Peng, Sheng-Lung</td>
      <td>2017</td>
      <td>2017-10-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123042" target="_blank">PMC7123042</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-981-10-5508-9_27" target="_blank">10.1007/978-981-10-5508-9_27</a></td>
      <td>t1e0tr4w</td>
      <td>0.366008</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_10.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="5q40uttg";>147</th>
      <td>Liu_Y_2014</td>
      <td>A FTA-based method for risk decision-making in emergency response</td>
      <td>Liu, Yang; Fan, Zhi-Ping; Yuan, Yuan; Li, Hongyan</td>
      <td>2014</td>
      <td>2014-02-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117036" target="_blank">PMC7117036</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cor.2012.08.015" target="_blank">10.1016/j.cor.2012.08.015</a></td>
      <td>5q40uttg</td>
      <td>0.365941</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="yxo8oanc";>148</th>
      <td>Punjani_2020</td>
      <td>3D Variability Analysis: Directly resolving continuous flexibility and discrete heterogeneity from single particle cryo-EM images</td>
      <td>Ali Punjani; David J. Fleet</td>
      <td>2020</td>
      <td>2020-04-09</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.08.032466" target="_blank">10.1101/2020.04.08.032466</a></td>
      <td>yxo8oanc</td>
      <td>0.365208</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#oohx14kl">Zhong_2020</a>, <a href="Topic_15.html#bqwmx5dy">Punjani_2019</a></td>
    </tr>
    <tr>
      <th id="e2coy5ms";>149</th>
      <td>Matsuzaw_2011</td>
      <td>KBDeX: A Platform for Exploring Discourse in Collaborative Learning</td>
      <td>Matsuzaw, Yoshiaki; Oshima, Jun; Oshima, Ritsuko; Niihara, Yusuke; Sakai, Sanshiro</td>
      <td>2011</td>
      <td>2011-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.sbspro.2011.10.576" target="_blank">10.1016/j.sbspro.2011.10.576</a></td>
      <td>e2coy5ms</td>
      <td>0.363154</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="6l688krc";>150</th>
      <td>Hung_2006</td>
      <td>Alignment using genetic programming with causal trees for identification of protein functions</td>
      <td>Hung, Chun-Min; Huang, Yueh-Min; Chang, Ming-Shi</td>
      <td>2006</td>
      <td>2006-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117053" target="_blank">PMC7117053</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.na.2005.09.048" target="_blank">10.1016/j.na.2005.09.048</a></td>
      <td>6l688krc</td>
      <td>0.362392</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="weoxwksv";>151</th>
      <td>Que_J_2012</td>
      <td>Spatial and Temporal Algorithm Evaluation for Detecting Over-The-Counter Thermometer Sale Increases during 2009 H1N1 Pandemic</td>
      <td>Que, Jialan; Tsui, Fu-Chiang</td>
      <td>2012</td>
      <td>2012-05-17</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3615801" target="_blank">PMC3615801</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23569624.0" target="_blank">23569624.0</a></td>
      <td><a href="https://doi.org/10.5210/ojphi.v4i1.3915" target="_blank">10.5210/ojphi.v4i1.3915</a></td>
      <td>weoxwksv</td>
      <td>0.358758</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_06.html#fpiew1jy">Takahashi_2008</a>, <a href="Topic_06.html#mrbi0a7r">Izakian_2012</a></td>
    </tr>
    <tr>
      <th id="hi7z3nob";>152</th>
      <td>Huang_2010</td>
      <td>SAROTUP: Scanner and Reporter of Target-Unrelated Peptides</td>
      <td>Huang, Jian; Ru, Beibei; Li, Shiyong; Lin, Hao; Guo, Feng-Biao</td>
      <td>2010</td>
      <td>2010-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842971" target="_blank">PMC2842971</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20339521.0" target="_blank">20339521.0</a></td>
      <td><a href="https://doi.org/10.1155/2010/101932" target="_blank">10.1155/2010/101932</a></td>
      <td>hi7z3nob</td>
      <td>0.357769</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4ylv92td";>153</th>
      <td>Kim_D_2018</td>
      <td>Mapping Oil Spills from Dual-Polarized SAR Images Using an Artificial Neural Network: Application to Oil Spill in the Kerch Strait in November 2007</td>
      <td>Kim, Daeseong; Jung, Hyung-Sup</td>
      <td>2018</td>
      <td>2018-07-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6069476" target="_blank">PMC6069476</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29997367.0" target="_blank">29997367.0</a></td>
      <td><a href="https://doi.org/10.3390/s18072237" target="_blank">10.3390/s18072237</a></td>
      <td>4ylv92td</td>
      <td>0.355221</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_14.html#3u81ntle">Acar_2016</a></td>
    </tr>
    <tr>
      <th id="sdgt2ms5";>154</th>
      <td>Kshirsagar_2013</td>
      <td>Multitask learning for host–pathogen protein interactions</td>
      <td>Kshirsagar, Meghana; Carbonell, Jaime; Klein-Seetharaman, Judith</td>
      <td>2013</td>
      <td>2013-07-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3694681" target="_blank">PMC3694681</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23812987.0" target="_blank">23812987.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btt245" target="_blank">10.1093/bioinformatics/btt245</a></td>
      <td>sdgt2ms5</td>
      <td>0.354207</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="hkvll3xh";>155</th>
      <td>Yang_2009</td>
      <td>Peptide Bioinformatics- Peptide Classification Using Peptide Machines</td>
      <td>Yang, Zheng Rong</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122642" target="_blank">PMC7122642</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19065810.0" target="_blank">19065810.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-60327-101-1_9" target="_blank">10.1007/978-1-60327-101-1_9</a></td>
      <td>hkvll3xh</td>
      <td>0.353670</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_04.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_04.html#85fqseal">Lee_B_2009</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="zqmdcj1z";>156</th>
      <td>Dandekar_2017</td>
      <td>Generating Fake but Realistic Headlines Using Deep Neural Networks</td>
      <td>Dandekar, Ashish; Zen, Remmy A. M.; Bressan, Stéphane</td>
      <td>2017</td>
      <td>2017-07-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121779" target="_blank">PMC7121779</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-64471-4_34" target="_blank">10.1007/978-3-319-64471-4_34</a></td>
      <td>zqmdcj1z</td>
      <td>0.352081</td>
      <td></td>
      <td><a href="Topic_06.html#k26orfrt">Jiang_2017</a>, <a href="Topic_01.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_08.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="aebmrvea";>157</th>
      <td>Davis_2007</td>
      <td>Assessing the Certainty of Locations Produced by an Address Geocoding System</td>
      <td>Davis, Clodoveu A.; Fonseca, Frederico T.</td>
      <td>2007</td>
      <td>2007-01-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087595" target="_blank">PMC7087595</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10707-006-0015-7" target="_blank">10.1007/s10707-006-0015-7</a></td>
      <td>aebmrvea</td>
      <td>0.351344</td>
      <td></td>
      <td><a href="Topic_09.html#eydkfqi2">Feng_2019</a>, <a href="Topic_01.html#ovuofl9q">Chen_2018</a>, <a href="Topic_06.html#f4q1r3vu">Keller_2009</a></td>
    </tr>
    <tr>
      <th id="2scgsmsc";>158</th>
      <td>Chou_2011</td>
      <td>Some remarks on protein attribute prediction and pseudo amino acid composition</td>
      <td>Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125570" target="_blank">PMC7125570</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21168420.0" target="_blank">21168420.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.12.024" target="_blank">10.1016/j.jtbi.2010.12.024</a></td>
      <td>2scgsmsc</td>
      <td>0.351340</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_04.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_04.html#fsh3gsvd">Xiao_2005</a>, <a href="Topic_04.html#yjrs7c38">De_Ferrari_2014</a></td>
    </tr>
    <tr>
      <th id="f1j6uhmj";>159</th>
      <td>Zhang_2014</td>
      <td>A Brief Review: The Z-curve Theory and its Application in Genome Analysis</td>
      <td>Zhang, Ren; Zhang, Chun-Ting</td>
      <td>2014</td>
      <td>2014-04-23</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4009844" target="_blank">PMC4009844</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24822026.0" target="_blank">24822026.0</a></td>
      <td><a href="https://doi.org/10.2174/1389202915999140328162433" target="_blank">10.2174/1389202915999140328162433</a></td>
      <td>f1j6uhmj</td>
      <td>0.351063</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="jy22na69";>160</th>
      <td>Vinga_2013</td>
      <td>Information theory applications for biological sequence analysis</td>
      <td>Vinga, Susana</td>
      <td>2013</td>
      <td>2013-09-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109941" target="_blank">PMC7109941</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24058049.0" target="_blank">24058049.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bbt068" target="_blank">10.1093/bib/bbt068</a></td>
      <td>jy22na69</td>
      <td>0.350884</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_04.html#8k7d1riv">Muppirala_2011</a></td>
    </tr>
    <tr>
      <th id="9gozwdng";>161</th>
      <td>Barbarini_2011</td>
      <td>Prediction of Peptide Reactivity with Human IVIg through a Knowledge-Based Approach</td>
      <td>Barbarini, Nicola; Tiengo, Alessandra; Bellazzi, Riccardo</td>
      <td>2011</td>
      <td>2011-08-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3160895" target="_blank">PMC3160895</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21887285.0" target="_blank">21887285.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0023616" target="_blank">10.1371/journal.pone.0023616</a></td>
      <td>9gozwdng</td>
      <td>0.350756</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_11.html#ml2luc1t">Sollner_2008</a></td>
    </tr>
    <tr>
      <th id="6zjf6qzz";>162</th>
      <td>Li_J_2015</td>
      <td>Concept learning via granular computing: A cognitive viewpoint</td>
      <td>Li, Jinhai; Mei, Changlin; Xu, Weihua; Qian, Yuhua</td>
      <td>2015</td>
      <td>2015-03-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094283" target="_blank">PMC7094283</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ins.2014.12.010" target="_blank">10.1016/j.ins.2014.12.010</a></td>
      <td>6zjf6qzz</td>
      <td>0.350044</td>
      <td></td>
      <td><a href="Topic_12.html#jzpt9ldx">Ren_R_2016</a>, <a href="Topic_09.html#e2coy5ms">Matsuzaw_2011</a>, <a href="Topic_09.html#6dcghdxf">Jiang_2009</a></td>
    </tr>
    <tr>
      <th id="r6bl7ls4";>163</th>
      <td>Zhang_2015</td>
      <td>Social network analysis on a topic‐based navigation guidance system in a public health portal</td>
      <td>Zhang, Jin; Zhai, Shanshan; Liu, Hongxia; Stevenson, Jennifer Ann</td>
      <td>2015</td>
      <td>2015-03-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7159533" target="_blank">PMC7159533</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1002/asi.23468" target="_blank">10.1002/asi.23468</a></td>
      <td>r6bl7ls4</td>
      <td>0.350044</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4grjr0h3";>164</th>
      <td>McClure_2019</td>
      <td>Unified feature association networks through integration of transcriptomic and proteomic data</td>
      <td>McClure, Ryan S.; Wendler, Jason P.; Adkins, Joshua N.; Swanstrom, Jesica; Baric, Ralph; Kaiser, Brooke L. Deatherage; Oxford, Kristie L.; Waters, Katrina M.; McDermott, Jason E.</td>
      <td>2019</td>
      <td>2019-09-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748406" target="_blank">PMC6748406</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31527878.0" target="_blank">31527878.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1007241" target="_blank">10.1371/journal.pcbi.1007241</a></td>
      <td>4grjr0h3</td>
      <td>0.349758</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_13.html#hs30wwiq">Gibbs_2014</a>, <a href="Topic_04.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_01.html#gmw88elf">Vella_2017</a>, <a href="Topic_13.html#g55witln">Gibbs_2013</a></td>
    </tr>
    <tr>
      <th id="do0gb0g6";>165</th>
      <td>van_Dijk_2008</td>
      <td>Predicting sub-Golgi localization of type II membrane proteins</td>
      <td>van Dijk, A. D. J.; Bosch, D.; ter Braak, C. J. F.; van der Krol, A. R.; van Ham, R. C. H. J.</td>
      <td>2008</td>
      <td>2008-08-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110242" target="_blank">PMC7110242</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18562268.0" target="_blank">18562268.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btn309" target="_blank">10.1093/bioinformatics/btn309</a></td>
      <td>do0gb0g6</td>
      <td>0.349481</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ufdgi67q";>166</th>
      <td>Khanna_2011</td>
      <td>In silico approach to screen compounds active against parasitic nematodes of major socio-economic importance</td>
      <td>Khanna, Varun; Ranganathan, Shoba</td>
      <td>2011</td>
      <td>2011-11-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3278842" target="_blank">PMC3278842</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22373185.0" target="_blank">22373185.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-s13-s25" target="_blank">10.1186/1471-2105-12-s13-s25</a></td>
      <td>ufdgi67q</td>
      <td>0.348892</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ctpgylnr";>167</th>
      <td>Yuan_2013</td>
      <td>Monitoring Influenza Epidemics in China with Search Query from Baidu</td>
      <td>Yuan, Qingyu; Nsoesie, Elaine O.; Lv, Benfu; Peng, Geng; Chunara, Rumi; Brownstein, John S.</td>
      <td>2013</td>
      <td>2013-05-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667820" target="_blank">PMC3667820</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23750192.0" target="_blank">23750192.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0064323" target="_blank">10.1371/journal.pone.0064323</a></td>
      <td>ctpgylnr</td>
      <td>0.348443</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_06.html#yupm6smb">Liang_2018</a></td>
    </tr>
    <tr>
      <th id="qaqvg453";>168</th>
      <td>Saw_A_2019</td>
      <td>Alignment-free method for DNA sequence clustering using Fuzzy integral similarity</td>
      <td>Saw, Ajay Kumar; Raj, Garima; Das, Manashi; Talukdar, Narayan Chandra; Tripathy, Binod Chandra; Nandi, Soumyadeep</td>
      <td>2019</td>
      <td>2019-03-06</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6403383" target="_blank">PMC6403383</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30842590.0" target="_blank">30842590.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-019-40452-6" target="_blank">10.1038/s41598-019-40452-6</a></td>
      <td>qaqvg453</td>
      <td>0.347587</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_05.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_05.html#buc4dd5y">Dong_2019</a></td>
    </tr>
    <tr>
      <th id="9zm4per4";>169</th>
      <td>Dawson_2007</td>
      <td>Prediction of RNA Pseudoknots Using Heuristic Modeling with Mapping and Sequential Folding</td>
      <td>Dawson, Wayne K.; Fujiwara, Kazuya; Kawai, Gota</td>
      <td>2007</td>
      <td>2007-09-19</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1975678" target="_blank">PMC1975678</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17878940.0" target="_blank">17878940.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0000905" target="_blank">10.1371/journal.pone.0000905</a></td>
      <td>9zm4per4</td>
      <td>0.347531</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="mxprt6x6";>170</th>
      <td>Souza_2020</td>
      <td>An Innovative Big Data Predictive Analytics Framework over Hybrid Big Data Sources with an Application for Disease Analytics</td>
      <td>Souza, Joglas; Leung, Carson K.; Cuzzocrea, Alfredo</td>
      <td>2020</td>
      <td>2020-03-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123615" target="_blank">PMC7123615</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-44041-1_59" target="_blank">10.1007/978-3-030-44041-1_59</a></td>
      <td>mxprt6x6</td>
      <td>0.346352</td>
      <td><a href="Topic_02.html#m0q52a2t">Kurc_2006</a></td>
      <td><a href="Topic_09.html#3gv6469y">Bello-Orgaz_2016</a>, <a href="Topic_06.html#03tx0rni">VanderWaal_2017</a>, <a href="Topic_06.html#uxmqlx4y">Yang_2019</a></td>
    </tr>
    <tr>
      <th id="6n88mcbf";>171</th>
      <td>Milano_2020</td>
      <td>Statistical and network-based analysis of Italian COVID-19 data: communities detection and temporal evolution</td>
      <td>Marianna Milano; Mario Cannataro</td>
      <td>2020</td>
      <td>2020-04-22</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.17.20068916" target="_blank">10.1101/2020.04.17.20068916</a></td>
      <td>6n88mcbf</td>
      <td>0.344769</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="w88hdlmr";>172</th>
      <td>Qi_X_2020</td>
      <td>A network-based method with privacy-preserving for identifying influential providers in large healthcare service systems</td>
      <td>Qi, Xiaoyu; Mei, Gang; Cuomo, Salvatore; Xiao, Lei</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.future.2020.04.004" target="_blank">10.1016/j.future.2020.04.004</a></td>
      <td>w88hdlmr</td>
      <td>0.344239</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="nray6171";>173</th>
      <td>Tan_T_2007</td>
      <td>A novel cognitive interpretation of breast cancer thermography with complementary learning fuzzy neural memory structure</td>
      <td>Tan, T.Z.; Quek, C.; Ng, G.S.; Ng, E.Y.K.</td>
      <td>2007</td>
      <td>2007-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126614" target="_blank">PMC7126614</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2006.06.012" target="_blank">10.1016/j.eswa.2006.06.012</a></td>
      <td>nray6171</td>
      <td>0.342953</td>
      <td></td>
      <td><a href="Topic_03.html#1r4gm2d4">Liu_Y_2020</a></td>
    </tr>
    <tr>
      <th id="8f1arjw1";>174</th>
      <td>Klein_2020</td>
      <td>A Chronological and Geographical Analysis of Personal Reports of COVID-19 on Twitter</td>
      <td>Ari Klein; Arjun Magge; Karen O'Connor; Haitao Cai; Davy Weissenbacher; Graciela Gonzalez-Hernandez</td>
      <td>2020</td>
      <td>2020-04-22</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.19.20069948" target="_blank">10.1101/2020.04.19.20069948</a></td>
      <td>8f1arjw1</td>
      <td>0.341829</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="ml2luc1t";>175</th>
      <td>Sollner_2008</td>
      <td>Analysis and prediction of protective continuous B-cell epitopes on pathogen proteins</td>
      <td>Sollner, Johannes; Grohmann, Rainer; Rapberger, Ronald; Perco, Paul; Lukas, Arno; Mayer, Bernd</td>
      <td>2008</td>
      <td>2008-01-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244602" target="_blank">PMC2244602</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18179690.0" target="_blank">18179690.0</a></td>
      <td><a href="https://doi.org/10.1186/1745-7580-4-1" target="_blank">10.1186/1745-7580-4-1</a></td>
      <td>ml2luc1t</td>
      <td>0.341533</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="p8n8va3y";>176</th>
      <td>Kim_M_2013</td>
      <td>A mass weighted chemical elastic network model elucidates closed form domain motions in proteins</td>
      <td>Kim, Min Hyeok; Seo, Sangjae; Jeong, Jay Il; Kim, Bum Joon; Liu, Wing Kam; Lim, Byeong Soo; Choi, Jae Boong; Kim, Moon Ki</td>
      <td>2013</td>
      <td>2013-03-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3649262" target="_blank">PMC3649262</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23456820.0" target="_blank">23456820.0</a></td>
      <td><a href="https://doi.org/10.1002/pro.2244" target="_blank">10.1002/pro.2244</a></td>
      <td>p8n8va3y</td>
      <td>0.340899</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="7izahzqy";>177</th>
      <td>Rochefort_2015</td>
      <td>Accuracy of using automated methods for detecting adverse events from electronic health record data: a research protocol</td>
      <td>Rochefort, Christian M; Buckeridge, David L; Forster, Alan J</td>
      <td>2015</td>
      <td>2015-01-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296680" target="_blank">PMC4296680</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25567422.0" target="_blank">25567422.0</a></td>
      <td><a href="https://doi.org/10.1186/s13012-014-0197-6" target="_blank">10.1186/s13012-014-0197-6</a></td>
      <td>7izahzqy</td>
      <td>0.339432</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="ckby80vf";>178</th>
      <td>Gardner_2005</td>
      <td>Software for optimization of SNP and PCR-RFLP genotyping to discriminate many genomes with the fewest assays</td>
      <td>Gardner, Shea N; Wagner, Mark C</td>
      <td>2005</td>
      <td>2005-05-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1156889" target="_blank">PMC1156889</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15904493.0" target="_blank">15904493.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2164-6-73" target="_blank">10.1186/1471-2164-6-73</a></td>
      <td>ckby80vf</td>
      <td>0.338658</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="qu4enz7m";>179</th>
      <td>Frenz_2019</td>
      <td>Automatically Fixing Errors in Glycoprotein Structures with Rosetta</td>
      <td>Frenz, Brandon; Rämisch, Sebastian; Borst, Andrew J.; Walls, Alexandra C.; Adolf-Bryfogle, Jared; Schief, William R.; Veesler, David; DiMaio, Frank</td>
      <td>2019</td>
      <td>2019-01-02</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6616339" target="_blank">PMC6616339</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30344107.0" target="_blank">30344107.0</a></td>
      <td><a href="https://doi.org/10.1016/j.str.2018.09.006" target="_blank">10.1016/j.str.2018.09.006</a></td>
      <td>qu4enz7m</td>
      <td>0.337598</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_10.html#9zm4per4">Dawson_2007</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="y0s4cf6g";>180</th>
      <td>Zou_Z_2013</td>
      <td>Semantic overlay network for large-scale spatial information indexing</td>
      <td>Zou, Zhiqiang; Wang, Yue; Cao, Kai; Qu, Tianshan; Wang, Zhongmin</td>
      <td>2013</td>
      <td>2013-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125740" target="_blank">PMC7125740</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cageo.2013.04.019" target="_blank">10.1016/j.cageo.2013.04.019</a></td>
      <td>y0s4cf6g</td>
      <td>0.336586</td>
      <td><a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_09.html#eydkfqi2">Feng_2019</a>, <a href="Topic_13.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_02.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_01.html#4wnoa4l7">Cai_Y_2019</a></td>
    </tr>
    <tr>
      <th id="5yk1j4ms";>181</th>
      <td>Gorbalenya_2008</td>
      <td>Phylogeny of Viruses</td>
      <td>Gorbalenya, A.E.</td>
      <td>2008</td>
      <td>2008-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7150205" target="_blank">PMC7150205</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-012374410-4.00712-3" target="_blank">10.1016/b978-012374410-4.00712-3</a></td>
      <td>5yk1j4ms</td>
      <td>0.336438</td>
      <td><a href="Topic_05.html#g6obo798">Zheng_2008</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_13.html#3atcqejc">Kinene_2016</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="z29xsj1o";>182</th>
      <td>Morfopoulou_2015</td>
      <td>Bayesian mixture analysis for metagenomic community profiling</td>
      <td>Morfopoulou, Sofia; Plagnol, Vincent</td>
      <td>2015</td>
      <td>2015-09-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4565032" target="_blank">PMC4565032</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26002885.0" target="_blank">26002885.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv317" target="_blank">10.1093/bioinformatics/btv317</a></td>
      <td>z29xsj1o</td>
      <td>0.336438</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_08.html#058r9486">Morfopoulou_2014</a></td>
    </tr>
    <tr>
      <th id="udzz15bk";>183</th>
      <td>Li_H_2009</td>
      <td>An effective docking strategy for virtual screening based on multi-objective optimization algorithm</td>
      <td>Li, Honglin; Zhang, Hailei; Zheng, Mingyue; Luo, Jie; Kang, Ling; Liu, Xiaofeng; Wang, Xicheng; Jiang, Hualiang</td>
      <td>2009</td>
      <td>2009-02-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753843" target="_blank">PMC2753843</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19210777.0" target="_blank">19210777.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-58" target="_blank">10.1186/1471-2105-10-58</a></td>
      <td>udzz15bk</td>
      <td>0.336122</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_07.html#7vwpvcdd">Liu_S_2012</a></td>
    </tr>
    <tr>
      <th id="9rx4jlw8";>184</th>
      <td>Kim_K_2007</td>
      <td>Selecting Genotyping Oligo Probes Via Logical Analysis of Data</td>
      <td>Kim, Kwangsoo; Ryoo, Hong Seo</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122177" target="_blank">PMC7122177</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72665-4_8" target="_blank">10.1007/978-3-540-72665-4_8</a></td>
      <td>9rx4jlw8</td>
      <td>0.334487</td>
      <td><a href="Topic_05.html#nfxcqsc2">Kim_K_2007</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_05.html#nfxcqsc2">Kim_K_2007</a></td>
    </tr>
    <tr>
      <th id="j97ugs3y";>185</th>
      <td>Santosh_2020</td>
      <td>AI-Driven Tools for Coronavirus Outbreak: Need of Active Learning and Cross-Population Train/Test Models on Multitudinal/Multimodal Data</td>
      <td>Santosh, K. C.</td>
      <td>2020</td>
      <td>2020-03-18</td>
      <td>PMC</td>
      <td>Y</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087612" target="_blank">PMC7087612</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32189081.0" target="_blank">32189081.0</a></td>
      <td><a href="https://doi.org/10.1007/s10916-020-01562-1" target="_blank">10.1007/s10916-020-01562-1</a></td>
      <td>j97ugs3y</td>
      <td>0.334438</td>
      <td></td>
      <td><a href="Topic_03.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_03.html#nvavj9gk">Andre_2020</a>, <a href="Topic_03.html#kjovtgua">Wu_J_2020</a></td>
    </tr>
    <tr>
      <th id="cjcvm3x7";>186</th>
      <td>Chung_2007</td>
      <td>RE-MuSiC: a tool for multiple sequence alignment with regular expression constraints</td>
      <td>Chung, Yun-Sheng; Lee, Wei-Hsun; Tang, Chuan Yi; Lu, Chin Lung</td>
      <td>2007</td>
      <td>2007-05-08</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933182" target="_blank">PMC1933182</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17488842.0" target="_blank">17488842.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm275" target="_blank">10.1093/nar/gkm275</a></td>
      <td>cjcvm3x7</td>
      <td>0.334085</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_10.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_10.html#gjild96e">Song_2005</a>, <a href="Topic_10.html#s7kzp795">Zemla_2011</a>, <a href="Topic_05.html#9zfct1yx">Altschul_1991</a></td>
    </tr>
    <tr>
      <th id="ldze0a22";>187</th>
      <td>Wang_2018</td>
      <td>Prediction of RNA-protein interactions using conjoint triad feature and chaos game representation</td>
      <td>Wang, Hongchu; Wu, Pengfei</td>
      <td>2018</td>
      <td>2018-08-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6984769" target="_blank">PMC6984769</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30117758.0" target="_blank">30117758.0</a></td>
      <td><a href="https://doi.org/10.1080/21655979.2018.1470721" target="_blank">10.1080/21655979.2018.1470721</a></td>
      <td>ldze0a22</td>
      <td>0.331245</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_04.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_04.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_04.html#6oca1mrm">Shen_2018</a>, <a href="Topic_04.html#qyd6qyjf">Xiao_2017</a></td>
    </tr>
    <tr>
      <th id="60udcsa7";>188</th>
      <td>Hou_Z_2016</td>
      <td>Linear Quantitative Profiling Method Fast Monitors Alkaloids of Sophora Flavescens That Was Verified by Tri-Marker Analyses</td>
      <td>Hou, Zhifei; Sun, Guoxiang; Guo, Yong</td>
      <td>2016</td>
      <td>2016-08-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987015" target="_blank">PMC4987015</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27529425.0" target="_blank">27529425.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0161146" target="_blank">10.1371/journal.pone.0161146</a></td>
      <td>60udcsa7</td>
      <td>0.331192</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="hjo78o1q";>189</th>
      <td>Shen_2006</td>
      <td>Probabilistic analysis of the frequencies of amino acid pairs within characterized protein sequences</td>
      <td>Shen, Shiyi; Kai, Bo; Ruan, Jishou; Torin Huzil, J.; Carpenter, Eric; Tuszynski, Jack A.</td>
      <td>2006</td>
      <td>2006-10-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127678" target="_blank">PMC7127678</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.physa.2006.03.004" target="_blank">10.1016/j.physa.2006.03.004</a></td>
      <td>hjo78o1q</td>
      <td>0.330567</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="u14tj0so";>190</th>
      <td>Jabbari_2014</td>
      <td>A fast and robust iterative algorithm for prediction of RNA pseudoknotted secondary structures</td>
      <td>Jabbari, Hosna; Condon, Anne</td>
      <td>2014</td>
      <td>2014-05-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4064103" target="_blank">PMC4064103</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24884954.0" target="_blank">24884954.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-147" target="_blank">10.1186/1471-2105-15-147</a></td>
      <td>u14tj0so</td>
      <td>0.330069</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_10.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="7qc1lqyp";>191</th>
      <td>Cheng_2012</td>
      <td>Segmentation of DNA using simple recurrent neural network</td>
      <td>Cheng, Wei-Chen; Huang, Jau-Chi; Liou, Cheng-Yuan</td>
      <td>2012</td>
      <td>2012-02-29</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126336" target="_blank">PMC7126336</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2011.09.001" target="_blank">10.1016/j.knosys.2011.09.001</a></td>
      <td>7qc1lqyp</td>
      <td>0.329424</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="s5k59eq8";>192</th>
      <td>Nooij_2018</td>
      <td>Overview of Virus Metagenomic Classification Methods and Their Biological Applications</td>
      <td>Nooij, Sam; Schmitz, Dennis; Vennema, Harry; Kroneman, Annelies; Koopmans, Marion P. G.</td>
      <td>2018</td>
      <td>2018-04-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924777" target="_blank">PMC5924777</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29740407.0" target="_blank">29740407.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2018.00749" target="_blank">10.3389/fmicb.2018.00749</a></td>
      <td>s5k59eq8</td>
      <td>0.327223</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="xurmzmwj";>193</th>
      <td>Lin_X_2020</td>
      <td>A Review on Applications of Computational Methods in Drug Screening and Design</td>
      <td>Lin, Xiaoqian; Li, Xiu; Lin, Xubo</td>
      <td>2020</td>
      <td>2020-03-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7144386" target="_blank">PMC7144386</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32197324.0" target="_blank">32197324.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules25061375" target="_blank">10.3390/molecules25061375</a></td>
      <td>xurmzmwj</td>
      <td>0.326818</td>
      <td></td>
      <td><a href="Topic_07.html#qah877pk">Ekins_2007</a></td>
    </tr>
    <tr>
      <th id="3m1zet9t";>194</th>
      <td>Cheng_2010</td>
      <td>Manifold construction based on local distance invariance</td>
      <td>Cheng, Wei-Chen; Liou, Cheng-Yuan</td>
      <td>2010</td>
      <td>2010-02-11</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091362" target="_blank">PMC7091362</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s12293-009-0030-y" target="_blank">10.1007/s12293-009-0030-y</a></td>
      <td>3m1zet9t</td>
      <td>0.326675</td>
      <td><a href="Topic_05.html#zocunore">Liao_2006</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_01.html#ear9cyri">Bakker_2018</a>, <a href="Topic_13.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_01.html#89xnnvuv">Saad_2005</a>, <a href="Topic_09.html#eydkfqi2">Feng_2019</a></td>
    </tr>
    <tr>
      <th id="uiqk356j";>195</th>
      <td>Kamvar_2019</td>
      <td>Epidemic curves made easy using the R package incidence</td>
      <td>Kamvar, Zhian N.; Cai, Jun; Pulliam, Juliet R.C.; Schumacher, Jakob; Jombart, Thibaut</td>
      <td>2019</td>
      <td>2019-01-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6509961" target="_blank">PMC6509961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31119031.0" target="_blank">31119031.0</a></td>
      <td><a href="https://doi.org/10.12688/f1000research.18002.1" target="_blank">10.12688/f1000research.18002.1</a></td>
      <td>uiqk356j</td>
      <td>0.325198</td>
      <td><a href="Topic_02.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_06.html#4524vg4x">Collier_2007</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ltf82dh3";>196</th>
      <td>Liu_W_2006</td>
      <td>Quantitative prediction of mouse class I MHC peptide binding affinity using support vector machine regression (SVR) models</td>
      <td>Liu, Wen; Meng, Xiangshan; Xu, Qiqi; Flower, Darren R; Li, Tongbin</td>
      <td>2006</td>
      <td>2006-03-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513606" target="_blank">PMC1513606</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16579851.0" target="_blank">16579851.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-182" target="_blank">10.1186/1471-2105-7-182</a></td>
      <td>ltf82dh3</td>
      <td>0.324577</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_11.html#71njvx7l">Soam_2010</a></td>
    </tr>
    <tr>
      <th id="j11yaga2";>197</th>
      <td>Meyerholz_2018</td>
      <td>Fundamental Concepts for Semiquantitative Tissue Scoring in Translational Research</td>
      <td>Meyerholz, David K; Beck, Amanda P</td>
      <td>2018</td>
      <td>2018-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927897" target="_blank">PMC6927897</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30715381.0" target="_blank">30715381.0</a></td>
      <td><a href="https://doi.org/10.1093/ilar/ily025" target="_blank">10.1093/ilar/ily025</a></td>
      <td>j11yaga2</td>
      <td>0.322260</td>
      <td><a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="yjon4plr";>198</th>
      <td>Dai_Q_2009</td>
      <td>Using Gaussian model to improve biological sequence comparison</td>
      <td>Dai, Qi; Liu, Xiaoqing; Li, Lihua; Yao, Yuhua; Han, Bin; Zhu, Lei</td>
      <td>2009</td>
      <td>2009-05-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7166749" target="_blank">PMC7166749</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19479732.0" target="_blank">19479732.0</a></td>
      <td><a href="https://doi.org/10.1002/jcc.21322" target="_blank">10.1002/jcc.21322</a></td>
      <td>yjon4plr</td>
      <td>0.319287</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="e9app2cv";>199</th>
      <td>Lu_C_2005</td>
      <td>A memory-efficient algorithm for multiple sequence alignment with constraints</td>
      <td>Lu, Chin Lung; Huang, Yen Pin</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109922" target="_blank">PMC7109922</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15374876.0" target="_blank">15374876.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bth468" target="_blank">10.1093/bioinformatics/bth468</a></td>
      <td>e9app2cv</td>
      <td>0.319068</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#gjild96e">Song_2005</a>, <a href="Topic_10.html#cjcvm3x7">Chung_2007</a>, <a href="Topic_01.html#zuh78iul">Pessia_2015</a>, <a href="Topic_10.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="qyd6qyjf";>200</th>
      <td>Xiao_2017</td>
      <td>Prediction of lncRNA-protein interactions using HeteSim scores based on heterogeneous networks</td>
      <td>Xiao, Yun; Zhang, Jingpu; Deng, Lei</td>
      <td>2017</td>
      <td>2017-06-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5473862" target="_blank">PMC5473862</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28623317.0" target="_blank">28623317.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-017-03986-1" target="_blank">10.1038/s41598-017-03986-1</a></td>
      <td>qyd6qyjf</td>
      <td>0.314903</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_04.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="kt9sbn40";>201</th>
      <td>Lee_G_2009</td>
      <td>Proportional fault-tolerant data mining with applications to bioinformatics</td>
      <td>Lee, Guanling; Peng, Sheng-Lung; Lin, Yuh-Tzu</td>
      <td>2009</td>
      <td>2009-02-19</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087812" target="_blank">PMC7087812</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10796-009-9158-z" target="_blank">10.1007/s10796-009-9158-z</a></td>
      <td>kt9sbn40</td>
      <td>0.312900</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_02.html#183n1lrf">Wilson_2003</a>, <a href="Topic_01.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_01.html#pdf3i2tt">Wu_Y_2020</a></td>
    </tr>
    <tr>
      <th id="04t18oqp";>202</th>
      <td>Kingsley_2016</td>
      <td>Ranking Protein-Protein Docking Results using Steered Molecular Dynamics and Potential of Mean Force Calculations</td>
      <td>Kingsley, Laura J.; Esquivel-Rodríguez, Juan; Yang, Ying; Kihara, Daisuke; Lill, Markus A.</td>
      <td>2016</td>
      <td>2016-05-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5015890" target="_blank">PMC5015890</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27232548.0" target="_blank">27232548.0</a></td>
      <td><a href="https://doi.org/10.1002/jcc.24412" target="_blank">10.1002/jcc.24412</a></td>
      <td>04t18oqp</td>
      <td>0.311383</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_07.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="8nw2al5o";>203</th>
      <td>Perez_2011</td>
      <td>Global animal disease surveillance</td>
      <td>Perez, A.; AlKhamis, M.; Carlsson, U.; Brito, B.; Carrasco-Medanic, R.; Whedbee, Z.; Willeberg, P.</td>
      <td>2011</td>
      <td>2011-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22748173.0" target="_blank">22748173.0</a></td>
      <td><a href="https://doi.org/10.1016/j.sste.2011.07.006" target="_blank">10.1016/j.sste.2011.07.006</a></td>
      <td>8nw2al5o</td>
      <td>0.310821</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="91fhb66m";>204</th>
      <td>Hashemian_2010</td>
      <td>Advanced Querying Features for Disease Surveillance Systems</td>
      <td>Hashemian, Mohammad R.</td>
      <td>2010</td>
      <td>2010-04-09</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3615752" target="_blank">PMC3615752</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23569575.0" target="_blank">23569575.0</a></td>
      <td><a href="https://doi.org/10.5210/ojphi.v2i1.2847" target="_blank">10.5210/ojphi.v2i1.2847</a></td>
      <td>91fhb66m</td>
      <td>0.310408</td>
      <td><a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_06.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_06.html#vkjz1orh">Chapman_2004</a></td>
    </tr>
    <tr>
      <th id="qk1pr335";>205</th>
      <td>Sun_P_2015</td>
      <td>Conformational B-Cell Epitope Prediction Method Based on Antigen Preprocessing and Mimotopes Analysis</td>
      <td>Sun, Pingping; Ju, Haixu; Zhang, Baowen; Gu, Yu; Liu, Bo; Huang, Yanxin; Zhang, Huijie; Li, Yuxin</td>
      <td>2015</td>
      <td>2015-01-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326220" target="_blank">PMC4326220</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25705652.0" target="_blank">25705652.0</a></td>
      <td><a href="https://doi.org/10.1155/2015/257030" target="_blank">10.1155/2015/257030</a></td>
      <td>qk1pr335</td>
      <td>0.310408</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="etifey83";>206</th>
      <td>Gao_L_2011</td>
      <td>Remote Data Retrieval for Bioinformatics Applications: An Agent Migration Approach</td>
      <td>Gao, Lei; Dai, Hua; Zhang, Tong-Liang; Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-06-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119054" target="_blank">PMC3119054</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21701677.0" target="_blank">21701677.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0020949" target="_blank">10.1371/journal.pone.0020949</a></td>
      <td>etifey83</td>
      <td>0.309687</td>
      <td><a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_02.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_02.html#60vrlrim">Lefkowitz_2008</a>, <a href="Topic_02.html#eawmuuye">Stokes_1993</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a></td>
    </tr>
    <tr>
      <th id="8dnyjuig";>207</th>
      <td>Alejandro_2018</td>
      <td>Ensemble Feature Selection and Meta-Analysis of Cancer miRNA Biomarkers</td>
      <td>Lopez-Rincon Alejandro; Martinez-Archundia Marlet; Martinez-Ruiz Gustavo Ulises; Tonda Alberto</td>
      <td>2018</td>
      <td>2018-06-21</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/353201" target="_blank">10.1101/353201</a></td>
      <td>8dnyjuig</td>
      <td>0.309327</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="kauk7ze0";>208</th>
      <td>Durrant_2013</td>
      <td>Comparing Neural-Network Scoring Functions and the State of the Art: Applications to Common Library Screening</td>
      <td>Durrant, Jacob D.; Friedman, Aaron\nJ.; Rogers, Kathleen\nE.; McCammon, J. Andrew</td>
      <td>2013</td>
      <td>2013-06-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3735370" target="_blank">PMC3735370</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23734946.0" target="_blank">23734946.0</a></td>
      <td><a href="https://doi.org/10.1021/ci400042y" target="_blank">10.1021/ci400042y</a></td>
      <td>kauk7ze0</td>
      <td>0.309120</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_07.html#udzz15bk">Li_H_2009</a></td>
    </tr>
    <tr>
      <th id="wcpc32w4";>209</th>
      <td>Johnson_2019</td>
      <td>An Ecological Framework for Modeling the Geography of Disease Transmission</td>
      <td>Johnson, Erica E.; Escobar, Luis E.; Zambrana-Torrelio, Carlos</td>
      <td>2019</td>
      <td>2019-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114676" target="_blank">PMC7114676</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31078330.0" target="_blank">31078330.0</a></td>
      <td><a href="https://doi.org/10.1016/j.tree.2019.03.004" target="_blank">10.1016/j.tree.2019.03.004</a></td>
      <td>wcpc32w4</td>
      <td>0.308169</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="bqwmx5dy";>210</th>
      <td>Punjani_2019</td>
      <td>Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</td>
      <td>Ali Punjani; Haowei Zhang; David J. Fleet</td>
      <td>2019</td>
      <td>2019-12-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2019.12.15.877092" target="_blank">10.1101/2019.12.15.877092</a></td>
      <td>bqwmx5dy</td>
      <td>0.306465</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_01.html#yxo8oanc">Punjani_2020</a>, <a href="Topic_01.html#oohx14kl">Zhong_2020</a>, <a href="Topic_14.html#3u81ntle">Acar_2016</a></td>
    </tr>
    <tr>
      <th id="vtktgyvh";>211</th>
      <td>Seo_J_2019</td>
      <td>An Exploration of Machine Learning Methods for Robust Boredom Classification Using EEG and GSR Data</td>
      <td>Seo, Jungryul; Laine, Teemu H.; Sohn, Kyung-Ah</td>
      <td>2019</td>
      <td>2019-10-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6832442" target="_blank">PMC6832442</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31635194.0" target="_blank">31635194.0</a></td>
      <td><a href="https://doi.org/10.3390/s19204561" target="_blank">10.3390/s19204561</a></td>
      <td>vtktgyvh</td>
      <td>0.306355</td>
      <td><a href="Topic_01.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_03.html#1r4gm2d4">Liu_Y_2020</a>, <a href="Topic_09.html#p7ga4h70">Jung_2017</a></td>
    </tr>
    <tr>
      <th id="oeokiay7";>212</th>
      <td>Fusani_2018</td>
      <td>Active learning strategies with COMBINE analysis: new tricks for an old dog</td>
      <td>Fusani, Lucia; Cabrera, Alvaro Cortes</td>
      <td>2018</td>
      <td>2018-12-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087723" target="_blank">PMC7087723</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30564994.0" target="_blank">30564994.0</a></td>
      <td><a href="https://doi.org/10.1007/s10822-018-0181-3" target="_blank">10.1007/s10822-018-0181-3</a></td>
      <td>oeokiay7</td>
      <td>0.305595</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#bs17igjs">Zhou_2010</a>, <a href="Topic_07.html#qah877pk">Ekins_2007</a>, <a href="Topic_07.html#kauk7ze0">Durrant_2013</a></td>
    </tr>
    <tr>
      <th id="d66rz6dc";>213</th>
      <td>Webb_2014</td>
      <td>Comparative Modeling of Drug Target Proteins☆</td>
      <td>Webb, B.; Eswar, N.; Fan, H.; Khuri, N.; Pieper, U.; Dong, G.Q.; Sali, A.</td>
      <td>2014</td>
      <td>2014-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7157477" target="_blank">PMC7157477</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-409547-2.11133-3" target="_blank">10.1016/b978-0-12-409547-2.11133-3</a></td>
      <td>d66rz6dc</td>
      <td>0.303950</td>
      <td><a href="Topic_07.html#oqxh7egj">Eswar_2007</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_07.html#oqxh7egj">Eswar_2007</a></td>
    </tr>
    <tr>
      <th id="81wig9xh";>214</th>
      <td>Chan_2010</td>
      <td>Surveillance and Epidemiology of Infectious Diseases using Spatial and Temporal Lustering Methods</td>
      <td>Chan, Ta-Chien; King, Chwan-Chuen</td>
      <td>2010</td>
      <td>2010-07-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120538" target="_blank">PMC7120538</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-1-4419-6892-0_10" target="_blank">10.1007/978-1-4419-6892-0_10</a></td>
      <td>81wig9xh</td>
      <td>0.303011</td>
      <td><a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="p8puu1jk";>215</th>
      <td>Li_M_2015</td>
      <td>Predicting RNA Secondary Structures: One-grammar-fits-all Solution</td>
      <td>Li, Menglu; Cheng, Micheal; Ye, Yongtao; Hon, Wk; Ting, Hf; Lam, Tw; Tang, Cy; Wong, Thomas; Yiu, Sm</td>
      <td>2015</td>
      <td>2015-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121278" target="_blank">PMC7121278</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-19048-8_18" target="_blank">10.1007/978-3-319-19048-8_18</a></td>
      <td>p8puu1jk</td>
      <td>0.301737</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_10.html#1lojd0xa">Gao_J_2010</a></td>
    </tr>
    <tr>
      <th id="fsh3gsvd";>216</th>
      <td>Xiao_2005</td>
      <td>Using cellular automata images and pseudo amino acid composition to predict protein subcellular location</td>
      <td>Xiao, X.; Shao, S.; Ding, Y.; Huang, Z.; Chou, K.-C.</td>
      <td>2005</td>
      <td>2005-07-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087770" target="_blank">PMC7087770</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16044193.0" target="_blank">16044193.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-005-0225-6" target="_blank">10.1007/s00726-005-0225-6</a></td>
      <td>fsh3gsvd</td>
      <td>0.301223</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_04.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_04.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_15.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_04.html#u1imic5l">Li_C_2018</a></td>
    </tr>
  </tbody>
</table>
</body>
</html>
