
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 11 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 11 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="y0wf456f";>1</th>
      <td>Zhang_2005</td>
      <td>PRED(BALB/c): a system for the prediction of peptide binding to H2(d) molecules, a haplotype of the BALB/c mouse</td>
      <td>Zhang, Guang Lan; Srinivasan, Kellathur N.; Veeramani, Anitha; August, J. Thomas; Brusic, Vladimir</td>
      <td>2005</td>
      <td>2005-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1160239" target="_blank">PMC1160239</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15980450.0" target="_blank">15980450.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gki479" target="_blank">10.1093/nar/gki479</a></td>
      <td>y0wf456f</td>
      <td>0.777179</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4hlcxccy";>2</th>
      <td>Agrawal_2018</td>
      <td>Prediction of Antimicrobial Potential of a Chemically Modified Peptide From Its Tertiary Structure</td>
      <td>Agrawal, Piyush; Raghava, Gajendra P. S.</td>
      <td>2018</td>
      <td>2018-10-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6212470" target="_blank">PMC6212470</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30416494.0" target="_blank">30416494.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2018.02551" target="_blank">10.3389/fmicb.2018.02551</a></td>
      <td>4hlcxccy</td>
      <td>0.773446</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="vmc0rrrk";>3</th>
      <td>Srinivasan_2004</td>
      <td>Prediction of class I T-cell epitopes: evidence of presence of immunological hot spots inside antigens</td>
      <td>Srinivasan, K.N.; Zhang, G.L.; Khan, A.M.; August, J.T.; Brusic, V.</td>
      <td>2004</td>
      <td>2004-08-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110022" target="_blank">PMC7110022</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15262812.0" target="_blank">15262812.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bth943" target="_blank">10.1093/bioinformatics/bth943</a></td>
      <td>vmc0rrrk</td>
      <td>0.757664</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="lal14r4s";>4</th>
      <td>Negi_2009</td>
      <td>Automated Detection of Conformational Epitopes Using Phage Display Peptide Sequences</td>
      <td>Negi, Surendra S; Braun, Werner</td>
      <td>2009</td>
      <td>2009-07-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808184" target="_blank">PMC2808184</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20140073.0" target="_blank">20140073.0</a></td>
      <td></td>
      <td>lal14r4s</td>
      <td>0.704959</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ap60vws9";>5</th>
      <td>Qureshi_2013</td>
      <td>VIRsiRNApred: a web server for predicting inhibition efficacy of siRNAs targeting human viruses</td>
      <td>Qureshi, Abid; Thakur, Nishant; Kumar, Manoj</td>
      <td>2013</td>
      <td>2013-12-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3878835" target="_blank">PMC3878835</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24330765.0" target="_blank">24330765.0</a></td>
      <td><a href="https://doi.org/10.1186/1479-5876-11-305" target="_blank">10.1186/1479-5876-11-305</a></td>
      <td>ap60vws9</td>
      <td>0.698262</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="f0ahybvs";>6</th>
      <td>Zhang_2008</td>
      <td>Hotspot Hunter: a computational system for large-scale screening and selection of candidate immunological hotspots in pathogen proteomes</td>
      <td>Zhang, Guang Lan; Khan, Asif M; Srinivasan, Kellathur N; Heiny, AT; Lee, KX; Kwoh, Chee Keong; August, J Thomas; Brusic, Vladimir</td>
      <td>2008</td>
      <td>2008-02-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2259420" target="_blank">PMC2259420</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18315850.0" target="_blank">18315850.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-s1-s19" target="_blank">10.1186/1471-2105-9-s1-s19</a></td>
      <td>f0ahybvs</td>
      <td>0.689700</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="k935v0z9";>7</th>
      <td>Beltrán_Lissabet_2019</td>
      <td>AntiVPP 1.0: A portable tool for prediction of antiviral peptides</td>
      <td>Beltrán Lissabet, Jorge Félix; Belén, Lisandra Herrera; Farias, Jorge G.</td>
      <td>2019</td>
      <td>2019-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094449" target="_blank">PMC7094449</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30802694.0" target="_blank">30802694.0</a></td>
      <td><a href="https://doi.org/10.1016/j.compbiomed.2019.02.011" target="_blank">10.1016/j.compbiomed.2019.02.011</a></td>
      <td>k935v0z9</td>
      <td>0.653569</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="7rs4jtfh";>8</th>
      <td>Mayrose_2007</td>
      <td>Pepitope: epitope mapping from affinity-selected peptides</td>
      <td>Mayrose, Itay; Penn, Osnat; Erez, Elana; Rubinstein, Nimrod D.; Shlomi, Tomer; Freund, Natalia Tarnovitski; Bublil, Erez M.; Ruppin, Eytan; Sharan, Roded; Gershoni, Jonathan M.; Martz, Eric; Pupko, Tal</td>
      <td>2007</td>
      <td>2007-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110138" target="_blank">PMC7110138</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17977889.0" target="_blank">17977889.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btm493" target="_blank">10.1093/bioinformatics/btm493</a></td>
      <td>7rs4jtfh</td>
      <td>0.636050</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="71njvx7l";>9</th>
      <td>Soam_2010</td>
      <td>Improved Prediction of MHC Class I Binders/Non-Binders Peptides Through Artificial Neural Network Using Variable Learning Rate: SARS Corona Virus, a Case Study</td>
      <td>Soam, Sudhir Singh; Bhasker, Bharat; Mishra, Bhartendu Nath</td>
      <td>2010</td>
      <td>2010-11-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123181" target="_blank">PMC7123181</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21431562.0" target="_blank">21431562.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-4419-7046-6_22" target="_blank">10.1007/978-1-4419-7046-6_22</a></td>
      <td>71njvx7l</td>
      <td>0.632325</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="t3zampk7";>10</th>
      <td>Chen_2012</td>
      <td>PepMapper: A Collaborative Web Tool for Mapping Epitopes from Affinity-Selected Peptides</td>
      <td>Chen, Wenhan; Guo, William W.; Huang, Yanxin; Ma, Zhiqiang</td>
      <td>2012</td>
      <td>2012-05-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3360666" target="_blank">PMC3360666</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22701536.0" target="_blank">22701536.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0037869" target="_blank">10.1371/journal.pone.0037869</a></td>
      <td>t3zampk7</td>
      <td>0.614924</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_11.html#7rs4jtfh">Mayrose_2007</a>, <a href="Topic_11.html#qk1pr335">Sun_P_2015</a></td>
    </tr>
    <tr>
      <th id="hi7z3nob";>11</th>
      <td>Huang_2010</td>
      <td>SAROTUP: Scanner and Reporter of Target-Unrelated Peptides</td>
      <td>Huang, Jian; Ru, Beibei; Li, Shiyong; Lin, Hao; Guo, Feng-Biao</td>
      <td>2010</td>
      <td>2010-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842971" target="_blank">PMC2842971</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20339521.0" target="_blank">20339521.0</a></td>
      <td><a href="https://doi.org/10.1155/2010/101932" target="_blank">10.1155/2010/101932</a></td>
      <td>hi7z3nob</td>
      <td>0.602839</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="qk1pr335";>12</th>
      <td>Sun_P_2015</td>
      <td>Conformational B-Cell Epitope Prediction Method Based on Antigen Preprocessing and Mimotopes Analysis</td>
      <td>Sun, Pingping; Ju, Haixu; Zhang, Baowen; Gu, Yu; Liu, Bo; Huang, Yanxin; Zhang, Huijie; Li, Yuxin</td>
      <td>2015</td>
      <td>2015-01-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326220" target="_blank">PMC4326220</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25705652.0" target="_blank">25705652.0</a></td>
      <td><a href="https://doi.org/10.1155/2015/257030" target="_blank">10.1155/2015/257030</a></td>
      <td>qk1pr335</td>
      <td>0.589863</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="wczw4vn1";>13</th>
      <td>He_Y_2010</td>
      <td>Vaxign: The First Web-Based Vaccine Design Program for Reverse Vaccinology and Applications for Vaccine Development</td>
      <td>He, Yongqun; Xiang, Zuoshuang; Mobley, Harry L. T.</td>
      <td>2010</td>
      <td>2010-07-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2910479" target="_blank">PMC2910479</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20671958.0" target="_blank">20671958.0</a></td>
      <td><a href="https://doi.org/10.1155/2010/297505" target="_blank">10.1155/2010/297505</a></td>
      <td>wczw4vn1</td>
      <td>0.582130</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="9lqh0evn";>14</th>
      <td>Zhang_2013</td>
      <td>A combination of epitope prediction and molecular docking allows for good identification of MHC class I restricted T-cell epitopes</td>
      <td>Zhang, Xue Wu</td>
      <td>2013</td>
      <td>2013-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106517" target="_blank">PMC7106517</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23666426.0" target="_blank">23666426.0</a></td>
      <td><a href="https://doi.org/10.1016/j.compbiolchem.2013.03.003" target="_blank">10.1016/j.compbiolchem.2013.03.003</a></td>
      <td>9lqh0evn</td>
      <td>0.580412</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="rtpbltkf";>15</th>
      <td>EL-Manzalawy_2008</td>
      <td>Predicting linear B-cell epitopes using string kernels</td>
      <td>EL-Manzalawy, Yasser; Dobbs, Drena; Honavar, Vasant</td>
      <td>2008</td>
      <td>2008-07-01</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683948" target="_blank">PMC2683948</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18496882.0" target="_blank">18496882.0</a></td>
      <td><a href="https://doi.org/10.1002/jmr.893" target="_blank">10.1002/jmr.893</a></td>
      <td>rtpbltkf</td>
      <td>0.564925</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="v4xue7ha";>16</th>
      <td>Xu_Y_2015</td>
      <td>Identification of Peptide Inhibitors of Enveloped Viruses Using Support Vector Machine</td>
      <td>Xu, Yongtao; Yu, Shui; Zou, Jian-Wei; Hu, Guixiang; Rahman, Noorsaadah A. B. D.; Othman, Rozana Binti; Tao, Xia; Huang, Meilan</td>
      <td>2015</td>
      <td>2015-12-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4670226" target="_blank">PMC4670226</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26636321.0" target="_blank">26636321.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0144171" target="_blank">10.1371/journal.pone.0144171</a></td>
      <td>v4xue7ha</td>
      <td>0.547108</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="a6y0eoov";>17</th>
      <td>Qureshi_2015</td>
      <td>AVP‐IC(50)Pred: Multiple machine learning techniques‐based prediction of peptide antiviral activity in terms of half maximal inhibitory concentration (IC(50))</td>
      <td>Qureshi, Abid; Tandon, Himani; Kumar, Manoj</td>
      <td>2015</td>
      <td>2015-11-26</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7161829" target="_blank">PMC7161829</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26213387.0" target="_blank">26213387.0</a></td>
      <td><a href="https://doi.org/10.1002/bip.22703" target="_blank">10.1002/bip.22703</a></td>
      <td>a6y0eoov</td>
      <td>0.540292</td>
      <td><a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ltf82dh3";>18</th>
      <td>Liu_W_2006</td>
      <td>Quantitative prediction of mouse class I MHC peptide binding affinity using support vector machine regression (SVR) models</td>
      <td>Liu, Wen; Meng, Xiangshan; Xu, Qiqi; Flower, Darren R; Li, Tongbin</td>
      <td>2006</td>
      <td>2006-03-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513606" target="_blank">PMC1513606</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16579851.0" target="_blank">16579851.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-182" target="_blank">10.1186/1471-2105-7-182</a></td>
      <td>ltf82dh3</td>
      <td>0.516329</td>
      <td><a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_11.html#71njvx7l">Soam_2010</a></td>
    </tr>
    <tr>
      <th id="2upc2spn";>19</th>
      <td>Qiu_T_2018</td>
      <td>CE-BLAST makes it possible to compute antigenic similarity for newly emerging pathogens</td>
      <td>Qiu, Tianyi; Yang, Yiyan; Qiu, Jingxuan; Huang, Yang; Xu, Tianlei; Xiao, Han; Wu, Dingfeng; Zhang, Qingchen; Zhou, Chen; Zhang, Xiaoyan; Tang, Kailin; Xu, Jianqing; Cao, Zhiwei</td>
      <td>2018</td>
      <td>2018-05-02</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5932059" target="_blank">PMC5932059</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29720583.0" target="_blank">29720583.0</a></td>
      <td><a href="https://doi.org/10.1038/s41467-018-04171-2" target="_blank">10.1038/s41467-018-04171-2</a></td>
      <td>2upc2spn</td>
      <td>0.485626</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="ajzk86rq";>20</th>
      <td>van_Weezep_2019</td>
      <td>PCR diagnostics: In silico validation by an automated tool using freely available software programs</td>
      <td>van Weezep, Erik; Kooi, Engbert A.; van Rijn, Piet A.</td>
      <td>2019</td>
      <td>2019-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7113775" target="_blank">PMC7113775</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31095975.0" target="_blank">31095975.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jviromet.2019.05.002" target="_blank">10.1016/j.jviromet.2019.05.002</a></td>
      <td>ajzk86rq</td>
      <td>0.450862</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="zwbc7nnn";>21</th>
      <td>Sathiamurthy_2005</td>
      <td>An ontology for immune epitopes: application to the design of a broad scope database of immune reactivities</td>
      <td>Sathiamurthy, Muthuraman; Peters, Bjoern; Bui, Huynh-Hoa; Sidney, John; Mokili, John; Wilson, Stephen S; Fleri, Ward; McGuinness, Deborah L; Bourne, Philip E; Sette, Alessandro</td>
      <td>2005</td>
      <td>2005-09-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1287064" target="_blank">PMC1287064</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16305755.0" target="_blank">16305755.0</a></td>
      <td><a href="https://doi.org/10.1186/1745-7580-1-2" target="_blank">10.1186/1745-7580-1-2</a></td>
      <td>zwbc7nnn</td>
      <td>0.432848</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="1ypgij14";>22</th>
      <td>Huang_2006</td>
      <td>MIMOX: a web tool for phage display based epitope mapping</td>
      <td>Huang, Jian; Gutteridge, Alex; Honda, Wataru; Kanehisa, Minoru</td>
      <td>2006</td>
      <td>2006-10-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1618411" target="_blank">PMC1618411</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17038191.0" target="_blank">17038191.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-451" target="_blank">10.1186/1471-2105-7-451</a></td>
      <td>1ypgij14</td>
      <td>0.411368</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_11.html#7rs4jtfh">Mayrose_2007</a></td>
    </tr>
    <tr>
      <th id="ml2luc1t";>23</th>
      <td>Sollner_2008</td>
      <td>Analysis and prediction of protective continuous B-cell epitopes on pathogen proteins</td>
      <td>Sollner, Johannes; Grohmann, Rainer; Rapberger, Ronald; Perco, Paul; Lukas, Arno; Mayer, Bernd</td>
      <td>2008</td>
      <td>2008-01-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2244602" target="_blank">PMC2244602</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18179690.0" target="_blank">18179690.0</a></td>
      <td><a href="https://doi.org/10.1186/1745-7580-4-1" target="_blank">10.1186/1745-7580-4-1</a></td>
      <td>ml2luc1t</td>
      <td>0.410577</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#qrnhp1ek">Kirillova_2009</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="gsv6dh36";>24</th>
      <td>Qureshi_2016</td>
      <td>AVCpred: an integrated web server for prediction and design of antiviral compounds</td>
      <td>Qureshi, Abid; Kaur, Gazaldeep; Kumar, Manoj</td>
      <td>2016</td>
      <td>2016-09-09</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162012" target="_blank">PMC7162012</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27490990.0" target="_blank">27490990.0</a></td>
      <td><a href="https://doi.org/10.1111/cbdd.12834" target="_blank">10.1111/cbdd.12834</a></td>
      <td>gsv6dh36</td>
      <td>0.399146</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#71tnv4lp">Santillana_2014</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="9gozwdng";>25</th>
      <td>Barbarini_2011</td>
      <td>Prediction of Peptide Reactivity with Human IVIg through a Knowledge-Based Approach</td>
      <td>Barbarini, Nicola; Tiengo, Alessandra; Bellazzi, Riccardo</td>
      <td>2011</td>
      <td>2011-08-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3160895" target="_blank">PMC3160895</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21887285.0" target="_blank">21887285.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0023616" target="_blank">10.1371/journal.pone.0023616</a></td>
      <td>9gozwdng</td>
      <td>0.368784</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_11.html#ml2luc1t">Sollner_2008</a></td>
    </tr>
    <tr>
      <th id="uh5rkjwx";>26</th>
      <td>González-Díaz_2014</td>
      <td>Model for Vaccine Design by Prediction of B-Epitopes of IEDB Given Perturbations in Peptide Sequence, In Vivo Process, Experimental Techniques, and Source or Host Organisms</td>
      <td>González-Díaz, Humberto; Pérez-Montoto, Lázaro G.; Ubeira, Florencio M.</td>
      <td>2014</td>
      <td>2014-01-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3987976" target="_blank">PMC3987976</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24741624.0" target="_blank">24741624.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/768515" target="_blank">10.1155/2014/768515</a></td>
      <td>uh5rkjwx</td>
      <td>0.365122</td>
      <td><a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="whoydkd5";>27</th>
      <td>Cao_Y_2005</td>
      <td>Information theory-based algorithm for in silico prediction of PCR products with whole genomic sequences as templates</td>
      <td>Cao, Youfang; Wang, Lianjie; Xu, Kexue; Kou, Chunhai; Zhang, Yulei; Wei, Guifang; He, Junjian; Wang, Yunfang; Zhao, Liping</td>
      <td>2005</td>
      <td>2005-07-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1183192" target="_blank">PMC1183192</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16042814.0" target="_blank">16042814.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-6-190" target="_blank">10.1186/1471-2105-6-190</a></td>
      <td>whoydkd5</td>
      <td>0.311430</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
  </tbody>
</table>
</body>
</html>
