
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 08 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 08 -- Abstracts with Biological Entities (English) - 75 Topics / Sub-Topic Model 25 - 15 Topics</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="qndr28n6";>1</th>
      <td>Bashir_2020</td>
      <td>An efficient pattern growth approach for mining fault tolerant frequent itemsets</td>
      <td>Bashir, Shariq</td>
      <td>2020</td>
      <td>2020-04-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126664" target="_blank">PMC7126664</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2019.113046" target="_blank">10.1016/j.eswa.2019.113046</a></td>
      <td>qndr28n6</td>
      <td>0.677543</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_01.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_08.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_01.html#pdf3i2tt">Wu_Y_2020</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="feq9grdc";>2</th>
      <td>Isa_Irawan_2019</td>
      <td>Application of Needleman-Wunch Algorithm to identify mutation in DNA sequences of Corona virus</td>
      <td>Isa Irawan, Mohammad; Mukhlash, Imam; Rizky, Abduh; Ririsati Dewi, Alfiana</td>
      <td>2019</td>
      <td>2019-05-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106772" target="_blank">PMC7106772</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1088/1742-6596/1218/1/012031" target="_blank">10.1088/1742-6596/1218/1/012031</a></td>
      <td>feq9grdc</td>
      <td>0.675180</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="csokkcqq";>3</th>
      <td>Shcherbina_2016</td>
      <td>Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim</td>
      <td>Anna Shcherbina; Darrell O. Ricke; Nelson Chiu</td>
      <td>2016</td>
      <td>2016-03-31</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/046532" target="_blank">10.1101/046532</a></td>
      <td>csokkcqq</td>
      <td>0.663562</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="skmxrkhf";>4</th>
      <td>Park_2018</td>
      <td>Technological opportunity discovery for technological convergence based on the prediction of technology knowledge flow in a citation network</td>
      <td>Park, Inchae; Yoon, Byungun</td>
      <td>2018</td>
      <td>2018-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.joi.2018.09.007" target="_blank">10.1016/j.joi.2018.09.007</a></td>
      <td>skmxrkhf</td>
      <td>0.641896</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="mpp3ctem";>5</th>
      <td>Zhang_2019</td>
      <td>Rapid identification of human‐infecting viruses</td>
      <td>Zhang, Zheng; Cai, Zena; Tan, Zhiying; Lu, Congyu; Jiang, Taijiao; Zhang, Gaihua; Peng, Yousong</td>
      <td>2019</td>
      <td>2019-08-12</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7168554" target="_blank">PMC7168554</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31373773.0" target="_blank">31373773.0</a></td>
      <td><a href="https://doi.org/10.1111/tbed.13314" target="_blank">10.1111/tbed.13314</a></td>
      <td>mpp3ctem</td>
      <td>0.631768</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="besvz92f";>6</th>
      <td>Schäffer_2019</td>
      <td>VADR: validation and annotation of virus sequence submissions to GenBank</td>
      <td>Alejandro A Schäffer; Eneida Hatcher; Linda Yankie; Lara Shonkwiler; J Rodney Brister; Ilene Karsch-Mizrachi; Eric P Nawrocki</td>
      <td>2019</td>
      <td>2019-11-22</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/852657" target="_blank">10.1101/852657</a></td>
      <td>besvz92f</td>
      <td>0.630349</td>
      <td><a href="Topic_02.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_15.html#a564l6vs">UNKNOWN_2016</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="o4y8ci0n";>7</th>
      <td>Mock_2019</td>
      <td>VIDHOP, viral host prediction with Deep Learning</td>
      <td>Florian Mock; Adrian Viehweger; Emanuel Barth; Manja Marz</td>
      <td>2019</td>
      <td>2019-03-12</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/575571" target="_blank">10.1101/575571</a></td>
      <td>o4y8ci0n</td>
      <td>0.554193</td>
      <td></td>
      <td><a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a></td>
    </tr>
    <tr>
      <th id="x8jytnj6";>8</th>
      <td>Crossman_2020</td>
      <td>Leverging Deep Learning to Simulate Coronavirus Spike proteins has the potential to predict future Zoonotic sequences</td>
      <td>Lisa Caroline Crossman</td>
      <td>2020</td>
      <td>2020-04-20</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.20.046920" target="_blank">10.1101/2020.04.20.046920</a></td>
      <td>x8jytnj6</td>
      <td>0.543863</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="z29xsj1o";>9</th>
      <td>Morfopoulou_2015</td>
      <td>Bayesian mixture analysis for metagenomic community profiling</td>
      <td>Morfopoulou, Sofia; Plagnol, Vincent</td>
      <td>2015</td>
      <td>2015-09-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4565032" target="_blank">PMC4565032</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26002885.0" target="_blank">26002885.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv317" target="_blank">10.1093/bioinformatics/btv317</a></td>
      <td>z29xsj1o</td>
      <td>0.536707</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_08.html#058r9486">Morfopoulou_2014</a></td>
    </tr>
    <tr>
      <th id="wtarrhf0";>10</th>
      <td>Broseus_2020</td>
      <td>TALC: Transcription Aware Long Read Correction</td>
      <td>Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie</td>
      <td>2020</td>
      <td>2020-01-11</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.10.901728" target="_blank">10.1101/2020.01.10.901728</a></td>
      <td>wtarrhf0</td>
      <td>0.522036</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="hphn7cs6";>11</th>
      <td>Cacciabue_2019</td>
      <td>A beginner’s guide for FMDV quasispecies analysis: sub-consensus variant detection and haplotype reconstruction using next-generation sequencing</td>
      <td>Cacciabue, Marco; Currá, Anabella; Carrillo, Elisa; König, Guido; Gismondi, María Inés</td>
      <td>2019</td>
      <td>2019-11-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110011" target="_blank">PMC7110011</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31697321.0" target="_blank">31697321.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bbz086" target="_blank">10.1093/bib/bbz086</a></td>
      <td>hphn7cs6</td>
      <td>0.511555</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="worgd0xu";>12</th>
      <td>Hatcher_2017</td>
      <td>Virus Variation Resource – improved response to emergent viral outbreaks</td>
      <td>Hatcher, Eneida L.; Zhdanov, Sergey A.; Bao, Yiming; Blinkova, Olga; Nawrocki, Eric P.; Ostapchuck, Yuri; Schäffer, Alejandro A.; Brister, J. Rodney</td>
      <td>2017</td>
      <td>2017-01-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210549" target="_blank">PMC5210549</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27899678.0" target="_blank">27899678.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1065" target="_blank">10.1093/nar/gkw1065</a></td>
      <td>worgd0xu</td>
      <td>0.508562</td>
      <td><a href="Topic_02.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_15.html#a564l6vs">UNKNOWN_2016</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="pd0cmefo";>13</th>
      <td>Shang_2020</td>
      <td>CHEER: hierarCHical taxonomic classification for viral mEtagEnomic data via deep leaRning</td>
      <td>Jiayu Shang; Yanni Sun</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.26.009001" target="_blank">10.1101/2020.03.26.009001</a></td>
      <td>pd0cmefo</td>
      <td>0.487917</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_08.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a></td>
    </tr>
    <tr>
      <th id="a0bbw3er";>14</th>
      <td>Waagmeester_2020</td>
      <td>A protocol for adding knowledge to Wikidata, a case report</td>
      <td>Andra Waagmeester; Egon L. Willighagen; Andrew I. Su; Martina Kutmon; Jose Emilio Labra Gayo; Daniel Fernández-Álvarez; Peter J. Schaap; Lisa M. Verhagen; Jasper J. Koehorst</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.05.026336" target="_blank">10.1101/2020.04.05.026336</a></td>
      <td>a0bbw3er</td>
      <td>0.483328</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#hgxhyg46">Greenbaum_2009</a>, <a href="Topic_06.html#4524vg4x">Collier_2007</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="058r9486";>15</th>
      <td>Morfopoulou_2014</td>
      <td>Bayesian mixture analysis for metagenomic community profiling.</td>
      <td>Sofia Morfopoulou; Vincent Plagnol</td>
      <td>2014</td>
      <td>2014-07-25</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/007476" target="_blank">10.1101/007476</a></td>
      <td>058r9486</td>
      <td>0.469642</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_08.html#z29xsj1o">Morfopoulou_2015</a></td>
    </tr>
    <tr>
      <th id="rxhygr8x";>16</th>
      <td>Li_H_2018</td>
      <td>Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences</td>
      <td>Li, Han; Sun, Fengzhu</td>
      <td>2018</td>
      <td>2018-07-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030160" target="_blank">PMC6030160</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29968780.0" target="_blank">29968780.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-018-28308-x" target="_blank">10.1038/s41598-018-28308-x</a></td>
      <td>rxhygr8x</td>
      <td>0.462619</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="aa88gtzr";>17</th>
      <td>Bao_Y_2008</td>
      <td>Virus Classification by Pairwise Sequence Comparison (PASC)</td>
      <td>Bao, Y.; Kapustin, Y.; Tatusova, T.</td>
      <td>2008</td>
      <td>2008-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-012374410-4.00710-x" target="_blank">10.1016/b978-012374410-4.00710-x</a></td>
      <td>aa88gtzr</td>
      <td>0.451682</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ac00tai9";>18</th>
      <td>Bartoszewicz_2020</td>
      <td>Interpretable detection of novel human viruses from genome sequencing data</td>
      <td>Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard</td>
      <td>2020</td>
      <td>2020-01-30</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.29.925354" target="_blank">10.1101/2020.01.29.925354</a></td>
      <td>ac00tai9</td>
      <td>0.443129</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a></td>
    </tr>
    <tr>
      <th id="55lkfivh";>19</th>
      <td>Cho_M_2019</td>
      <td>Prediction of cross-species infection propensities of viruses with receptor similarity</td>
      <td>Cho, Myeongji; Son, Hyeon Seok</td>
      <td>2019</td>
      <td>2019-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106226" target="_blank">PMC7106226</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31026604.0" target="_blank">31026604.0</a></td>
      <td><a href="https://doi.org/10.1016/j.meegid.2019.04.016" target="_blank">10.1016/j.meegid.2019.04.016</a></td>
      <td>55lkfivh</td>
      <td>0.442290</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#7qc1lqyp">Cheng_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="llv3cvdr";>20</th>
      <td>Qiang_2020</td>
      <td>Using the spike protein feature to predict infection risk and monitor the evolutionary dynamic of coronavirus</td>
      <td>Qiang, Xiao-Li; Xu, Peng; Fang, Gang; Liu, Wen-Bin; Kou, Zheng</td>
      <td>2020</td>
      <td>2020-03-25</td>
      <td>COMM-USE</td>
      <td>Y</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093988" target="_blank">PMC7093988</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32209118.0" target="_blank">32209118.0</a></td>
      <td><a href="https://doi.org/10.1186/s40249-020-00649-8" target="_blank">10.1186/s40249-020-00649-8</a></td>
      <td>llv3cvdr</td>
      <td>0.426641</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="pdf3i2tt";>21</th>
      <td>Wu_Y_2020</td>
      <td>NetNCSP: Nonoverlapping closed sequential pattern mining</td>
      <td>Wu, Youxi; Zhu, Changrui; Li, Yan; Guo, Lei; Wu, Xindong</td>
      <td>2020</td>
      <td>2020-03-31</td>
      <td>PMC</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2020.105812" target="_blank">10.1016/j.knosys.2020.105812</a></td>
      <td>pdf3i2tt</td>
      <td>0.418818</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_08.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_01.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="uw76y16g";>22</th>
      <td>Tithi_2017</td>
      <td>Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data</td>
      <td>Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang</td>
      <td>2017</td>
      <td>2017-10-02</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/196998" target="_blank">10.1101/196998</a></td>
      <td>uw76y16g</td>
      <td>0.418711</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="nz4qpumr";>23</th>
      <td>Sahli_2012</td>
      <td>Arapan-S: a fast and highly accurate whole-genome assembly software for viruses and small genomes</td>
      <td>Sahli, Mohammed; Shibuya, Tetsuo</td>
      <td>2012</td>
      <td>2012-05-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441218" target="_blank">PMC3441218</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22591859.0" target="_blank">22591859.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-5-243" target="_blank">10.1186/1756-0500-5-243</a></td>
      <td>nz4qpumr</td>
      <td>0.413332</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="oinumv1z";>24</th>
      <td>Kalantar_2020</td>
      <td>IDseq – An Open Source Cloud-based Pipeline and Analysis Service for Metagenomic Pathogen Detection and Monitoring</td>
      <td>Katrina L. Kalantar; Tiago Carvalho; Charles F.A. de Bourcy; Boris Dimitrov; Greg Dingle; Rebecca Egger; Julie Han; Olivia B. Holmes; Yun-Fang Juan; Ryan King; Andrey Kislyuk; Maria Mariano; Lucia V. Reynoso; David Rissato Cruz; Jonathan Sheu; Jennifer Tang; James Wang; Mark A. Zhang; Emily Zhong; Vida Ahyong; Sreyngim Lay; Sophana Chea; Jennifer A. Bohl; Jessica E. Manning; Cristina M. Tato; Joseph L. DeRisi</td>
      <td>2020</td>
      <td>2020-04-09</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.07.030551" target="_blank">10.1101/2020.04.07.030551</a></td>
      <td>oinumv1z</td>
      <td>0.399420</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_02.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_02.html#m0q52a2t">Kurc_2006</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="4uvezfqm";>25</th>
      <td>Zhou_2018</td>
      <td>A generalized approach to predicting protein-protein interactions between virus and host</td>
      <td>Zhou, Xiang; Park, Byungkyu; Choi, Daesik; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-08-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6101077" target="_blank">PMC6101077</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30367586.0" target="_blank">30367586.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-018-4924-2" target="_blank">10.1186/s12864-018-4924-2</a></td>
      <td>4uvezfqm</td>
      <td>0.399118</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="bmcku2yb";>26</th>
      <td>Wilm_2012</td>
      <td>LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets</td>
      <td>Wilm, Andreas; Aw, Pauline Poh Kim; Bertrand, Denis; Yeo, Grace Hui Ting; Ong, Swee Hoe; Wong, Chang Hua; Khor, Chiea Chuen; Petric, Rosemary; Hibberd, Martin Lloyd; Nagarajan, Niranjan</td>
      <td>2012</td>
      <td>2012-10-12</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" target="_blank">PMC3526318</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23066108.0" target="_blank">23066108.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gks918" target="_blank">10.1093/nar/gks918</a></td>
      <td>bmcku2yb</td>
      <td>0.395284</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="ddu6yl8r";>27</th>
      <td>Adetiba_2018</td>
      <td>Alignment-Free Z-Curve Genomic Cepstral Coefficients and Machine Learning for Classification of Viruses</td>
      <td>Adetiba, Emmanuel; Olugbara, Oludayo O.; Taiwo, Tunmike B.; Adebiyi, Marion O.; Badejo, Joke A.; Akanle, Matthew B.; Matthews, Victor O.</td>
      <td>2018</td>
      <td>2018-03-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120486" target="_blank">PMC7120486</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-78723-7_25" target="_blank">10.1007/978-3-319-78723-7_25</a></td>
      <td>ddu6yl8r</td>
      <td>0.395051</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_03.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_08.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_03.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_03.html#2l1zw19o">Ozturk_2020</a></td>
    </tr>
    <tr>
      <th id="p6075gva";>28</th>
      <td>Scheuch_2015</td>
      <td>RIEMS: a software pipeline for sensitive and comprehensive taxonomic classification of reads from metagenomics datasets</td>
      <td>Scheuch, Matthias; Höper, Dirk; Beer, Martin</td>
      <td>2015</td>
      <td>2015-03-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4351923" target="_blank">PMC4351923</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25886935.0" target="_blank">25886935.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-015-0503-6" target="_blank">10.1186/s12859-015-0503-6</a></td>
      <td>p6075gva</td>
      <td>0.393667</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_02.html#m0q52a2t">Kurc_2006</a></td>
      <td><a href="Topic_08.html#csokkcqq">Shcherbina_2016</a>, <a href="Topic_08.html#z29xsj1o">Morfopoulou_2015</a>, <a href="Topic_08.html#058r9486">Morfopoulou_2014</a></td>
    </tr>
    <tr>
      <th id="xn2wulnp";>29</th>
      <td>Fonseca_2019</td>
      <td>A computational method for the identification of Dengue, Zika and Chikungunya virus species and genotypes</td>
      <td>Fonseca, Vagner; Libin, Pieter J. K.; Theys, Kristof; Faria, Nuno R.; Nunes, Marcio R. T.; Restovic, Maria I.; Freire, Murilo; Giovanetti, Marta; Cuypers, Lize; Nowé, Ann; Abecasis, Ana; Deforche, Koen; Santiago, Gilberto A.; de Siqueira, Isadora C.; San, Emmanuel J.; Machado, Kaliane C. B.; Azevedo, Vasco; Filippis, Ana Maria Bispo-de; da Cunha, Rivaldo Venâncio; Pybus, Oliver G.; Vandamme, Anne-Mieke; Alcantara, Luiz C. J.; de Oliveira, Tulio</td>
      <td>2019</td>
      <td>2019-05-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6527240" target="_blank">PMC6527240</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31067235.0" target="_blank">31067235.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pntd.0007231" target="_blank">10.1371/journal.pntd.0007231</a></td>
      <td>xn2wulnp</td>
      <td>0.389101</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <th id="0dxrai3j";>30</th>
      <td>Alguwaizani_2018</td>
      <td>Predicting Interactions between Virus and Host Proteins Using Repeat Patterns and Composition of Amino Acids</td>
      <td>Alguwaizani, Saud; Park, Byungkyu; Zhou, Xiang; Huang, De-Shuang; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-05-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5966669" target="_blank">PMC5966669</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29854357.0" target="_blank">29854357.0</a></td>
      <td><a href="https://doi.org/10.1155/2018/1391265" target="_blank">10.1155/2018/1391265</a></td>
      <td>0dxrai3j</td>
      <td>0.362645</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_08.html#4uvezfqm">Zhou_2018</a></td>
    </tr>
    <tr>
      <th id="s96d3ust";>31</th>
      <td>Hu_Y_2006</td>
      <td>Mining association rules with multiple minimum supports: a new mining algorithm and a support tuning mechanism</td>
      <td>Hu, Ya-Han; Chen, Yen-Liang</td>
      <td>2006</td>
      <td>2006-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127670" target="_blank">PMC7127670</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.dss.2004.09.007" target="_blank">10.1016/j.dss.2004.09.007</a></td>
      <td>s96d3ust</td>
      <td>0.361892</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_08.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_01.html#pdf3i2tt">Wu_Y_2020</a>, <a href="Topic_01.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="kt9sbn40";>32</th>
      <td>Lee_G_2009</td>
      <td>Proportional fault-tolerant data mining with applications to bioinformatics</td>
      <td>Lee, Guanling; Peng, Sheng-Lung; Lin, Yuh-Tzu</td>
      <td>2009</td>
      <td>2009-02-19</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087812" target="_blank">PMC7087812</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10796-009-9158-z" target="_blank">10.1007/s10796-009-9158-z</a></td>
      <td>kt9sbn40</td>
      <td>0.344724</td>
      <td><a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#w8luu86h">Liou_2013</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_08.html#qndr28n6">Bashir_2020</a>, <a href="Topic_02.html#183n1lrf">Wilson_2003</a>, <a href="Topic_01.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_01.html#pdf3i2tt">Wu_Y_2020</a></td>
    </tr>
    <tr>
      <th id="l9a7upui";>33</th>
      <td>Moshiri_2020</td>
      <td>ViralMSA: Massively scalable reference-guided multiple sequence alignment of viral genomes</td>
      <td>Niema Moshiri</td>
      <td>2020</td>
      <td>2020-04-21</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.20.052068" target="_blank">10.1101/2020.04.20.052068</a></td>
      <td>l9a7upui</td>
      <td>0.320829</td>
      <td><a href="Topic_02.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="2n5vetzw";>34</th>
      <td>Li_K_2010</td>
      <td>ANDES: Statistical tools for the ANalyses of DEep Sequencing</td>
      <td>Li, Kelvin; Venter, Eli; Yooseph, Shibu; Stockwell, Timothy B; Eckerle, Lance D; Denison, Mark R; Spiro, David J; Methé, Barbara A</td>
      <td>2010</td>
      <td>2010-07-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921379" target="_blank">PMC2921379</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20633290.0" target="_blank">20633290.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-3-199" target="_blank">10.1186/1756-0500-3-199</a></td>
      <td>2n5vetzw</td>
      <td>0.309025</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_05.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_05.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="g22ggyxb";>35</th>
      <td>Weissenbacher_2017</td>
      <td>Extracting geographic locations from the literature for virus phylogeography using supervised and distant supervision methods</td>
      <td>Weissenbacher, Davy; Sarker, Abeed; Tahsin, Tasnia; Scotch, Matthew; Gonzalez, Graciela</td>
      <td>2017</td>
      <td>2017-07-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5543364" target="_blank">PMC5543364</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28815119.0" target="_blank">28815119.0</a></td>
      <td></td>
      <td>g22ggyxb</td>
      <td>0.307808</td>
      <td><a href="Topic_05.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_05.html#t1e0tr4w">Lee_C_2017</a></td>
      <td></td>
    </tr>
  </tbody>
</table>
</body>
</html>
