
<!DOCTYPE html>
<html>
<meta charset="UTF-8">
<head>
  <title>Topic 17</title>

<style>
table {
  font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #ddd;
  padding: 8px;
}

tr:nth-child(even){background-color: #f2f2f2;}

tr:hover {background-color: #ddd;}

th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #0099FF;
  color: white;
}
</style>

</head>
<body>
<h2>Topic 17</h2>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>cite ad</th>
      <th>title</th>
      <th>authors</th>
      <th>publish year</th>
      <th>publish time</th>
      <th>dataset</th>
      <th>abstract mentions covid</th>
      <th>pmcid</th>
      <th>pubmed id</th>
      <th>doi</th>
      <th>cord uid</th>
      <th>topic weight</th>
      <th>Similarity scispacy</th>
      <th>Similarity specter</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="qndr28n6";>1</th>
      <td>Bashir_2020</td>
      <td>An efficient pattern growth approach for mining fault tolerant frequent itemsets</td>
      <td>Bashir, Shariq</td>
      <td>2020</td>
      <td>2020-04-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126664" target="_blank">PMC7126664</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2019.113046" target="_blank">10.1016/j.eswa.2019.113046</a></td>
      <td>qndr28n6</td>
      <td>0.918291</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_17.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_17.html#pdf3i2tt">Wu_Y_2020</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="qy4kirx1";>2</th>
      <td>Mendizabal-Ruiz_2018</td>
      <td>Genomic signal processing for DNA sequence clustering</td>
      <td>Mendizabal-Ruiz, Gerardo; Román-Godínez, Israel; Torres-Ramos, Sulema; Salido-Ruiz, Ricardo A.; Vélez-Pérez, Hugo; Morales, J. Alejandro</td>
      <td>2018</td>
      <td>2018-01-24</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5786891" target="_blank">PMC5786891</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29379686.0" target="_blank">29379686.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.4264" target="_blank">10.7717/peerj.4264</a></td>
      <td>qy4kirx1</td>
      <td>0.906269</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#7u5frg8w">Zhao_2011</a>, <a href="Topic_17.html#nw3tybw9">Hoang_2015</a>, <a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#ut9z9eo7">Mu_Z_2019</a></td>
    </tr>
    <tr>
      <th id="ts1eb8fk";>3</th>
      <td>Montgomery_2004</td>
      <td>Sockeye: A 3D Environment for Comparative Genomics</td>
      <td>Montgomery, Stephen B.; Astakhova, Tamara; Bilenky, Mikhail; Birney, Ewan; Fu, Tony; Hassel, Maik; Melsopp, Craig; Rak, Marcin; Robertson, A. Gordon; Sleumer, Monica; Siddiqui, Asim S.; Jones, Steven J.M.</td>
      <td>2004</td>
      <td>2004-05-01</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC479126" target="_blank">PMC479126</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15123592.0" target="_blank">15123592.0</a></td>
      <td><a href="https://doi.org/10.1101/gr.1890304" target="_blank">10.1101/gr.1890304</a></td>
      <td>ts1eb8fk</td>
      <td>0.871561</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_17.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="gkek599u";>4</th>
      <td>Zou_S_2014</td>
      <td>A 2D graphical representation of the sequences of DNA based on triplets and its application</td>
      <td>Zou, Sai; Wang, Lei; Wang, Junfeng</td>
      <td>2014</td>
      <td>2014-01-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3896961" target="_blank">PMC3896961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24383852.0" target="_blank">24383852.0</a></td>
      <td><a href="https://doi.org/10.1186/1687-4153-2014-1" target="_blank">10.1186/1687-4153-2014-1</a></td>
      <td>gkek599u</td>
      <td>0.870527</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#i5fs2yye">Yu_J_2009</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_02.html#ajz4ef44">Nandy_2009</a>, <a href="Topic_17.html#5cxaiw4k">Xie_G_2011</a></td>
    </tr>
    <tr>
      <th id="pdf3i2tt";>5</th>
      <td>Wu_Y_2020</td>
      <td>NetNCSP: Nonoverlapping closed sequential pattern mining</td>
      <td>Wu, Youxi; Zhu, Changrui; Li, Yan; Guo, Lei; Wu, Xindong</td>
      <td>2020</td>
      <td>2020-03-31</td>
      <td>PMC</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2020.105812" target="_blank">10.1016/j.knosys.2020.105812</a></td>
      <td>pdf3i2tt</td>
      <td>0.860926</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#qndr28n6">Bashir_2020</a>, <a href="Topic_17.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_17.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="jvfjf7aw";>6</th>
      <td>Feng_2010</td>
      <td>New method for comparing DNA primary sequences based on a discrimination measure</td>
      <td>Feng, Jie; Hu, Yong; Wan, Ping; Zhang, Aibing; Zhao, Weizhong</td>
      <td>2010</td>
      <td>2010-10-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094107" target="_blank">PMC7094107</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20688082.0" target="_blank">20688082.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.07.040" target="_blank">10.1016/j.jtbi.2010.07.040</a></td>
      <td>jvfjf7aw</td>
      <td>0.847269</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_07.html#le7v89mp">Chang_2011</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a></td>
    </tr>
    <tr>
      <th id="dc45h6v3";>7</th>
      <td>Wang_2014</td>
      <td>ADLD: A Novel Graphical Representation of Protein Sequences and Its Application</td>
      <td>Wang, Lei; Peng, Hui; Zheng, Jinhua</td>
      <td>2014</td>
      <td>2014-10-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4230005" target="_blank">PMC4230005</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25530796.0" target="_blank">25530796.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/959753" target="_blank">10.1155/2014/959753</a></td>
      <td>dc45h6v3</td>
      <td>0.843354</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#8j04y50s">Deng_2014</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
    </tr>
    <tr>
      <th id="lrcxwtx7";>8</th>
      <td>Peng_2018</td>
      <td>Algorithms for interval-valued fuzzy soft sets in emergency decision making based on WDBA and CODAS with new information measure</td>
      <td>Peng, Xindong; Garg, Harish</td>
      <td>2018</td>
      <td>2018-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7116931" target="_blank">PMC7116931</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cie.2018.04.001" target="_blank">10.1016/j.cie.2018.04.001</a></td>
      <td>lrcxwtx7</td>
      <td>0.834568</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_17.html#5q40uttg">Liu_Y_2014</a>, <a href="Topic_71.html#zlzi9197">Zhong_2010</a>, <a href="Topic_19.html#x1zq2i9h">Al-Qaness_2020</a>, <a href="Topic_17.html#mxdlii77">Arji_2019</a></td>
    </tr>
    <tr>
      <th id="odupmsen";>9</th>
      <td>Krantz_2020</td>
      <td>True epidemic growth construction through harmonic analysis</td>
      <td>Krantz, Steven G.; Polyakov, Peter; Rao, Arni S.R. Srinivasa</td>
      <td>2020</td>
      <td>2020-06-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118632" target="_blank">PMC7118632</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32173304.0" target="_blank">32173304.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2020.110243" target="_blank">10.1016/j.jtbi.2020.110243</a></td>
      <td>odupmsen</td>
      <td>0.819905</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_06.html#w4g8mzuy">Groendyke_2012</a>, <a href="Topic_06.html#g8laoz7j">Wang_2014</a>, <a href="Topic_06.html#vt6nh3vs">Kenah_2007</a></td>
    </tr>
    <tr>
      <th id="eydkfqi2";>10</th>
      <td>Feng_2019</td>
      <td>Relative space-based GIS data model to analyze the group dynamics of moving objects</td>
      <td>Feng, Mingxiang; Shaw, Shih-Lung; Fang, Zhixiang; Cheng, Hao</td>
      <td>2019</td>
      <td>2019-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111340" target="_blank">PMC7111340</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.isprsjprs.2019.05.002" target="_blank">10.1016/j.isprsjprs.2019.05.002</a></td>
      <td>eydkfqi2</td>
      <td>0.813014</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_02.html#aebmrvea">Davis_2007</a>, <a href="Topic_71.html#cvlq4i8c">Li_M_2015</a>, <a href="Topic_17.html#y0s4cf6g">Zou_Z_2013</a></td>
    </tr>
    <tr>
      <th id="y0s4cf6g";>11</th>
      <td>Zou_Z_2013</td>
      <td>Semantic overlay network for large-scale spatial information indexing</td>
      <td>Zou, Zhiqiang; Wang, Yue; Cao, Kai; Qu, Tianshan; Wang, Zhongmin</td>
      <td>2013</td>
      <td>2013-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125740" target="_blank">PMC7125740</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cageo.2013.04.019" target="_blank">10.1016/j.cageo.2013.04.019</a></td>
      <td>y0s4cf6g</td>
      <td>0.810842</td>
      <td><a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_17.html#eydkfqi2">Feng_2019</a>, <a href="Topic_17.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_17.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_02.html#4wnoa4l7">Cai_Y_2019</a></td>
    </tr>
    <tr>
      <th id="e9app2cv";>12</th>
      <td>Lu_C_2005</td>
      <td>A memory-efficient algorithm for multiple sequence alignment with constraints</td>
      <td>Lu, Chin Lung; Huang, Yen Pin</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109922" target="_blank">PMC7109922</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15374876.0" target="_blank">15374876.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bth468" target="_blank">10.1093/bioinformatics/bth468</a></td>
      <td>e9app2cv</td>
      <td>0.803370</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_17.html#cjcvm3x7">Chung_2007</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="vhvwxvwq";>13</th>
      <td>Shikder_2019</td>
      <td>An OpenMP-based tool for finding longest common subsequence in bioinformatics</td>
      <td>Shikder, Rayhan; Thulasiraman, Parimala; Irani, Pourang; Hu, Pingzhao</td>
      <td>2019</td>
      <td>2019-04-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6458724" target="_blank">PMC6458724</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30971295.0" target="_blank">30971295.0</a></td>
      <td><a href="https://doi.org/10.1186/s13104-019-4256-6" target="_blank">10.1186/s13104-019-4256-6</a></td>
      <td>vhvwxvwq</td>
      <td>0.784949</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_17.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a></td>
    </tr>
    <tr>
      <th id="w8luu86h";>14</th>
      <td>Liou_2013</td>
      <td>Structural Complexity of DNA Sequence</td>
      <td>Liou, Cheng-Yuan; Tseng, Shen-Han; Cheng, Wei-Chen; Tsai, Huai-Ying</td>
      <td>2013</td>
      <td>2013-04-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3638703" target="_blank">PMC3638703</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23662161.0" target="_blank">23662161.0</a></td>
      <td><a href="https://doi.org/10.1155/2013/628036" target="_blank">10.1155/2013/628036</a></td>
      <td>w8luu86h</td>
      <td>0.779979</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_07.html#le7v89mp">Chang_2011</a>, <a href="Topic_17.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="6dcghdxf";>15</th>
      <td>Jiang_2009</td>
      <td>Reasoning with rough description logics: An approximate concepts approach</td>
      <td>Jiang, Yuncheng; Wang, Ju; Tang, Suqin; Xiao, Bao</td>
      <td>2009</td>
      <td>2009-02-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094356" target="_blank">PMC7094356</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ins.2008.10.021" target="_blank">10.1016/j.ins.2008.10.021</a></td>
      <td>6dcghdxf</td>
      <td>0.778578</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#elf16dno">Jiang_2009</a>, <a href="Topic_02.html#6zjf6qzz">Li_J_2015</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#elf16dno">Jiang_2009</a>, <a href="Topic_17.html#gi8c1omv">Capotorti_2006</a>, <a href="Topic_02.html#6zjf6qzz">Li_J_2015</a></td>
    </tr>
    <tr>
      <th id="i5fs2yye";>16</th>
      <td>Yu_J_2009</td>
      <td>TN curve: A novel 3D graphical representation of DNA sequence based on trinucleotides and its applications</td>
      <td>Yu, Jia-Feng; Sun, Xiao; Wang, Ji-Hua</td>
      <td>2009</td>
      <td>2009-12-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094124" target="_blank">PMC7094124</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19679137.0" target="_blank">19679137.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2009.08.005" target="_blank">10.1016/j.jtbi.2009.08.005</a></td>
      <td>i5fs2yye</td>
      <td>0.773062</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#gkek599u">Zou_S_2014</a>, <a href="Topic_17.html#5cxaiw4k">Xie_G_2011</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#8j04y50s">Deng_2014</a></td>
    </tr>
    <tr>
      <th id="egbkl2v2";>17</th>
      <td>Liu_C_2005</td>
      <td>Profiling and Searching for RNA Pseudoknot Structures in Genomes</td>
      <td>Liu, Chunmei; Song, Yinglei; Malmberg, Russell L.; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122704" target="_blank">PMC7122704</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11428848_123" target="_blank">10.1007/11428848_123</a></td>
      <td>egbkl2v2</td>
      <td>0.763279</td>
      <td><a href="Topic_17.html#jms7hrmp">Liu_C_2005</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#jms7hrmp">Liu_C_2005</a>, <a href="Topic_61.html#a5htfx76">Dost_2006</a>, <a href="Topic_61.html#o7mt8hr6">Kim_J_2007</a></td>
    </tr>
    <tr>
      <th id="6f1vqkqm";>18</th>
      <td>Zhao_2013</td>
      <td>SSW Library: An SIMD Smith-Waterman C/C++ Library for Use in Genomic Applications</td>
      <td>Zhao, Mengyao; Lee, Wan-Ping; Garrison, Erik P.; Marth, Gabor T.</td>
      <td>2013</td>
      <td>2013-12-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3852983" target="_blank">PMC3852983</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24324759.0" target="_blank">24324759.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0082138" target="_blank">10.1371/journal.pone.0082138</a></td>
      <td>6f1vqkqm</td>
      <td>0.761049</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_17.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_17.html#xyru2q9o">Shepard_2016</a>, <a href="Topic_17.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a></td>
    </tr>
    <tr>
      <th id="oim1ifw0";>19</th>
      <td>Hou_W_2017</td>
      <td>A new method to analyze protein sequence similarity using Dynamic Time Warping</td>
      <td>Hou, Wenbing; Pan, Qiuhui; Peng, Qianying; He, Mingfeng</td>
      <td>2017</td>
      <td>2017-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125777" target="_blank">PMC7125777</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27974244.0" target="_blank">27974244.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ygeno.2016.12.002" target="_blank">10.1016/j.ygeno.2016.12.002</a></td>
      <td>oim1ifw0</td>
      <td>0.760903</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_17.html#41atidai">Abo-Elkhier_2019</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#nw3tybw9">Hoang_2015</a></td>
    </tr>
    <tr>
      <th id="pd9a1em0";>20</th>
      <td>Liu_K_2007</td>
      <td>Extended model of topological relations between spatial objects in geographic information systems</td>
      <td>Liu, Kimfung; Shi, Wenzhong</td>
      <td>2007</td>
      <td>2007-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jag.2006.09.004" target="_blank">10.1016/j.jag.2006.09.004</a></td>
      <td>pd9a1em0</td>
      <td>0.758399</td>
      <td><a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#eydkfqi2">Feng_2019</a>, <a href="Topic_17.html#y0s4cf6g">Zou_Z_2013</a>, <a href="Topic_39.html#f6sl2k3n">Grubesic_2009</a>, <a href="Topic_39.html#94iieejz">Dai_L_2018</a></td>
    </tr>
    <tr>
      <th id="s96d3ust";>21</th>
      <td>Hu_Y_2006</td>
      <td>Mining association rules with multiple minimum supports: a new mining algorithm and a support tuning mechanism</td>
      <td>Hu, Ya-Han; Chen, Yen-Liang</td>
      <td>2006</td>
      <td>2006-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127670" target="_blank">PMC7127670</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.dss.2004.09.007" target="_blank">10.1016/j.dss.2004.09.007</a></td>
      <td>s96d3ust</td>
      <td>0.757095</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#qndr28n6">Bashir_2020</a>, <a href="Topic_17.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_17.html#pdf3i2tt">Wu_Y_2020</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a></td>
    </tr>
    <tr>
      <th id="7vfo98ho";>22</th>
      <td>Zhao_2012</td>
      <td>Using Support Vector Machine and Evolutionary Profiles to Predict Antifreeze Protein Sequences</td>
      <td>Zhao, Xiaowei; Ma, Zhiqiang; Yin, Minghao</td>
      <td>2012</td>
      <td>2012-02-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3292016" target="_blank">PMC3292016</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22408447.0" target="_blank">22408447.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms13022196" target="_blank">10.3390/ijms13022196</a></td>
      <td>7vfo98ho</td>
      <td>0.755864</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a>, <a href="Topic_53.html#v4xue7ha">Xu_Y_2015</a></td>
    </tr>
    <tr>
      <th id="61djk4bs";>23</th>
      <td>Wei_D_2012</td>
      <td>A novel hierarchical clustering algorithm for gene sequences</td>
      <td>Wei, Dan; Jiang, Qingshan; Wei, Yanjie; Wang, Shengrui</td>
      <td>2012</td>
      <td>2012-07-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3443659" target="_blank">PMC3443659</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22823405.0" target="_blank">22823405.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-13-174" target="_blank">10.1186/1471-2105-13-174</a></td>
      <td>61djk4bs</td>
      <td>0.752201</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a>, <a href="Topic_17.html#7u5frg8w">Zhao_2011</a></td>
    </tr>
    <tr>
      <th id="y7697hx8";>24</th>
      <td>Dai_Q_2013</td>
      <td>Study of LZ-word distribution and its application for sequence comparison</td>
      <td>Dai, Qi; Yan, Zhaofang; Shi, Zhuoxing; Liu, Xiaoqing; Yao, Yuhua; He, Pingan</td>
      <td>2013</td>
      <td>2013-11-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094135" target="_blank">PMC7094135</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23876763.0" target="_blank">23876763.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2013.07.008" target="_blank">10.1016/j.jtbi.2013.07.008</a></td>
      <td>y7697hx8</td>
      <td>0.749689</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_17.html#d6wcgkln">Wang_2008</a>, <a href="Topic_02.html#ajz4ef44">Nandy_2009</a>, <a href="Topic_17.html#p1i7bx94">Zhang_2013</a></td>
    </tr>
    <tr>
      <th id="6qpsxmgi";>25</th>
      <td>Wang_2006</td>
      <td>MMDB: annotating protein sequences with Entrez's 3D-structure database</td>
      <td>Wang, Yanli; Addess, Kenneth J.; Chen, Jie; Geer, Lewis Y.; He, Jane; He, Siqian; Lu, Shennan; Madej, Thomas; Marchler-Bauer, Aron; Thiessen, Paul A.; Zhang, Naigong; Bryant, Stephen H.</td>
      <td>2006</td>
      <td>2006-11-29</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1751549" target="_blank">PMC1751549</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17135201.0" target="_blank">17135201.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl952" target="_blank">10.1093/nar/gkl952</a></td>
      <td>6qpsxmgi</td>
      <td>0.747743</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a></td>
      <td><a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_17.html#s7kzp795">Zemla_2011</a></td>
    </tr>
    <tr>
      <th id="sgw66xhn";>26</th>
      <td>Saw_A_2019</td>
      <td>Alignment-free similarity analysis for protein sequences based on fuzzy integral</td>
      <td>Saw, Ajay Kumar; Tripathy, Binod Chandra; Nandi, Soumyadeep</td>
      <td>2019</td>
      <td>2019-02-26</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391537" target="_blank">PMC6391537</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30808983.0" target="_blank">30808983.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-019-39477-8" target="_blank">10.1038/s41598-019-39477-8</a></td>
      <td>sgw66xhn</td>
      <td>0.741068</td>
      <td><a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#g1ob16b9">Xie_X_2012</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="t1e0tr4w";>27</th>
      <td>Lee_C_2017</td>
      <td>A Pairwise Alignment Algorithm for Long Sequences of High Similarity</td>
      <td>Lee, Chien-Tai; Peng, Sheng-Lung</td>
      <td>2017</td>
      <td>2017-10-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123042" target="_blank">PMC7123042</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-981-10-5508-9_27" target="_blank">10.1007/978-981-10-5508-9_27</a></td>
      <td>t1e0tr4w</td>
      <td>0.740407</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="jn3aqdck";>28</th>
      <td>Rizzi_2010</td>
      <td>Hierarchical Clustering Using the Arithmetic-Harmonic Cut: Complexity and Experiments</td>
      <td>Rizzi, Romeo; Mahata, Pritha; Mathieson, Luke; Moscato, Pablo</td>
      <td>2010</td>
      <td>2010-12-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2997101" target="_blank">PMC2997101</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21151943.0" target="_blank">21151943.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0014067" target="_blank">10.1371/journal.pone.0014067</a></td>
      <td>jn3aqdck</td>
      <td>0.739820</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_17.html#7u5frg8w">Zhao_2011</a>, <a href="Topic_17.html#fsh3gsvd">Xiao_2005</a></td>
    </tr>
    <tr>
      <th id="x4z2iz7b";>29</th>
      <td>Wąż_P_2013</td>
      <td>Descriptors of 2D-dynamic graphs as a classification tool of DNA sequences</td>
      <td>Wąż, Piotr; Bielińska-Wąż, Dorota; Nandy, Ashesh</td>
      <td>2013</td>
      <td>2013-09-03</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088265" target="_blank">PMC7088265</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10910-013-0249-1" target="_blank">10.1007/s10910-013-0249-1</a></td>
      <td>x4z2iz7b</td>
      <td>0.734776</td>
      <td><a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#i5fs2yye">Yu_J_2009</a>, <a href="Topic_17.html#gkek599u">Zou_S_2014</a>, <a href="Topic_17.html#8j04y50s">Deng_2014</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
    </tr>
    <tr>
      <th id="ut9z9eo7";>30</th>
      <td>Mu_Z_2019</td>
      <td>DCGR: feature extractions from protein sequences based on CGR via remodeling multiple information</td>
      <td>Mu, Zengchao; Yu, Ting; Qi, Enfeng; Liu, Juntao; Li, Guojun</td>
      <td>2019</td>
      <td>2019-06-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6587251" target="_blank">PMC6587251</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31221087.0" target="_blank">31221087.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-2943-x" target="_blank">10.1186/s12859-019-2943-x</a></td>
      <td>ut9z9eo7</td>
      <td>0.734238</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#c3ewuebx">Jia_J_2016</a></td>
    </tr>
    <tr>
      <th id="nw3tybw9";>31</th>
      <td>Hoang_2015</td>
      <td>A new method to cluster DNA sequences using Fourier power spectrum</td>
      <td>Hoang, Tung; Yin, Changchuan; Zheng, Hui; Yu, Chenglong; Lucy He, Rong; Yau, Stephen S.-T.</td>
      <td>2015</td>
      <td>2015-05-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094126" target="_blank">PMC7094126</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25747773.0" target="_blank">25747773.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2015.02.026" target="_blank">10.1016/j.jtbi.2015.02.026</a></td>
      <td>nw3tybw9</td>
      <td>0.733955</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#7u5frg8w">Zhao_2011</a>, <a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a></td>
    </tr>
    <tr>
      <th id="qaqvg453";>32</th>
      <td>Saw_A_2019</td>
      <td>Alignment-free method for DNA sequence clustering using Fuzzy integral similarity</td>
      <td>Saw, Ajay Kumar; Raj, Garima; Das, Manashi; Talukdar, Narayan Chandra; Tripathy, Binod Chandra; Nandi, Soumyadeep</td>
      <td>2019</td>
      <td>2019-03-06</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6403383" target="_blank">PMC6403383</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30842590.0" target="_blank">30842590.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-019-40452-6" target="_blank">10.1038/s41598-019-40452-6</a></td>
      <td>qaqvg453</td>
      <td>0.731566</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a></td>
    </tr>
    <tr>
      <th id="2lg8i9r1";>33</th>
      <td>Dai_Q_2012</td>
      <td>Sequence comparison via polar coordinates representation and curve tree</td>
      <td>Dai, Qi; Guo, Xiaodong; Li, Lihua</td>
      <td>2012</td>
      <td>2012-01-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094155" target="_blank">PMC7094155</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22001081.0" target="_blank">22001081.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2011.09.030" target="_blank">10.1016/j.jtbi.2011.09.030</a></td>
      <td>2lg8i9r1</td>
      <td>0.728987</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_17.html#8j04y50s">Deng_2014</a></td>
    </tr>
    <tr>
      <th id="wvw6q7px";>34</th>
      <td>Jonić_2016</td>
      <td>Versatility of Approximating Single-Particle Electron Microscopy Density Maps Using Pseudoatoms and Approximation-Accuracy Control</td>
      <td>Jonić, Slavica; Sorzano, Carlos Oscar S.</td>
      <td>2016</td>
      <td>2016-12-21</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5209604" target="_blank">PMC5209604</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28097146.0" target="_blank">28097146.0</a></td>
      <td><a href="https://doi.org/10.1155/2016/7060348" target="_blank">10.1155/2016/7060348</a></td>
      <td>wvw6q7px</td>
      <td>0.725514</td>
      <td><a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_17.html#wsgb5s6z">Quintana_1998</a>, <a href="Topic_17.html#bqwmx5dy">Punjani_2019</a>, <a href="Topic_17.html#oohx14kl">Zhong_2020</a></td>
    </tr>
    <tr>
      <th id="2ai20pd1";>35</th>
      <td>HESS_2007</td>
      <td>An empirical test of the midpoint rooting method</td>
      <td>HESS, PABLO N.; DE MORAES RUSSO, CLAUDIA A.</td>
      <td>2007</td>
      <td>2007-12-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110036" target="_blank">PMC7110036</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1111/j.1095-8312.2007.00864.x" target="_blank">10.1111/j.1095-8312.2007.00864.x</a></td>
      <td>2ai20pd1</td>
      <td>0.725233</td>
      <td><a href="Topic_17.html#3atcqejc">Kinene_2016</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#ne9d0siv">Banks_2006</a>, <a href="Topic_07.html#r97x0paz">Khan_2008</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a></td>
    </tr>
    <tr>
      <th id="09a32vyg";>36</th>
      <td>Dong_2017</td>
      <td>Virus Database and Online Inquiry System Based on Natural Vectors</td>
      <td>Dong, Rui; Zheng, Hui; Tian, Kun; Yau, Shek-Chung; Mao, Weiguang; Yu, Wenping; Yin, Changchuan; Yu, Chenglong; He, Rong Lucy; Yang, Jie; Yau, Stephen ST</td>
      <td>2017</td>
      <td>2017-12-17</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5751915" target="_blank">PMC5751915</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29308007.0" target="_blank">29308007.0</a></td>
      <td><a href="https://doi.org/10.1177/1176934317746667" target="_blank">10.1177/1176934317746667</a></td>
      <td>09a32vyg</td>
      <td>0.718287</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a></td>
      <td><a href="Topic_17.html#pbhvz1k0">Hirahata_2006</a>, <a href="Topic_17.html#sbnnh2mm">Pellet_2009</a>, <a href="Topic_17.html#88l9bq4i">Han_L_2005</a></td>
    </tr>
    <tr>
      <th id="mxprt6x6";>37</th>
      <td>Souza_2020</td>
      <td>An Innovative Big Data Predictive Analytics Framework over Hybrid Big Data Sources with an Application for Disease Analytics</td>
      <td>Souza, Joglas; Leung, Carson K.; Cuzzocrea, Alfredo</td>
      <td>2020</td>
      <td>2020-03-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123615" target="_blank">PMC7123615</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-44041-1_59" target="_blank">10.1007/978-3-030-44041-1_59</a></td>
      <td>mxprt6x6</td>
      <td>0.715447</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a></td>
      <td><a href="Topic_17.html#3gv6469y">Bello-Orgaz_2016</a>, <a href="Topic_04.html#03tx0rni">VanderWaal_2017</a>, <a href="Topic_17.html#uxmqlx4y">Yang_2019</a></td>
    </tr>
    <tr>
      <th id="gc9hevy5";>38</th>
      <td>Cho_Y_2005</td>
      <td>Cleavage Site Analysis Using Rule Extraction from Neural Networks</td>
      <td>Cho, Yeun-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114972" target="_blank">PMC7114972</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11539087_132" target="_blank">10.1007/11539087_132</a></td>
      <td>gc9hevy5</td>
      <td>0.709168</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_21.html#sm7gsmvf">Teng_2002</a>, <a href="Topic_16.html#5gsbtfag">Kiemer_2004</a></td>
    </tr>
    <tr>
      <th id="41atidai";>39</th>
      <td>Abo-Elkhier_2019</td>
      <td>Measuring Similarity among Protein Sequences Using a New Descriptor</td>
      <td>Abo-Elkhier, Mervat M.; Abd Elwahaab, Marwa A.; Abo El Maaty, Moheb I.</td>
      <td>2019</td>
      <td>2019-11-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6893242" target="_blank">PMC6893242</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31886192.0" target="_blank">31886192.0</a></td>
      <td><a href="https://doi.org/10.1155/2019/2796971" target="_blank">10.1155/2019/2796971</a></td>
      <td>41atidai</td>
      <td>0.703561</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a></td>
    </tr>
    <tr>
      <th id="nz4qpumr";>40</th>
      <td>Sahli_2012</td>
      <td>Arapan-S: a fast and highly accurate whole-genome assembly software for viruses and small genomes</td>
      <td>Sahli, Mohammed; Shibuya, Tetsuo</td>
      <td>2012</td>
      <td>2012-05-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3441218" target="_blank">PMC3441218</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22591859.0" target="_blank">22591859.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-5-243" target="_blank">10.1186/1756-0500-5-243</a></td>
      <td>nz4qpumr</td>
      <td>0.702435</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_21.html#3g3apbon">Lauber_2013</a></td>
    </tr>
    <tr>
      <th id="ym9j0ei0";>41</th>
      <td>Tao_J_2005</td>
      <td>Probabilistic Reasoning for Closed-Room People Monitoring</td>
      <td>Tao, Ji; Tan, Yap-Peng</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123562" target="_blank">PMC7123562</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/3-540-32367-8_15" target="_blank">10.1007/3-540-32367-8_15</a></td>
      <td>ym9j0ei0</td>
      <td>0.701265</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#dslwklo4">Tao_J_2004</a>, <a href="Topic_17.html#flgc5l04">Budzan_2013</a>, <a href="Topic_17.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_17.html#gm1e9j80">Thi_T_2014</a></td>
    </tr>
    <tr>
      <th id="gi8c1omv";>42</th>
      <td>Capotorti_2006</td>
      <td>Management of uncertainty orderings through ASP</td>
      <td>Capotorti, Andrea; Formisano, Andrea</td>
      <td>2006</td>
      <td>2006-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-044452075-3/50012-1" target="_blank">10.1016/b978-044452075-3/50012-1</a></td>
      <td>gi8c1omv</td>
      <td>0.694942</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_17.html#elf16dno">Jiang_2009</a>, <a href="Topic_17.html#6dcghdxf">Jiang_2009</a>, <a href="Topic_02.html#nouieh2s">Fox_J_2003</a>, <a href="Topic_17.html#jsyuh368">Chu_H_2008</a></td>
    </tr>
    <tr>
      <th id="ldze0a22";>43</th>
      <td>Wang_2018</td>
      <td>Prediction of RNA-protein interactions using conjoint triad feature and chaos game representation</td>
      <td>Wang, Hongchu; Wu, Pengfei</td>
      <td>2018</td>
      <td>2018-08-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6984769" target="_blank">PMC6984769</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30117758.0" target="_blank">30117758.0</a></td>
      <td><a href="https://doi.org/10.1080/21655979.2018.1470721" target="_blank">10.1080/21655979.2018.1470721</a></td>
      <td>ldze0a22</td>
      <td>0.690939</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_17.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_17.html#6oca1mrm">Shen_2018</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a></td>
    </tr>
    <tr>
      <th id="82820j9a";>44</th>
      <td>Nagraj_2018</td>
      <td>epicontacts: Handling, visualisation and analysis of epidemiological contacts</td>
      <td>Nagraj, VP; Randhawa, Nistara; Campbell, Finlay; Crellen, Thomas; Sudre, Bertrand; Jombart, Thibaut</td>
      <td>2018</td>
      <td>2018-10-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6572866" target="_blank">PMC6572866</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31240097.0" target="_blank">31240097.0</a></td>
      <td><a href="https://doi.org/10.12688/f1000research.14492.2" target="_blank">10.12688/f1000research.14492.2</a></td>
      <td>82820j9a</td>
      <td>0.689407</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a></td>
      <td><a href="Topic_06.html#z8ix4kz7">Robert_2020</a>, <a href="Topic_06.html#og0b2l6b">Chen_2007</a>, <a href="Topic_06.html#zejxb953">Chen_2010</a>, <a href="Topic_06.html#l4soafvr">Cori_2018</a></td>
    </tr>
    <tr>
      <th id="1ul9zqhb";>45</th>
      <td>Mousavi_2012</td>
      <td>An enhanced beam search algorithm for the Shortest Common Supersequence Problem</td>
      <td>Mousavi, Sayyed Rasoul; Bahri, Fateme; Tabataba, Farzaneh Sadat</td>
      <td>2012</td>
      <td>2012-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127599" target="_blank">PMC7127599</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.engappai.2011.08.006" target="_blank">10.1016/j.engappai.2011.08.006</a></td>
      <td>1ul9zqhb</td>
      <td>0.683368</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#j51gaesp">Amiri_2018</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a>, <a href="Topic_17.html#jxgd1yc4">Kai_Z_2018</a>, <a href="Topic_17.html#z1xao2ic">Masoudi-Sobhanzadeh_2016</a></td>
    </tr>
    <tr>
      <th id="g4k6pc82";>46</th>
      <td>Munteanu_2008</td>
      <td>Natural/random protein classification models based on star network topological indices</td>
      <td>Munteanu, Cristian Robert; González-Díaz, Humberto; Borges, Fernanda; de Magalhães, Alexandre Lopes</td>
      <td>2008</td>
      <td>2008-10-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094162" target="_blank">PMC7094162</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18692072.0" target="_blank">18692072.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2008.07.018" target="_blank">10.1016/j.jtbi.2008.07.018</a></td>
      <td>g4k6pc82</td>
      <td>0.675993</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_16.html#twmrilk8">Liu_X_2009</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#us6ay892">Perez-Bello_2009</a>, <a href="Topic_17.html#fsh3gsvd">Xiao_2005</a></td>
    </tr>
    <tr>
      <th id="88eqc5ah";>47</th>
      <td>Qi_Z_2010</td>
      <td>New method for global alignment of 2 DNA sequences by the tree data structure</td>
      <td>Qi, Zhao-Hui; Qi, Xiao-Qin; Liu, Chen-Chen</td>
      <td>2010</td>
      <td>2010-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094160" target="_blank">PMC7094160</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20025888.0" target="_blank">20025888.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2009.12.012" target="_blank">10.1016/j.jtbi.2009.12.012</a></td>
      <td>88eqc5ah</td>
      <td>0.675867</td>
      <td><a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a></td>
    </tr>
    <tr>
      <th id="54ioyzbc";>48</th>
      <td>Xiao_2005</td>
      <td>Using cellular automata to generate image representation for biological sequences</td>
      <td>Xiao, X.; Shao, S.; Ding, Y.; Huang, Z.; Chen, X.; Chou, K.-C.</td>
      <td>2005</td>
      <td>2005-02-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088382" target="_blank">PMC7088382</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15700108.0" target="_blank">15700108.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-004-0154-9" target="_blank">10.1007/s00726-004-0154-9</a></td>
      <td>54ioyzbc</td>
      <td>0.675555</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#fsh3gsvd">Xiao_2005</a>, <a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_61.html#1u96c10x">Gupta_2019</a></td>
    </tr>
    <tr>
      <th id="dc63kane";>49</th>
      <td>Andreatta_2011</td>
      <td>NNAlign: A Web-Based Prediction Method Allowing Non-Expert End-User Discovery of Sequence Motifs in Quantitative Peptide Data</td>
      <td>Andreatta, Massimo; Schafer-Nielsen, Claus; Lund, Ole; Buus, Søren; Nielsen, Morten</td>
      <td>2011</td>
      <td>2011-11-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3206854" target="_blank">PMC3206854</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22073191.0" target="_blank">22073191.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0026781" target="_blank">10.1371/journal.pone.0026781</a></td>
      <td>dc63kane</td>
      <td>0.674497</td>
      <td><a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#6oca1mrm">Shen_2018</a></td>
    </tr>
    <tr>
      <th id="vj9tv3or";>50</th>
      <td>Guo_F_2017</td>
      <td>Accurate prediction of human essential genes using only nucleotide composition and association information</td>
      <td>Guo, Feng-Biao; Dong, Chuan; Hua, Hong-Li; Liu, Shuo; Luo, Hao; Zhang, Hong-Wan; Jin, Yan-Ting; Zhang, Kai-Yue</td>
      <td>2017</td>
      <td>2017-06-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110051" target="_blank">PMC7110051</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28158612.0" target="_blank">28158612.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btx055" target="_blank">10.1093/bioinformatics/btx055</a></td>
      <td>vj9tv3or</td>
      <td>0.674142</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#w49x7kca">Biao_2016</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_17.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_17.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="u14tj0so";>51</th>
      <td>Jabbari_2014</td>
      <td>A fast and robust iterative algorithm for prediction of RNA pseudoknotted secondary structures</td>
      <td>Jabbari, Hosna; Condon, Anne</td>
      <td>2014</td>
      <td>2014-05-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4064103" target="_blank">PMC4064103</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24884954.0" target="_blank">24884954.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-147" target="_blank">10.1186/1471-2105-15-147</a></td>
      <td>u14tj0so</td>
      <td>0.673960</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="6oca1mrm";>52</th>
      <td>Shen_2018</td>
      <td>RPiRLS: Quantitative Predictions of RNA Interacting with Any Protein of Known Sequence</td>
      <td>Shen, Wen-Jun; Cui, Wenjuan; Chen, Danze; Zhang, Jieming; Xu, Jianzhen</td>
      <td>2018</td>
      <td>2018-02-28</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6017498" target="_blank">PMC6017498</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29495575.0" target="_blank">29495575.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules23030540" target="_blank">10.3390/molecules23030540</a></td>
      <td>6oca1mrm</td>
      <td>0.673245</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_17.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_16.html#vhsipxs7">Järvelin_2016</a>, <a href="Topic_21.html#wo462anq">Xia_H_2015</a></td>
    </tr>
    <tr>
      <th id="lz0gy9el";>53</th>
      <td>Liu_L_2012</td>
      <td>A relative Lempel–Ziv complexity: Application to comparing biological sequences</td>
      <td>Liu, Liwei; Li, Dongbo; Bai, Fenglan</td>
      <td>2012</td>
      <td>2012-03-19</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094452" target="_blank">PMC7094452</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cplett.2012.01.061" target="_blank">10.1016/j.cplett.2012.01.061</a></td>
      <td>lz0gy9el</td>
      <td>0.671916</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a></td>
    </tr>
    <tr>
      <th id="t94aufs3";>54</th>
      <td>Aurrecoechea_2017</td>
      <td>EuPathDB: the eukaryotic pathogen genomics database resource</td>
      <td>Aurrecoechea, Cristina; Barreto, Ana; Basenko, Evelina Y.; Brestelli, John; Brunk, Brian P.; Cade, Shon; Crouch, Kathryn; Doherty, Ryan; Falke, Dave; Fischer, Steve; Gajria, Bindu; Harb, Omar S.; Heiges, Mark; Hertz-Fowler, Christiane; Hu, Sufen; Iodice, John; Kissinger, Jessica C.; Lawrence, Cris; Li, Wei; Pinney, Deborah F.; Pulman, Jane A.; Roos, David S.; Shanmugasundram, Achchuthan; Silva-Franco, Fatima; Steinbiss, Sascha; Stoeckert, Christian J.; Spruill, Drew; Wang, Haiming; Warrenfeltz, Susanne; Zheng, Jie</td>
      <td>2017</td>
      <td>2017-01-04</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210576" target="_blank">PMC5210576</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27903906.0" target="_blank">27903906.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1105" target="_blank">10.1093/nar/gkw1105</a></td>
      <td>t94aufs3</td>
      <td>0.668669</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_17.html#0kkm0tgj">Zhang_2007</a>, <a href="Topic_17.html#3ulketgy">Snyder_2006</a>, <a href="Topic_01.html#p19e9uom">Myler_2008</a></td>
    </tr>
    <tr>
      <th id="kt9sbn40";>55</th>
      <td>Lee_G_2009</td>
      <td>Proportional fault-tolerant data mining with applications to bioinformatics</td>
      <td>Lee, Guanling; Peng, Sheng-Lung; Lin, Yuh-Tzu</td>
      <td>2009</td>
      <td>2009-02-19</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087812" target="_blank">PMC7087812</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10796-009-9158-z" target="_blank">10.1007/s10796-009-9158-z</a></td>
      <td>kt9sbn40</td>
      <td>0.666552</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#qndr28n6">Bashir_2020</a>, <a href="Topic_17.html#183n1lrf">Wilson_2003</a>, <a href="Topic_17.html#s96d3ust">Hu_Y_2006</a>, <a href="Topic_17.html#pdf3i2tt">Wu_Y_2020</a></td>
    </tr>
    <tr>
      <th id="jms7hrmp";>56</th>
      <td>Liu_C_2005</td>
      <td>Profiling and Searching for RNA Pseudoknot Structures in Genomes</td>
      <td>Liu, Chunmei; Song, Yinglei; Malmberg, Russell L.; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120494" target="_blank">PMC7120494</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11567752_2" target="_blank">10.1007/11567752_2</a></td>
      <td>jms7hrmp</td>
      <td>0.666431</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#egbkl2v2">Liu_C_2005</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#egbkl2v2">Liu_C_2005</a>, <a href="Topic_61.html#a5htfx76">Dost_2006</a>, <a href="Topic_61.html#o7mt8hr6">Kim_J_2007</a></td>
    </tr>
    <tr>
      <th id="5ge7ozpd";>57</th>
      <td>Zheng_2020</td>
      <td>Comparative study of the lymphocyte change between COVID-19 and non-COVID-19 pneumonia cases suggesting uncontrolled inflammation might not be the main reason of tissue injury</td>
      <td>Yishan Zheng; Zhen Huang; Guoping Ying; Xia Zhang; Wei Ye; Zhiliang Hu; Chunmei Hu; Hongxia Wei; Yi Zeng; Yun Chi; Cong Cheng; Feishen Lin; Hu Lu; Lingyan Xiao; Yan Song; Chunming Wang; Yongxiang Yi; Lei Dong</td>
      <td>2020</td>
      <td>2020-02-23</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.19.20024885" target="_blank">10.1101/2020.02.19.20024885</a></td>
      <td>5ge7ozpd</td>
      <td>0.666137</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a></td>
      <td><a href="Topic_09.html#hkv4cnw7">Rendon_2016</a>, <a href="Topic_54.html#arhqix9h">Heydari_2020</a>, <a href="Topic_67.html#6lngz7y1">Zhao_2020</a></td>
    </tr>
    <tr>
      <th id="jzpt9ldx";>58</th>
      <td>Ren_R_2016</td>
      <td>The attribute reductions of three-way concept lattices</td>
      <td>Ren, Ruisi; Wei, Ling</td>
      <td>2016</td>
      <td>2016-05-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126604" target="_blank">PMC7126604</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2016.01.045" target="_blank">10.1016/j.knosys.2016.01.045</a></td>
      <td>jzpt9ldx</td>
      <td>0.663341</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_02.html#6zjf6qzz">Li_J_2015</a>, <a href="Topic_17.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_17.html#j22q89l9">Vanderwende_2007</a>, <a href="Topic_02.html#4wnoa4l7">Cai_Y_2019</a></td>
    </tr>
    <tr>
      <th id="maf96bof";>59</th>
      <td>Bitard-Feildel_2018</td>
      <td>HCAtk and pyHCA: A Toolkit and Python API for the Hydrophobic Cluster Analysis of Protein Sequences</td>
      <td>Tristan Bitard-Feildel; Isabelle Callebaut</td>
      <td>2018</td>
      <td>2018-01-18</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/249995" target="_blank">10.1101/249995</a></td>
      <td>maf96bof</td>
      <td>0.660145</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_69.html#tvxpckxo">Pridgeon_2003</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_16.html#cz1bi0ym">Yu_L_2017</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="ufuin9r4";>60</th>
      <td>Ren_H_2017</td>
      <td>Anomaly detection based on a dynamic Markov model</td>
      <td>Ren, Huorong; Ye, Zhixing; Li, Zhiwu</td>
      <td>2017</td>
      <td>2017-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094635" target="_blank">PMC7094635</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ins.2017.05.021" target="_blank">10.1016/j.ins.2017.05.021</a></td>
      <td>ufuin9r4</td>
      <td>0.659332</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_06.html#ue8yn570">Wieland_2007</a>, <a href="Topic_06.html#fl6wlhjs">Craigmile_2007</a>, <a href="Topic_06.html#dze1vp3a">Mohamad_Mohsin_2014</a></td>
    </tr>
    <tr>
      <th id="pbhvz1k0";>61</th>
      <td>Hirahata_2006</td>
      <td>Genome Information Broker for Viruses (GIB-V): database for comparative analysis of virus genomes</td>
      <td>Hirahata, Masaki; Abe, Takashi; Tanaka, Naoto; Kuwana, Yoshikazu; Shigemoto, Yasumasa; Miyazaki, Satoru; Suzuki, Yoshiyuki; Sugawara, Hideaki</td>
      <td>2006</td>
      <td>2006-12-07</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1781101" target="_blank">PMC1781101</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17158166.0" target="_blank">17158166.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl1004" target="_blank">10.1093/nar/gkl1004</a></td>
      <td>pbhvz1k0</td>
      <td>0.657735</td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_21.html#2jlz4c58">Crucière_1988</a></td>
    </tr>
    <tr>
      <th id="1ypgij14";>62</th>
      <td>Huang_2006</td>
      <td>MIMOX: a web tool for phage display based epitope mapping</td>
      <td>Huang, Jian; Gutteridge, Alex; Honda, Wataru; Kanehisa, Minoru</td>
      <td>2006</td>
      <td>2006-10-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1618411" target="_blank">PMC1618411</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17038191.0" target="_blank">17038191.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-451" target="_blank">10.1186/1471-2105-7-451</a></td>
      <td>1ypgij14</td>
      <td>0.650532</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_01.html#92esdjy9">Delhalle_2012</a>, <a href="Topic_17.html#7rs4jtfh">Mayrose_2007</a>, <a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_69.html#71940dh7">Kumar_2019</a></td>
    </tr>
    <tr>
      <th id="3gv6469y";>63</th>
      <td>Bello-Orgaz_2016</td>
      <td>Social big data: Recent achievements and new challenges</td>
      <td>Bello-Orgaz, Gema; Jung, Jason J.; Camacho, David</td>
      <td>2016</td>
      <td>2016-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106299" target="_blank">PMC7106299</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.inffus.2015.08.005" target="_blank">10.1016/j.inffus.2015.08.005</a></td>
      <td>3gv6469y</td>
      <td>0.648767</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#tnoybbfx">Wagner_2020</a></td>
      <td><a href="Topic_17.html#mxprt6x6">Souza_2020</a>, <a href="Topic_17.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_17.html#kt9sbn40">Lee_G_2009</a>, <a href="Topic_17.html#uxmqlx4y">Yang_2019</a></td>
    </tr>
    <tr>
      <th id="ihh521ur";>64</th>
      <td>Lu_Q_2006</td>
      <td>KDE Bioscience: Platform for bioinformatics analysis workflows</td>
      <td>Lu, Qiang; Hao, Pei; Curcin, Vasa; He, Weizhong; Li, Yuan-Yuan; Luo, Qing-Ming; Guo, Yi-Ke; Li, Yi-Xue</td>
      <td>2006</td>
      <td>2006-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106075" target="_blank">PMC7106075</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16260186.0" target="_blank">16260186.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbi.2005.09.001" target="_blank">10.1016/j.jbi.2005.09.001</a></td>
      <td>ihh521ur</td>
      <td>0.648348</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_17.html#hgxhyg46">Greenbaum_2009</a>, <a href="Topic_17.html#vret3sh1">Aziz_2012</a>, <a href="Topic_17.html#3ulketgy">Snyder_2006</a></td>
    </tr>
    <tr>
      <th id="buc4dd5y";>65</th>
      <td>Dong_2019</td>
      <td>A Novel Approach to Clustering Genome Sequences Using Inter-nucleotide Covariance</td>
      <td>Dong, Rui; He, Lily; He, Rong Lucy; Yau, Stephen S.-T.</td>
      <td>2019</td>
      <td>2019-04-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6465635" target="_blank">PMC6465635</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31024610.0" target="_blank">31024610.0</a></td>
      <td><a href="https://doi.org/10.3389/fgene.2019.00234" target="_blank">10.3389/fgene.2019.00234</a></td>
      <td>buc4dd5y</td>
      <td>0.648226</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_13.html#4lou2mb6">Kapoor_2020</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="ch4kere3";>66</th>
      <td>Yan_C_2019</td>
      <td>IILLS: predicting virus-receptor interactions based on similarity and semi-supervised learning</td>
      <td>Yan, Cheng; Duan, Guihua; Wu, Fang-Xiang; Wang, Jianxin</td>
      <td>2019</td>
      <td>2019-12-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6933616" target="_blank">PMC6933616</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31881820.0" target="_blank">31881820.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-3278-3" target="_blank">10.1186/s12859-019-3278-3</a></td>
      <td>ch4kere3</td>
      <td>0.645755</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_17.html#55lkfivh">Cho_M_2019</a>, <a href="Topic_10.html#x6m02voy">Bae_S_2011</a></td>
    </tr>
    <tr>
      <th id="etifey83";>67</th>
      <td>Gao_L_2011</td>
      <td>Remote Data Retrieval for Bioinformatics Applications: An Agent Migration Approach</td>
      <td>Gao, Lei; Dai, Hua; Zhang, Tong-Liang; Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-06-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3119054" target="_blank">PMC3119054</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21701677.0" target="_blank">21701677.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0020949" target="_blank">10.1371/journal.pone.0020949</a></td>
      <td>etifey83</td>
      <td>0.645716</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_17.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_17.html#60vrlrim">Lefkowitz_2008</a>, <a href="Topic_54.html#eawmuuye">Stokes_1993</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
    </tr>
    <tr>
      <th id="lqviqh3m";>68</th>
      <td>Höhl_2007</td>
      <td>Is Multiple-Sequence Alignment Required for Accurate Inference of Phylogeny?</td>
      <td>Höhl, Michael; Ragan, Mark A.</td>
      <td>2007</td>
      <td>2007-04-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107264" target="_blank">PMC7107264</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17454975.0" target="_blank">17454975.0</a></td>
      <td><a href="https://doi.org/10.1080/10635150701294741" target="_blank">10.1080/10635150701294741</a></td>
      <td>lqviqh3m</td>
      <td>0.640653</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_06.html#zw5kt090">Hilton_2018</a></td>
    </tr>
    <tr>
      <th id="f7vhyau9";>69</th>
      <td>Basu_2010</td>
      <td>AMS 3.0: prediction of post-translational modifications</td>
      <td>Basu, Subhadip; Plewczynski, Dariusz</td>
      <td>2010</td>
      <td>2010-04-28</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2874555" target="_blank">PMC2874555</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20423529.0" target="_blank">20423529.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-11-210" target="_blank">10.1186/1471-2105-11-210</a></td>
      <td>f7vhyau9</td>
      <td>0.640210</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_10.html#38c28tw1">Fung_2018</a></td>
    </tr>
    <tr>
      <th id="6yvdv1xk";>70</th>
      <td>David_2010</td>
      <td>Using simple artificial intelligence methods for predicting amyloidogenesis in antibodies</td>
      <td>David, Maria Pamela C; Concepcion, Gisela P; Padlan, Eduardo A</td>
      <td>2010</td>
      <td>2010-02-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3098112" target="_blank">PMC3098112</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20144194.0" target="_blank">20144194.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-11-79" target="_blank">10.1186/1471-2105-11-79</a></td>
      <td>6yvdv1xk</td>
      <td>0.639727</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_02.html#7h828tvv">Endres_2020</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_69.html#n184795x">Zamanzadeh_2017</a></td>
    </tr>
    <tr>
      <th id="besvz92f";>71</th>
      <td>Schäffer_2019</td>
      <td>VADR: validation and annotation of virus sequence submissions to GenBank</td>
      <td>Alejandro A Schäffer; Eneida Hatcher; Linda Yankie; Lara Shonkwiler; J Rodney Brister; Ilene Karsch-Mizrachi; Eric P Nawrocki</td>
      <td>2019</td>
      <td>2019-11-22</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/852657" target="_blank">10.1101/852657</a></td>
      <td>besvz92f</td>
      <td>0.638137</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a></td>
      <td><a href="Topic_17.html#u3q36o3e">Shean_2019</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a>, <a href="Topic_21.html#27fd59qe">Meulenberg_1995</a></td>
    </tr>
    <tr>
      <th id="ear9cyri";>72</th>
      <td>Bakker_2018</td>
      <td>Dynamic graphs, community detection, and Riemannian geometry</td>
      <td>Bakker, Craig; Halappanavar, Mahantesh; Visweswara Sathanur, Arun</td>
      <td>2018</td>
      <td>2018-03-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6214282" target="_blank">PMC6214282</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30839776.0" target="_blank">30839776.0</a></td>
      <td><a href="https://doi.org/10.1007/s41109-018-0059-2" target="_blank">10.1007/s41109-018-0059-2</a></td>
      <td>ear9cyri</td>
      <td>0.637245</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_06.html#k5wcibdk">Pacheco_2017</a>, <a href="Topic_17.html#397hp1yt">Coelho_2008</a>, <a href="Topic_06.html#xdlk8557">Han_X_2014</a>, <a href="Topic_06.html#w4g8mzuy">Groendyke_2012</a></td>
    </tr>
    <tr>
      <th id="rzzsmuoc";>73</th>
      <td>Watson_2007</td>
      <td>DetectiV: visualization, normalization and significance testing for pathogen-detection microarray data</td>
      <td>Watson, Michael; Dukes, Juliet; Abu-Median, Abu-Bakr; King, Donald P; Britton, Paul</td>
      <td>2007</td>
      <td>2007-09-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2375028" target="_blank">PMC2375028</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17868443.0" target="_blank">17868443.0</a></td>
      <td><a href="https://doi.org/10.1186/gb-2007-8-9-r190" target="_blank">10.1186/gb-2007-8-9-r190</a></td>
      <td>rzzsmuoc</td>
      <td>0.636123</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_14.html#xo1cyzrg">Giles_2015</a>, <a href="Topic_14.html#ueyaw8pd">Wong_2007</a>, <a href="Topic_14.html#9ntgqyhw">McLoughlin_2011</a></td>
    </tr>
    <tr>
      <th id="kt6wqvam";>74</th>
      <td>Xiao_2013</td>
      <td>iGPCR-Drug: A Web Server for Predicting Interaction between GPCRs and Drugs in Cellular Networking</td>
      <td>Xiao, Xuan; Min, Jian-Liang; Wang, Pu; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-08-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3754978" target="_blank">PMC3754978</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24015221.0" target="_blank">24015221.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0072234" target="_blank">10.1371/journal.pone.0072234</a></td>
      <td>kt6wqvam</td>
      <td>0.635064</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_17.html#qdld7hdc">Fan_Y_2014</a>, <a href="Topic_17.html#jpj1czhg">Min_J_2013</a></td>
    </tr>
    <tr>
      <th id="j51gaesp";>75</th>
      <td>Amiri_2018</td>
      <td>Time-constrained maximal covering routing problem</td>
      <td>Amiri, Afsaneh; Salari, Majid</td>
      <td>2018</td>
      <td>2018-12-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7079968" target="_blank">PMC7079968</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s00291-018-0541-3" target="_blank">10.1007/s00291-018-0541-3</a></td>
      <td>j51gaesp</td>
      <td>0.634993</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#1ul9zqhb">Mousavi_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_79.html#it5pj8ub">Feng_2017</a>, <a href="Topic_17.html#1ul9zqhb">Mousavi_2012</a>, <a href="Topic_71.html#5xlrcjtr">Xiao_2006</a>, <a href="Topic_39.html#ho0qvatz">Yoon_2015</a></td>
    </tr>
    <tr>
      <th id="vm5zjr64";>76</th>
      <td>Zuo_G_2011</td>
      <td>Jackknife and Bootstrap Tests of the Composition Vector Trees</td>
      <td>Zuo, Guanghong; Xu, Zhao; Yu, Hongjie; Hao, Bailin</td>
      <td>2011</td>
      <td>2011-03-05</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5054193" target="_blank">PMC5054193</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21382595.0" target="_blank">21382595.0</a></td>
      <td><a href="https://doi.org/10.1016/s1672-0229(10)60028-9" target="_blank">10.1016/s1672-0229(10)60028-9</a></td>
      <td>vm5zjr64</td>
      <td>0.634414</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a></td>
      <td><a href="Topic_17.html#ne9d0siv">Banks_2006</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_02.html#bkfkmtld">Redelings_2007</a></td>
    </tr>
    <tr>
      <th id="qyd6qyjf";>77</th>
      <td>Xiao_2017</td>
      <td>Prediction of lncRNA-protein interactions using HeteSim scores based on heterogeneous networks</td>
      <td>Xiao, Yun; Zhang, Jingpu; Deng, Lei</td>
      <td>2017</td>
      <td>2017-06-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5473862" target="_blank">PMC5473862</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28623317.0" target="_blank">28623317.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-017-03986-1" target="_blank">10.1038/s41598-017-03986-1</a></td>
      <td>qyd6qyjf</td>
      <td>0.634062</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_34.html#fmb5nyxu">Liui_2020</a>, <a href="Topic_34.html#er3kkn9m">Trypsteen_2016</a>, <a href="Topic_17.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="g1ob16b9";>78</th>
      <td>Xie_X_2012</td>
      <td>Protein sequence analysis based on hydropathy profile of amino acids</td>
      <td>Xie, Xiao-li; Zheng, Li-fei; Yu, Ying; Liang, Li-ping; Guo, Man-cai; Song, John; Yuan, Zhi-fa</td>
      <td>2012</td>
      <td>2012-01-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274743" target="_blank">PMC3274743</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22302429.0" target="_blank">22302429.0</a></td>
      <td><a href="https://doi.org/10.1631/jzus.b1100052" target="_blank">10.1631/jzus.b1100052</a></td>
      <td>g1ob16b9</td>
      <td>0.633161</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#2kpxhzbe">Das_J_2017</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a></td>
    </tr>
    <tr>
      <th id="f3uo372d";>79</th>
      <td>Sher_2017</td>
      <td>DRREP: deep ridge regressed epitope predictor</td>
      <td>Sher, Gene; Zhi, Degui; Zhang, Shaojie</td>
      <td>2017</td>
      <td>2017-10-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5629616" target="_blank">PMC5629616</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28984193.0" target="_blank">28984193.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-017-4024-8" target="_blank">10.1186/s12864-017-4024-8</a></td>
      <td>f3uo372d</td>
      <td>0.632192</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_17.html#ml2luc1t">Sollner_2008</a>, <a href="Topic_69.html#n184795x">Zamanzadeh_2017</a>, <a href="Topic_17.html#rtpbltkf">EL-Manzalawy_2008</a></td>
    </tr>
    <tr>
      <th id="jsg0yc2t";>80</th>
      <td>O'Leary_2016</td>
      <td>Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation</td>
      <td>O'Leary, Nuala A.; Wright, Mathew W.; Brister, J. Rodney; Ciufo, Stacy; Haddad, Diana; McVeigh, Rich; Rajput, Bhanu; Robbertse, Barbara; Smith-White, Brian; Ako-Adjei, Danso; Astashyn, Alexander; Badretdin, Azat; Bao, Yiming; Blinkova, Olga; Brover, Vyacheslav; Chetvernin, Vyacheslav; Choi, Jinna; Cox, Eric; Ermolaeva, Olga; Farrell, Catherine M.; Goldfarb, Tamara; Gupta, Tripti; Haft, Daniel; Hatcher, Eneida; Hlavina, Wratko; Joardar, Vinita S.; Kodali, Vamsi K.; Li, Wenjun; Maglott, Donna; Masterson, Patrick; McGarvey, Kelly M.; Murphy, Michael R.; O'Neill, Kathleen; Pujar, Shashikant; Rangwala, Sanjida H.; Rausch, Daniel; Riddick, Lillian D.; Schoch, Conrad; Shkeda, Andrei; Storz, Susan S.; Sun, Hanzhen; Thibaud-Nissen, Francoise; Tolstoy, Igor; Tully, Raymond E.; Vatsan, Anjana R.; Wallin, Craig; Webb, David; Wu, Wendy; Landrum, Melissa J.; Kimchi, Avi; Tatusova, Tatiana; DiCuccio, Michael; Kitts, Paul; Murphy, Terence D.; Pruitt, Kim D.</td>
      <td>2016</td>
      <td>2016-01-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702849" target="_blank">PMC4702849</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26553804.0" target="_blank">26553804.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkv1189" target="_blank">10.1093/nar/gkv1189</a></td>
      <td>jsg0yc2t</td>
      <td>0.629300</td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wj7yonjw">Gardner_2008</a></td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#besvz92f">Schäffer_2019</a></td>
    </tr>
    <tr>
      <th id="ak33vta6";>81</th>
      <td>Du_X_2019</td>
      <td>Language model-based automatic prefix abbreviation expansion method for biomedical big data analysis</td>
      <td>Du, Xiaokun; Zhu, Rongbo; Li, Yanhong; Anjum, Ashiq</td>
      <td>2019</td>
      <td>2019-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103364" target="_blank">PMC7103364</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.future.2019.01.016" target="_blank">10.1016/j.future.2019.01.016</a></td>
      <td>ak33vta6</td>
      <td>0.628951</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_17.html#abrweljg">Yu_A_2011</a>, <a href="Topic_54.html#bep0xtxa">Li_Y_2020</a></td>
    </tr>
    <tr>
      <th id="lpe49hcj";>82</th>
      <td>Wang_2014</td>
      <td>Identification of indoor contaminant source location by a single concentration sensor</td>
      <td>Wang, Lin-Lin; You, Xue-Yi</td>
      <td>2014</td>
      <td>2014-07-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089341" target="_blank">PMC7089341</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11869-014-0280-9" target="_blank">10.1007/s11869-014-0280-9</a></td>
      <td>lpe49hcj</td>
      <td>0.627554</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_64.html#k6ovqpsn">Cai_H_2014</a>, <a href="Topic_17.html#t54c2si1">Bastani_2012</a>, <a href="Topic_64.html#ah3jclgw">Feng_2019</a>, <a href="Topic_01.html#t628fcbq">Zulkifli_2018</a></td>
    </tr>
    <tr>
      <th id="my2l85bn";>83</th>
      <td>Cho_Y_2005</td>
      <td>Rule Generation Using NN and GA for SARS-CoV Cleavage Site Prediction</td>
      <td>Cho, Yeon-Jin; Kim, Hyeoncheol</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122303" target="_blank">PMC7122303</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11553939_111" target="_blank">10.1007/11553939_111</a></td>
      <td>my2l85bn</td>
      <td>0.627256</td>
      <td><a href="Topic_17.html#gc9hevy5">Cho_Y_2005</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_16.html#02o93wlh">Fan_K_2005</a>, <a href="Topic_16.html#ukco9p7s">Shan_2004</a>, <a href="Topic_16.html#n0fa076h">Kuo_C_2004</a></td>
    </tr>
    <tr>
      <th id="60vrlrim";>84</th>
      <td>Lefkowitz_2008</td>
      <td>Virus Databases</td>
      <td>Lefkowitz, E.J.; Odom, M.R.; Upton, C.</td>
      <td>2008</td>
      <td>2008-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-012374410-4.00719-6" target="_blank">10.1016/b978-012374410-4.00719-6</a></td>
      <td>60vrlrim</td>
      <td>0.626163</td>
      <td><a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_01.html#tv2ntug6">Gautam_2019</a>, <a href="Topic_17.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_01.html#p19e9uom">Myler_2008</a></td>
    </tr>
    <tr>
      <th id="7qriyc1a";>85</th>
      <td>Yang_2007</td>
      <td>Predicting Palmitoylation Sites Using a Regularised Bio-basis Function Neural Network</td>
      <td>Yang, Zheng Rong</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120480" target="_blank">PMC7120480</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72031-7_37" target="_blank">10.1007/978-3-540-72031-7_37</a></td>
      <td>7qriyc1a</td>
      <td>0.624889</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_16.html#40pe21lm">Zhang_2018</a>, <a href="Topic_53.html#o2z7n3v5">Tsukamoto_2013</a>, <a href="Topic_16.html#0pmo3opx">Joshi_2017</a></td>
    </tr>
    <tr>
      <th id="fvuwick5";>86</th>
      <td>Qi_Z_2018</td>
      <td>Novel Method of 3-Dimensional Graphical Representation for Proteins and Its Application</td>
      <td>Qi, Zhao-Hui; Li, Ke-Cheng; Ma, Jin-Long; Yao, Yu-Hua; Liu, Ling-Yun</td>
      <td>2018</td>
      <td>2018-06-12</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6024350" target="_blank">PMC6024350</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29977111.0" target="_blank">29977111.0</a></td>
      <td><a href="https://doi.org/10.1177/1176934318777755" target="_blank">10.1177/1176934318777755</a></td>
      <td>fvuwick5</td>
      <td>0.624847</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_17.html#8j04y50s">Deng_2014</a>, <a href="Topic_01.html#m3k8o573">Ghosh_2011</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="5svtmk4a";>87</th>
      <td>Qi_Z_2009</td>
      <td>Classification analysis of dual nucleotides using dimension reduction</td>
      <td>Qi, Zhao-Hui; Wang, Jian-Min; Qi, Xiao-Qin</td>
      <td>2009</td>
      <td>2009-09-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126582" target="_blank">PMC7126582</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19481099.0" target="_blank">19481099.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2009.05.011" target="_blank">10.1016/j.jtbi.2009.05.011</a></td>
      <td>5svtmk4a</td>
      <td>0.624847</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a></td>
    </tr>
    <tr>
      <th id="elf16dno";>88</th>
      <td>Jiang_2009</td>
      <td>Reasoning within expressive fuzzy rough description logics</td>
      <td>Jiang, Yuncheng; Wang, Ju; Deng, Peimin; Tang, Suqin</td>
      <td>2009</td>
      <td>2009-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7133619" target="_blank">PMC7133619</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.fss.2009.01.004" target="_blank">10.1016/j.fss.2009.01.004</a></td>
      <td>elf16dno</td>
      <td>0.623251</td>
      <td><a href="Topic_17.html#6dcghdxf">Jiang_2009</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_02.html#6zjf6qzz">Li_J_2015</a></td>
      <td><a href="Topic_17.html#6dcghdxf">Jiang_2009</a>, <a href="Topic_17.html#gi8c1omv">Capotorti_2006</a>, <a href="Topic_17.html#lrcxwtx7">Peng_2018</a>, <a href="Topic_02.html#nouieh2s">Fox_J_2003</a></td>
    </tr>
    <tr>
      <th id="wo6356vq";>89</th>
      <td>Khanna_2019</td>
      <td>Prediction of novel mouse TLR9 agonists using a random forest approach</td>
      <td>Khanna, Varun; Li, Lei; Fung, Johnson; Ranganathan, Shoba; Petrovsky, Nikolai</td>
      <td>2019</td>
      <td>2019-12-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6924143" target="_blank">PMC6924143</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31856726.0" target="_blank">31856726.0</a></td>
      <td><a href="https://doi.org/10.1186/s12860-019-0241-0" target="_blank">10.1186/s12860-019-0241-0</a></td>
      <td>wo6356vq</td>
      <td>0.618074</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_42.html#l1ykm6sw">Li_Y_2013</a>, <a href="Topic_31.html#cmdwd67m">Villanueva_2011</a>, <a href="Topic_74.html#cwkwnydk">Shen_2017</a></td>
    </tr>
    <tr>
      <th id="lhriiv4u";>90</th>
      <td>Molodenskiy_2019</td>
      <td>An automated data processing and analysis pipeline for transmembrane proteins in detergent solutions</td>
      <td>D. Molodenskiy; H. Mertens; D. Svergun</td>
      <td>2019</td>
      <td>2019-07-24</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/714303" target="_blank">10.1101/714303</a></td>
      <td>lhriiv4u</td>
      <td>0.616460</td>
      <td><a href="Topic_17.html#1voq7zxz">DiMaio_2016</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_26.html#jljpb7rv">Voß_D_2009</a>, <a href="Topic_01.html#9k2z7xri">Pandey_2016</a>, <a href="Topic_01.html#3riv0oh3">Pedro_2019</a></td>
    </tr>
    <tr>
      <th id="pt7v3gb4";>91</th>
      <td>UNKNOWN_2019</td>
      <td>UniProt: a worldwide hub of protein knowledge</td>
      <td>None</td>
      <td>2019</td>
      <td>2019-01-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323992" target="_blank">PMC6323992</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30395287.0" target="_blank">30395287.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gky1049" target="_blank">10.1093/nar/gky1049</a></td>
      <td>pt7v3gb4</td>
      <td>0.614157</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a></td>
      <td><a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_01.html#p19e9uom">Myler_2008</a>, <a href="Topic_16.html#ampip7od">Bagowski_2010</a></td>
    </tr>
    <tr>
      <th id="02opdk0m";>92</th>
      <td>Xu_Z_2009</td>
      <td>CVTree update: a newly designed phylogenetic study platform using composition vectors and whole genomes</td>
      <td>Xu, Zhao; Hao, Bailin</td>
      <td>2009</td>
      <td>2009-07-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2703908" target="_blank">PMC2703908</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19398429.0" target="_blank">19398429.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkp278" target="_blank">10.1093/nar/gkp278</a></td>
      <td>02opdk0m</td>
      <td>0.613370</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a></td>
    </tr>
    <tr>
      <th id="zgv4kd6j";>93</th>
      <td>Reeder_2007</td>
      <td>pknotsRG: RNA pseudoknot folding including near-optimal structures and sliding windows</td>
      <td>Reeder, Jens; Steffen, Peter; Giegerich, Robert</td>
      <td>2007</td>
      <td>2007-05-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933184" target="_blank">PMC1933184</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17478505.0" target="_blank">17478505.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm258" target="_blank">10.1093/nar/gkm258</a></td>
      <td>zgv4kd6j</td>
      <td>0.612154</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="hohrbxyj";>94</th>
      <td>Lespinats_2011</td>
      <td>ColorPhylo: A Color Code to Accurately Display Taxonomic Classifications</td>
      <td>Lespinats, Sylvain; Fertil, Bernard</td>
      <td>2011</td>
      <td>2011-11-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3255521" target="_blank">PMC3255521</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22253532.0" target="_blank">22253532.0</a></td>
      <td><a href="https://doi.org/10.4137/ebo.s7565" target="_blank">10.4137/ebo.s7565</a></td>
      <td>hohrbxyj</td>
      <td>0.610620</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_02.html#jl3uah1i">Harper_2013</a>, <a href="Topic_45.html#tquokpxk">Lung_2013</a></td>
    </tr>
    <tr>
      <th id="pd0cmefo";>95</th>
      <td>Shang_2020</td>
      <td>CHEER: hierarCHical taxonomic classification for viral mEtagEnomic data via deep leaRning</td>
      <td>Jiayu Shang; Yanni Sun</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.26.009001" target="_blank">10.1101/2020.03.26.009001</a></td>
      <td>pd0cmefo</td>
      <td>0.610434</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_29.html#k4ra1job">Yadav_2013</a>, <a href="Topic_17.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a></td>
    </tr>
    <tr>
      <th id="nfxcqsc2";>96</th>
      <td>Kim_K_2007</td>
      <td>A LAD-based method for selecting short oligo probes for genotyping applications</td>
      <td>Kim, Kwangsoo; Ryoo, Hong Seo</td>
      <td>2007</td>
      <td>2007-06-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7080176" target="_blank">PMC7080176</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s00291-007-0089-0" target="_blank">10.1007/s00291-007-0089-0</a></td>
      <td>nfxcqsc2</td>
      <td>0.610276</td>
      <td><a href="Topic_17.html#9rx4jlw8">Kim_K_2007</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_14.html#16tii0ha">Pandya_2007</a></td>
      <td><a href="Topic_17.html#9rx4jlw8">Kim_K_2007</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a>, <a href="Topic_60.html#kt9pmfdr">Kutyavin_2006</a>, <a href="Topic_34.html#hlvfg1b2">Tan_J_2009</a></td>
    </tr>
    <tr>
      <th id="ddu6yl8r";>97</th>
      <td>Adetiba_2018</td>
      <td>Alignment-Free Z-Curve Genomic Cepstral Coefficients and Machine Learning for Classification of Viruses</td>
      <td>Adetiba, Emmanuel; Olugbara, Oludayo O.; Taiwo, Tunmike B.; Adebiyi, Marion O.; Badejo, Joke A.; Akanle, Matthew B.; Matthews, Victor O.</td>
      <td>2018</td>
      <td>2018-03-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120486" target="_blank">PMC7120486</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-78723-7_25" target="_blank">10.1007/978-3-319-78723-7_25</a></td>
      <td>ddu6yl8r</td>
      <td>0.607759</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_17.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_17.html#2l1zw19o">Ozturk_2020</a></td>
    </tr>
    <tr>
      <th id="8j04y50s";>98</th>
      <td>Deng_2014</td>
      <td>DV-Curve Representation of Protein Sequences and Its Application</td>
      <td>Deng, Wei; Luan, Yihui</td>
      <td>2014</td>
      <td>2014-05-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4034481" target="_blank">PMC4034481</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24899916.0" target="_blank">24899916.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/203871" target="_blank">10.1155/2014/203871</a></td>
      <td>8j04y50s</td>
      <td>0.605516</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_21.html#5rovh1xf">Godeny_1990</a></td>
    </tr>
    <tr>
      <th id="j1p4nmsa";>99</th>
      <td>Fenga_2020</td>
      <td>CoViD19 Meta heuristic optimization based forecast method on time dependent bootstrapped data</td>
      <td>Livio Fenga; Carlo Del Castello</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.02.20050153" target="_blank">10.1101/2020.04.02.20050153</a></td>
      <td>j1p4nmsa</td>
      <td>0.605043</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_06.html#o3hytzwu">Chong_2020</a>, <a href="Topic_06.html#z19umr4q">Chernyshev_2020</a>, <a href="Topic_06.html#6vty0pmk">Lega_2016</a>, <a href="Topic_19.html#8b1esll4">Huang_2020</a></td>
    </tr>
    <tr>
      <th id="hs30wwiq";>100</th>
      <td>Gibbs_2014</td>
      <td>Multi-omic network signatures of disease</td>
      <td>Gibbs, David L.; Gralinski, Lisa; Baric, Ralph S.; McWeeney, Shannon K.</td>
      <td>2014</td>
      <td>2014-01-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882664" target="_blank">PMC3882664</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24432028.0" target="_blank">24432028.0</a></td>
      <td><a href="https://doi.org/10.3389/fgene.2013.00309" target="_blank">10.3389/fgene.2013.00309</a></td>
      <td>hs30wwiq</td>
      <td>0.603131</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_34.html#qh98622y">Xu_P_2019</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_01.html#x4hrwhkz">Prokop_2020</a>, <a href="Topic_17.html#4grjr0h3">McClure_2019</a></td>
    </tr>
    <tr>
      <th id="43igjn6x";>101</th>
      <td>Kylberg_2009</td>
      <td>Towards Automated TEM for Virus Diagnostics: Segmentation of Grid Squares and Detection of Regions of Interest</td>
      <td>Kylberg, Gustaf; Sintorn, Ida-Maria; Borgefors, Gunilla</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120801" target="_blank">PMC7120801</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-642-02230-2_18" target="_blank">10.1007/978-3-642-02230-2_18</a></td>
      <td>43igjn6x</td>
      <td>0.602459</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_14.html#bn6t8tcu">Zhang_2013</a>, <a href="Topic_60.html#e61vrp7h">Graham_2007</a>, <a href="Topic_68.html#8brxpotx">Field_1982</a>, <a href="Topic_26.html#bxv4wjyg">Pelchen‐Matthews_2007</a></td>
    </tr>
    <tr>
      <th id="jxd60j4o";>102</th>
      <td>Li_Y_2017</td>
      <td>A novel fast vector method for genetic sequence comparison</td>
      <td>Li, Yongkun; He, Lily; Lucy He, Rong; Yau, Stephen S.-T.</td>
      <td>2017</td>
      <td>2017-09-22</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610321" target="_blank">PMC5610321</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28939913.0" target="_blank">28939913.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-017-12493-2" target="_blank">10.1038/s41598-017-12493-2</a></td>
      <td>jxd60j4o</td>
      <td>0.599924</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_02.html#ajz4ef44">Nandy_2009</a></td>
    </tr>
    <tr>
      <th id="r9diwa8y";>103</th>
      <td>Bendaña_2008</td>
      <td>Colorstock, SScolor, Ratón: RNA alignment visualization tools</td>
      <td>Bendaña, Yuri R.; Holmes, Ian H.</td>
      <td>2008</td>
      <td>2008-02-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109877" target="_blank">PMC7109877</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18218657.0" target="_blank">18218657.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btm635" target="_blank">10.1093/bioinformatics/btm635</a></td>
      <td>r9diwa8y</td>
      <td>0.599886</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_17.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a></td>
    </tr>
    <tr>
      <th id="jpj1czhg";>104</th>
      <td>Min_J_2013</td>
      <td>iEzy-Drug: A Web Server for Identifying the Interaction between Enzymes and Drugs in Cellular Networking</td>
      <td>Min, Jian-Liang; Xiao, Xuan; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-11-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3858977" target="_blank">PMC3858977</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24371828.0" target="_blank">24371828.0</a></td>
      <td><a href="https://doi.org/10.1155/2013/701317" target="_blank">10.1155/2013/701317</a></td>
      <td>jpj1czhg</td>
      <td>0.596573</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_41.html#utfeiviy">McDowell_1988</a>, <a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a></td>
    </tr>
    <tr>
      <th id="hltgn9aj";>105</th>
      <td>Gatherer_2007</td>
      <td>Genome Signatures, Self-Organizing Maps and Higher Order Phylogenies: A Parametric Analysis</td>
      <td>Gatherer, Derek</td>
      <td>2007</td>
      <td>2007-09-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2684143" target="_blank">PMC2684143</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19468314.0" target="_blank">19468314.0</a></td>
      <td></td>
      <td>hltgn9aj</td>
      <td>0.595882</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_45.html#uf7b9azg">Kepler_2010</a>, <a href="Topic_21.html#5d64esp4">Walker_2015</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="ip0fxvhp";>106</th>
      <td>Schuck_2009</td>
      <td>On computational approaches for size-and-shape distributions from sedimentation velocity analytical ultracentrifugation</td>
      <td>Schuck, Peter</td>
      <td>2009</td>
      <td>2009-10-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892069" target="_blank">PMC2892069</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19806353.0" target="_blank">19806353.0</a></td>
      <td><a href="https://doi.org/10.1007/s00249-009-0545-7" target="_blank">10.1007/s00249-009-0545-7</a></td>
      <td>ip0fxvhp</td>
      <td>0.595147</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_17.html#cw2opick">Chou_2011</a>, <a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_17.html#wvw6q7px">Jonić_2016</a></td>
    </tr>
    <tr>
      <th id="gjild96e";>107</th>
      <td>Song_2005</td>
      <td>Efficient Parameterized Algorithm for Biopolymer Structure-Sequence Alignment</td>
      <td>Song, Yinglei; Liu, Chunmei; Huang, Xiuzhen; Malmberg, Russell L.; Xu, Ying; Cai, Liming</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121179" target="_blank">PMC7121179</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11557067_31" target="_blank">10.1007/11557067_31</a></td>
      <td>gjild96e</td>
      <td>0.592523</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_16.html#0a3okta0">Myllykoski_2017</a></td>
    </tr>
    <tr>
      <th id="e2coy5ms";>108</th>
      <td>Matsuzaw_2011</td>
      <td>KBDeX: A Platform for Exploring Discourse in Collaborative Learning</td>
      <td>Matsuzaw, Yoshiaki; Oshima, Jun; Oshima, Ritsuko; Niihara, Yusuke; Sakai, Sanshiro</td>
      <td>2011</td>
      <td>2011-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.sbspro.2011.10.576" target="_blank">10.1016/j.sbspro.2011.10.576</a></td>
      <td>e2coy5ms</td>
      <td>0.591240</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_28.html#0k4tsgus">Pearson_2006</a>, <a href="Topic_02.html#xr067v2n">van_Aalst_2009</a>, <a href="Topic_54.html#q0e9qw5k">Ojima_2006</a>, <a href="Topic_28.html#dmlrutu6">Pewnim_2011</a></td>
    </tr>
    <tr>
      <th id="abrweljg";>109</th>
      <td>Yu_A_2011</td>
      <td>A comparison of two methods for retrieving ICD-9-CM data: The effect of using an ontology-based method for handling terminology changes</td>
      <td>Yu, Alexander C.; Cimino, James J.</td>
      <td>2011</td>
      <td>2011-04-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3440000" target="_blank">PMC3440000</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21262390.0" target="_blank">21262390.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbi.2011.01.005" target="_blank">10.1016/j.jbi.2011.01.005</a></td>
      <td>abrweljg</td>
      <td>0.588849</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_17.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_17.html#ak33vta6">Du_X_2019</a>, <a href="Topic_17.html#uwbmox8a">Robson_2016</a>, <a href="Topic_02.html#iczb5z4e">McComb_2003</a></td>
    </tr>
    <tr>
      <th id="407bthqj";>110</th>
      <td>Huang_2011</td>
      <td>Bioinformatics Resources and Tools for Phage Display</td>
      <td>Huang, Jian; Ru, Beibei; Dai, Ping</td>
      <td>2011</td>
      <td>2011-01-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6259106" target="_blank">PMC6259106</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21245805.0" target="_blank">21245805.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules16010694" target="_blank">10.3390/molecules16010694</a></td>
      <td>407bthqj</td>
      <td>0.586803</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_01.html#j8umlbds">Wu_C_2016</a>, <a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_17.html#1ypgij14">Huang_2006</a></td>
    </tr>
    <tr>
      <th id="5cxaiw4k";>111</th>
      <td>Xie_G_2011</td>
      <td>Three 3D graphical representations of DNA primary sequences based on the classifications of DNA bases and their applications</td>
      <td>Xie, Guosen; Mo, Zhongxi</td>
      <td>2011</td>
      <td>2011-01-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126940" target="_blank">PMC7126940</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20969878.0" target="_blank">20969878.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.10.018" target="_blank">10.1016/j.jtbi.2010.10.018</a></td>
      <td>5cxaiw4k</td>
      <td>0.586126</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#i5fs2yye">Yu_J_2009</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#gkek599u">Zou_S_2014</a>, <a href="Topic_02.html#ajz4ef44">Nandy_2009</a></td>
    </tr>
    <tr>
      <th id="pb1e2w7s";>112</th>
      <td>Kolatkar_2008</td>
      <td>C-ME: A 3D Community-Based, Real-Time Collaboration Tool for Scientific Research and Training</td>
      <td>Kolatkar, Anand; Kennedy, Kevin; Halabuk, Dan; Kunken, Josh; Marrinucci, Dena; Bethel, Kelly; Guzman, Rodney; Huckaby, Tim; Kuhn, Peter</td>
      <td>2008</td>
      <td>2008-02-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2229842" target="_blank">PMC2229842</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18286178.0" target="_blank">18286178.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0001621" target="_blank">10.1371/journal.pone.0001621</a></td>
      <td>pb1e2w7s</td>
      <td>0.584840</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a></td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#lhriiv4u">Molodenskiy_2019</a>, <a href="Topic_17.html#5pvln91x">Asbury_2010</a></td>
    </tr>
    <tr>
      <th id="u1imic5l";>113</th>
      <td>Li_C_2018</td>
      <td>Protein Sequence Comparison and DNA-binding Protein Identification with Generalized PseAAC and Graphical Representation</td>
      <td>Li, Chun; Zhao, Jialing; Wang, Changzhong; Yao, Yuhua</td>
      <td>2018</td>
      <td>2018-02-10</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5930480" target="_blank">PMC5930480</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29380690.0" target="_blank">29380690.0</a></td>
      <td><a href="https://doi.org/10.2174/1386207321666180130100838" target="_blank">10.2174/1386207321666180130100838</a></td>
      <td>u1imic5l</td>
      <td>0.584662</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a></td>
    </tr>
    <tr>
      <th id="df5mfd82";>114</th>
      <td>Hoffman_2016</td>
      <td>Automatically quantifying the scientific quality and sensationalism of news records mentioning pandemics: validating a maximum entropy machine-learning model</td>
      <td>Hoffman, Steven J.; Justicz, Victoria</td>
      <td>2016</td>
      <td>2016-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127105" target="_blank">PMC7127105</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26854419.0" target="_blank">26854419.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jclinepi.2015.12.010" target="_blank">10.1016/j.jclinepi.2015.12.010</a></td>
      <td>df5mfd82</td>
      <td>0.582685</td>
      <td><a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_54.html#qweqi0et">Kobres_2019</a>, <a href="Topic_54.html#1ru3wgkk">Robinson_2013</a>, <a href="Topic_22.html#dafll0s6">Ak_Ç_2020</a>, <a href="Topic_54.html#osv673g3">Kobres_2019</a></td>
    </tr>
    <tr>
      <th id="3u81ntle";>115</th>
      <td>Acar_2016</td>
      <td>Adaptive multiresolution method for MAP reconstruction in electron tomography</td>
      <td>Acar, Erman; Peltonen, Sari; Ruotsalainen, Ulla</td>
      <td>2016</td>
      <td>2016-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115799" target="_blank">PMC7115799</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27522477.0" target="_blank">27522477.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ultramic.2016.08.002" target="_blank">10.1016/j.ultramic.2016.08.002</a></td>
      <td>3u81ntle</td>
      <td>0.580834</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#89xnnvuv">Saad_2005</a>, <a href="Topic_17.html#bqwmx5dy">Punjani_2019</a>, <a href="Topic_60.html#0jyzk5kt">Zhang_2011</a>, <a href="Topic_20.html#set51q5h">Chichón_2012</a></td>
    </tr>
    <tr>
      <th id="jbmj21c5";>116</th>
      <td>Du_Q_2017</td>
      <td>2L-PCA: a two-level principal component analyzer for quantitative drug design and its applications</td>
      <td>Du, Qi-Shi; Wang, Shu-Qing; Xie, Neng-Zhong; Wang, Qing-Yan; Huang, Ri-Bo; Chou, Kuo-Chen</td>
      <td>2017</td>
      <td>2017-08-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5642577" target="_blank">PMC5642577</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29050302.0" target="_blank">29050302.0</a></td>
      <td><a href="https://doi.org/10.18632/oncotarget.19757" target="_blank">10.18632/oncotarget.19757</a></td>
      <td>jbmj21c5</td>
      <td>0.579869</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#cx3ex9ru">Leach_2010</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_18.html#bs17igjs">Zhou_2010</a>, <a href="Topic_17.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_17.html#ni6iyzdn">He_Z_2010</a>, <a href="Topic_17.html#fewy8y4a">Wang_2018</a></td>
    </tr>
    <tr>
      <th id="rtpbltkf";>117</th>
      <td>EL-Manzalawy_2008</td>
      <td>Predicting linear B-cell epitopes using string kernels</td>
      <td>EL-Manzalawy, Yasser; Dobbs, Drena; Honavar, Vasant</td>
      <td>2008</td>
      <td>2008-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2683948" target="_blank">PMC2683948</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18496882.0" target="_blank">18496882.0</a></td>
      <td><a href="https://doi.org/10.1002/jmr.893" target="_blank">10.1002/jmr.893</a></td>
      <td>rtpbltkf</td>
      <td>0.579199</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_69.html#60xkxdhe">Hua_R_2013</a>, <a href="Topic_69.html#fmumym1x">Lon_J_2020</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_69.html#ehhotvqc">Huang_2008</a></td>
    </tr>
    <tr>
      <th id="fsh3gsvd";>118</th>
      <td>Xiao_2005</td>
      <td>Using cellular automata images and pseudo amino acid composition to predict protein subcellular location</td>
      <td>Xiao, X.; Shao, S.; Ding, Y.; Huang, Z.; Chou, K.-C.</td>
      <td>2005</td>
      <td>2005-07-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087770" target="_blank">PMC7087770</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16044193.0" target="_blank">16044193.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-005-0225-6" target="_blank">10.1007/s00726-005-0225-6</a></td>
      <td>fsh3gsvd</td>
      <td>0.578368</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_17.html#98oo0bkl">Nakai_1992</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="zocunore";>119</th>
      <td>Liao_2006</td>
      <td>Coronavirus phylogeny based on triplets of nucleic acids bases</td>
      <td>Liao, Bo; Liu, Yanshu; Li, Renfa; Zhu, Wen</td>
      <td>2006</td>
      <td>2006-04-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094651" target="_blank">PMC7094651</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cplett.2006.01.030" target="_blank">10.1016/j.cplett.2006.01.030</a></td>
      <td>zocunore</td>
      <td>0.577635</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_21.html#mj9ea464">Lapps_1987</a>, <a href="Topic_21.html#wdi66k53">Jouvenne_1992</a>, <a href="Topic_21.html#f9otvby6">Boursnell_1984</a></td>
    </tr>
    <tr>
      <th id="7vwpvcdd";>120</th>
      <td>Liu_S_2012</td>
      <td>Application of Consensus Scoring and Principal Component Analysis for Virtual Screening against β-Secretase (BACE-1)</td>
      <td>Liu, Shu; Fu, Rao; Zhou, Li-Hua; Chen, Sheng-Ping</td>
      <td>2012</td>
      <td>2012-06-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372491" target="_blank">PMC3372491</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22701601.0" target="_blank">22701601.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0038086" target="_blank">10.1371/journal.pone.0038086</a></td>
      <td>7vwpvcdd</td>
      <td>0.576888</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_18.html#8fwxa8um">Lee_H_2014</a>, <a href="Topic_16.html#et00ovrr">Grazioso_2012</a>, <a href="Topic_18.html#upzmzauj">Theerawatanasirikul_2020</a></td>
    </tr>
    <tr>
      <th id="nray6171";>121</th>
      <td>Tan_T_2007</td>
      <td>A novel cognitive interpretation of breast cancer thermography with complementary learning fuzzy neural memory structure</td>
      <td>Tan, T.Z.; Quek, C.; Ng, G.S.; Ng, E.Y.K.</td>
      <td>2007</td>
      <td>2007-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126614" target="_blank">PMC7126614</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2006.06.012" target="_blank">10.1016/j.eswa.2006.06.012</a></td>
      <td>nray6171</td>
      <td>0.576842</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#zkc4cqvt">Shaw_2012</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a></td>
      <td><a href="Topic_01.html#85axgasj">Bandalakunta_Gururajarao_2019</a>, <a href="Topic_14.html#1r4gm2d4">Liu_Y_2020</a>, <a href="Topic_01.html#0avs6g6a">Lahiri_2012</a>, <a href="Topic_09.html#hmd3quxa">Zeeneldin_2018</a></td>
    </tr>
    <tr>
      <th id="gr108zuo";>122</th>
      <td>Deng_2015</td>
      <td>An ensemble strategy that significantly improves de novo assembly of microbial genomes from metagenomic next-generation sequencing data</td>
      <td>Deng, Xutao; Naccache, Samia N.; Ng, Terry; Federman, Scot; Li, Linlin; Chiu, Charles Y.; Delwart, Eric L.</td>
      <td>2015</td>
      <td>2015-04-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4402509" target="_blank">PMC4402509</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25586223.0" target="_blank">25586223.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkv002" target="_blank">10.1093/nar/gkv002</a></td>
      <td>gr108zuo</td>
      <td>0.576520</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_17.html#rbwgi0ql">Lin_Y_2017</a>, <a href="Topic_34.html#cg463fbn">Wang_2013</a>, <a href="Topic_21.html#fcvddgvy">Viehweger_2019</a></td>
    </tr>
    <tr>
      <th id="d6wcgkln";>123</th>
      <td>Wang_2008</td>
      <td>WSE, a new sequence distance measure based on word frequencies</td>
      <td>Wang, Jun; Zheng, Xiaoqi</td>
      <td>2008</td>
      <td>2008-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.mbs.2008.06.001" target="_blank">10.1016/j.mbs.2008.06.001</a></td>
      <td>d6wcgkln</td>
      <td>0.576164</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#8qnq02gd">Zheng_2009</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#y7697hx8">Dai_Q_2013</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a>, <a href="Topic_61.html#3mj670c2">Gu_W_2004</a></td>
    </tr>
    <tr>
      <th id="fridlqhv";>124</th>
      <td>AlMoammar_2018</td>
      <td>Selecting Accurate Classifier Models for a MERS-CoV Dataset</td>
      <td>AlMoammar, Afnan; AlHenaki, Lubna; Kurdi, Heba</td>
      <td>2018</td>
      <td>2018-11-09</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123473" target="_blank">PMC7123473</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-01054-6_74" target="_blank">10.1007/978-3-030-01054-6_74</a></td>
      <td>fridlqhv</td>
      <td>0.574948</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#vm5zjr64">Zuo_G_2011</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_55.html#dl194hu8">Faridi_2016</a>, <a href="Topic_55.html#4hk736ev">Alfaraj_2018</a></td>
    </tr>
    <tr>
      <th id="worgd0xu";>125</th>
      <td>Hatcher_2017</td>
      <td>Virus Variation Resource – improved response to emergent viral outbreaks</td>
      <td>Hatcher, Eneida L.; Zhdanov, Sergey A.; Bao, Yiming; Blinkova, Olga; Nawrocki, Eric P.; Ostapchuck, Yuri; Schäffer, Alejandro A.; Brister, J. Rodney</td>
      <td>2017</td>
      <td>2017-01-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210549" target="_blank">PMC5210549</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27899678.0" target="_blank">27899678.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1065" target="_blank">10.1093/nar/gkw1065</a></td>
      <td>worgd0xu</td>
      <td>0.574196</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_05.html#adzvn69b">Garcia–Moliner_2013</a>, <a href="Topic_66.html#uee3lqrx">Davaro_2013</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a></td>
      <td><a href="Topic_10.html#89stvkqn">Brass_2009</a>, <a href="Topic_17.html#wd3ir3wg">Wang_2012</a></td>
    </tr>
    <tr>
      <th id="r4n2is6u";>126</th>
      <td>Sharma_2017</td>
      <td>Study of Sentiment Analysis Using Hadoop</td>
      <td>Sharma, Dipty</td>
      <td>2017</td>
      <td>2017-10-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122945" target="_blank">PMC7122945</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-981-10-6620-7_35" target="_blank">10.1007/978-981-10-6620-7_35</a></td>
      <td>r4n2is6u</td>
      <td>0.573097</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a></td>
      <td><a href="Topic_17.html#p7ga4h70">Jung_2017</a>, <a href="Topic_19.html#q856rx6b">Zhao_2020</a>, <a href="Topic_54.html#nm0i0rtz">Choi_2017</a>, <a href="Topic_17.html#3gv6469y">Bello-Orgaz_2016</a></td>
    </tr>
    <tr>
      <th id="735m0znd";>127</th>
      <td>Tien_2008</td>
      <td>Methods for simultaneously identifying coherent local clusters with smooth global patterns in gene expression profiles</td>
      <td>Tien, Yin-Jing; Lee, Yun-Shien; Wu, Han-Ming; Chen, Chun-Houh</td>
      <td>2008</td>
      <td>2008-03-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2322988" target="_blank">PMC2322988</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18366693.0" target="_blank">18366693.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-155" target="_blank">10.1186/1471-2105-9-155</a></td>
      <td>735m0znd</td>
      <td>0.571386</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#qi630tpo">Ragonnet-Cronin_2013</a>, <a href="Topic_17.html#hltgn9aj">Gatherer_2007</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="azpg5yrh";>128</th>
      <td>Mead_2019</td>
      <td>Visualization of protein sequence space with force-directed graphs, and their application to the choice of target-template pairs for homology modelling</td>
      <td>Mead, Dylan J.T.; Lunagomez, Simón; Gatherer, Derek</td>
      <td>2019</td>
      <td>2019-11-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110651" target="_blank">PMC7110651</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31377535.0" target="_blank">31377535.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmgm.2019.07.014" target="_blank">10.1016/j.jmgm.2019.07.014</a></td>
      <td>azpg5yrh</td>
      <td>0.571386</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#s7kzp795">Zemla_2011</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a>, <a href="Topic_17.html#bhtl73z9">Suresh_2015</a></td>
    </tr>
    <tr>
      <th id="iuglkdcp";>129</th>
      <td>Sperschneider_2012</td>
      <td>Predicting pseudoknotted structures across two RNA sequences</td>
      <td>Sperschneider, Jana; Datta, Amitava; Wise, Michael J.</td>
      <td>2012</td>
      <td>2012-12-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516145" target="_blank">PMC3516145</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23044552.0" target="_blank">23044552.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bts575" target="_blank">10.1093/bioinformatics/bts575</a></td>
      <td>iuglkdcp</td>
      <td>0.571372</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_61.html#9zm4per4">Dawson_2007</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_61.html#tv8hi2wy">Low_J_2010</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="tg06vxza";>130</th>
      <td>Cheng_2013</td>
      <td>CGAP: a new comprehensive platform for the comparative analysis of chloroplast genomes</td>
      <td>Cheng, Jinkui; Zeng, Xu; Ren, Guomin; Liu, Zhihua</td>
      <td>2013</td>
      <td>2013-03-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3636126" target="_blank">PMC3636126</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23496817.0" target="_blank">23496817.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-14-95" target="_blank">10.1186/1471-2105-14-95</a></td>
      <td>tg06vxza</td>
      <td>0.569301</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a></td>
    </tr>
    <tr>
      <th id="srtkqr13";>131</th>
      <td>Zhang_2005</td>
      <td>Web-based electronic patient records for collaborative medical applications</td>
      <td>Zhang, Jianguo; Sun, Jianyong; Yang, Yuanyuan; Chen, Xiaomeng; Meng, Lili; Lian, Ping</td>
      <td>2005</td>
      <td>2005-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126510" target="_blank">PMC7126510</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15755531.0" target="_blank">15755531.0</a></td>
      <td><a href="https://doi.org/10.1016/j.compmedimag.2004.09.005" target="_blank">10.1016/j.compmedimag.2004.09.005</a></td>
      <td>srtkqr13</td>
      <td>0.569059</td>
      <td><a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_24.html#jw1lxwyd">Lapinsky_2004</a>, <a href="Topic_04.html#8uinbp0r">Séroussi_2015</a>, <a href="Topic_17.html#1lgi1e60">Freifeld_2008</a></td>
    </tr>
    <tr>
      <th id="ycbyx6vu";>132</th>
      <td>McCrae_2008</td>
      <td>Synonym set extraction from the biomedical literature by lexical pattern discovery</td>
      <td>McCrae, John; Collier, Nigel</td>
      <td>2008</td>
      <td>2008-03-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2335115" target="_blank">PMC2335115</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18366721.0" target="_blank">18366721.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-159" target="_blank">10.1186/1471-2105-9-159</a></td>
      <td>ycbyx6vu</td>
      <td>0.568138</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#ak33vta6">Du_X_2019</a>, <a href="Topic_17.html#abrweljg">Yu_A_2011</a>, <a href="Topic_17.html#hgxhyg46">Greenbaum_2009</a>, <a href="Topic_17.html#uwbmox8a">Robson_2016</a></td>
    </tr>
    <tr>
      <th id="s1gwh4qg";>133</th>
      <td>Sidorov_2009</td>
      <td>SNAD: sequence name annotation-based designer</td>
      <td>Sidorov, Igor A; Reshetov, Denis A; Gorbalenya, Alexander E</td>
      <td>2009</td>
      <td>2009-08-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2739203" target="_blank">PMC2739203</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19682364.0" target="_blank">19682364.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-251" target="_blank">10.1186/1471-2105-10-251</a></td>
      <td>s1gwh4qg</td>
      <td>0.567516</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a></td>
      <td><a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#besvz92f">Schäffer_2019</a></td>
    </tr>
    <tr>
      <th id="h640qddv";>134</th>
      <td>Guo_F_2006</td>
      <td>ZCURVE_V: a new self-training system for recognizing protein-coding genes in viral and phage genomes</td>
      <td>Guo, Feng-Biao; Zhang, Chun-Ting</td>
      <td>2006</td>
      <td>2006-01-10</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1352377" target="_blank">PMC1352377</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16401352.0" target="_blank">16401352.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-9" target="_blank">10.1186/1471-2105-7-9</a></td>
      <td>h640qddv</td>
      <td>0.564384</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_17.html#f3nbz3o1">Chen_2003</a>, <a href="Topic_21.html#sik7m86l">Tzeng_2006</a>, <a href="Topic_37.html#2se4d1yp">Yun_S_2015</a></td>
    </tr>
    <tr>
      <th id="6l688krc";>135</th>
      <td>Hung_2006</td>
      <td>Alignment using genetic programming with causal trees for identification of protein functions</td>
      <td>Hung, Chun-Min; Huang, Yueh-Min; Chang, Ming-Shi</td>
      <td>2006</td>
      <td>2006-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117053" target="_blank">PMC7117053</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.na.2005.09.048" target="_blank">10.1016/j.na.2005.09.048</a></td>
      <td>6l688krc</td>
      <td>0.564056</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_34.html#3ttlrmbx">Neer_1987</a>, <a href="Topic_17.html#u1imic5l">Li_C_2018</a></td>
    </tr>
    <tr>
      <th id="geiq55mq";>136</th>
      <td>Xiong_2016</td>
      <td>Weighted multifractal cross-correlation analysis based on Shannon entropy</td>
      <td>Xiong, Hui; Shang, Pengjian</td>
      <td>2016</td>
      <td>2016-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128505" target="_blank">PMC7128505</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.cnsns.2015.06.029" target="_blank">10.1016/j.cnsns.2015.06.029</a></td>
      <td>geiq55mq</td>
      <td>0.562445</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#b25mi0sp">Zhao_2013</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#a2m46j0z">Tian_2016</a>, <a href="Topic_71.html#tjnt01ld">Argyroudis_2019</a>, <a href="Topic_06.html#7umn0vkv">Tao_Y_2020</a>, <a href="Topic_06.html#hi5afmbw">Pan_J_2008</a></td>
    </tr>
    <tr>
      <th id="qlwc8g2s";>137</th>
      <td>Gamermann_2014</td>
      <td>New Approach for Phylogenetic Tree Recovery Based on Genome-Scale Metabolic Networks</td>
      <td>Gamermann, Daniel; Montagud, Arnaud; Conejero, J. Alberto; Urchueguía, Javier F.; de Córdoba, Pedro Fernández</td>
      <td>2014</td>
      <td>2014-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4082356" target="_blank">PMC4082356</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24611553.0" target="_blank">24611553.0</a></td>
      <td><a href="https://doi.org/10.1089/cmb.2013.0150" target="_blank">10.1089/cmb.2013.0150</a></td>
      <td>qlwc8g2s</td>
      <td>0.562389</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a>, <a href="Topic_34.html#e0wmnfyn">McGuire_2012</a></td>
    </tr>
    <tr>
      <th id="3fwd4guk";>138</th>
      <td>Xu_Y_2013</td>
      <td>iSNO-AAPair: incorporating amino acid pairwise coupling into PseAAC for predicting cysteine S-nitrosylation sites in proteins</td>
      <td>Xu, Yan; Shao, Xiao-Jian; Wu, Ling-Yun; Deng, Nai-Yang; Chou, Kuo-Chen</td>
      <td>2013</td>
      <td>2013-10-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3792191" target="_blank">PMC3792191</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24109555.0" target="_blank">24109555.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.171" target="_blank">10.7717/peerj.171</a></td>
      <td>3fwd4guk</td>
      <td>0.561357</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_16.html#g0h0itzs">Buller_2012</a>, <a href="Topic_21.html#4gjo72hx">Tremblay-Létourneau_2011</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a></td>
    </tr>
    <tr>
      <th id="qgit5qqo";>139</th>
      <td>Reeder_2006</td>
      <td>Beyond Mfold: Recent advances in RNA bioinformatics</td>
      <td>Reeder, Jens; Höchsmann, Matthias; Rehmsmeier, Marc; Voss, Björn; Giegerich, Robert</td>
      <td>2006</td>
      <td>2006-06-25</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125698" target="_blank">PMC7125698</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16530285.0" target="_blank">16530285.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbiotec.2006.01.034" target="_blank">10.1016/j.jbiotec.2006.01.034</a></td>
      <td>qgit5qqo</td>
      <td>0.559897</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_61.html#9zm4per4">Dawson_2007</a></td>
      <td><a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a>, <a href="Topic_16.html#vhsipxs7">Järvelin_2016</a>, <a href="Topic_01.html#igkhn4m4">Ding_2012</a>, <a href="Topic_21.html#wo462anq">Xia_H_2015</a></td>
    </tr>
    <tr>
      <th id="8k7d1riv";>140</th>
      <td>Muppirala_2011</td>
      <td>Predicting RNA-Protein Interactions Using Only Sequence Information</td>
      <td>Muppirala, Usha K; Honavar, Vasant G; Dobbs, Drena</td>
      <td>2011</td>
      <td>2011-12-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322362" target="_blank">PMC3322362</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22192482.0" target="_blank">22192482.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-489" target="_blank">10.1186/1471-2105-12-489</a></td>
      <td>8k7d1riv</td>
      <td>0.558228</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_17.html#6oca1mrm">Shen_2018</a>, <a href="Topic_21.html#06qfbvlr">Li_Z_2009</a></td>
    </tr>
    <tr>
      <th id="yupm6smb";>141</th>
      <td>Liang_2018</td>
      <td>Forecasting influenza epidemics by integrating internet search queries and traditional surveillance data with the support vector machine regression model in Liaoning, from 2011 to 2015</td>
      <td>Liang, Feng; Guan, Peng; Wu, Wei; Huang, Desheng</td>
      <td>2018</td>
      <td>2018-06-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6022725" target="_blank">PMC6022725</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29967755.0" target="_blank">29967755.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.5134" target="_blank">10.7717/peerj.5134</a></td>
      <td>yupm6smb</td>
      <td>0.556074</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#ctpgylnr">Yuan_2013</a>, <a href="Topic_22.html#fp76ssuu">Chen_2019</a>, <a href="Topic_06.html#okfzkidc">Martin_2011</a>, <a href="Topic_19.html#qpt5dj8p">Li_M_2020</a></td>
    </tr>
    <tr>
      <th id="qutbqcwb";>142</th>
      <td>Zuo_G_2015</td>
      <td>CVTree3 Web Server for Whole-genome-based and Alignment-free Prokaryotic Phylogeny and Taxonomy</td>
      <td>Zuo, Guanghong; Hao, Bailin</td>
      <td>2015</td>
      <td>2015-11-10</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4678791" target="_blank">PMC4678791</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26563468.0" target="_blank">26563468.0</a></td>
      <td><a href="https://doi.org/10.1016/j.gpb.2015.08.004" target="_blank">10.1016/j.gpb.2015.08.004</a></td>
      <td>qutbqcwb</td>
      <td>0.555511</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a></td>
      <td><a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a></td>
    </tr>
    <tr>
      <th id="ev6acl7b";>143</th>
      <td>Lam_H_2017</td>
      <td>Improved Algorithmic Complexity for the 3SEQ Recombination Detection Algorithm</td>
      <td>Lam, Ha Minh; Ratmann, Oliver; Boni, Maciej F</td>
      <td>2017</td>
      <td>2017-10-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850291" target="_blank">PMC5850291</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29029186.0" target="_blank">29029186.0</a></td>
      <td><a href="https://doi.org/10.1093/molbev/msx263" target="_blank">10.1093/molbev/msx263</a></td>
      <td>ev6acl7b</td>
      <td>0.555502</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_02.html#xk6s2bp8">Muslin_2019</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a></td>
    </tr>
    <tr>
      <th id="wtarrhf0";>144</th>
      <td>Broseus_2020</td>
      <td>TALC: Transcription Aware Long Read Correction</td>
      <td>Lucile Broseus; Aubin Thomas; Andrew J. Oldfield; Dany Severac; Emeric Dubois; William Ritchie</td>
      <td>2020</td>
      <td>2020-01-11</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.10.901728" target="_blank">10.1101/2020.01.10.901728</a></td>
      <td>wtarrhf0</td>
      <td>0.552887</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a></td>
      <td><a href="Topic_34.html#tjxtifao">Lee_V_2020</a>, <a href="Topic_34.html#er3kkn9m">Trypsteen_2016</a>, <a href="Topic_34.html#qinv80fu">Donovan-Banfield_2020</a>, <a href="Topic_21.html#soxxnnk8">Habjan_2013</a></td>
    </tr>
    <tr>
      <th id="bf5rct4q";>145</th>
      <td>Jalali-Heravi_2009</td>
      <td>Shuffling multivariate adaptive regression splines and adaptive neuro-fuzzy inference system as tools for QSAR study of SARS inhibitors</td>
      <td>Jalali-Heravi, M.; Asadollahi-Baboli, M.; Mani-Varnosfaderani, A.</td>
      <td>2009</td>
      <td>2009-12-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126869" target="_blank">PMC7126869</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19665859.0" target="_blank">19665859.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jpba.2009.07.009" target="_blank">10.1016/j.jpba.2009.07.009</a></td>
      <td>bf5rct4q</td>
      <td>0.551918</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_06.html#63wqn2fg">Höhle_2007</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_18.html#bs17igjs">Zhou_2010</a>, <a href="Topic_17.html#fewy8y4a">Wang_2018</a>, <a href="Topic_18.html#gqnzwyn3">Mukherjee_2011</a></td>
    </tr>
    <tr>
      <th id="w49x7kca";>146</th>
      <td>Biao_2016</td>
      <td>Accurate prediction of human essential genes using only nucleotide composition and association information</td>
      <td>Feng-Biao Biao; Chuan Dong; Hong-Li Hua; Shuo Liu; Hao Luo; Hong-Wan Zhang; Yan-Ting Jin; Kai-Yue Zhang</td>
      <td>2016</td>
      <td>2016-10-28</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/084129" target="_blank">10.1101/084129</a></td>
      <td>w49x7kca</td>
      <td>0.550990</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#vj9tv3or">Guo_F_2017</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_43.html#i66vbuab">Kalia_2016</a>, <a href="Topic_52.html#3ury4hnv">Levy_2000</a></td>
    </tr>
    <tr>
      <th id="zamrnne9";>147</th>
      <td>Pérez-Montoto_2009</td>
      <td>Study of peptide fingerprints of parasite proteins and drug–DNA interactions with Markov-Mean-Energy invariants of biopolymer molecular-dynamic lattice networks</td>
      <td>Pérez-Montoto, Lázaro Guillermo; Dea-Ayuela, María Auxiliadora; Prado-Prado, Francisco J.; Bolas-Fernández, Francisco; Ubeira, Florencio M.; González-Díaz, Humberto</td>
      <td>2009</td>
      <td>2009-07-17</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111648" target="_blank">PMC7111648</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.polymer.2009.05.055" target="_blank">10.1016/j.polymer.2009.05.055</a></td>
      <td>zamrnne9</td>
      <td>0.548998</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_02.html#i2mac4xm">Bissaro_2020</a>, <a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_10.html#gxp4sqq0">Kumar_2013</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a></td>
    </tr>
    <tr>
      <th id="k7hm3aow";>148</th>
      <td>Kuhring_2019</td>
      <td>An iterative and automated computational pipeline for untargeted strain-level identification using MS/MS spectra from pathogenic samples</td>
      <td>Mathias Kuhring; Joerg Doellinger; Andreas Nitsche; Thilo Muth; Bernhard Y. Renard</td>
      <td>2019</td>
      <td>2019-10-24</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/812313" target="_blank">10.1101/812313</a></td>
      <td>k7hm3aow</td>
      <td>0.548839</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#j6wdclpu">Naidoo_2014</a>, <a href="Topic_07.html#15cxc32n">Shifman_2019</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a></td>
    </tr>
    <tr>
      <th id="e17qi1v2";>149</th>
      <td>Raj_A_2011</td>
      <td>Identifying Hosts of Families of Viruses: A Machine Learning Approach</td>
      <td>Raj, Anil; Dewar, Michael; Palacios, Gustavo; Rabadan, Raul; Wiggins, Christopher H.</td>
      <td>2011</td>
      <td>2011-12-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3235098" target="_blank">PMC3235098</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22174744.0" target="_blank">22174744.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0027631" target="_blank">10.1371/journal.pone.0027631</a></td>
      <td>e17qi1v2</td>
      <td>0.548727</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_36.html#wa5il5go">Brierley_2019</a>, <a href="Topic_36.html#e0plqpgb">Brierley_2019</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a>, <a href="Topic_17.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="7u5frg8w";>150</th>
      <td>Zhao_2011</td>
      <td>A novel clustering method via nucleotide-based Fourier power spectrum analysis</td>
      <td>Zhao, Bo; Duan, Victor; Yau, Stephen S.-T.</td>
      <td>2011</td>
      <td>2011-06-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094093" target="_blank">PMC7094093</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21443881.0" target="_blank">21443881.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2011.03.029" target="_blank">10.1016/j.jtbi.2011.03.029</a></td>
      <td>7u5frg8w</td>
      <td>0.548352</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#nw3tybw9">Hoang_2015</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a>, <a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#buc4dd5y">Dong_2019</a></td>
    </tr>
    <tr>
      <th id="sjl8x1jt";>151</th>
      <td>Tithi_2018</td>
      <td>FastViromeExplorer: a pipeline for virus and phage identification and abundance profiling in metagenomics data</td>
      <td>Tithi, Saima Sultana; Aylward, Frank O.; Jensen, Roderick V.; Zhang, Liqing</td>
      <td>2018</td>
      <td>2018-01-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5768174" target="_blank">PMC5768174</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29340239.0" target="_blank">29340239.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.4227" target="_blank">10.7717/peerj.4227</a></td>
      <td>sjl8x1jt</td>
      <td>0.548317</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#uw76y16g">Tithi_2017</a>, <a href="Topic_35.html#aa4s7b21">Li_C_2020</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a>, <a href="Topic_50.html#eifztaov">Selvin_2019</a></td>
    </tr>
    <tr>
      <th id="wyrvxocw";>152</th>
      <td>Porter_2014</td>
      <td>A multivariate CAR model for mismatched lattices</td>
      <td>Porter, Aaron T.; Oleson, Jacob J.</td>
      <td>2014</td>
      <td>2014-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25457598.0" target="_blank">25457598.0</a></td>
      <td><a href="https://doi.org/10.1016/j.sste.2014.08.001" target="_blank">10.1016/j.sste.2014.08.001</a></td>
      <td>wyrvxocw</td>
      <td>0.547886</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_06.html#s7q1829b">Liccardo_2013</a>, <a href="Topic_06.html#w4g8mzuy">Groendyke_2012</a>, <a href="Topic_06.html#ygdau2px">Roy_M_2006</a></td>
    </tr>
    <tr>
      <th id="59yvbupx";>153</th>
      <td>Scott_2011</td>
      <td>NoD: a Nucleolar localization sequence detector for eukaryotic and viral proteins</td>
      <td>Scott, Michelle S; Troshin, Peter V; Barton, Geoffrey J</td>
      <td>2011</td>
      <td>2011-08-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3166288" target="_blank">PMC3166288</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21812952.0" target="_blank">21812952.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-317" target="_blank">10.1186/1471-2105-12-317</a></td>
      <td>59yvbupx</td>
      <td>0.547414</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a></td>
      <td><a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_10.html#wggzeqwe">Fulcher_2011</a>, <a href="Topic_10.html#s935ttyn">Wulan_2015</a></td>
    </tr>
    <tr>
      <th id="a0pzw26l";>154</th>
      <td>Yu_Z_2010</td>
      <td>Proper Distance Metrics for Phylogenetic Analysis Using Complete Genomes without Sequence Alignment</td>
      <td>Yu, Zu-Guo; Zhan, Xiao-Wen; Han, Guo-Sheng; Wang, Roger W.; Anh, Vo; Chu, Ka Hou</td>
      <td>2010</td>
      <td>2010-03-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2869232" target="_blank">PMC2869232</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20480005.0" target="_blank">20480005.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms11031141" target="_blank">10.3390/ijms11031141</a></td>
      <td>a0pzw26l</td>
      <td>0.546611</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#gf718lbl">Yu_Z_2010</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_34.html#telmxmp4">Hawkins_2019</a></td>
    </tr>
    <tr>
      <th id="f6lfykot";>155</th>
      <td>Wu_C_2018</td>
      <td>A novel model for protein sequence similarity analysis based on spectral radius</td>
      <td>Wu, Chuanyan; Gao, Rui; De Marinis, Yang; Zhang, Yusen</td>
      <td>2018</td>
      <td>2018-06-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094169" target="_blank">PMC7094169</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29524440.0" target="_blank">29524440.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2018.03.001" target="_blank">10.1016/j.jtbi.2018.03.001</a></td>
      <td>f6lfykot</td>
      <td>0.546349</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_17.html#u1imic5l">Li_C_2018</a>, <a href="Topic_17.html#41atidai">Abo-Elkhier_2019</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#9t3kneoo">Abd_Elwahaab_2019</a></td>
    </tr>
    <tr>
      <th id="z1xao2ic";>156</th>
      <td>Masoudi-Sobhanzadeh_2016</td>
      <td>World Competitive Contests (WCC) algorithm: A novel intelligent optimization algorithm for biological and non-biological problems</td>
      <td>Masoudi-Sobhanzadeh, Yosef; Motieghader, Habib</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.imu.2016.06.002" target="_blank">10.1016/j.imu.2016.06.002</a></td>
      <td>z1xao2ic</td>
      <td>0.545421</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#1ul9zqhb">Mousavi_2012</a>, <a href="Topic_17.html#j51gaesp">Amiri_2018</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a>, <a href="Topic_17.html#udzz15bk">Li_H_2009</a></td>
    </tr>
    <tr>
      <th id="5pvln91x";>157</th>
      <td>Asbury_2010</td>
      <td>Genome3D: A viewer-model framework for integrating and visualizing multi-scale epigenomic information within a three-dimensional genome</td>
      <td>Asbury, Thomas M; Mitman, Matt; Tang, Jijun; Zheng, W Jim</td>
      <td>2010</td>
      <td>2010-09-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2941692" target="_blank">PMC2941692</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20813045.0" target="_blank">20813045.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-11-444" target="_blank">10.1186/1471-2105-11-444</a></td>
      <td>5pvln91x</td>
      <td>0.544800</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_17.html#vret3sh1">Aziz_2012</a></td>
    </tr>
    <tr>
      <th id="sdgt2ms5";>158</th>
      <td>Kshirsagar_2013</td>
      <td>Multitask learning for host–pathogen protein interactions</td>
      <td>Kshirsagar, Meghana; Carbonell, Jaime; Klein-Seetharaman, Judith</td>
      <td>2013</td>
      <td>2013-07-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3694681" target="_blank">PMC3694681</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23812987.0" target="_blank">23812987.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btt245" target="_blank">10.1093/bioinformatics/btt245</a></td>
      <td>sdgt2ms5</td>
      <td>0.544422</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#8ef74ltg">Arnold_2012</a>, <a href="Topic_30.html#9rpoll7y">Mitchell_2019</a>, <a href="Topic_34.html#zlqh436d">Chasman_2016</a>, <a href="Topic_10.html#nt60lv2k">Mitchell_2013</a></td>
    </tr>
    <tr>
      <th id="vret3sh1";>159</th>
      <td>Aziz_2012</td>
      <td>SEED Servers: High-Performance Access to the SEED Genomes, Annotations, and Metabolic Models</td>
      <td>Aziz, Ramy K.; Devoid, Scott; Disz, Terrence; Edwards, Robert A.; Henry, Christopher S.; Olsen, Gary J.; Olson, Robert; Overbeek, Ross; Parrello, Bruce; Pusch, Gordon D.; Stevens, Rick L.; Vonstein, Veronika; Xia, Fangfang</td>
      <td>2012</td>
      <td>2012-10-24</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3480482" target="_blank">PMC3480482</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23110173.0" target="_blank">23110173.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0048053" target="_blank">10.1371/journal.pone.0048053</a></td>
      <td>vret3sh1</td>
      <td>0.543569</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a></td>
    </tr>
    <tr>
      <th id="zuh78iul";>160</th>
      <td>Pessia_2015</td>
      <td>K-Pax2: Bayesian identification of cluster-defining amino acid positions in large sequence datasets</td>
      <td>Pessia, Alberto; Grad, Yonatan; Cobey, Sarah; Puranen, Juha Santeri; Corander, Jukka</td>
      <td>2015</td>
      <td>2015-07-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5320600" target="_blank">PMC5320600</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28348810.0" target="_blank">28348810.0</a></td>
      <td><a href="https://doi.org/10.1099/mgen.0.000025" target="_blank">10.1099/mgen.0.000025</a></td>
      <td>zuh78iul</td>
      <td>0.543069</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_45.html#uf7b9azg">Kepler_2010</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="ca6pff0p";>161</th>
      <td>UNKNOWN_2018</td>
      <td>Database resources of the National Center for Biotechnology Information</td>
      <td>None</td>
      <td>2018</td>
      <td>2018-01-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5753372" target="_blank">PMC5753372</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29140470.0" target="_blank">29140470.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkx1095" target="_blank">10.1093/nar/gkx1095</a></td>
      <td>ca6pff0p</td>
      <td>0.542820</td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a></td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
    </tr>
    <tr>
      <th id="58p5b2vw";>162</th>
      <td>Pérez-Montoto_2009</td>
      <td>Scoring function for DNA–drug docking of anticancer and antiparasitic compounds based on spectral moments of 2D lattice graphs for molecular dynamics trajectories</td>
      <td>Pérez-Montoto, Lázaro G.; Santana, Lourdes; González-Díaz, Humberto</td>
      <td>2009</td>
      <td>2009-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127518" target="_blank">PMC7127518</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19604606.0" target="_blank">19604606.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ejmech.2009.06.011" target="_blank">10.1016/j.ejmech.2009.06.011</a></td>
      <td>58p5b2vw</td>
      <td>0.542694</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#8qnq02gd">Zheng_2009</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_17.html#vk8n8s2q">González-Díaz_2009</a>, <a href="Topic_16.html#et00ovrr">Grazioso_2012</a>, <a href="Topic_17.html#zamrnne9">Pérez-Montoto_2009</a></td>
    </tr>
    <tr>
      <th id="8gyejoc5";>163</th>
      <td>Finnie_2016</td>
      <td>EpiJSON: A unified data-format for epidemiology</td>
      <td>Finnie, Thomas J.R.; South, Andy; Bento, Ana; Sherrard-Smith, Ellie; Jombart, Thibaut</td>
      <td>2016</td>
      <td>2016-06-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104924" target="_blank">PMC7104924</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27266846.0" target="_blank">27266846.0</a></td>
      <td><a href="https://doi.org/10.1016/j.epidem.2015.12.002" target="_blank">10.1016/j.epidem.2015.12.002</a></td>
      <td>8gyejoc5</td>
      <td>0.542340</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a></td>
      <td><a href="Topic_06.html#l4soafvr">Cori_2018</a>, <a href="Topic_04.html#20zr7mtt">Grad_2014</a>, <a href="Topic_06.html#z8ix4kz7">Robert_2020</a>, <a href="Topic_54.html#t02jngq0">Ramshaw_2019</a></td>
    </tr>
    <tr>
      <th id="l72x4wn3";>164</th>
      <td>Tagashira_2020</td>
      <td>PhyloFold: Precise and Swift Prediction of RNA Secondary Structures to Incorporate Phylogeny among Homologs</td>
      <td>Masaki Tagashira</td>
      <td>2020</td>
      <td>2020-03-06</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.05.975797" target="_blank">10.1101/2020.03.05.975797</a></td>
      <td>l72x4wn3</td>
      <td>0.540921</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_17.html#p1i7bx94">Zhang_2013</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a></td>
    </tr>
    <tr>
      <th id="8c75hp7r";>165</th>
      <td>Muthukrishnan_2014</td>
      <td>Support vector machine (SVM) based multiclass prediction with basic statistical analysis of plasminogen activators</td>
      <td>Muthukrishnan, Selvaraj; Puri, Munish; Lefevre, Christophe</td>
      <td>2014</td>
      <td>2014-01-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3924408" target="_blank">PMC3924408</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24468032.0" target="_blank">24468032.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-7-63" target="_blank">10.1186/1756-0500-7-63</a></td>
      <td>8c75hp7r</td>
      <td>0.540520</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_37.html#g4s0v8ra">Rimstad_1995</a>, <a href="Topic_17.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_52.html#2haxmyur">Huang_2012</a>, <a href="Topic_16.html#8myzo7hg">Böttcher-Friebertshäuser_2018</a></td>
    </tr>
    <tr>
      <th id="5k8np850";>166</th>
      <td>Munteanu_2009</td>
      <td>Multi-target QPDR classification model for human breast and colon cancer-related proteins using star graph topological indices</td>
      <td>Munteanu, Cristian Robert; Magalhães, Alexandre L.; Uriarte, Eugenio; González-Díaz, Humberto</td>
      <td>2009</td>
      <td>2009-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094125" target="_blank">PMC7094125</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19111559.0" target="_blank">19111559.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2008.11.017" target="_blank">10.1016/j.jtbi.2008.11.017</a></td>
      <td>5k8np850</td>
      <td>0.538981</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_17.html#g4k6pc82">Munteanu_2008</a>, <a href="Topic_17.html#8dnyjuig">Alejandro_2018</a>, <a href="Topic_17.html#g60hnqlf">Wang_2010</a></td>
    </tr>
    <tr>
      <th id="b8jlvkg7";>167</th>
      <td>Ai_Y_2013</td>
      <td>GenomeFingerprinter: The Genome Fingerprint and the Universal Genome Fingerprint Analysis for Systematic Comparative Genomics</td>
      <td>Ai, Yuncan; Ai, Hannan; Meng, Fanmei; Zhao, Lei</td>
      <td>2013</td>
      <td>2013-10-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3812135" target="_blank">PMC3812135</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24205026.0" target="_blank">24205026.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0077912" target="_blank">10.1371/journal.pone.0077912</a></td>
      <td>b8jlvkg7</td>
      <td>0.538024</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a>, <a href="Topic_21.html#5d64esp4">Walker_2015</a></td>
    </tr>
    <tr>
      <th id="s0ps9rd0";>168</th>
      <td>Xu_J_2010</td>
      <td>QSAR Studies on Andrographolide Derivatives as α-Glucosidase Inhibitors</td>
      <td>Xu, Jun; Huang, Sichao; Luo, Haibin; Li, Guoji; Bao, Jiaolin; Cai, Shaohui; Wang, Yuqiang</td>
      <td>2010</td>
      <td>2010-03-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2869241" target="_blank">PMC2869241</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20479989.0" target="_blank">20479989.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms11030880" target="_blank">10.3390/ijms11030880</a></td>
      <td>s0ps9rd0</td>
      <td>0.537456</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_18.html#rhld75dy">Zhong_2015</a>, <a href="Topic_49.html#1vbkttzx">Li_X_2019</a>, <a href="Topic_25.html#prnpojgb">Li_W_2011</a>, <a href="Topic_58.html#8hyw5z99">Xie_J_2018</a></td>
    </tr>
    <tr>
      <th id="zqmdcj1z";>169</th>
      <td>Dandekar_2017</td>
      <td>Generating Fake but Realistic Headlines Using Deep Neural Networks</td>
      <td>Dandekar, Ashish; Zen, Remmy A. M.; Bressan, Stéphane</td>
      <td>2017</td>
      <td>2017-07-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121779" target="_blank">PMC7121779</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-64471-4_34" target="_blank">10.1007/978-3-319-64471-4_34</a></td>
      <td>zqmdcj1z</td>
      <td>0.537391</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a></td>
      <td><a href="Topic_17.html#k26orfrt">Jiang_2017</a>, <a href="Topic_54.html#nm0i0rtz">Choi_2017</a>, <a href="Topic_02.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_17.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="qdld7hdc";>170</th>
      <td>Fan_Y_2014</td>
      <td>iNR-Drug: Predicting the Interaction of Drugs with Nuclear Receptors in Cellular Networking</td>
      <td>Fan, Yue-Nong; Xiao, Xuan; Min, Jian-Liang; Chou, Kuo-Chen</td>
      <td>2014</td>
      <td>2014-03-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3975431" target="_blank">PMC3975431</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24651462.0" target="_blank">24651462.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms15034915" target="_blank">10.3390/ijms15034915</a></td>
      <td>qdld7hdc</td>
      <td>0.537161</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#cx3ex9ru">Leach_2010</a></td>
      <td><a href="Topic_17.html#kt6wqvam">Xiao_2013</a>, <a href="Topic_18.html#qmg58cgr">Tang_2020</a>, <a href="Topic_62.html#can4cvkx">Chen_2019</a></td>
    </tr>
    <tr>
      <th id="ap60vws9";>171</th>
      <td>Qureshi_2013</td>
      <td>VIRsiRNApred: a web server for predicting inhibition efficacy of siRNAs targeting human viruses</td>
      <td>Qureshi, Abid; Thakur, Nishant; Kumar, Manoj</td>
      <td>2013</td>
      <td>2013-12-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3878835" target="_blank">PMC3878835</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24330765.0" target="_blank">24330765.0</a></td>
      <td><a href="https://doi.org/10.1186/1479-5876-11-305" target="_blank">10.1186/1479-5876-11-305</a></td>
      <td>ap60vws9</td>
      <td>0.536065</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_37.html#b5u5hp2r">Liu_Y_2008</a></td>
      <td><a href="Topic_17.html#l8tx7tyy">Thakur_2011</a>, <a href="Topic_29.html#3mw66vxm">Zhou_2012</a>, <a href="Topic_18.html#qhhdjgeq">Zhu_X_2015</a>, <a href="Topic_37.html#tex6bgab">Sui_H_2009</a></td>
    </tr>
    <tr>
      <th id="2rpr7aph";>172</th>
      <td>Bhandari_2020</td>
      <td>Solubility-Weighted Index: fast and accurate prediction of protein solubility</td>
      <td>Bikash K. Bhandari; Paul P. Gardner; Chun Shen Lim</td>
      <td>2020</td>
      <td>2020-02-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.15.951012" target="_blank">10.1101/2020.02.15.951012</a></td>
      <td>2rpr7aph</td>
      <td>0.536047</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_37.html#54hvxhzd">Kwon_2019</a>, <a href="Topic_37.html#iesysf3l">Eshaghi_2005</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_16.html#njlqgih3">Robinson_2013</a></td>
    </tr>
    <tr>
      <th id="183n1lrf";>173</th>
      <td>Wilson_2003</td>
      <td>Application of data mining techniques in pharmacovigilance</td>
      <td>Wilson, Andrew M; Thabane, Lehana; Holbrook, Anne</td>
      <td>2003</td>
      <td>2003-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1884444" target="_blank">PMC1884444</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/14748811.0" target="_blank">14748811.0</a></td>
      <td><a href="https://doi.org/10.1046/j.1365-2125.2003.01968.x" target="_blank">10.1046/j.1365-2125.2003.01968.x</a></td>
      <td>183n1lrf</td>
      <td>0.535893</td>
      <td><a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_17.html#j5nmifhj">Bai_J_2016</a>, <a href="Topic_43.html#zwaxie0c">Vecoso_2019</a>, <a href="Topic_19.html#aoi4iqkf">Sun_P_2020</a></td>
    </tr>
    <tr>
      <th id="qi630tpo";>174</th>
      <td>Ragonnet-Cronin_2013</td>
      <td>Automated analysis of phylogenetic clusters</td>
      <td>Ragonnet-Cronin, Manon; Hodcroft, Emma; Hué, Stéphane; Fearnhill, Esther; Delpech, Valerie; Brown, Andrew J Leigh; Lycett, Samantha</td>
      <td>2013</td>
      <td>2013-11-06</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228337" target="_blank">PMC4228337</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24191891.0" target="_blank">24191891.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-14-317" target="_blank">10.1186/1471-2105-14-317</a></td>
      <td>qi630tpo</td>
      <td>0.534858</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#aj352do0">Tibayrenc_2015</a>, <a href="Topic_06.html#wich35l7">Velsko_2014</a>, <a href="Topic_07.html#sjxa8wn0">Duarte_2009</a></td>
    </tr>
    <tr>
      <th id="1e2kkhht";>175</th>
      <td>Tcherepanov_2006</td>
      <td>Genome Annotation Transfer Utility (GATU): rapid annotation of viral genomes using a closely related reference genome</td>
      <td>Tcherepanov, Vasily; Ehlers, Angelika; Upton, Chris</td>
      <td>2006</td>
      <td>2006-06-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1534038" target="_blank">PMC1534038</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16772042.0" target="_blank">16772042.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2164-7-150" target="_blank">10.1186/1471-2164-7-150</a></td>
      <td>1e2kkhht</td>
      <td>0.532241</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a></td>
    </tr>
    <tr>
      <th id="oeokiay7";>176</th>
      <td>Fusani_2018</td>
      <td>Active learning strategies with COMBINE analysis: new tricks for an old dog</td>
      <td>Fusani, Lucia; Cabrera, Alvaro Cortes</td>
      <td>2018</td>
      <td>2018-12-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087723" target="_blank">PMC7087723</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30564994.0" target="_blank">30564994.0</a></td>
      <td><a href="https://doi.org/10.1007/s10822-018-0181-3" target="_blank">10.1007/s10822-018-0181-3</a></td>
      <td>oeokiay7</td>
      <td>0.531226</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_18.html#qmg58cgr">Tang_2020</a>, <a href="Topic_18.html#bs17igjs">Zhou_2010</a>, <a href="Topic_17.html#qah877pk">Ekins_2007</a>, <a href="Topic_18.html#kauk7ze0">Durrant_2013</a></td>
    </tr>
    <tr>
      <th id="z29xsj1o";>177</th>
      <td>Morfopoulou_2015</td>
      <td>Bayesian mixture analysis for metagenomic community profiling</td>
      <td>Morfopoulou, Sofia; Plagnol, Vincent</td>
      <td>2015</td>
      <td>2015-09-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4565032" target="_blank">PMC4565032</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26002885.0" target="_blank">26002885.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv317" target="_blank">10.1093/bioinformatics/btv317</a></td>
      <td>z29xsj1o</td>
      <td>0.526295</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#058r9486">Morfopoulou_2014</a>, <a href="Topic_14.html#xlqdn0c7">Baldwin_2014</a>, <a href="Topic_50.html#eifztaov">Selvin_2019</a>, <a href="Topic_14.html#xo1cyzrg">Giles_2015</a></td>
    </tr>
    <tr>
      <th id="4hlcxccy";>178</th>
      <td>Agrawal_2018</td>
      <td>Prediction of Antimicrobial Potential of a Chemically Modified Peptide From Its Tertiary Structure</td>
      <td>Agrawal, Piyush; Raghava, Gajendra P. S.</td>
      <td>2018</td>
      <td>2018-10-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6212470" target="_blank">PMC6212470</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30416494.0" target="_blank">30416494.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2018.02551" target="_blank">10.3389/fmicb.2018.02551</a></td>
      <td>4hlcxccy</td>
      <td>0.525161</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_16.html#0pmo3opx">Joshi_2017</a>, <a href="Topic_18.html#gxbrgy6g">Lee_C_2019</a>, <a href="Topic_49.html#gl55zhgw">Alvin_2014</a>, <a href="Topic_49.html#rx1z4orm">Patra_2012</a></td>
    </tr>
    <tr>
      <th id="y5xsf3ij";>179</th>
      <td>Tapinos_2019</td>
      <td>The Utility of Data Transformation for Alignment, De Novo Assembly and Classification of Short Read Virus Sequences</td>
      <td>Tapinos, Avraam; Constantinides, Bede; Phan, My V. T.; Kouchaki, Samaneh; Cotten, Matthew; Robertson, David L.</td>
      <td>2019</td>
      <td>2019-04-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563281" target="_blank">PMC6563281</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31035503.0" target="_blank">31035503.0</a></td>
      <td><a href="https://doi.org/10.3390/v11050394" target="_blank">10.3390/v11050394</a></td>
      <td>y5xsf3ij</td>
      <td>0.524978</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a></td>
    </tr>
    <tr>
      <th id="p6075gva";>180</th>
      <td>Scheuch_2015</td>
      <td>RIEMS: a software pipeline for sensitive and comprehensive taxonomic classification of reads from metagenomics datasets</td>
      <td>Scheuch, Matthias; Höper, Dirk; Beer, Martin</td>
      <td>2015</td>
      <td>2015-03-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4351923" target="_blank">PMC4351923</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25886935.0" target="_blank">25886935.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-015-0503-6" target="_blank">10.1186/s12859-015-0503-6</a></td>
      <td>p6075gva</td>
      <td>0.524457</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_50.html#qg2r691d">Twin_2013</a>, <a href="Topic_17.html#csokkcqq">Shcherbina_2016</a>, <a href="Topic_17.html#z29xsj1o">Morfopoulou_2015</a>, <a href="Topic_17.html#058r9486">Morfopoulou_2014</a></td>
    </tr>
    <tr>
      <th id="l56eboio";>181</th>
      <td>Zhao_2013</td>
      <td>Design and Experimental Approach to the Construction of a Human Signal-Molecule-Profiling Database</td>
      <td>Zhao, Xinyan; Dong, Tao</td>
      <td>2013</td>
      <td>2013-12-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3881147" target="_blank">PMC3881147</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24351788.0" target="_blank">24351788.0</a></td>
      <td><a href="https://doi.org/10.3390/ijerph10126887" target="_blank">10.3390/ijerph10126887</a></td>
      <td>l56eboio</td>
      <td>0.523789</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_49.html#5ef2no83">Deiab_2013</a>, <a href="Topic_17.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_23.html#2601n7a9">Leu_S_2010</a>, <a href="Topic_18.html#s50g754b">Chiba_2012</a></td>
    </tr>
    <tr>
      <th id="8qnq02gd";>182</th>
      <td>Zheng_2009</td>
      <td>A Poisson model of sequence comparison and its application to coronavirus phylogeny</td>
      <td>Zheng, Xiaoqi; Qin, Yufang; Wang, Jun</td>
      <td>2009</td>
      <td>2009-02-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094598" target="_blank">PMC7094598</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19073197.0" target="_blank">19073197.0</a></td>
      <td><a href="https://doi.org/10.1016/j.mbs.2008.11.006" target="_blank">10.1016/j.mbs.2008.11.006</a></td>
      <td>8qnq02gd</td>
      <td>0.523748</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_07.html#z9p1ylsr">Sheikh_2020</a>, <a href="Topic_07.html#490n4rxk">Kusters_1989</a>, <a href="Topic_07.html#4b3pfwsp">Li_Z_2013</a>, <a href="Topic_07.html#vbadg8ki">Jeong_2005</a></td>
    </tr>
    <tr>
      <th id="2l1zw19o";>183</th>
      <td>Ozturk_2020</td>
      <td>Classification of Coronavirus Images using Shrunken Features</td>
      <td>Saban Ozturk; Umut Ozkaya; Mucahid Barstugan</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.03.20048868" target="_blank">10.1101/2020.04.03.20048868</a></td>
      <td>2l1zw19o</td>
      <td>0.522041</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_14.html#u2uswgy3">Wang_2020</a>, <a href="Topic_17.html#zqsy6r4i">Song_2020</a></td>
    </tr>
    <tr>
      <th id="nzv96tjh";>184</th>
      <td>Stark_2019</td>
      <td>Neural organization of speech production: A lesion-based study of error patterns in connected speech</td>
      <td>Brielle C Stark; Alexandra Basilakos; Gregory Hickok; Chris Rorden; Leonardo Bonilha; Julius Fridriksson</td>
      <td>2019</td>
      <td>2019-02-08</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/544841" target="_blank">10.1101/544841</a></td>
      <td>nzv96tjh</td>
      <td>0.521726</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_74.html#pxvdx7p2">Meissner-Bernard_2018</a>, <a href="Topic_28.html#gd656u2q">An_S_2017</a>, <a href="Topic_02.html#6zjf6qzz">Li_J_2015</a></td>
      <td><a href="Topic_46.html#8uw6h6ge">Mohammed_1990</a>, <a href="Topic_46.html#ekze7vj0">Gilani_2020</a>, <a href="Topic_47.html#b3wiay6w">Pietrowsky_1996</a>, <a href="Topic_08.html#0muvg348">Volk_2017</a></td>
    </tr>
    <tr>
      <th id="48mtdwuv";>185</th>
      <td>Davis_2020</td>
      <td>Vorpal: A novel RNA virus feature-extraction algorithm demonstrated through interpretable genotype-to-phenotype linear models</td>
      <td>Phillip Davis; John Bagnoli; David Yarmosh; Alan Shteyman; Lance Presser; Sharon Altmann; Shelton Bradrick; Joseph A. Russell</td>
      <td>2020</td>
      <td>2020-03-02</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.28.969782" target="_blank">10.1101/2020.02.28.969782</a></td>
      <td>48mtdwuv</td>
      <td>0.521040</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_07.html#thsq6pnk">Lachheb_2019</a></td>
    </tr>
    <tr>
      <th id="5wtrij3m";>186</th>
      <td>Zhou_2018</td>
      <td>Early detection of dynamic harmful cascades in large-scale networks</td>
      <td>Zhou, Chuan; Lu, Wei-Xue; Zhang, Jingzun; Li, Lei; Hu, Yue; Guo, Li</td>
      <td>2018</td>
      <td>2018-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102699" target="_blank">PMC7102699</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jocs.2017.10.014" target="_blank">10.1016/j.jocs.2017.10.014</a></td>
      <td>5wtrij3m</td>
      <td>0.520817</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_06.html#xbdrf8h9">Kan_J_2017</a>, <a href="Topic_06.html#934xczf3">Zhan_2018</a>, <a href="Topic_06.html#51n89hvc">Shi_T_2019</a>, <a href="Topic_06.html#xdlk8557">Han_X_2014</a></td>
    </tr>
    <tr>
      <th id="mgmbdfhe";>187</th>
      <td>Paez-Espino_2019</td>
      <td>IMG/VR v.2.0: an integrated data management and analysis system for cultivated and environmental viral genomes</td>
      <td>Paez-Espino, David; Roux, Simon; Chen, I-Min A; Palaniappan, Krishna; Ratner, Anna; Chu, Ken; Huntemann, Marcel; Reddy, T B K; Pons, Joan Carles; Llabrés, Mercè; Eloe-Fadrosh, Emiley A; Ivanova, Natalia N; Kyrpides, Nikos C</td>
      <td>2019</td>
      <td>2019-01-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323928" target="_blank">PMC6323928</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30407573.0" target="_blank">30407573.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gky1127" target="_blank">10.1093/nar/gky1127</a></td>
      <td>mgmbdfhe</td>
      <td>0.519362</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_45.html#hmgmsviu">Paez-Espino_2017</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_44.html#h31fi573">Murtaugh_1995</a></td>
    </tr>
    <tr>
      <th id="03uonbrv";>188</th>
      <td>Ludwig_2019</td>
      <td>ZODIAC: database-independent molecular formula annotation using Gibbs sampling reveals unknown small molecules</td>
      <td>Marcus Ludwig; Louis-Félix Nothias; Kai Dührkop; Irina Koester; Markus Fleischauer; Martin A. Hoffmann; Daniel Petras; Fernando Vargas; Mustafa Morsy; Lihini Aluwihare; Pieter C. Dorrestein; Sebastian Böcker</td>
      <td>2019</td>
      <td>2019-11-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/842740" target="_blank">10.1101/842740</a></td>
      <td>03uonbrv</td>
      <td>0.518498</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_01.html#p19e9uom">Myler_2008</a>, <a href="Topic_18.html#mw7athol">Plewczynski_2007</a>, <a href="Topic_18.html#cyp148on">Li_J_2006</a>, <a href="Topic_17.html#vk8n8s2q">González-Díaz_2009</a></td>
    </tr>
    <tr>
      <th id="60udcsa7";>189</th>
      <td>Hou_Z_2016</td>
      <td>Linear Quantitative Profiling Method Fast Monitors Alkaloids of Sophora Flavescens That Was Verified by Tri-Marker Analyses</td>
      <td>Hou, Zhifei; Sun, Guoxiang; Guo, Yong</td>
      <td>2016</td>
      <td>2016-08-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987015" target="_blank">PMC4987015</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27529425.0" target="_blank">27529425.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0161146" target="_blank">10.1371/journal.pone.0161146</a></td>
      <td>60udcsa7</td>
      <td>0.516650</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_49.html#gtcr19qr">Fang_2019</a>, <a href="Topic_49.html#yhnk7iyq">Xing_2014</a>, <a href="Topic_49.html#zl4abcn0">Subramaniam_2014</a></td>
    </tr>
    <tr>
      <th id="rbwgi0ql";>190</th>
      <td>Lin_Y_2017</td>
      <td>De novo assembly of highly polymorphic metagenomic data using in situ generated reference sequences and a novel BLAST-based assembly pipeline</td>
      <td>Lin, You-Yu; Hsieh, Chia-Hung; Chen, Jiun-Hong; Lu, Xuemei; Kao, Jia-Horng; Chen, Pei-Jer; Chen, Ding-Shinn; Wang, Hurng-Yi</td>
      <td>2017</td>
      <td>2017-04-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5406902" target="_blank">PMC5406902</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28446139.0" target="_blank">28446139.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-017-1630-z" target="_blank">10.1186/s12859-017-1630-z</a></td>
      <td>rbwgi0ql</td>
      <td>0.514515</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a></td>
      <td><a href="Topic_34.html#cg463fbn">Wang_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#xyru2q9o">Shepard_2016</a>, <a href="Topic_34.html#f5u2o0rq">Bassano_2017</a></td>
    </tr>
    <tr>
      <th id="5nc8f3a4";>191</th>
      <td>Pathegama_2005</td>
      <td>Interactive Real-time Image Analysis System for Distant Operation</td>
      <td>Pathegama, Mahinda P.; Göl, Özdemir</td>
      <td>2005</td>
      <td>2005-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121714" target="_blank">PMC7121714</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/0-387-26325-x_3" target="_blank">10.1007/0-387-26325-x_3</a></td>
      <td>5nc8f3a4</td>
      <td>0.513134</td>
      <td><a href="Topic_01.html#lzdjfvh7">May_E_2016</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#43igjn6x">Kylberg_2009</a>, <a href="Topic_60.html#e61vrp7h">Graham_2007</a>, <a href="Topic_60.html#bssazy93">Liu_Q_2005</a>, <a href="Topic_17.html#3u81ntle">Acar_2016</a></td>
    </tr>
    <tr>
      <th id="yts4xppb";>192</th>
      <td>Dórea_2013</td>
      <td>Exploratory Analysis of Methods for Automated Classification of Laboratory Test Orders into Syndromic Groups in Veterinary Medicine</td>
      <td>Dórea, Fernanda C.; Muckle, C. Anne; Kelton, David; McClure, JT.; McEwen, Beverly J.; McNab, W. Bruce; Sanchez, Javier; Revie, Crawford W.</td>
      <td>2013</td>
      <td>2013-03-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3591392" target="_blank">PMC3591392</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23505427.0" target="_blank">23505427.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0057334" target="_blank">10.1371/journal.pone.0057334</a></td>
      <td>yts4xppb</td>
      <td>0.512811</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_14.html#kjovtgua">Wu_J_2020</a></td>
    </tr>
    <tr>
      <th id="71tnv4lp";>193</th>
      <td>Santillana_2014</td>
      <td>Using Clinicians’ Search Query Data to Monitor Influenza Epidemics</td>
      <td>Santillana, Mauricio; Nsoesie, Elaine O.; Mekaru, Sumiko R.; Scales, David; Brownstein, John S.</td>
      <td>2014</td>
      <td>2014-08-12</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296132" target="_blank">PMC4296132</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25115873.0" target="_blank">25115873.0</a></td>
      <td><a href="https://doi.org/10.1093/cid/ciu647" target="_blank">10.1093/cid/ciu647</a></td>
      <td>71tnv4lp</td>
      <td>0.512775</td>
      <td><a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a></td>
      <td><a href="Topic_06.html#wgrsaf0k">Sebastiani_2006</a>, <a href="Topic_04.html#rpwccvqi">Rojek_2020</a>, <a href="Topic_22.html#wrhr237o">Meerhoff_2006</a>, <a href="Topic_17.html#1lgi1e60">Freifeld_2008</a></td>
    </tr>
    <tr>
      <th id="9qv11m4f";>194</th>
      <td>Heo_L_2020</td>
      <td>Modeling of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) Proteins by Machine Learning and Physics-Based Refinement</td>
      <td>Lim Heo; Michael Feig</td>
      <td>2020</td>
      <td>2020-03-28</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.25.008904" target="_blank">10.1101/2020.03.25.008904</a></td>
      <td>9qv11m4f</td>
      <td>0.512181</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_27.html#zrbesm5b">Rockx_2020</a>, <a href="Topic_45.html#h5ire02h">Tan_Y_2020</a>, <a href="Topic_41.html#dqxfcwyu">Park_2020</a></td>
    </tr>
    <tr>
      <th id="2se42vec";>195</th>
      <td>Sareen_2016</td>
      <td>IoT-based cloud framework to control Ebola virus outbreak</td>
      <td>Sareen, Sanjay; Sood, Sandeep K.; Gupta, Sunil Kumar</td>
      <td>2016</td>
      <td>2016-10-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091278" target="_blank">PMC7091278</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s12652-016-0427-7" target="_blank">10.1007/s12652-016-0427-7</a></td>
      <td>2se42vec</td>
      <td>0.511798</td>
      <td><a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#3l2dbnjb">Sood_2017</a>, <a href="Topic_14.html#ityhcak7">Zhu_H_2020</a>, <a href="Topic_32.html#4rulshop">Huang_2013</a>, <a href="Topic_04.html#ckpq0pvo">Lwin_2017</a></td>
    </tr>
    <tr>
      <th id="i5jk0407";>196</th>
      <td>Abbas_2020</td>
      <td>Classification of COVID-19 in chest X-ray images using DeTraC deep convolutional neural network</td>
      <td>Asmaa Abbas; Mohammed Abdelsamea; Mohamed Gaber</td>
      <td>2020</td>
      <td>2020-04-01</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.30.20047456" target="_blank">10.1101/2020.03.30.20047456</a></td>
      <td>i5jk0407</td>
      <td>0.511615</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_17.html#k1lg8c7q">Jin_C_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a>, <a href="Topic_17.html#ll4rxd9p">Zheng_2020</a></td>
    </tr>
    <tr>
      <th id="eabho73n";>197</th>
      <td>Shahid_2009</td>
      <td>Comparison of distance measures in spatial analytical modeling for health service planning</td>
      <td>Shahid, Rizwan; Bertazzon, Stefania; Knudtson, Merril L; Ghali, William A</td>
      <td>2009</td>
      <td>2009-11-06</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2781002" target="_blank">PMC2781002</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19895692.0" target="_blank">19895692.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6963-9-200" target="_blank">10.1186/1472-6963-9-200</a></td>
      <td>eabho73n</td>
      <td>0.511203</td>
      <td><a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#8k22red9">So_M_2008</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_39.html#01tncjq0">STRESMAN_2014</a>, <a href="Topic_22.html#rz9cwhm3">Hellmich_2017</a>, <a href="Topic_39.html#5e1xn7kj">Falcón-Lezama_2017</a></td>
    </tr>
    <tr>
      <th id="jg13scgo";>198</th>
      <td>Tsui_2003</td>
      <td>Technical description of RODS: a real-time public health surveillance system</td>
      <td>Tsui, Fu-Chiang; Espino, Jeremy U.; Dato, Virginia M.; Gesteland, Per H.; Hutman, Judith; Wagner, Michael M.</td>
      <td>2003</td>
      <td>2003-10-31</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC212776" target="_blank">PMC212776</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12807803.0" target="_blank">12807803.0</a></td>
      <td><a href="https://doi.org/10.1197/jamia.m1345" target="_blank">10.1197/jamia.m1345</a></td>
      <td>jg13scgo</td>
      <td>0.508185</td>
      <td><a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a></td>
      <td><a href="Topic_04.html#tfp2idq2">Hale_2019</a>, <a href="Topic_04.html#21hahh03">Fan_S_2010</a>, <a href="Topic_04.html#1fu1blu0">Lazarus_2006</a></td>
    </tr>
    <tr>
      <th id="ermuiujq";>199</th>
      <td>Bottomly_2015</td>
      <td>plethy: management of whole body plethysmography data in R</td>
      <td>Bottomly, Daniel; Wilmot, Beth; McWeeney, Shannon K</td>
      <td>2015</td>
      <td>2015-04-29</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4434826" target="_blank">PMC4434826</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25924931.0" target="_blank">25924931.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-015-0547-7" target="_blank">10.1186/s12859-015-0547-7</a></td>
      <td>ermuiujq</td>
      <td>0.506485</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a></td>
      <td><a href="Topic_14.html#s578t1fk">van_Oort_2017</a>, <a href="Topic_14.html#5agqknvu">Fan_R_2008</a></td>
    </tr>
    <tr>
      <th id="cxzlmfst";>200</th>
      <td>Malanoski_2006</td>
      <td>Automated identification of multiple micro-organisms from resequencing DNA microarrays</td>
      <td>Malanoski, Anthony P.; Lin, Baochuan; Wang, Zheng; Schnur, Joel M.; Stenger, David A.</td>
      <td>2006</td>
      <td>2006-09-29</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1636417" target="_blank">PMC1636417</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17012284.0" target="_blank">17012284.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl565" target="_blank">10.1093/nar/gkl565</a></td>
      <td>cxzlmfst</td>
      <td>0.504938</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_14.html#xo1cyzrg">Giles_2015</a>, <a href="Topic_14.html#l4b5pezm">Lin_B_2006</a>, <a href="Topic_14.html#g0yknytb">Wang_2017</a></td>
    </tr>
    <tr>
      <th id="4mad9xws";>201</th>
      <td>Ming_2014</td>
      <td>Multistep-Ahead Air Passengers Traffic Prediction with Hybrid ARIMA-SVMs Models</td>
      <td>Ming, Wei; Bao, Yukun; Hu, Zhongyi; Xiong, Tao</td>
      <td>2014</td>
      <td>2014-02-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3958729" target="_blank">PMC3958729</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24723814.0" target="_blank">24723814.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/567246" target="_blank">10.1155/2014/567246</a></td>
      <td>4mad9xws</td>
      <td>0.504576</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_71.html#vioymdp0">Xie_G_2020</a>, <a href="Topic_71.html#94qom8px">Yang_2018</a></td>
    </tr>
    <tr>
      <th id="x70501t3";>202</th>
      <td>Pourhomayoun_2020</td>
      <td>Predicting Mortality Risk in Patients with COVID-19 Using Artificial Intelligence to Help Medical Decision-Making</td>
      <td>Mohammad Pourhomayoun; Mahdi Shakibi</td>
      <td>2020</td>
      <td>2020-04-01</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.30.20047308" target="_blank">10.1101/2020.03.30.20047308</a></td>
      <td>x70501t3</td>
      <td>0.503867</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a></td>
      <td><a href="Topic_17.html#nvavj9gk">Andre_2020</a>, <a href="Topic_12.html#s9j21zsy">Yan_L_2020</a>, <a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_19.html#0agldesf">Sarkar_2020</a></td>
    </tr>
    <tr>
      <th id="t40ybhgb";>203</th>
      <td>Gardner_2005</td>
      <td>Draft versus finished sequence data for DNA and protein diagnostic signature development</td>
      <td>Gardner, Shea N.; Lam, Marisa W.; Smith, Jason R.; Torres, Clinton L.; Slezak, Tom R.</td>
      <td>2005</td>
      <td>2005-10-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1266063" target="_blank">PMC1266063</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16243783.0" target="_blank">16243783.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gki896" target="_blank">10.1093/nar/gki896</a></td>
      <td>t40ybhgb</td>
      <td>0.503641</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_10.html#sd81bz0m">Gardner_2004</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_21.html#fcvddgvy">Viehweger_2019</a></td>
    </tr>
    <tr>
      <th id="78vjzrpx";>204</th>
      <td>Song_2018</td>
      <td>iProt-Sub: a comprehensive package for accurately mapping and predicting protease-specific substrates and cleavage sites</td>
      <td>Song, Jiangning; Wang, Yanan; Li, Fuyi; Akutsu, Tatsuya; Rawlings, Neil D; Webb, Geoffrey I; Chou, Kuo-Chen</td>
      <td>2018</td>
      <td>2018-04-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6556904" target="_blank">PMC6556904</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29897410.0" target="_blank">29897410.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bby028" target="_blank">10.1093/bib/bby028</a></td>
      <td>78vjzrpx</td>
      <td>0.503394</td>
      <td><a href="Topic_17.html#my2l85bn">Cho_Y_2005</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#hiy7f4wh">Someya_2008</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="csokkcqq";>205</th>
      <td>Shcherbina_2016</td>
      <td>Evaluating performance of metagenomic characterization algorithms using in silico datasets generated with FASTQSim</td>
      <td>Anna Shcherbina; Darrell O. Ricke; Nelson Chiu</td>
      <td>2016</td>
      <td>2016-03-31</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/046532" target="_blank">10.1101/046532</a></td>
      <td>csokkcqq</td>
      <td>0.499995</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_14.html#xlqdn0c7">Baldwin_2014</a>, <a href="Topic_50.html#qg2r691d">Twin_2013</a>, <a href="Topic_70.html#0gi6qzw0">Takhampunya_2019</a>, <a href="Topic_14.html#28nlawnb">Zinter_2018</a></td>
    </tr>
    <tr>
      <th id="6frnbfh4";>206</th>
      <td>Chen_2019</td>
      <td>TAR-VIR: a pipeline for TARgeted VIRal strain reconstruction from metagenomic data</td>
      <td>Chen, Jiao; Huang, Jiating; Sun, Yanni</td>
      <td>2019</td>
      <td>2019-06-04</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6549370" target="_blank">PMC6549370</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31164077.0" target="_blank">31164077.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-2878-2" target="_blank">10.1186/s12859-019-2878-2</a></td>
      <td>6frnbfh4</td>
      <td>0.499994</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_45.html#0l33i6s4">Anderson_2020</a></td>
    </tr>
    <tr>
      <th id="feq9grdc";>207</th>
      <td>Isa_Irawan_2019</td>
      <td>Application of Needleman-Wunch Algorithm to identify mutation in DNA sequences of Corona virus</td>
      <td>Isa Irawan, Mohammad; Mukhlash, Imam; Rizky, Abduh; Ririsati Dewi, Alfiana</td>
      <td>2019</td>
      <td>2019-05-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106772" target="_blank">PMC7106772</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1088/1742-6596/1218/1/012031" target="_blank">10.1088/1742-6596/1218/1/012031</a></td>
      <td>feq9grdc</td>
      <td>0.499991</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_37.html#kihn83ny">Armesto_2007</a>, <a href="Topic_10.html#je6oe1h4">Auewarakul_2005</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="jb447ndz";>208</th>
      <td>ponti_2018</td>
      <td>A method for RNA structure prediction shows evidence for structure in lncRNAs</td>
      <td>Riccardo Delli ponti; Alexandros Armaos; Stefanie Marti; Gian Gaetano Tartaglia</td>
      <td>2018</td>
      <td>2018-04-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/284869" target="_blank">10.1101/284869</a></td>
      <td>jb447ndz</td>
      <td>0.499991</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#scn7b4c6">Delli_Ponti_2018</a>, <a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="abn1i3ir";>209</th>
      <td>Modak_2019</td>
      <td>Application of Support Vector Machines in Viral Biology</td>
      <td>Modak, Sonal; Mehta, Swati; Sehgal, Deepak; Valadi, Jayaraman</td>
      <td>2019</td>
      <td>2019-11-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114997" target="_blank">PMC7114997</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-29022-1_12" target="_blank">10.1007/978-3-030-29022-1_12</a></td>
      <td>abn1i3ir</td>
      <td>0.499991</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a></td>
      <td><a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_01.html#058o3nmn">Zheng_2011</a>, <a href="Topic_29.html#zduf4f73">Zhong_2007</a></td>
    </tr>
    <tr>
      <th id="scn7b4c6";>210</th>
      <td>Delli_Ponti_2018</td>
      <td>A Method for RNA Structure Prediction Shows Evidence for Structure in lncRNAs</td>
      <td>Delli Ponti, Riccardo; Armaos, Alexandros; Marti, Stefanie; Tartaglia, Gian Gaetano</td>
      <td>2018</td>
      <td>2018-12-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6286970" target="_blank">PMC6286970</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30560136.0" target="_blank">30560136.0</a></td>
      <td><a href="https://doi.org/10.3389/fmolb.2018.00111" target="_blank">10.3389/fmolb.2018.00111</a></td>
      <td>scn7b4c6</td>
      <td>0.499991</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#jb447ndz">ponti_2018</a>, <a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a></td>
    </tr>
    <tr>
      <th id="fn929z9y";>211</th>
      <td>Carrillo-de-Gea_2016</td>
      <td>A Computer-Aided Detection System for Digital Chest Radiographs</td>
      <td>Carrillo-de-Gea, Juan Manuel; García-Mateos, Ginés; Fernández-Alemán, José Luis; Hernández-Hernández, José Luis</td>
      <td>2016</td>
      <td>2016-05-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5058572" target="_blank">PMC5058572</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27372536.0" target="_blank">27372536.0</a></td>
      <td><a href="https://doi.org/10.1155/2016/8208923" target="_blank">10.1155/2016/8208923</a></td>
      <td>fn929z9y</td>
      <td>0.499989</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_65.html#yzqcet2l">Javadi_2006</a>, <a href="Topic_17.html#45dpoepu">Hassanien_2020</a>, <a href="Topic_11.html#0zjdsaji">Shuyue_2006</a>, <a href="Topic_67.html#n881myx0">Upchurch_2018</a></td>
    </tr>
    <tr>
      <th id="nwl5i9h6";>212</th>
      <td>Hjortland_2016</td>
      <td>Steady-state kinetic studies reveal that the anti-cancer target Ubiquitin-Specific Protease 17 (USP17) is a highly efficient deubiquitinating enzyme</td>
      <td>Hjortland, Nicole M.; Mesecar, Andrew D.</td>
      <td>2016</td>
      <td>2016-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5868344" target="_blank">PMC5868344</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27756680.0" target="_blank">27756680.0</a></td>
      <td><a href="https://doi.org/10.1016/j.abb.2016.10.008" target="_blank">10.1016/j.abb.2016.10.008</a></td>
      <td>nwl5i9h6</td>
      <td>0.499786</td>
      <td><a href="Topic_06.html#5sv3dwbs">Banik_2016</a>, <a href="Topic_17.html#a2e3f161">Alqahtani_2016</a>, <a href="Topic_60.html#9dj07sac">Liu_L_2016</a>, <a href="Topic_60.html#q1ggl78u">Gogineni_2015</a></td>
      <td><a href="Topic_01.html#oukcmch5">UNKNOWN_2015</a></td>
    </tr>
    <tr>
      <th id="emeb5ge7";>213</th>
      <td>Chang_2020</td>
      <td>The 10th Anniversary of Asia Pacific Allergy</td>
      <td>Chang, Yoon-Seok</td>
      <td>2020</td>
      <td>2020-01-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7016318" target="_blank">PMC7016318</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32099832.0" target="_blank">32099832.0</a></td>
      <td><a href="https://doi.org/10.5415/apallergy.2020.10.e10" target="_blank">10.5415/apallergy.2020.10.e10</a></td>
      <td>emeb5ge7</td>
      <td>0.499786</td>
      <td></td>
      <td><a href="Topic_56.html#d21rxlsy">Yu_W_2004</a>, <a href="Topic_05.html#t9yizht7">Marks_2016</a>, <a href="Topic_02.html#o9skj94r">Plouffe_2004</a></td>
    </tr>
    <tr>
      <th id="a2e3f161";>214</th>
      <td>Alqahtani_2016</td>
      <td>Camel exposure and knowledge about MERS-CoV among Australian Hajj pilgrims in 2014</td>
      <td>Alqahtani, Amani Salem; Wiley, Kerrie Elizabeth; Tashani, Mohamed; Heywood, Anita Elizabeth; Willaby, Harold Wayne; BinDhim, Nasser Fahad; Booy, Robert; Rashid, Harunor</td>
      <td>2016</td>
      <td>2016-01-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090863" target="_blank">PMC7090863</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26810660.0" target="_blank">26810660.0</a></td>
      <td><a href="https://doi.org/10.1007/s12250-015-3669-1" target="_blank">10.1007/s12250-015-3669-1</a></td>
      <td>a2e3f161</td>
      <td>0.499786</td>
      <td><a href="Topic_06.html#5sv3dwbs">Banik_2016</a>, <a href="Topic_60.html#9dj07sac">Liu_L_2016</a>, <a href="Topic_17.html#nwl5i9h6">Hjortland_2016</a>, <a href="Topic_60.html#q1ggl78u">Gogineni_2015</a></td>
      <td><a href="Topic_65.html#code9hzz">Ebrahim_2020</a>, <a href="Topic_16.html#f812ewhb">Memish_2014</a>, <a href="Topic_32.html#1xnhuinm">Vora_2014</a></td>
    </tr>
    <tr>
      <th id="7my2q928";>215</th>
      <td>Largo_2016</td>
      <td>Ion channel activity of the CSFV p7 viroporin in surrogates of the ER lipid bilayer</td>
      <td>Largo, Eneko; Verdiá-Báguena, Carmina; Aguilella, Vicente M.; Nieva, José L.; Alcaraz, Antonio</td>
      <td>2016</td>
      <td>2016-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094309" target="_blank">PMC7094309</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26464198.0" target="_blank">26464198.0</a></td>
      <td><a href="https://doi.org/10.1016/j.bbamem.2015.10.007" target="_blank">10.1016/j.bbamem.2015.10.007</a></td>
      <td>7my2q928</td>
      <td>0.499786</td>
      <td><a href="Topic_06.html#vs12l0uv">Choudhury_2011</a>, <a href="Topic_19.html#tgyxiwuo">Abou-Hamdan_2018</a>, <a href="Topic_60.html#xdihvg6k">David_2013</a>, <a href="Topic_17.html#a2e3f161">Alqahtani_2016</a></td>
      <td><a href="Topic_54.html#8sb5xqxc">Jiang_2013</a></td>
    </tr>
    <tr>
      <th id="2n5vetzw";>216</th>
      <td>Li_K_2010</td>
      <td>ANDES: Statistical tools for the ANalyses of DEep Sequencing</td>
      <td>Li, Kelvin; Venter, Eli; Yooseph, Shibu; Stockwell, Timothy B; Eckerle, Lance D; Denison, Mark R; Spiro, David J; Methé, Barbara A</td>
      <td>2010</td>
      <td>2010-07-15</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2921379" target="_blank">PMC2921379</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20633290.0" target="_blank">20633290.0</a></td>
      <td><a href="https://doi.org/10.1186/1756-0500-3-199" target="_blank">10.1186/1756-0500-3-199</a></td>
      <td>2n5vetzw</td>
      <td>0.497234</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_50.html#qg2r691d">Twin_2013</a>, <a href="Topic_14.html#3jxyipbn">Lewandowska_2017</a>, <a href="Topic_14.html#xlqdn0c7">Baldwin_2014</a></td>
    </tr>
    <tr>
      <th id="9gozwdng";>217</th>
      <td>Barbarini_2011</td>
      <td>Prediction of Peptide Reactivity with Human IVIg through a Knowledge-Based Approach</td>
      <td>Barbarini, Nicola; Tiengo, Alessandra; Bellazzi, Riccardo</td>
      <td>2011</td>
      <td>2011-08-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3160895" target="_blank">PMC3160895</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21887285.0" target="_blank">21887285.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0023616" target="_blank">10.1371/journal.pone.0023616</a></td>
      <td>9gozwdng</td>
      <td>0.496398</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#ml2luc1t">Sollner_2008</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_69.html#ftf67ykd">Irving_2010</a>, <a href="Topic_69.html#60xkxdhe">Hua_R_2013</a></td>
    </tr>
    <tr>
      <th id="7rs4jtfh";>218</th>
      <td>Mayrose_2007</td>
      <td>Pepitope: epitope mapping from affinity-selected peptides</td>
      <td>Mayrose, Itay; Penn, Osnat; Erez, Elana; Rubinstein, Nimrod D.; Shlomi, Tomer; Freund, Natalia Tarnovitski; Bublil, Erez M.; Ruppin, Eytan; Sharan, Roded; Gershoni, Jonathan M.; Martz, Eric; Pupko, Tal</td>
      <td>2007</td>
      <td>2007-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110138" target="_blank">PMC7110138</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17977889.0" target="_blank">17977889.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btm493" target="_blank">10.1093/bioinformatics/btm493</a></td>
      <td>7rs4jtfh</td>
      <td>0.494618</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_69.html#ftf67ykd">Irving_2010</a>, <a href="Topic_23.html#9ne5iwny">Li_S_2016</a>, <a href="Topic_23.html#dv6oeldo">Niesters_1987</a>, <a href="Topic_69.html#6moxjy9j">Yu_M_2000</a></td>
    </tr>
    <tr>
      <th id="0dxrai3j";>219</th>
      <td>Alguwaizani_2018</td>
      <td>Predicting Interactions between Virus and Host Proteins Using Repeat Patterns and Composition of Amino Acids</td>
      <td>Alguwaizani, Saud; Park, Byungkyu; Zhou, Xiang; Huang, De-Shuang; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-05-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5966669" target="_blank">PMC5966669</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29854357.0" target="_blank">29854357.0</a></td>
      <td><a href="https://doi.org/10.1155/2018/1391265" target="_blank">10.1155/2018/1391265</a></td>
      <td>0dxrai3j</td>
      <td>0.494377</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#kixzxs68">Naito_2007</a></td>
      <td><a href="Topic_17.html#4uvezfqm">Zhou_2018</a>, <a href="Topic_10.html#7rnf2iyw">Goodacre_2020</a>, <a href="Topic_10.html#629kl04a">Bahir_2009</a></td>
    </tr>
    <tr>
      <th id="us6ay892";>220</th>
      <td>Perez-Bello_2009</td>
      <td>Alignment-free prediction of mycobacterial DNA promoters based on pseudo-folding lattice network or star-graph topological indices</td>
      <td>Perez-Bello, Alcides; Munteanu, Cristian Robert; Ubeira, Florencio M.; Lopes De Magalhães, Alexandre; Uriarte, Eugenio; González-Díaz, Humberto</td>
      <td>2009</td>
      <td>2009-02-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126577" target="_blank">PMC7126577</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18992259.0" target="_blank">18992259.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2008.09.035" target="_blank">10.1016/j.jtbi.2008.09.035</a></td>
      <td>us6ay892</td>
      <td>0.494112</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_69.html#9pkvj14m">Singleton_2018</a>, <a href="Topic_10.html#wrztpeb8">Zhang_2015</a></td>
    </tr>
    <tr>
      <th id="jz2mrn4t";>221</th>
      <td>Vila_2014</td>
      <td>(13)C Chemical Shifts in Proteins: A Rich Source of Encoded Structural Information</td>
      <td>Vila, Jorge A.; Arnautova, Yelena A.</td>
      <td>2014</td>
      <td>2014-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121069" target="_blank">PMC7121069</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-642-28554-7_19" target="_blank">10.1007/978-3-642-28554-7_19</a></td>
      <td>jz2mrn4t</td>
      <td>0.492955</td>
      <td><a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_17.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_16.html#gytg4iku">Vila_2018</a>, <a href="Topic_16.html#3398ymw9">Lee_G_2012</a>, <a href="Topic_16.html#g0h0itzs">Buller_2012</a></td>
    </tr>
    <tr>
      <th id="g60hnqlf";>222</th>
      <td>Wang_2010</td>
      <td>Method of regulatory network that can explore protein regulations for disease classification</td>
      <td>Wang, Hong Qiang; Zhu, Hai Long; Cho, William C.S.; Yip, Timothy T.C.; Ngan, Roger K.C.; Law, Stephen C.K.</td>
      <td>2010</td>
      <td>2010-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126395" target="_blank">PMC7126395</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19962281.0" target="_blank">19962281.0</a></td>
      <td><a href="https://doi.org/10.1016/j.artmed.2009.07.011" target="_blank">10.1016/j.artmed.2009.07.011</a></td>
      <td>g60hnqlf</td>
      <td>0.490272</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a></td>
      <td><a href="Topic_34.html#zlqh436d">Chasman_2016</a>, <a href="Topic_34.html#z12o62r9">Parolo_2018</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_34.html#j80hnhpb">Noh_H_2018</a></td>
    </tr>
    <tr>
      <th id="3atcqejc";>223</th>
      <td>Kinene_2016</td>
      <td>Rooting Trees, Methods for</td>
      <td>Kinene, T.; Wainaina, J.; Maina, S.; Boykin, L.M.</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-800049-6.00215-8" target="_blank">10.1016/b978-0-12-800049-6.00215-8</a></td>
      <td>3atcqejc</td>
      <td>0.490193</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_01.html#isbc1r2o">Munjal_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#le7v89mp">Chang_2011</a></td>
      <td><a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#2ai20pd1">HESS_2007</a>, <a href="Topic_17.html#ne9d0siv">Banks_2006</a></td>
    </tr>
    <tr>
      <th id="2basllfv";>224</th>
      <td>Chen_2019</td>
      <td>A binning tool to reconstruct viral haplotypes from assembled contigs</td>
      <td>Jiao Chen; Jiayu Shang; Jianrong Wang; Yanni Sun</td>
      <td>2019</td>
      <td>2019-07-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/704288" target="_blank">10.1101/704288</a></td>
      <td>2basllfv</td>
      <td>0.490065</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a></td>
      <td><a href="Topic_17.html#6tss9ydx">Chen_2019</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a></td>
    </tr>
    <tr>
      <th id="uwbmox8a";>225</th>
      <td>Robson_2016</td>
      <td>Data-mining to build a knowledge representation store for clinical decision support. Studies on curation and validation based on machine performance in multiple choice medical licensing examinations</td>
      <td>Robson, Barry; Boray, Srinidhi</td>
      <td>2016</td>
      <td>2016-06-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094475" target="_blank">PMC7094475</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27089305.0" target="_blank">27089305.0</a></td>
      <td><a href="https://doi.org/10.1016/j.compbiomed.2016.02.010" target="_blank">10.1016/j.compbiomed.2016.02.010</a></td>
      <td>uwbmox8a</td>
      <td>0.487180</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a></td>
      <td><a href="Topic_17.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_17.html#abrweljg">Yu_A_2011</a>, <a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_17.html#98oo0bkl">Nakai_1992</a></td>
    </tr>
    <tr>
      <th id="88l9bq4i";>226</th>
      <td>Han_L_2005</td>
      <td>Prediction of functional class of novel viral proteins by a statistical learning method irrespective of sequence similarity</td>
      <td>Han, L.Y.; Cai, C.Z.; Ji, Z.L.; Chen, Y.Z.</td>
      <td>2005</td>
      <td>2005-01-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111859" target="_blank">PMC7111859</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15582660.0" target="_blank">15582660.0</a></td>
      <td><a href="https://doi.org/10.1016/j.virol.2004.10.020" target="_blank">10.1016/j.virol.2004.10.020</a></td>
      <td>88l9bq4i</td>
      <td>0.486667</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_16.html#r7ofnxf1">Rozanov_1995</a></td>
    </tr>
    <tr>
      <th id="vtktgyvh";>227</th>
      <td>Seo_J_2019</td>
      <td>An Exploration of Machine Learning Methods for Robust Boredom Classification Using EEG and GSR Data</td>
      <td>Seo, Jungryul; Laine, Teemu H.; Sohn, Kyung-Ah</td>
      <td>2019</td>
      <td>2019-10-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6832442" target="_blank">PMC6832442</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31635194.0" target="_blank">31635194.0</a></td>
      <td><a href="https://doi.org/10.3390/s19204561" target="_blank">10.3390/s19204561</a></td>
      <td>vtktgyvh</td>
      <td>0.486302</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_47.html#b3wiay6w">Pietrowsky_1996</a>, <a href="Topic_11.html#64dr62ls">Drake_2000</a>, <a href="Topic_14.html#1r4gm2d4">Liu_Y_2020</a>, <a href="Topic_17.html#p7ga4h70">Jung_2017</a></td>
    </tr>
    <tr>
      <th id="bhtl73z9";>228</th>
      <td>Suresh_2015</td>
      <td>RPI-Pred: predicting ncRNA-protein interaction using sequence and structural information</td>
      <td>Suresh, V.; Liu, Liang; Adjeroh, Donald; Zhou, Xiaobo</td>
      <td>2015</td>
      <td>2015-02-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4330382" target="_blank">PMC4330382</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25609700.0" target="_blank">25609700.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkv020" target="_blank">10.1093/nar/gkv020</a></td>
      <td>bhtl73z9</td>
      <td>0.485983</td>
      <td><a href="Topic_34.html#2f9nc2to">Bansal_2020</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#hsh2yynq">Kinjo_2012</a>, <a href="Topic_61.html#a5htfx76">Dost_2006</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_17.html#6oca1mrm">Shen_2018</a>, <a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a></td>
    </tr>
    <tr>
      <th id="j5uufd63";>229</th>
      <td>Sandhu_2015</td>
      <td>An intelligent system for predicting and preventing MERS-CoV infection outbreak</td>
      <td>Sandhu, Rajinder; Sood, Sandeep K.; Kaur, Gurpreet</td>
      <td>2015</td>
      <td>2015-07-08</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089482" target="_blank">PMC7089482</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11227-015-1474-0" target="_blank">10.1007/s11227-015-1474-0</a></td>
      <td>j5uufd63</td>
      <td>0.485716</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_06.html#sk2n2gxw">Eifan_2017</a>, <a href="Topic_06.html#tu06vd27">Sardar_2020</a>, <a href="Topic_17.html#wqa98m4v">Al-Turaiki_2016</a></td>
    </tr>
    <tr>
      <th id="hkvll3xh";>230</th>
      <td>Yang_2009</td>
      <td>Peptide Bioinformatics- Peptide Classification Using Peptide Machines</td>
      <td>Yang, Zheng Rong</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122642" target="_blank">PMC7122642</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19065810.0" target="_blank">19065810.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-60327-101-1_9" target="_blank">10.1007/978-1-60327-101-1_9</a></td>
      <td>hkvll3xh</td>
      <td>0.485297</td>
      <td><a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_17.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="9t3kneoo";>231</th>
      <td>Abd_Elwahaab_2019</td>
      <td>A Statistical Similarity/Dissimilarity Analysis of Protein Sequences Based on a Novel Group Representative Vector</td>
      <td>Abd Elwahaab, Marwa A.; Abo-Elkhier, Mervat M.; Abo el Maaty, Moheb I.</td>
      <td>2019</td>
      <td>2019-05-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6530227" target="_blank">PMC6530227</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31205946.0" target="_blank">31205946.0</a></td>
      <td><a href="https://doi.org/10.1155/2019/8702968" target="_blank">10.1155/2019/8702968</a></td>
      <td>9t3kneoo</td>
      <td>0.484213</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a></td>
      <td><a href="Topic_17.html#88l9bq4i">Han_L_2005</a>, <a href="Topic_17.html#61djk4bs">Wei_D_2012</a>, <a href="Topic_17.html#41atidai">Abo-Elkhier_2019</a>, <a href="Topic_17.html#f6lfykot">Wu_C_2018</a></td>
    </tr>
    <tr>
      <th id="47kmwl6s";>232</th>
      <td>Lin_H_2017</td>
      <td>drVM: a new tool for efficient genome assembly of known eukaryotic viruses from metagenomes</td>
      <td>Lin, Hsin-Hung; Liao, Yu-Chieh</td>
      <td>2017</td>
      <td>2017-01-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466706" target="_blank">PMC5466706</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28369462.0" target="_blank">28369462.0</a></td>
      <td><a href="https://doi.org/10.1093/gigascience/gix003" target="_blank">10.1093/gigascience/gix003</a></td>
      <td>47kmwl6s</td>
      <td>0.483873</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_10.html#jw4n2j6y">Smits_2014</a></td>
    </tr>
    <tr>
      <th id="ojy6s49w";>233</th>
      <td>Warren_2007</td>
      <td>Assembling millions of short DNA sequences using SSAKE</td>
      <td>Warren, René L.; Sutton, Granger G.; Jones, Steven J. M.; Holt, Robert A.</td>
      <td>2007</td>
      <td>2007-02-15</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109930" target="_blank">PMC7109930</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17158514.0" target="_blank">17158514.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btl629" target="_blank">10.1093/bioinformatics/btl629</a></td>
      <td>ojy6s49w</td>
      <td>0.483150</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_21.html#fcvddgvy">Viehweger_2019</a>, <a href="Topic_21.html#xr2bs4bk">Viehweger_2018</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a></td>
    </tr>
    <tr>
      <th id="tnoybbfx";>234</th>
      <td>Wagner_2020</td>
      <td>Real-time biomedical knowledge synthesis of the exponentially growing world wide web using unsupervised neural networks</td>
      <td>Tyler Wagner; Samir Awasthi; Gayle Wittenberg; AJ Venkatakrishnan; Dan Tarjan; Anuli Anyanwu-Ofili; Andrew Badley; John Halamka; Christopher Flores; Najat Khan; Rakesh Barve; Venky Soundararajan</td>
      <td>2020</td>
      <td>2020-04-04</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.03.020602" target="_blank">10.1101/2020.04.03.020602</a></td>
      <td>tnoybbfx</td>
      <td>0.483150</td>
      <td><a href="Topic_04.html#lgcmamfb">Dentico_2005</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a></td>
      <td><a href="Topic_17.html#j7t9nebs">Venkatakrishnan_2020</a>, <a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_06.html#yzwsqlb2">Ray_B_2016</a></td>
    </tr>
    <tr>
      <th id="xyru2q9o";>235</th>
      <td>Shepard_2016</td>
      <td>Viral deep sequencing needs an adaptive approach: IRMA, the iterative refinement meta-assembler</td>
      <td>Shepard, Samuel S.; Meno, Sarah; Bahl, Justin; Wilson, Malania M.; Barnes, John; Neuhaus, Elizabeth</td>
      <td>2016</td>
      <td>2016-09-05</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011931" target="_blank">PMC5011931</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27595578.0" target="_blank">27595578.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-016-3030-6" target="_blank">10.1186/s12864-016-3030-6</a></td>
      <td>xyru2q9o</td>
      <td>0.482929</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_21.html#d9urjb9e">Masante_2015</a></td>
    </tr>
    <tr>
      <th id="wfn50oqq";>236</th>
      <td>Adoteye_2015</td>
      <td>Optimal design of non-equilibrium experiments for genetic network interrogation</td>
      <td>Adoteye, Kaska; Banks, H.T.; Flores, Kevin B.</td>
      <td>2015</td>
      <td>2015-02-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4281269" target="_blank">PMC4281269</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25558126.0" target="_blank">25558126.0</a></td>
      <td><a href="https://doi.org/10.1016/j.aml.2014.09.013" target="_blank">10.1016/j.aml.2014.09.013</a></td>
      <td>wfn50oqq</td>
      <td>0.482765</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_10.html#shukyqmx">Dirmeier_2020</a>, <a href="Topic_02.html#uvcdt42w">Domingo_2020</a>, <a href="Topic_02.html#7fnjbewx">Domingo_2016</a>, <a href="Topic_08.html#x1ajgymw">Ferris_2014</a></td>
    </tr>
    <tr>
      <th id="k4eetalp";>237</th>
      <td>Cármenes_1989</td>
      <td>Predict7, a program for protein structure prediction</td>
      <td>Cármenes, R.S.; Freije, J.P.; Molina, M.M.; Martín, J.M.</td>
      <td>1989</td>
      <td>1989-03-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111062" target="_blank">PMC7111062</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/2539121.0" target="_blank">2539121.0</a></td>
      <td><a href="https://doi.org/10.1016/0006-291x(89)90049-1" target="_blank">10.1016/0006-291x(89)90049-1</a></td>
      <td>k4eetalp</td>
      <td>0.482765</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a></td>
    </tr>
    <tr>
      <th id="4uvezfqm";>238</th>
      <td>Zhou_2018</td>
      <td>A generalized approach to predicting protein-protein interactions between virus and host</td>
      <td>Zhou, Xiang; Park, Byungkyu; Choi, Daesik; Han, Kyungsook</td>
      <td>2018</td>
      <td>2018-08-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6101077" target="_blank">PMC6101077</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30367586.0" target="_blank">30367586.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-018-4924-2" target="_blank">10.1186/s12864-018-4924-2</a></td>
      <td>4uvezfqm</td>
      <td>0.482762</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a>, <a href="Topic_10.html#sblrsi6y">Trobaugh_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_10.html#7rnf2iyw">Goodacre_2020</a>, <a href="Topic_10.html#sx1fyqw0">Durmuş_2017</a>, <a href="Topic_26.html#cz42gvhy">Popescu_2011</a>, <a href="Topic_10.html#f59c4idd">Han_L_2017</a></td>
    </tr>
    <tr>
      <th id="89xnnvuv";>239</th>
      <td>Saad_2005</td>
      <td>Orientation determination by wavelets matching for 3D reconstruction of very noisy electron microscopic virus images</td>
      <td>Saad, Ali Samir</td>
      <td>2005</td>
      <td>2005-03-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC555555" target="_blank">PMC555555</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15743519.0" target="_blank">15743519.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6807-5-5" target="_blank">10.1186/1472-6807-5-5</a></td>
      <td>89xnnvuv</td>
      <td>0.482236</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#3u81ntle">Acar_2016</a>, <a href="Topic_20.html#set51q5h">Chichón_2012</a>, <a href="Topic_26.html#0zchxz00">Laine_2015</a>, <a href="Topic_10.html#evn5c4ve">Rochat_2012</a></td>
    </tr>
    <tr>
      <th id="0ec2xunt";>240</th>
      <td>Plewczyński_2008</td>
      <td>The interactome: Predicting the protein-protein interactions in cells</td>
      <td>Plewczyński, Dariusz; Ginalski, Krzysztof</td>
      <td>2008</td>
      <td>2008-10-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6275871" target="_blank">PMC6275871</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18839074.0" target="_blank">18839074.0</a></td>
      <td><a href="https://doi.org/10.2478/s11658-008-0024-7" target="_blank">10.2478/s11658-008-0024-7</a></td>
      <td>0ec2xunt</td>
      <td>0.481933</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_10.html#gxp4sqq0">Kumar_2013</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_01.html#fc24n7f8">He_R_2008</a></td>
    </tr>
    <tr>
      <th id="7kskldmn";>241</th>
      <td>Sloane_2020</td>
      <td>Chapter 83 Artificial intelligence in medical devices and clinical decision support systems</td>
      <td>Sloane, Elliot B.; J. Silva, Ricardo</td>
      <td>2020</td>
      <td>2020-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-813467-2.00084-5" target="_blank">10.1016/b978-0-12-813467-2.00084-5</a></td>
      <td>7kskldmn</td>
      <td>0.481498</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#6dcghdxf">Jiang_2009</a>, <a href="Topic_17.html#7my2q928">Largo_2016</a>, <a href="Topic_06.html#vs12l0uv">Choudhury_2011</a></td>
      <td><a href="Topic_17.html#x70501t3">Pourhomayoun_2020</a>, <a href="Topic_17.html#yts4xppb">Dórea_2013</a></td>
    </tr>
    <tr>
      <th id="3ulketgy";>242</th>
      <td>Snyder_2006</td>
      <td>PATRIC: The VBI PathoSystems Resource Integration Center</td>
      <td>Snyder, E. E.; Kampanya, N.; Lu, J.; Nordberg, E. K.; Karur, H. R.; Shukla, M.; Soneja, J.; Tian, Y.; Xue, T.; Yoo, H.; Zhang, F.; Dharmanolla, C.; Dongre, N. V.; Gillespie, J. J.; Hamelius, J.; Hance, M.; Huntington, K. I.; Jukneliene, D.; Koziski, J.; Mackasmiel, L.; Mane, S. P.; Nguyen, V.; Purkayastha, A.; Shallom, J.; Yu, G.; Guo, Y.; Gabbard, J.; Hix, D.; Azad, A. F.; Baker, S. C.; Boyle, S. M.; Khudyakov, Y.; Meng, X. J.; Rupprecht, C.; Vinje, J.; Crasta, O. R.; Czar, M. J.; Dickerman, A.; Eckart, J. D.; Kenyon, R.; Will, R.; Setubal, J. C.; Sobral, B. W. S.</td>
      <td>2006</td>
      <td>2006-11-16</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1669763" target="_blank">PMC1669763</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17142235.0" target="_blank">17142235.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl858" target="_blank">10.1093/nar/gkl858</a></td>
      <td>3ulketgy</td>
      <td>0.481485</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_17.html#0kkm0tgj">Zhang_2007</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_07.html#3b3k9ksy">Crawford_2019</a>, <a href="Topic_10.html#07nfb69o">Sun_D_2016</a></td>
    </tr>
    <tr>
      <th id="udzz15bk";>243</th>
      <td>Li_H_2009</td>
      <td>An effective docking strategy for virtual screening based on multi-objective optimization algorithm</td>
      <td>Li, Honglin; Zhang, Hailei; Zheng, Mingyue; Luo, Jie; Kang, Ling; Liu, Xiaofeng; Wang, Xicheng; Jiang, Hualiang</td>
      <td>2009</td>
      <td>2009-02-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753843" target="_blank">PMC2753843</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19210777.0" target="_blank">19210777.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-58" target="_blank">10.1186/1471-2105-10-58</a></td>
      <td>udzz15bk</td>
      <td>0.479734</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_18.html#gqnzwyn3">Mukherjee_2011</a>, <a href="Topic_18.html#8l1yg8vm">Kumar_2015</a>, <a href="Topic_18.html#8fwxa8um">Lee_H_2014</a>, <a href="Topic_17.html#7vwpvcdd">Liu_S_2012</a></td>
    </tr>
    <tr>
      <th id="9zfct1yx";>244</th>
      <td>Altschul_1991</td>
      <td>Amino acid substitution matrices from an information theoretic perspective</td>
      <td>Altschul, Stephen F.</td>
      <td>1991</td>
      <td>1991-06-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7130686" target="_blank">PMC7130686</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/2051488.0" target="_blank">2051488.0</a></td>
      <td><a href="https://doi.org/10.1016/0022-2836(91)90193-a" target="_blank">10.1016/0022-2836(91)90193-a</a></td>
      <td>9zfct1yx</td>
      <td>0.479598</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_34.html#3ttlrmbx">Neer_1987</a></td>
    </tr>
    <tr>
      <th id="ni6iyzdn";>245</th>
      <td>He_Z_2010</td>
      <td>Predicting Drug-Target Interaction Networks Based on Functional Groups and Biological Features</td>
      <td>He, Zhisong; Zhang, Jian; Shi, Xiao-He; Hu, Le-Le; Kong, Xiangyin; Cai, Yu-Dong; Chou, Kuo-Chen</td>
      <td>2010</td>
      <td>2010-03-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2836373" target="_blank">PMC2836373</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20300175.0" target="_blank">20300175.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0009603" target="_blank">10.1371/journal.pone.0009603</a></td>
      <td>ni6iyzdn</td>
      <td>0.478267</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#fewy8y4a">Wang_2018</a>, <a href="Topic_18.html#bs17igjs">Zhou_2010</a>, <a href="Topic_18.html#s50g754b">Chiba_2012</a></td>
    </tr>
    <tr>
      <th id="uo1d8xka";>246</th>
      <td>Singh_2008</td>
      <td>Comparative phyloinformatics of virus genes at micro and macro levels in a distributed computing environment</td>
      <td>Singh, Dadabhai T; Trehan, Rahul; Schmidt, Bertil; Bretschneider, Timo</td>
      <td>2008</td>
      <td>2008-02-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2259424" target="_blank">PMC2259424</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18315855.0" target="_blank">18315855.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-9-s1-s23" target="_blank">10.1186/1471-2105-9-s1-s23</a></td>
      <td>uo1d8xka</td>
      <td>0.477836</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_30.html#kvfffe0b">Gutiérrez_2013</a>, <a href="Topic_02.html#aj352do0">Tibayrenc_2015</a>, <a href="Topic_07.html#thsq6pnk">Lachheb_2019</a></td>
    </tr>
    <tr>
      <th id="c3ewuebx";>247</th>
      <td>Jia_J_2016</td>
      <td>iPPBS-Opt: A Sequence-Based Ensemble Classifier for Identifying Protein-Protein Binding Sites by Optimizing Imbalanced Training Datasets</td>
      <td>Jia, Jianhua; Liu, Zi; Xiao, Xuan; Liu, Bingxiang; Chou, Kuo-Chen</td>
      <td>2016</td>
      <td>2016-01-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6274413" target="_blank">PMC6274413</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26797600.0" target="_blank">26797600.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules21010095" target="_blank">10.3390/molecules21010095</a></td>
      <td>c3ewuebx</td>
      <td>0.477617</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_16.html#40pe21lm">Zhang_2018</a>, <a href="Topic_69.html#n184795x">Zamanzadeh_2017</a></td>
    </tr>
    <tr>
      <th id="jxgd1yc4";>248</th>
      <td>Kai_Z_2018</td>
      <td>A Novel Efficient Simulated Annealing Algorithm for the RNA Secondary Structure Predicting with Pseudoknots</td>
      <td>Kai, Zhang; Yulin, Lv</td>
      <td>2018</td>
      <td>2018-07-06</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121944" target="_blank">PMC7121944</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-95933-7_44" target="_blank">10.1007/978-3-319-95933-7_44</a></td>
      <td>jxgd1yc4</td>
      <td>0.476200</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a>, <a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="2dbpm0nt";>249</th>
      <td>Choo_2009</td>
      <td>A comprehensive assessment of N-terminal signal peptides prediction methods</td>
      <td>Choo, Khar Heng; Tan, Tin Wee; Ranganathan, Shoba</td>
      <td>2009</td>
      <td>2009-12-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2788353" target="_blank">PMC2788353</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19958512.0" target="_blank">19958512.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-s15-s2" target="_blank">10.1186/1471-2105-10-s15-s2</a></td>
      <td>2dbpm0nt</td>
      <td>0.475531</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_26.html#jljpb7rv">Voß_D_2009</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="afx7hacm";>250</th>
      <td>Xie_H_2014</td>
      <td>3D QSAR Studies, Pharmacophore Modeling and Virtual Screening on a Series of Steroidal Aromatase Inhibitors</td>
      <td>Xie, Huiding; Qiu, Kaixiong; Xie, Xiaoguang</td>
      <td>2014</td>
      <td>2014-11-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4264204" target="_blank">PMC4264204</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25405729.0" target="_blank">25405729.0</a></td>
      <td><a href="https://doi.org/10.3390/ijms151120927" target="_blank">10.3390/ijms151120927</a></td>
      <td>afx7hacm</td>
      <td>0.475008</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#kiwi87c9">Zhou_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_16.html#olfpcndb">Thanigaimalai_2013</a>, <a href="Topic_49.html#376hfuee">de_Oliveira_2013</a></td>
    </tr>
    <tr>
      <th id="7ch054mt";>251</th>
      <td>Lin_C_2009</td>
      <td>Developing a hierarchy relation with an expert decision analysis process for selecting the optimal resort type for a Taiwanese international resort park</td>
      <td>Lin, Chin-Tsai; Juan, Pin-Ju</td>
      <td>2009</td>
      <td>2009-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125690" target="_blank">PMC7125690</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2007.11.044" target="_blank">10.1016/j.eswa.2007.11.044</a></td>
      <td>7ch054mt</td>
      <td>0.473711</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_71.html#g247rjh9">Guirao_2016</a>, <a href="Topic_71.html#528telf6">Huang_2012</a>, <a href="Topic_04.html#rxvnn7p5">Ugwu_2006</a>, <a href="Topic_04.html#jhbx4sdm">Paraskevas_2013</a></td>
    </tr>
    <tr>
      <th id="wtvfow2f";>252</th>
      <td>Seemann_2008</td>
      <td>Unifying evolutionary and thermodynamic information for RNA folding of multiple alignments</td>
      <td>Seemann, Stefan E.; Gorodkin, Jan; Backofen, Rolf</td>
      <td>2008</td>
      <td>2008-10-04</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2582601" target="_blank">PMC2582601</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18836192.0" target="_blank">18836192.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkn544" target="_blank">10.1093/nar/gkn544</a></td>
      <td>wtvfow2f</td>
      <td>0.473693</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_61.html#213ly4xo">Fricke_2019</a></td>
    </tr>
    <tr>
      <th id="u3q36o3e";>253</th>
      <td>Shean_2019</td>
      <td>VAPiD: a lightweight cross-platform viral annotation pipeline and identification tool to facilitate virus genome submissions to NCBI GenBank</td>
      <td>Shean, Ryan C.; Makhsous, Negar; Stoddard, Graham D.; Lin, Michelle J.; Greninger, Alexander L.</td>
      <td>2019</td>
      <td>2019-01-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6343335" target="_blank">PMC6343335</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30674273.0" target="_blank">30674273.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-2606-y" target="_blank">10.1186/s12859-019-2606-y</a></td>
      <td>u3q36o3e</td>
      <td>0.473410</td>
      <td><a href="Topic_05.html#adzvn69b">Garcia–Moliner_2013</a>, <a href="Topic_66.html#uee3lqrx">Davaro_2013</a>, <a href="Topic_21.html#59emvigh">Ryu_W_2017</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a></td>
      <td><a href="Topic_17.html#wd3ir3wg">Wang_2012</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="rktwtqtq";>254</th>
      <td>Cai_C_2006</td>
      <td>Prediction of Transmembrane Proteins from Their Primary Sequence by Support Vector Machine Approach</td>
      <td>Cai, C. Z.; Yuan, Q. F.; Xiao, H. G.; Liu, X. H.; Han, L. Y.; Chen, Y. Z.</td>
      <td>2006</td>
      <td>2006-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121931" target="_blank">PMC7121931</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/11816102_56" target="_blank">10.1007/11816102_56</a></td>
      <td>rktwtqtq</td>
      <td>0.473124</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a></td>
      <td><a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_17.html#88l9bq4i">Han_L_2005</a></td>
    </tr>
    <tr>
      <th id="grs9eqql";>255</th>
      <td>Rumiński_2016</td>
      <td>Analysis of the parameters of respiration patterns extracted from thermal image sequences</td>
      <td>Rumiński, Jacek</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127106" target="_blank">PMC7127106</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.bbe.2016.07.006" target="_blank">10.1016/j.bbe.2016.07.006</a></td>
      <td>grs9eqql</td>
      <td>0.472747</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_06.html#kry1sejo">Lloyd_2009</a></td>
      <td><a href="Topic_17.html#flgc5l04">Budzan_2013</a>, <a href="Topic_01.html#0avs6g6a">Lahiri_2012</a>, <a href="Topic_65.html#37vyazby">Sun_G_2017</a>, <a href="Topic_64.html#n44h10tr">Hassan_2020</a></td>
    </tr>
    <tr>
      <th id="c6ttke66";>256</th>
      <td>Sandhu_2016</td>
      <td>Smart monitoring and controlling of Pandemic Influenza A (H1N1) using Social Network Analysis and cloud computing</td>
      <td>Sandhu, Rajinder; Gill, Harsuminder K.; Sood, Sandeep K.</td>
      <td>2016</td>
      <td>2016-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jocs.2015.11.001" target="_blank">10.1016/j.jocs.2015.11.001</a></td>
      <td>c6ttke66</td>
      <td>0.472009</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_22.html#iwtevhmq">Abdalla_2020</a>, <a href="Topic_13.html#ap92lgv7">dos_Santos_2018</a>, <a href="Topic_30.html#uj1vby14">Li_H_2015</a></td>
    </tr>
    <tr>
      <th id="6tss9ydx";>257</th>
      <td>Chen_2019</td>
      <td>A binning tool to reconstruct viral haplotypes from assembled contigs</td>
      <td>Chen, Jiao; Shang, Jiayu; Wang, Jianrong; Sun, Yanni</td>
      <td>2019</td>
      <td>2019-11-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6829986" target="_blank">PMC6829986</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31684876.0" target="_blank">31684876.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-3138-1" target="_blank">10.1186/s12859-019-3138-1</a></td>
      <td>6tss9ydx</td>
      <td>0.471597</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2basllfv">Chen_2019</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_17.html#2basllfv">Chen_2019</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a></td>
    </tr>
    <tr>
      <th id="vtt1k6ph";>258</th>
      <td>Liu_W_2015</td>
      <td>State-of-the-art methods for inverse design of an enclosed environment</td>
      <td>Liu, Wei; Zhang, Tengfei; Xue, Yu; Zhai, Zhiqiang (John); Wang, Jihong; Wei, Yun; Chen, Qingyan</td>
      <td>2015</td>
      <td>2015-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127361" target="_blank">PMC7127361</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2015.02.041" target="_blank">10.1016/j.buildenv.2015.02.041</a></td>
      <td>vtt1k6ph</td>
      <td>0.471437</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_64.html#nbbcacy4">Yuan_2014</a>, <a href="Topic_64.html#21ee4wxk">Chen_2013</a>, <a href="Topic_64.html#scwhwurr">Zhu_S_2010</a>, <a href="Topic_64.html#xf8160dx">Liu_W_2012</a></td>
    </tr>
    <tr>
      <th id="thtsulcy";>259</th>
      <td>Catanzaro_2013</td>
      <td>A mixed integer linear programming model to reconstruct phylogenies from single nucleotide polymorphism haplotypes under the maximum parsimony criterion</td>
      <td>Catanzaro, Daniele; Ravi, Ramamoorthi; Schwartz, Russell</td>
      <td>2013</td>
      <td>2013-01-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3599976" target="_blank">PMC3599976</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23343437.0" target="_blank">23343437.0</a></td>
      <td><a href="https://doi.org/10.1186/1748-7188-8-3" target="_blank">10.1186/1748-7188-8-3</a></td>
      <td>thtsulcy</td>
      <td>0.471116</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#aj352do0">Tibayrenc_2015</a>, <a href="Topic_45.html#uf7b9azg">Kepler_2010</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a>, <a href="Topic_06.html#zw5kt090">Hilton_2018</a></td>
    </tr>
    <tr>
      <th id="ngow9w7p";>260</th>
      <td>Lu_H_2009</td>
      <td>Multilingual chief complaint classification for syndromic surveillance: An experiment with Chinese chief complaints</td>
      <td>Lu, Hsin-Min; Chen, Hsinchun; Zeng, Daniel; King, Chwan-Chuen; Shih, Fuh-Yuan; Wu, Tsung-Shu; Hsiao, Jin-Yi</td>
      <td>2009</td>
      <td>2009-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108263" target="_blank">PMC7108263</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18838292.0" target="_blank">18838292.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ijmedinf.2008.08.004" target="_blank">10.1016/j.ijmedinf.2008.08.004</a></td>
      <td>ngow9w7p</td>
      <td>0.470595</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a></td>
      <td><a href="Topic_09.html#vkjz1orh">Chapman_2004</a>, <a href="Topic_17.html#r0lduvs1">Zhang_2009</a>, <a href="Topic_22.html#mohb5szt">Xin_H_2019</a>, <a href="Topic_22.html#kdymzbv2">Marsden-Haug_2007</a></td>
    </tr>
    <tr>
      <th id="udvrdglk";>261</th>
      <td>Barlow_2012</td>
      <td>In-silico studies in Chinese herbal medicines’ research: Evaluation of in-silico methodologies and phytochemical data sources, and a review of research to date</td>
      <td>Barlow, D.J.; Buriani, A.; Ehrman, T.; Bosisio, E.; Eberini, I.; Hylands, P.J.</td>
      <td>2012</td>
      <td>2012-04-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126886" target="_blank">PMC7126886</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22326356.0" target="_blank">22326356.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jep.2012.01.041" target="_blank">10.1016/j.jep.2012.01.041</a></td>
      <td>udvrdglk</td>
      <td>0.469717</td>
      <td><a href="Topic_01.html#s6x0igia">Dove_2004</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_01.html#lc7haf6u">Fathelrahman_2018</a>, <a href="Topic_01.html#4pjdutgg">Awaisu_2019</a></td>
      <td><a href="Topic_49.html#tjydn8k2">Lee_C_2020</a>, <a href="Topic_49.html#y5et8pqg">Dong_2017</a>, <a href="Topic_47.html#8rzibpbl">Eng_Y_2019</a>, <a href="Topic_62.html#ywnr8d48">Zhang_2019</a></td>
    </tr>
    <tr>
      <th id="yr8r080l";>262</th>
      <td>Pavlović-Lažetić_2009</td>
      <td>n-Gram characterization of genomic islands in bacterial genomes</td>
      <td>Pavlović-Lažetić, Gordana M.; Mitić, Nenad S.; Beljanski, Miloš V.</td>
      <td>2009</td>
      <td>2009-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19101056.0" target="_blank">19101056.0</a></td>
      <td><a href="https://doi.org/10.1016/j.cmpb.2008.10.014" target="_blank">10.1016/j.cmpb.2008.10.014</a></td>
      <td>yr8r080l</td>
      <td>0.469577</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_69.html#3v9cp89p">Chen_2020</a>, <a href="Topic_34.html#e0wmnfyn">McGuire_2012</a>, <a href="Topic_10.html#0abutzb3">Kang_2017</a>, <a href="Topic_07.html#v4odd4pm">Zhang_2006</a></td>
    </tr>
    <tr>
      <th id="facjsy92";>263</th>
      <td>Pickett_2012</td>
      <td>Virus Pathogen Database and Analysis Resource (ViPR): A Comprehensive Bioinformatics Database and Analysis Resource for the Coronavirus Research Community</td>
      <td>Pickett, Brett E.; Greer, Douglas S.; Zhang, Yun; Stewart, Lucy; Zhou, Liwei; Sun, Guangyu; Gu, Zhiping; Kumar, Sanjeev; Zaremba, Sam; Larsen, Christopher N.; Jen, Wei; Klem, Edward B.; Scheuermann, Richard H.</td>
      <td>2012</td>
      <td>2012-11-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3509690" target="_blank">PMC3509690</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23202522.0" target="_blank">23202522.0</a></td>
      <td><a href="https://doi.org/10.3390/v4113209" target="_blank">10.3390/v4113209</a></td>
      <td>facjsy92</td>
      <td>0.469397</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_17.html#48ym7eti">Pickett_2011</a>, <a href="Topic_07.html#v38tjof3">Fahmi_2020</a></td>
    </tr>
    <tr>
      <th id="whoydkd5";>264</th>
      <td>Cao_Y_2005</td>
      <td>Information theory-based algorithm for in silico prediction of PCR products with whole genomic sequences as templates</td>
      <td>Cao, Youfang; Wang, Lianjie; Xu, Kexue; Kou, Chunhai; Zhang, Yulei; Wei, Guifang; He, Junjian; Wang, Yunfang; Zhao, Liping</td>
      <td>2005</td>
      <td>2005-07-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1183192" target="_blank">PMC1183192</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16042814.0" target="_blank">16042814.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-6-190" target="_blank">10.1186/1471-2105-6-190</a></td>
      <td>whoydkd5</td>
      <td>0.469152</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_21.html#x2w73spd">Goldberg_1991</a></td>
    </tr>
    <tr>
      <th id="xu4camqf";>265</th>
      <td>Su_H_2012</td>
      <td>Framing the structure of global open innovation research</td>
      <td>Su, Hsin-Ning; Lee, Pei-Chun</td>
      <td>2012</td>
      <td>2012-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.joi.2011.12.001" target="_blank">10.1016/j.joi.2011.12.001</a></td>
      <td>xu4camqf</td>
      <td>0.467753</td>
      <td><a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_04.html#cvvyf3cb">Kelley_2011</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a></td>
      <td><a href="Topic_54.html#vl6gkoin">Lee_P_2011</a>, <a href="Topic_54.html#t28a3k6j">Fu_H_2013</a>, <a href="Topic_54.html#5ezqv94a">Annisa_2019</a></td>
    </tr>
    <tr>
      <th id="9z8z03rp";>266</th>
      <td>Kai_Z_2019</td>
      <td>An efficient simulated annealing algorithm for the RNA secondary structure prediction with Pseudoknots</td>
      <td>Kai, Zhang; Yuting, Wang; Yulin, Lv; Jun, Liu; Juanjuan, He</td>
      <td>2019</td>
      <td>2019-12-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6933665" target="_blank">PMC6933665</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31881969.0" target="_blank">31881969.0</a></td>
      <td><a href="https://doi.org/10.1186/s12864-019-6300-2" target="_blank">10.1186/s12864-019-6300-2</a></td>
      <td>9z8z03rp</td>
      <td>0.467303</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#jxgd1yc4">Kai_Z_2018</a>, <a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="yw81pkrq";>267</th>
      <td>Huang_2020</td>
      <td>Multiple-Input Deep Convolutional Neural Network Model for COVID-19 Forecasting in China</td>
      <td>Chiou-Jye Huang; Yung-Hsiang Chen; Yuxuan Ma; Ping-Huan Kuo</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.23.20041608" target="_blank">10.1101/2020.03.23.20041608</a></td>
      <td>yw81pkrq</td>
      <td>0.466683</td>
      <td><a href="Topic_06.html#h1i29kcq">Li_C_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#8qnq02gd">Zheng_2009</a></td>
      <td><a href="Topic_19.html#qpt5dj8p">Li_M_2020</a>, <a href="Topic_17.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="98oo0bkl";>268</th>
      <td>Nakai_1992</td>
      <td>A knowledge base for predicting protein localization sites in eukaryotic cells</td>
      <td>Nakai, Kenta; Kanehisa, Minoru</td>
      <td>1992</td>
      <td>1992-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134799" target="_blank">PMC7134799</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/1478671.0" target="_blank">1478671.0</a></td>
      <td><a href="https://doi.org/10.1016/s0888-7543(05)80111-9" target="_blank">10.1016/s0888-7543(05)80111-9</a></td>
      <td>98oo0bkl</td>
      <td>0.466681</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_17.html#85fqseal">Lee_B_2009</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#59yvbupx">Scott_2011</a></td>
    </tr>
    <tr>
      <th id="ppzxvykb";>269</th>
      <td>Dinkel_2011</td>
      <td>ELM—the database of eukaryotic linear motifs</td>
      <td>Dinkel, Holger; Michael, Sushama; Weatheritt, Robert J.; Davey, Norman E.; Van Roey, Kim; Altenberg, Brigitte; Toedt, Grischa; Uyar, Bora; Seiler, Markus; Budd, Aidan; Jödicke, Lisa; Dammert, Marcel A.; Schroeter, Christian; Hammer, Maria; Schmidt, Tobias; Jehl, Peter; McGuigan, Caroline; Dymecka, Magdalena; Chica, Claudia; Luck, Katja; Via, Allegra; Chatr-aryamontri, Andrew; Haslam, Niall; Grebnev, Gleb; Edwards, Richard J.; Steinmetz, Michel O.; Meiselbach, Heike; Diella, Francesca; Gibson, Toby J.</td>
      <td>2011</td>
      <td>2011-11-21</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245074" target="_blank">PMC3245074</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22110040.0" target="_blank">22110040.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkr1064" target="_blank">10.1093/nar/gkr1064</a></td>
      <td>ppzxvykb</td>
      <td>0.466177</td>
      <td><a href="Topic_16.html#ov1f1uyt">Ivarsson_2012</a>, <a href="Topic_16.html#hsh2yynq">Kinjo_2012</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a></td>
      <td><a href="Topic_16.html#0a3okta0">Myllykoski_2017</a></td>
    </tr>
    <tr>
      <th id="zbz88pqx";>270</th>
      <td>Lopes_2017</td>
      <td>Computational Comparison and Visualization of Viruses in the Perspective of Clinical Information</td>
      <td>Lopes, António M.; Machado, J. A. Tenreiro; Galhano, Alexandra M.</td>
      <td>2017</td>
      <td>2017-04-08</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090701" target="_blank">PMC7090701</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28391493.0" target="_blank">28391493.0</a></td>
      <td><a href="https://doi.org/10.1007/s12539-017-0229-4" target="_blank">10.1007/s12539-017-0229-4</a></td>
      <td>zbz88pqx</td>
      <td>0.465545</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_17.html#r7tg0s45">John_2019</a>, <a href="Topic_14.html#u0rjntlc">Davidson_2019</a>, <a href="Topic_54.html#arhqix9h">Heydari_2020</a></td>
    </tr>
    <tr>
      <th id="1v3wl6ky";>271</th>
      <td>Majumdar_2018</td>
      <td>Kyasanur Forest Disease Classification Framework Using Novel Extremal Optimization Tuned Neural Network in Fog Computing Environment</td>
      <td>Majumdar, Abhishek; Debnath, Tapas; Sood, Sandeep K.; Baishnab, Krishna Lal</td>
      <td>2018</td>
      <td>2018-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088392" target="_blank">PMC7088392</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30173290.0" target="_blank">30173290.0</a></td>
      <td><a href="https://doi.org/10.1007/s10916-018-1041-3" target="_blank">10.1007/s10916-018-1041-3</a></td>
      <td>1v3wl6ky</td>
      <td>0.465531</td>
      <td><a href="Topic_04.html#e0sqppeq">Guery_2014</a>, <a href="Topic_01.html#3b2adfbi">Wiwanitkit_2014</a>, <a href="Topic_06.html#h1i29kcq">Li_C_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a></td>
      <td><a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_55.html#kaip0d1i">John_2019</a>, <a href="Topic_04.html#j6h8u8gb">Zhang_2015</a>, <a href="Topic_17.html#o4y8ci0n">Mock_2019</a></td>
    </tr>
    <tr>
      <th id="ud3x78u3";>272</th>
      <td>Pereira_2014</td>
      <td>A Chemoinformatics Approach to the Discovery of Lead-Like Molecules from Marine and Microbial Sources En Route to Antitumor and Antibiotic Drugs</td>
      <td>Pereira, Florbela; Latino, Diogo A. R. S.; Gaudêncio, Susana P.</td>
      <td>2014</td>
      <td>2014-01-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3944514" target="_blank">PMC3944514</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24473174.0" target="_blank">24473174.0</a></td>
      <td><a href="https://doi.org/10.3390/md12020757" target="_blank">10.3390/md12020757</a></td>
      <td>ud3x78u3</td>
      <td>0.465135</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_18.html#l36rzqnx">Park_2012</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_49.html#gl55zhgw">Alvin_2014</a>, <a href="Topic_49.html#okp8ofl0">Pereira_2015</a>, <a href="Topic_18.html#rhg5pbsq">van_der_Kooy_2013</a>, <a href="Topic_49.html#bmdpbkni">Tabares_2011</a></td>
    </tr>
    <tr>
      <th id="bqwmx5dy";>273</th>
      <td>Punjani_2019</td>
      <td>Non-uniform refinement: Adaptive regularization improves single particle cryo-EM reconstruction</td>
      <td>Ali Punjani; Haowei Zhang; David J. Fleet</td>
      <td>2019</td>
      <td>2019-12-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2019.12.15.877092" target="_blank">10.1101/2019.12.15.877092</a></td>
      <td>bqwmx5dy</td>
      <td>0.464979</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#1voq7zxz">DiMaio_2016</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#oohx14kl">Zhong_2020</a>, <a href="Topic_17.html#3u81ntle">Acar_2016</a>, <a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_16.html#qu4enz7m">Frenz_2019</a></td>
    </tr>
    <tr>
      <th id="hgxhyg46";>274</th>
      <td>Greenbaum_2009</td>
      <td>Ontology Development for the Immune Epitope Database</td>
      <td>Greenbaum, Jason A.; Vita, Randi; Zarebski, Laura M.; Sette, Alessandro; Peters, Bjoern</td>
      <td>2009</td>
      <td>2009-08-08</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121563" target="_blank">PMC7121563</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-1-4419-0540-6_4" target="_blank">10.1007/978-1-4419-0540-6_4</a></td>
      <td>hgxhyg46</td>
      <td>0.464663</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_17.html#zwbc7nnn">Sathiamurthy_2005</a>, <a href="Topic_69.html#n184795x">Zamanzadeh_2017</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a></td>
    </tr>
    <tr>
      <th id="wt0lvhfl";>275</th>
      <td>Wu_T_2009</td>
      <td>IRSS: a web-based tool for automatic layout and analysis of IRES secondary structure prediction and searching system in silico</td>
      <td>Wu, Tzong-Yuan; Hsieh, Chi-Chun; Hong, Jun-Jie; Chen, Chung-Yung; Tsai, Yuh-Show</td>
      <td>2009</td>
      <td>2009-05-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2698906" target="_blank">PMC2698906</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19473520.0" target="_blank">19473520.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-160" target="_blank">10.1186/1471-2105-10-160</a></td>
      <td>wt0lvhfl</td>
      <td>0.464014</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a></td>
      <td><a href="Topic_61.html#7pdh7alh">Hong_2013</a>, <a href="Topic_61.html#ywkxxye5">Fajardo_2012</a>, <a href="Topic_21.html#cciyb05w">Yang_2015</a></td>
    </tr>
    <tr>
      <th id="397hp1yt";>276</th>
      <td>Coelho_2008</td>
      <td>Epigrass: a tool to study disease spread in complex networks</td>
      <td>Coelho, Flávio C; Cruz, Oswaldo G; Codeço, Cláudia T</td>
      <td>2008</td>
      <td>2008-02-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2275240" target="_blank">PMC2275240</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18302744.0" target="_blank">18302744.0</a></td>
      <td><a href="https://doi.org/10.1186/1751-0473-3-3" target="_blank">10.1186/1751-0473-3-3</a></td>
      <td>397hp1yt</td>
      <td>0.463652</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_06.html#w4g8mzuy">Groendyke_2012</a>, <a href="Topic_04.html#jrvl0ykn">Nelson_2019</a>, <a href="Topic_02.html#7a5nxxar">Bjørnstad_2019</a>, <a href="Topic_06.html#q1imoe7k">Zhong_2009</a></td>
    </tr>
    <tr>
      <th id="uv7g3liw";>277</th>
      <td>Heine_2011</td>
      <td>Statistical learning techniques applied to epidemiology: a simulated case-control comparison study with logistic regression</td>
      <td>Heine, John J; Land, Walker H; Egan, Kathleen M</td>
      <td>2011</td>
      <td>2011-01-27</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045299" target="_blank">PMC3045299</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21272346.0" target="_blank">21272346.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-37" target="_blank">10.1186/1471-2105-12-37</a></td>
      <td>uv7g3liw</td>
      <td>0.463429</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_06.html#auasueaw">Kenah_2015</a>, <a href="Topic_22.html#i5m3ax7g">Wang_2012</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
    </tr>
    <tr>
      <th id="t3zampk7";>278</th>
      <td>Chen_2012</td>
      <td>PepMapper: A Collaborative Web Tool for Mapping Epitopes from Affinity-Selected Peptides</td>
      <td>Chen, Wenhan; Guo, William W.; Huang, Yanxin; Ma, Zhiqiang</td>
      <td>2012</td>
      <td>2012-05-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3360666" target="_blank">PMC3360666</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22701536.0" target="_blank">22701536.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0037869" target="_blank">10.1371/journal.pone.0037869</a></td>
      <td>t3zampk7</td>
      <td>0.462979</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#7rs4jtfh">Mayrose_2007</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_69.html#ftf67ykd">Irving_2010</a>, <a href="Topic_17.html#qk1pr335">Sun_P_2015</a></td>
    </tr>
    <tr>
      <th id="1voq7zxz";>279</th>
      <td>DiMaio_2016</td>
      <td>Chapter Ten Tools for Model Building and Optimization into Near-Atomic Resolution Electron Cryo-Microscopy Density Maps</td>
      <td>DiMaio, F.; Chiu, W.</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5103630" target="_blank">PMC5103630</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27572730.0" target="_blank">27572730.0</a></td>
      <td><a href="https://doi.org/10.1016/bs.mie.2016.06.003" target="_blank">10.1016/bs.mie.2016.06.003</a></td>
      <td>1voq7zxz</td>
      <td>0.462713</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_01.html#n6ett8uo">Subramaniam_2016</a>, <a href="Topic_60.html#gz12gn0u">Yadav_2005</a></td>
    </tr>
    <tr>
      <th id="ljli6a2z";>280</th>
      <td>Balasubramanian_2020</td>
      <td>Automatic Identification of SARS Coronavirus using Compression-Complexity Measures</td>
      <td>Karthi Balasubramanian; Nithin Nagaraj</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.24.006007" target="_blank">10.1101/2020.03.24.006007</a></td>
      <td>ljli6a2z</td>
      <td>0.461999</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_45.html#gccmkxly">Phan_2018</a>, <a href="Topic_17.html#gxstlzuk">Cleemput_2020</a>, <a href="Topic_17.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_07.html#t3t20wji">Qin_E_2003</a></td>
    </tr>
    <tr>
      <th id="kiwi87c9";>281</th>
      <td>Zhou_2011</td>
      <td>Multi-conformation 3D QSAR study of benzenesulfonyl-pyrazol-ester compounds and their analogs as cathepsin B inhibitors</td>
      <td>Zhou, Zhigang; Wang, Yanli; Bryant, Stephen H.</td>
      <td>2011</td>
      <td>2011-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167229" target="_blank">PMC3167229</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21798778.0" target="_blank">21798778.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmgm.2011.06.013" target="_blank">10.1016/j.jmgm.2011.06.013</a></td>
      <td>kiwi87c9</td>
      <td>0.461089</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_16.html#et00ovrr">Grazioso_2012</a>, <a href="Topic_16.html#32ju2h1g">Niu_C_2008</a>, <a href="Topic_18.html#cyp148on">Li_J_2006</a></td>
    </tr>
    <tr>
      <th id="gf718lbl";>282</th>
      <td>Yu_Z_2010</td>
      <td>Whole-proteome phylogeny of large dsDNA viruses and parvoviruses through a composition vector method related to dynamical language model</td>
      <td>Yu, Zu-Guo; Chu, Ka Hou; Li, Chi Pang; Anh, Vo; Zhou, Li-Qian; Wang, Roger Wei</td>
      <td>2010</td>
      <td>2010-06-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2898692" target="_blank">PMC2898692</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20565983.0" target="_blank">20565983.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2148-10-192" target="_blank">10.1186/1471-2148-10-192</a></td>
      <td>gf718lbl</td>
      <td>0.461006</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a></td>
      <td><a href="Topic_07.html#44h6rdwg">Zhou_2017</a>, <a href="Topic_07.html#fciota2z">Suzuki_2020</a></td>
    </tr>
    <tr>
      <th id="ll4rxd9p";>283</th>
      <td>Zheng_2020</td>
      <td>Deep Learning-based Detection for COVID-19 from Chest CT using Weak Label</td>
      <td>Chuansheng Zheng; Xianbo Deng; Qing Fu; Qiang Zhou; Jiapei Feng; Hui Ma; Wenyu Liu; Xinggang Wang</td>
      <td>2020</td>
      <td>2020-03-17</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.12.20027185" target="_blank">10.1101/2020.03.12.20027185</a></td>
      <td>ll4rxd9p</td>
      <td>0.460135</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_06.html#63wqn2fg">Höhle_2007</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_17.html#k1lg8c7q">Jin_C_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="pq1vaknj";>284</th>
      <td>Heal_2018</td>
      <td>Applying graph theory to protein structures: an Atlas of coiled coils</td>
      <td>Heal, Jack W; Bartlett, Gail J; Wood, Christopher W; Thomson, Andrew R; Woolfson, Derek N</td>
      <td>2018</td>
      <td>2018-10-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6157074" target="_blank">PMC6157074</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29722888.0" target="_blank">29722888.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bty347" target="_blank">10.1093/bioinformatics/bty347</a></td>
      <td>pq1vaknj</td>
      <td>0.460135</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_16.html#l7p84aur">Apgar_2008</a>, <a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_17.html#g4k6pc82">Munteanu_2008</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="ufdgi67q";>285</th>
      <td>Khanna_2011</td>
      <td>In silico approach to screen compounds active against parasitic nematodes of major socio-economic importance</td>
      <td>Khanna, Varun; Ranganathan, Shoba</td>
      <td>2011</td>
      <td>2011-11-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3278842" target="_blank">PMC3278842</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22373185.0" target="_blank">22373185.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-s13-s25" target="_blank">10.1186/1471-2105-12-s13-s25</a></td>
      <td>ufdgi67q</td>
      <td>0.457639</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_49.html#gl55zhgw">Alvin_2014</a>, <a href="Topic_40.html#cgcf7mgj">Zhuo_2017</a>, <a href="Topic_49.html#bmdpbkni">Tabares_2011</a>, <a href="Topic_01.html#8m7zbv6b">Roeber_2013</a></td>
    </tr>
    <tr>
      <th id="p1i7bx94";>286</th>
      <td>Zhang_2013</td>
      <td>Enhancement of accuracy and efficiency for RNA secondary structure prediction by sequence segmentation and MapReduce</td>
      <td>Zhang, Boyu; Yehdego, Daniel T; Johnson, Kyle L; Leung, Ming-Ying; Taufer, Michela</td>
      <td>2013</td>
      <td>2013-11-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3952952" target="_blank">PMC3952952</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24564983.0" target="_blank">24564983.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6807-13-s1-s3" target="_blank">10.1186/1472-6807-13-s1-s3</a></td>
      <td>p1i7bx94</td>
      <td>0.456232</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#bhtl73z9">Suresh_2015</a>, <a href="Topic_61.html#7cwp2xbw">Yamasaki_2019</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a>, <a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="qrnhp1ek";>287</th>
      <td>Kirillova_2009</td>
      <td>Protein Domain Boundary Predictions: A Structural Biology Perspective</td>
      <td>Kirillova, Svetlana; Kumar, Suresh; Carugo, Oliviero</td>
      <td>2009</td>
      <td>2009-01-21</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2669640" target="_blank">PMC2669640</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19401756.0" target="_blank">19401756.0</a></td>
      <td><a href="https://doi.org/10.2174/1874091x00903010001" target="_blank">10.2174/1874091x00903010001</a></td>
      <td>qrnhp1ek</td>
      <td>0.455916</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_16.html#gbnns8np">RAMAKRISHNA_2015</a></td>
    </tr>
    <tr>
      <th id="4qbdytun";>288</th>
      <td>Foulger_2015</td>
      <td>Representing virus-host interactions and other multi-organism processes in the Gene Ontology</td>
      <td>Foulger, R. E.; Osumi-Sutherland, D.; McIntosh, B. K.; Hulo, C.; Masson, P.; Poux, S.; Le Mercier, P.; Lomax, J.</td>
      <td>2015</td>
      <td>2015-07-28</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4517558" target="_blank">PMC4517558</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26215368.0" target="_blank">26215368.0</a></td>
      <td><a href="https://doi.org/10.1186/s12866-015-0481-x" target="_blank">10.1186/s12866-015-0481-x</a></td>
      <td>4qbdytun</td>
      <td>0.455378</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a></td>
      <td><a href="Topic_10.html#629kl04a">Bahir_2009</a>, <a href="Topic_10.html#lpbma80s">Vigerust_2011</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_10.html#12dee0lv">Malik_2017</a></td>
    </tr>
    <tr>
      <th id="qk1pr335";>289</th>
      <td>Sun_P_2015</td>
      <td>Conformational B-Cell Epitope Prediction Method Based on Antigen Preprocessing and Mimotopes Analysis</td>
      <td>Sun, Pingping; Ju, Haixu; Zhang, Baowen; Gu, Yu; Liu, Bo; Huang, Yanxin; Zhang, Huijie; Li, Yuxin</td>
      <td>2015</td>
      <td>2015-01-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326220" target="_blank">PMC4326220</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25705652.0" target="_blank">25705652.0</a></td>
      <td><a href="https://doi.org/10.1155/2015/257030" target="_blank">10.1155/2015/257030</a></td>
      <td>qk1pr335</td>
      <td>0.454569</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_69.html#fmumym1x">Lon_J_2020</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a>, <a href="Topic_23.html#9ne5iwny">Li_S_2016</a></td>
    </tr>
    <tr>
      <th id="f1j6uhmj";>290</th>
      <td>Zhang_2014</td>
      <td>A Brief Review: The Z-curve Theory and its Application in Genome Analysis</td>
      <td>Zhang, Ren; Zhang, Chun-Ting</td>
      <td>2014</td>
      <td>2014-04-10</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4009844" target="_blank">PMC4009844</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24822026.0" target="_blank">24822026.0</a></td>
      <td><a href="https://doi.org/10.2174/1389202915999140328162433" target="_blank">10.2174/1389202915999140328162433</a></td>
      <td>f1j6uhmj</td>
      <td>0.454569</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_69.html#9pkvj14m">Singleton_2018</a>, <a href="Topic_17.html#l2pfl776">Zhang_2011</a></td>
    </tr>
    <tr>
      <th id="t54c2si1";>291</th>
      <td>Bastani_2012</td>
      <td>Contaminant source identification within a building: Toward design of immune buildings</td>
      <td>Bastani, Arash; Haghighat, Fariborz; Kozinski, Janusz A.</td>
      <td>2012</td>
      <td>2012-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127283" target="_blank">PMC7127283</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2011.12.002" target="_blank">10.1016/j.buildenv.2011.12.002</a></td>
      <td>t54c2si1</td>
      <td>0.453865</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_64.html#k6ovqpsn">Cai_H_2014</a>, <a href="Topic_64.html#fne8p8h8">Wu_Y_2017</a>, <a href="Topic_64.html#z3ktm7mv">Bolashikov_2009</a>, <a href="Topic_64.html#nldu46fg">Tung_2009</a></td>
    </tr>
    <tr>
      <th id="hphn7cs6";>292</th>
      <td>Cacciabue_2019</td>
      <td>A beginner’s guide for FMDV quasispecies analysis: sub-consensus variant detection and haplotype reconstruction using next-generation sequencing</td>
      <td>Cacciabue, Marco; Currá, Anabella; Carrillo, Elisa; König, Guido; Gismondi, María Inés</td>
      <td>2019</td>
      <td>2019-11-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110011" target="_blank">PMC7110011</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31697321.0" target="_blank">31697321.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bbz086" target="_blank">10.1093/bib/bbz086</a></td>
      <td>hphn7cs6</td>
      <td>0.453366</td>
      <td><a href="Topic_01.html#84gvwb35">Siridechadilok_2013</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a>, <a href="Topic_10.html#dkfrk71a">Smith_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_07.html#thsq6pnk">Lachheb_2019</a>, <a href="Topic_45.html#1a9u53za">Shabman_2016</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a>, <a href="Topic_07.html#t4215imq">Alquezar-Planas_2013</a></td>
    </tr>
    <tr>
      <th id="raycnjbl";>293</th>
      <td>Fontanini_2015</td>
      <td>Constructing Markov matrices for real-time transient contaminant transport analysis for indoor environments</td>
      <td>Fontanini, Anthony D.; Vaidya, Umesh; Ganapathysubramanian, Baskar</td>
      <td>2015</td>
      <td>2015-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125716" target="_blank">PMC7125716</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2015.07.020" target="_blank">10.1016/j.buildenv.2015.07.020</a></td>
      <td>raycnjbl</td>
      <td>0.451871</td>
      <td><a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_06.html#8k22red9">So_M_2008</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_64.html#kyq5n601">Chen_2015</a>, <a href="Topic_64.html#61gfqqhy">Hathway_2011</a>, <a href="Topic_64.html#k6ovqpsn">Cai_H_2014</a>, <a href="Topic_64.html#21ee4wxk">Chen_2013</a></td>
    </tr>
    <tr>
      <th id="v5uiuhih";>294</th>
      <td>Lemmin_2019</td>
      <td>Glycosylator: a Python framework for the rapid modeling of glycans</td>
      <td>Lemmin, Thomas; Soto, Cinque</td>
      <td>2019</td>
      <td>2019-10-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6806574" target="_blank">PMC6806574</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31640540.0" target="_blank">31640540.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-019-3097-6" target="_blank">10.1186/s12859-019-3097-6</a></td>
      <td>v5uiuhih</td>
      <td>0.450882</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_41.html#3mujx7j4">Cao_L_2018</a>, <a href="Topic_17.html#y9y7d96c">Besançon_2020</a>, <a href="Topic_41.html#utfeiviy">McDowell_1988</a>, <a href="Topic_17.html#30foefr9">Liu_G_2015</a></td>
    </tr>
    <tr>
      <th id="ac00tai9";>295</th>
      <td>Bartoszewicz_2020</td>
      <td>Interpretable detection of novel human viruses from genome sequencing data</td>
      <td>Jakub M Bartoszewicz; Anja Seidel; Bernhard Y Renard</td>
      <td>2020</td>
      <td>2020-01-30</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.29.925354" target="_blank">10.1101/2020.01.29.925354</a></td>
      <td>ac00tai9</td>
      <td>0.450579</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#qdy29oym">Denison_2008</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_17.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_07.html#thsq6pnk">Lachheb_2019</a>, <a href="Topic_13.html#k0f9fser">Shi_W_2017</a></td>
    </tr>
    <tr>
      <th id="5z3s5pjl";>296</th>
      <td>Jiang_2010</td>
      <td>The calculation of information and organismal complexity</td>
      <td>Jiang, Yun; Xu, Cunshuan</td>
      <td>2010</td>
      <td>2010-10-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2973933" target="_blank">PMC2973933</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20937149.0" target="_blank">20937149.0</a></td>
      <td><a href="https://doi.org/10.1186/1745-6150-5-59" target="_blank">10.1186/1745-6150-5-59</a></td>
      <td>5z3s5pjl</td>
      <td>0.449566</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_10.html#kjyuxc3g">Koonin_2013</a>, <a href="Topic_21.html#5d64esp4">Walker_2015</a>, <a href="Topic_10.html#goidlh08">Walker_2011</a></td>
    </tr>
    <tr>
      <th id="7bh8222u";>297</th>
      <td>Zhang_2007</td>
      <td>Estimating residue evolutionary conservation by introducing von Neumann entropy and a novel gap-treating approach</td>
      <td>Zhang, S.-W.; Zhang, Y.-L.; Pan, Q.; Cheng, Y.-M.; Chou, K.-C.</td>
      <td>2007</td>
      <td>2007-08-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088136" target="_blank">PMC7088136</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17710364.0" target="_blank">17710364.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0586-0" target="_blank">10.1007/s00726-007-0586-0</a></td>
      <td>7bh8222u</td>
      <td>0.449315</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_17.html#9zfct1yx">Altschul_1991</a>, <a href="Topic_06.html#zw5kt090">Hilton_2018</a></td>
    </tr>
    <tr>
      <th id="er6lz09f";>298</th>
      <td>Pfab_2020</td>
      <td>DeepTracer: Predicting Backbone Atomic Structure from High Resolution Cryo-EM Density Maps of Protein Complexes</td>
      <td>Jonas Pfab; Dong Si</td>
      <td>2020</td>
      <td>2020-02-13</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.12.946772" target="_blank">10.1101/2020.02.12.946772</a></td>
      <td>er6lz09f</td>
      <td>0.449213</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#oohx14kl">Zhong_2020</a>, <a href="Topic_01.html#n6ett8uo">Subramaniam_2016</a></td>
    </tr>
    <tr>
      <th id="uqvjx98j";>299</th>
      <td>Dwith_Chenna_2018</td>
      <td>Free-Form Deformation Approach for Registration of Visible and Infrared Facial Images in Fever Screening †</td>
      <td>Dwith Chenna, Yedukondala Narendra; Ghassemi, Pejhman; Pfefer, T. Joshua; Casamento, Jon; Wang, Quanzeng</td>
      <td>2018</td>
      <td>2018-01-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5795541" target="_blank">PMC5795541</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29300320.0" target="_blank">29300320.0</a></td>
      <td><a href="https://doi.org/10.3390/s18010125" target="_blank">10.3390/s18010125</a></td>
      <td>uqvjx98j</td>
      <td>0.447573</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#43igjn6x">Kylberg_2009</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_65.html#g5neg8wx">Ghassemi_2018</a>, <a href="Topic_17.html#flgc5l04">Budzan_2013</a>, <a href="Topic_65.html#37vyazby">Sun_G_2017</a>, <a href="Topic_01.html#0avs6g6a">Lahiri_2012</a></td>
    </tr>
    <tr>
      <th id="6vuvhf8v";>300</th>
      <td>Banakar_2016</td>
      <td>An intelligent device for diagnosing avian diseases: Newcastle, infectious bronchitis, avian influenza</td>
      <td>Banakar, Ahmad; Sadeghi, Mohammad; Shushtari, Abdolhamid</td>
      <td>2016</td>
      <td>2016-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125684" target="_blank">PMC7125684</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.compag.2016.08.006" target="_blank">10.1016/j.compag.2016.08.006</a></td>
      <td>6vuvhf8v</td>
      <td>0.447223</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_63.html#n6k1t3c1">Balestrin_2014</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_63.html#9zs3qqo4">Alsultan_2019</a></td>
    </tr>
    <tr>
      <th id="a2m46j0z";>301</th>
      <td>Tian_2016</td>
      <td>The similarity analysis of financial stocks based on information clustering</td>
      <td>Tian, Qiang; Shang, Pengjian; Feng, Guochen</td>
      <td>2016</td>
      <td>2016-05-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088863" target="_blank">PMC7088863</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11071-016-2851-9" target="_blank">10.1007/s11071-016-2851-9</a></td>
      <td>a2m46j0z</td>
      <td>0.446177</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_71.html#e4shf7qq">Tian_2014</a>, <a href="Topic_71.html#oezkz55m">Tung_2011</a>, <a href="Topic_71.html#tjnt01ld">Argyroudis_2019</a>, <a href="Topic_71.html#4ezebg45">Hui_E_2016</a></td>
    </tr>
    <tr>
      <th id="51khqqwx";>302</th>
      <td>Thomas_2007</td>
      <td>GraphDNA: a Java program for graphical display of DNA composition analyses</td>
      <td>Thomas, Jamie M; Horspool, Daniel; Brown, Gordon; Tcherepanov, Vasily; Upton, Chris</td>
      <td>2007</td>
      <td>2007-01-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1783863" target="_blank">PMC1783863</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17244370.0" target="_blank">17244370.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-8-21" target="_blank">10.1186/1471-2105-8-21</a></td>
      <td>51khqqwx</td>
      <td>0.445482</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a></td>
      <td><a href="Topic_21.html#ov92qh6h">Nowak_1984</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a></td>
    </tr>
    <tr>
      <th id="3c4dttrt";>303</th>
      <td>Yu_C_2010</td>
      <td>A Novel Construction of Genome Space with Biological Geometry</td>
      <td>Yu, Chenglong; Liang, Qian; Yin, Changchuan; He, Rong L.; Yau, Stephen S.-T.</td>
      <td>2010</td>
      <td>2010-04-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885272" target="_blank">PMC2885272</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20360268.0" target="_blank">20360268.0</a></td>
      <td><a href="https://doi.org/10.1093/dnares/dsq008" target="_blank">10.1093/dnares/dsq008</a></td>
      <td>3c4dttrt</td>
      <td>0.445482</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_21.html#3g3apbon">Lauber_2013</a></td>
    </tr>
    <tr>
      <th id="q26f8pv4";>304</th>
      <td>Sperschneider_2010</td>
      <td>DotKnot: pseudoknot prediction using the probability dot plot under a refined energy model</td>
      <td>Sperschneider, Jana; Datta, Amitava</td>
      <td>2010</td>
      <td>2010-01-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2853144" target="_blank">PMC2853144</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20123730.0" target="_blank">20123730.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkq021" target="_blank">10.1093/nar/gkq021</a></td>
      <td>q26f8pv4</td>
      <td>0.445404</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_61.html#9zm4per4">Dawson_2007</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#gjild96e">Song_2005</a></td>
      <td><a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a>, <a href="Topic_17.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_17.html#9z8z03rp">Kai_Z_2019</a></td>
    </tr>
    <tr>
      <th id="l9wrrapv";>305</th>
      <td>Duchêne_2015</td>
      <td>Evaluating the Adequacy of Molecular Clock Models Using Posterior Predictive Simulations</td>
      <td>Duchêne, David A.; Duchêne, Sebastian; Holmes, Edward C.; Ho, Simon Y.W.</td>
      <td>2015</td>
      <td>2015-07-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107558" target="_blank">PMC7107558</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26163668.0" target="_blank">26163668.0</a></td>
      <td><a href="https://doi.org/10.1093/molbev/msv154" target="_blank">10.1093/molbev/msv154</a></td>
      <td>l9wrrapv</td>
      <td>0.445278</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_06.html#zw5kt090">Hilton_2018</a></td>
    </tr>
    <tr>
      <th id="85fqseal";>306</th>
      <td>Lee_B_2009</td>
      <td>Identification of protein functions using a machine-learning approach based on sequence-derived properties</td>
      <td>Lee, Bum Ju; Shin, Moon Sun; Oh, Young Joon; Oh, Hae Seok; Ryu, Keun Ho</td>
      <td>2009</td>
      <td>2009-08-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2731080" target="_blank">PMC2731080</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19664241.0" target="_blank">19664241.0</a></td>
      <td><a href="https://doi.org/10.1186/1477-5956-7-27" target="_blank">10.1186/1477-5956-7-27</a></td>
      <td>85fqseal</td>
      <td>0.444968</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_17.html#88l9bq4i">Han_L_2005</a></td>
    </tr>
    <tr>
      <th id="3xq5jwof";>307</th>
      <td>Kulkarni-Kale_2006</td>
      <td>Curation of viral genomes: challenges, applications and the way forward</td>
      <td>Kulkarni-Kale, Urmila; Bhosle, Shriram G; Manjari, G Sunitha; Joshi, Manali; Bansode, Sandeep; Kolaskar, Ashok S</td>
      <td>2006</td>
      <td>2006-12-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1764468" target="_blank">PMC1764468</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17254296.0" target="_blank">17254296.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-s5-s12" target="_blank">10.1186/1471-2105-7-s5-s12</a></td>
      <td>3xq5jwof</td>
      <td>0.444968</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a></td>
    </tr>
    <tr>
      <th id="wr64fzjf";>308</th>
      <td>Serrano-Guerrero_2020</td>
      <td>A relevance and quality-based ranking algorithm applied to evidence-based medicine</td>
      <td>Serrano-Guerrero, Jesus; Romero, Francisco P.; Olivas, Jose A.</td>
      <td>2020</td>
      <td>2020-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114639" target="_blank">PMC7114639</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32114416.0" target="_blank">32114416.0</a></td>
      <td><a href="https://doi.org/10.1016/j.cmpb.2020.105415" target="_blank">10.1016/j.cmpb.2020.105415</a></td>
      <td>wr64fzjf</td>
      <td>0.444473</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_54.html#41hjajw3">Rosenberg_2010</a>, <a href="Topic_54.html#89d2gzhy">Rodríguez-Terol_2009</a>, <a href="Topic_54.html#jmjtr1pb">Luo_C_2018</a>, <a href="Topic_54.html#4poiud3q">Wang_2016</a></td>
    </tr>
    <tr>
      <th id="4wnoa4l7";>309</th>
      <td>Cai_Y_2019</td>
      <td>Temporal event searches based on event maps and relationships</td>
      <td>Cai, Yi; Xie, Haoran; Lau, Raymond Y.K.; Li, Qing; Wong, Tak-Lam; Wang, Fu Lee</td>
      <td>2019</td>
      <td>2019-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7105191" target="_blank">PMC7105191</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.asoc.2019.105750" target="_blank">10.1016/j.asoc.2019.105750</a></td>
      <td>4wnoa4l7</td>
      <td>0.444473</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_02.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_17.html#y0s4cf6g">Zou_Z_2013</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_17.html#a0bbw3er">Waagmeester_2020</a></td>
    </tr>
    <tr>
      <th id="30foefr9";>310</th>
      <td>Liu_G_2015</td>
      <td>INTEGRATION OF SYSTEMS GLYCOBIOLOGY WITH BIOINFORMATICS TOOLBOXES, GLYCOINFORMATICS RESOURCES AND GLYCOPROTEOMICS DATA</td>
      <td>Liu, Gang; Neelamegham, Sriram</td>
      <td>2015</td>
      <td>2015-04-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4457600" target="_blank">PMC4457600</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25871730.0" target="_blank">25871730.0</a></td>
      <td><a href="https://doi.org/10.1002/wsbm.1296" target="_blank">10.1002/wsbm.1296</a></td>
      <td>30foefr9</td>
      <td>0.444465</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_41.html#3mujx7j4">Cao_L_2018</a>, <a href="Topic_41.html#jexxd6jl">Langereis_2015</a>, <a href="Topic_16.html#jatetuim">Satoh_2015</a>, <a href="Topic_08.html#1zjjg7n6">Karav_2017</a></td>
    </tr>
    <tr>
      <th id="s8ohts1j";>311</th>
      <td>Stanstrup_2019</td>
      <td>The metaRbolomics Toolbox in Bioconductor and beyond</td>
      <td>Stanstrup, Jan; Broeckling, Corey D.; Helmus, Rick; Hoffmann, Nils; Mathé, Ewy; Naake, Thomas; Nicolotti, Luca; Peters, Kristian; Rainer, Johannes; Salek, Reza M.; Schulze, Tobias; Schymanski, Emma L.; Stravs, Michael A.; Thévenot, Etienne A.; Treutler, Hendrik; Weber, Ralf J. M.; Willighagen, Egon; Witting, Michael; Neumann, Steffen</td>
      <td>2019</td>
      <td>2019-09-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6835268" target="_blank">PMC6835268</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31548506.0" target="_blank">31548506.0</a></td>
      <td><a href="https://doi.org/10.3390/metabo9100200" target="_blank">10.3390/metabo9100200</a></td>
      <td>s8ohts1j</td>
      <td>0.443633</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_01.html#lzdjfvh7">May_E_2016</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_17.html#30foefr9">Liu_G_2015</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_49.html#drg733k9">Chen_2013</a></td>
    </tr>
    <tr>
      <th id="61va8q92";>312</th>
      <td>Niethammer_2013</td>
      <td>Segmentation with area constraints</td>
      <td>Niethammer, Marc; Zach, Christopher</td>
      <td>2013</td>
      <td>2013-01-31</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3656501" target="_blank">PMC3656501</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23084504.0" target="_blank">23084504.0</a></td>
      <td><a href="https://doi.org/10.1016/j.media.2012.09.002" target="_blank">10.1016/j.media.2012.09.002</a></td>
      <td>61va8q92</td>
      <td>0.443218</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a></td>
      <td><a href="Topic_17.html#3u81ntle">Acar_2016</a>, <a href="Topic_17.html#bqwmx5dy">Punjani_2019</a>, <a href="Topic_17.html#89xnnvuv">Saad_2005</a>, <a href="Topic_17.html#wvw6q7px">Jonić_2016</a></td>
    </tr>
    <tr>
      <th id="zz1hkbpc";>313</th>
      <td>Gardner_2014</td>
      <td>Simulate_PCR for amplicon prediction and annotation from multiplex, degenerate primers and probes</td>
      <td>Gardner, Shea N; Slezak, Tom</td>
      <td>2014</td>
      <td>2014-07-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4226945" target="_blank">PMC4226945</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25005023.0" target="_blank">25005023.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-237" target="_blank">10.1186/1471-2105-15-237</a></td>
      <td>zz1hkbpc</td>
      <td>0.442199</td>
      <td><a href="Topic_07.html#g4sjyytr">Phillippy_2009</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a></td>
      <td><a href="Topic_29.html#7658dmvk">Gardner_2009</a>, <a href="Topic_29.html#xjfb83is">Li_K_2012</a></td>
    </tr>
    <tr>
      <th id="ygr1kv9r";>314</th>
      <td>Hu_P_2007</td>
      <td>A Web-Based System for Infectious Disease Data Integration and Sharing: Evaluating Outcome, Task Performance Efficiency, User Information Satisfaction, and Usability</td>
      <td>Hu, Paul Jen-Hwa; Zeng, Daniel; Chen, Hsinchun; Larson, Catherine A.; Tseng, Chunju</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122142" target="_blank">PMC7122142</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72608-1_13" target="_blank">10.1007/978-3-540-72608-1_13</a></td>
      <td>ygr1kv9r</td>
      <td>0.441601</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_06.html#2z32ln7g">Daughton_2017</a>, <a href="Topic_04.html#6q8eg9z4">Keller_2009</a>, <a href="Topic_17.html#jx8wulcg">Seo_D_2017</a></td>
    </tr>
    <tr>
      <th id="vho70jcx";>315</th>
      <td>Minot_2014</td>
      <td>SIANN: Strain Identification by Alignment to Near Neighbors</td>
      <td>Samuel Minot; Stephen D Turner; Krista L Ternus; Dana R Kadavy</td>
      <td>2014</td>
      <td>2014-01-10</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/001727" target="_blank">10.1101/001727</a></td>
      <td>vho70jcx</td>
      <td>0.441471</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_07.html#15cxc32n">Shifman_2019</a>, <a href="Topic_29.html#7dcl9txk">ZHANG_2018</a>, <a href="Topic_17.html#058r9486">Morfopoulou_2014</a>, <a href="Topic_17.html#z29xsj1o">Morfopoulou_2015</a></td>
    </tr>
    <tr>
      <th id="s7kzp795";>316</th>
      <td>Zemla_2011</td>
      <td>StralSV: assessment of sequence variability within similar 3D structures and application to polio RNA-dependent RNA polymerase</td>
      <td>Zemla, Adam T; Lang, Dorothy M; Kostova, Tanya; Andino, Raul; Ecale Zhou, Carol L</td>
      <td>2011</td>
      <td>2011-06-02</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3121648" target="_blank">PMC3121648</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21635786.0" target="_blank">21635786.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-226" target="_blank">10.1186/1471-2105-12-226</a></td>
      <td>s7kzp795</td>
      <td>0.441196</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_21.html#s90geszi">Lang_2012</a>, <a href="Topic_16.html#ovy03sfy">Scrima_2004</a></td>
    </tr>
    <tr>
      <th id="oohx14kl";>317</th>
      <td>Zhong_2020</td>
      <td>CryoDRGN: Reconstruction of heterogeneous structures from cryo-electron micrographs using neural networks</td>
      <td>Ellen D. Zhong; Tristan Bepler; Bonnie Berger; Joseph H Davis</td>
      <td>2020</td>
      <td>2020-03-29</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.27.003871" target="_blank">10.1101/2020.03.27.003871</a></td>
      <td>oohx14kl</td>
      <td>0.440045</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_17.html#er6lz09f">Pfab_2020</a>, <a href="Topic_17.html#1voq7zxz">DiMaio_2016</a>, <a href="Topic_17.html#bqwmx5dy">Punjani_2019</a></td>
    </tr>
    <tr>
      <th id="f21dknmb";>318</th>
      <td>Ji_X_2020</td>
      <td>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</td>
      <td>Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Yiqing Xue; Chunli Zhang; Guangming Tan; Gang Niu</td>
      <td>2020</td>
      <td>2020-02-26</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.24.20025437" target="_blank">10.1101/2020.02.24.20025437</a></td>
      <td>f21dknmb</td>
      <td>0.439420</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_06.html#cx3ex9ru">Leach_2010</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a></td>
      <td><a href="Topic_17.html#eaglecq7">Ji_X_2020</a>, <a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_54.html#nm0i0rtz">Choi_2017</a></td>
    </tr>
    <tr>
      <th id="5igorndt";>319</th>
      <td>Janssen_2015</td>
      <td>The RNA shapes studio</td>
      <td>Janssen, Stefan; Giegerich, Robert</td>
      <td>2015</td>
      <td>2015-02-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4308662" target="_blank">PMC4308662</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25273103.0" target="_blank">25273103.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btu649" target="_blank">10.1093/bioinformatics/btu649</a></td>
      <td>5igorndt</td>
      <td>0.438242</td>
      <td><a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_61.html#1lojd0xa">Gao_J_2010</a>, <a href="Topic_17.html#zgv4kd6j">Reeder_2007</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a>, <a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="1c515h51";>320</th>
      <td>Pérez_2016</td>
      <td>Glyco3D: A Suite of Interlinked Databases of 3D Structures of Complex Carbohydrates, Lectins, Antibodies, and Glycosyltransferases</td>
      <td>Pérez, Serge; Sarkar, Anita; Rivet, Alain; Drouillard, Sophie; Breton, Christelle; Imberty, Anne</td>
      <td>2016</td>
      <td>2016-09-08</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121943" target="_blank">PMC7121943</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-4-431-56454-6_7" target="_blank">10.1007/978-4-431-56454-6_7</a></td>
      <td>1c515h51</td>
      <td>0.438242</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wj7yonjw">Gardner_2008</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a></td>
      <td><a href="Topic_41.html#utfeiviy">McDowell_1988</a>, <a href="Topic_16.html#jatetuim">Satoh_2015</a>, <a href="Topic_41.html#3mujx7j4">Cao_L_2018</a>, <a href="Topic_16.html#cijyfa4j">Tong_2009</a></td>
    </tr>
    <tr>
      <th id="maggme5z";>321</th>
      <td>Cai_Y_2013</td>
      <td>Introduction to 3D Immersive and Interactive Learning</td>
      <td>Cai, Yiyu; Tay, Chor Ter; Ngo, Boon Keong</td>
      <td>2013</td>
      <td>2013-02-12</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122101" target="_blank">PMC7122101</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-981-4021-90-6_1" target="_blank">10.1007/978-981-4021-90-6_1</a></td>
      <td>maggme5z</td>
      <td>0.437545</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_01.html#f3lr0lwj">UNKNOWN_2020</a>, <a href="Topic_13.html#4lou2mb6">Kapoor_2020</a></td>
      <td><a href="Topic_28.html#0k4tsgus">Pearson_2006</a>, <a href="Topic_24.html#rds8o5z4">Grant_2016</a>, <a href="Topic_24.html#ic3l0gi8">Chen_2008</a></td>
    </tr>
    <tr>
      <th id="vvlsqy36";>322</th>
      <td>Peters_2007</td>
      <td>Integrating epitope data into the emerging web of biomedical knowledge resources</td>
      <td>Peters, Bjoern; Sette, Alessandro</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097317" target="_blank">PMC7097317</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17479127.0" target="_blank">17479127.0</a></td>
      <td><a href="https://doi.org/10.1038/nri2092" target="_blank">10.1038/nri2092</a></td>
      <td>vvlsqy36</td>
      <td>0.437545</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_15.html#hfa9w2dg">Jin_J_2018</a>, <a href="Topic_69.html#p37j3olh">Khan_2006</a>, <a href="Topic_69.html#fmumym1x">Lon_J_2020</a>, <a href="Topic_69.html#b6r9zdwa">Lollier_2011</a></td>
    </tr>
    <tr>
      <th id="yjrs7c38";>323</th>
      <td>De_Ferrari_2014</td>
      <td>From sequence to enzyme mechanism using multi-label machine learning</td>
      <td>De Ferrari, Luna; Mitchell, John BO</td>
      <td>2014</td>
      <td>2014-05-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4229970" target="_blank">PMC4229970</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24885296.0" target="_blank">24885296.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-150" target="_blank">10.1186/1471-2105-15-150</a></td>
      <td>yjrs7c38</td>
      <td>0.436732</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_13.html#4lou2mb6">Kapoor_2020</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_21.html#4gjo72hx">Tremblay-Létourneau_2011</a></td>
    </tr>
    <tr>
      <th id="wsgb5s6z";>324</th>
      <td>Quintana_1998</td>
      <td>Optimization of phosphorus localization by EFTEM of nucleic acid containing structures</td>
      <td>Quintana, C.; Marco, S.; Bonnet, N.; Risco, C.; Gutiérrez, M.L.; Guerrero, A.; Carrascosa, J.L.</td>
      <td>1998</td>
      <td>1998-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7135716" target="_blank">PMC7135716</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/9744088.0" target="_blank">9744088.0</a></td>
      <td><a href="https://doi.org/10.1016/s0968-4328(98)00011-0" target="_blank">10.1016/s0968-4328(98)00011-0</a></td>
      <td>wsgb5s6z</td>
      <td>0.434236</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#43igjn6x">Kylberg_2009</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#anp238gu">Effantin_2016</a>, <a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_20.html#k7qzzixm">Dea_S_1989</a></td>
    </tr>
    <tr>
      <th id="641pmd1d";>325</th>
      <td>Zhang_2017</td>
      <td>Viral Phylogenomics Using an Alignment-Free Method: A Three-Step Approach to Determine Optimal Length of k-mer</td>
      <td>Zhang, Qian; Jun, Se-Ran; Leuze, Michael; Ussery, David; Nookaew, Intawat</td>
      <td>2017</td>
      <td>2017-01-19</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5244389" target="_blank">PMC5244389</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28102365.0" target="_blank">28102365.0</a></td>
      <td><a href="https://doi.org/10.1038/srep40712" target="_blank">10.1038/srep40712</a></td>
      <td>641pmd1d</td>
      <td>0.433852</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_45.html#1a9u53za">Shabman_2016</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_21.html#5rovh1xf">Godeny_1990</a>, <a href="Topic_21.html#x2w73spd">Goldberg_1991</a></td>
    </tr>
    <tr>
      <th id="cetdqgff";>326</th>
      <td>Randhawa_2020</td>
      <td>Machine learning using intrinsic genomic signatures for rapid classification of novel pathogens: COVID-19 case study</td>
      <td>Gurjit S. Randhawa; Maximillian P.M. Soltysiak; Hadi El Roz; Camila P.E. de Souza; Kathleen A. Hill; Lila Kari</td>
      <td>2020</td>
      <td>2020-02-04</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.03.932350" target="_blank">10.1101/2020.02.03.932350</a></td>
      <td>cetdqgff</td>
      <td>0.432312</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a></td>
      <td><a href="Topic_27.html#zrbesm5b">Rockx_2020</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_20.html#7ztlskf2">Galmès_2012</a></td>
    </tr>
    <tr>
      <th id="g22ggyxb";>327</th>
      <td>Weissenbacher_2017</td>
      <td>Extracting geographic locations from the literature for virus phylogeography using supervised and distant supervision methods</td>
      <td>Weissenbacher, Davy; Sarker, Abeed; Tahsin, Tasnia; Scotch, Matthew; Gonzalez, Graciela</td>
      <td>2017</td>
      <td>2017-07-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5543364" target="_blank">PMC5543364</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28815119.0" target="_blank">28815119.0</a></td>
      <td></td>
      <td>g22ggyxb</td>
      <td>0.431621</td>
      <td><a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_06.html#wich35l7">Velsko_2014</a>, <a href="Topic_17.html#y62ha34a">Dellicour_2016</a>, <a href="Topic_06.html#l0uya9dh">Lemey_2009</a>, <a href="Topic_17.html#u2t1x89m">Chan_2013</a></td>
    </tr>
    <tr>
      <th id="058r9486";>328</th>
      <td>Morfopoulou_2014</td>
      <td>Bayesian mixture analysis for metagenomic community profiling.</td>
      <td>Sofia Morfopoulou; Vincent Plagnol</td>
      <td>2014</td>
      <td>2014-07-25</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/007476" target="_blank">10.1101/007476</a></td>
      <td>058r9486</td>
      <td>0.430687</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#z29xsj1o">Morfopoulou_2015</a>, <a href="Topic_14.html#xo1cyzrg">Giles_2015</a>, <a href="Topic_14.html#p2vuztsb">Miller_2013</a>, <a href="Topic_14.html#xlqdn0c7">Baldwin_2014</a></td>
    </tr>
    <tr>
      <th id="p8puu1jk";>329</th>
      <td>Li_M_2015</td>
      <td>Predicting RNA Secondary Structures: One-grammar-fits-all Solution</td>
      <td>Li, Menglu; Cheng, Micheal; Ye, Yongtao; Hon, Wk; Ting, Hf; Lam, Tw; Tang, Cy; Wong, Thomas; Yiu, Sm</td>
      <td>2015</td>
      <td>2015-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121278" target="_blank">PMC7121278</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-19048-8_18" target="_blank">10.1007/978-3-319-19048-8_18</a></td>
      <td>p8puu1jk</td>
      <td>0.430612</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_61.html#a5htfx76">Dost_2006</a></td>
      <td><a href="Topic_61.html#213ly4xo">Fricke_2019</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a>, <a href="Topic_61.html#1lojd0xa">Gao_J_2010</a></td>
    </tr>
    <tr>
      <th id="hjo78o1q";>330</th>
      <td>Shen_2006</td>
      <td>Probabilistic analysis of the frequencies of amino acid pairs within characterized protein sequences</td>
      <td>Shen, Shiyi; Kai, Bo; Ruan, Jishou; Torin Huzil, J.; Carpenter, Eric; Tuszynski, Jack A.</td>
      <td>2006</td>
      <td>2006-10-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127678" target="_blank">PMC7127678</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.physa.2006.03.004" target="_blank">10.1016/j.physa.2006.03.004</a></td>
      <td>hjo78o1q</td>
      <td>0.430152</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_02.html#ajz4ef44">Nandy_2009</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a></td>
    </tr>
    <tr>
      <th id="48ym7eti";>331</th>
      <td>Pickett_2011</td>
      <td>ViPR: an open bioinformatics database and analysis resource for virology research</td>
      <td>Pickett, Brett E.; Sadat, Eva L.; Zhang, Yun; Noronha, Jyothi M.; Squires, R. Burke; Hunt, Victoria; Liu, Mengya; Kumar, Sanjeev; Zaremba, Sam; Gu, Zhiping; Zhou, Liwei; Larson, Christopher N.; Dietrich, Jonathan; Klem, Edward B.; Scheuermann, Richard H.</td>
      <td>2011</td>
      <td>2011-10-17</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245011" target="_blank">PMC3245011</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22006842.0" target="_blank">22006842.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkr859" target="_blank">10.1093/nar/gkr859</a></td>
      <td>48ym7eti</td>
      <td>0.429660</td>
      <td><a href="Topic_05.html#adzvn69b">Garcia–Moliner_2013</a>, <a href="Topic_21.html#59emvigh">Ryu_W_2017</a>, <a href="Topic_05.html#d9zmdm23">Crowe_2014</a>, <a href="Topic_66.html#uee3lqrx">Davaro_2013</a></td>
      <td><a href="Topic_34.html#e0wmnfyn">McGuire_2012</a>, <a href="Topic_17.html#facjsy92">Pickett_2012</a>, <a href="Topic_69.html#ft1fy6ly">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="gytg4iku";>332</th>
      <td>Vila_2018</td>
      <td>(13)C Chemical Shifts in Proteins: A Rich Source of Encoded Structural Information</td>
      <td>Vila, Jorge A.; Arnautova, Yelena A.</td>
      <td>2018</td>
      <td>2018-07-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123919" target="_blank">PMC7123919</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-95843-9_20" target="_blank">10.1007/978-3-319-95843-9_20</a></td>
      <td>gytg4iku</td>
      <td>0.429607</td>
      <td><a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_17.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#jz2mrn4t">Vila_2014</a>, <a href="Topic_16.html#3398ymw9">Lee_G_2012</a>, <a href="Topic_16.html#g0h0itzs">Buller_2012</a></td>
    </tr>
    <tr>
      <th id="clagd6rh";>333</th>
      <td>Sen_D_2018</td>
      <td>Graphical representation methods: How well do they discriminate between homologous gene sequences?</td>
      <td>Sen, Dwaipayan; Roy, Proyasha; Nandy, Ashesh; Basak, Subhash C.; Das, Sukhen</td>
      <td>2018</td>
      <td>2018-09-24</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7116999" target="_blank">PMC7116999</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.chemphys.2018.07.031" target="_blank">10.1016/j.chemphys.2018.07.031</a></td>
      <td>clagd6rh</td>
      <td>0.428772</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
    </tr>
    <tr>
      <th id="flznxurb";>334</th>
      <td>Coutinho_2015</td>
      <td>Homology-Independent Metrics for Comparative Genomics</td>
      <td>Coutinho, Tarcisio José Domingos; Franco, Glória Regina; Lobo, Francisco Pereira</td>
      <td>2015</td>
      <td>2015-05-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4446528" target="_blank">PMC4446528</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26029354.0" target="_blank">26029354.0</a></td>
      <td><a href="https://doi.org/10.1016/j.csbj.2015.04.005" target="_blank">10.1016/j.csbj.2015.04.005</a></td>
      <td>flznxurb</td>
      <td>0.427660</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#tiyjmih7">Francis_2016</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_37.html#eri1nk2i">Narra_2008</a></td>
    </tr>
    <tr>
      <th id="f9w310tp";>335</th>
      <td>Ajayi_2019</td>
      <td>Forecasting herd-level porcine epidemic diarrhea (PED) frequency trends in Ontario (Canada)</td>
      <td>Ajayi, Toluwalope; Dara, Rozita; Poljak, Zvonimir</td>
      <td>2019</td>
      <td>2019-03-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125872" target="_blank">PMC7125872</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30771890.0" target="_blank">30771890.0</a></td>
      <td><a href="https://doi.org/10.1016/j.prevetmed.2019.01.005" target="_blank">10.1016/j.prevetmed.2019.01.005</a></td>
      <td>f9w310tp</td>
      <td>0.427567</td>
      <td><a href="Topic_06.html#h5zfmhqj">batista_2020</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_06.html#63wqn2fg">Höhle_2007</a>, <a href="Topic_06.html#h1i29kcq">Li_C_2018</a></td>
      <td><a href="Topic_44.html#up1q3k6q">Dortmans_2018</a>, <a href="Topic_44.html#k0mdkews">YAMANE_2016</a>, <a href="Topic_44.html#n4ezxnjc">Bertasio_2016</a>, <a href="Topic_44.html#g11vfemb">Lee_C_2015</a></td>
    </tr>
    <tr>
      <th id="a564l6vs";>336</th>
      <td>UNKNOWN_2016</td>
      <td>Database resources of the National Center for Biotechnology Information</td>
      <td>None</td>
      <td>2016</td>
      <td>2016-01-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702911" target="_blank">PMC4702911</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26615191.0" target="_blank">26615191.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkv1290" target="_blank">10.1093/nar/gkv1290</a></td>
      <td>a564l6vs</td>
      <td>0.427514</td>
      <td><a href="Topic_66.html#5386g91w">Kradin_2013</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_05.html#adzvn69b">Garcia–Moliner_2013</a>, <a href="Topic_12.html#g8w2kpuw">UNKNOWN_2013</a></td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#t94aufs3">Aurrecoechea_2017</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
    </tr>
    <tr>
      <th id="qtrquuvv";>337</th>
      <td>Wu_T_2020</td>
      <td>Open-source analytics tools for studying the COVID-19 coronavirus outbreak</td>
      <td>Tianzhi Wu; Xijin Ge; Guangchuang Yu; Erqiang Hu</td>
      <td>2020</td>
      <td>2020-02-27</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.25.20027433" target="_blank">10.1101/2020.02.25.20027433</a></td>
      <td>qtrquuvv</td>
      <td>0.426008</td>
      <td><a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_17.html#71tnv4lp">Santillana_2014</a></td>
      <td><a href="Topic_04.html#yiy2y2x3">de_Sousa_2014</a>, <a href="Topic_06.html#96wkqutc">Hilton_2020</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a></td>
    </tr>
    <tr>
      <th id="8cixsjrf";>338</th>
      <td>Yang_2007</td>
      <td>IDBD: Infectious Disease Biomarker Database</td>
      <td>Yang, In Seok; Ryu, Chunsun; Cho, Ki Joon; Kim, Jin Kwang; Ong, Swee Hoe; Mitchell, Wayne P.; Kim, Bong Su; Oh, Hee-Bok; Kim, Kyung Hyun</td>
      <td>2007</td>
      <td>2007-11-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238845" target="_blank">PMC2238845</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17982173.0" target="_blank">17982173.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm925" target="_blank">10.1093/nar/gkm925</a></td>
      <td>8cixsjrf</td>
      <td>0.425967</td>
      <td><a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_01.html#w77cmljm">Sharon_2010</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_09.html#chxi7ph5">Mejias_2014</a>, <a href="Topic_31.html#urlvaswn">Zav'yalov_2019</a>, <a href="Topic_09.html#k8u2tidd">Oved_2015</a></td>
    </tr>
    <tr>
      <th id="f3nbz3o1";>339</th>
      <td>Chen_2003</td>
      <td>ZCURVE_CoV: a new system to recognize protein coding genes in coronavirus genomes, and its applications in analyzing SARS-CoV genomes</td>
      <td>Chen, Ling-Ling; Ou, Hong-Yu; Zhang, Ren; Zhang, Chun-Ting</td>
      <td>2003</td>
      <td>2003-07-25</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134609" target="_blank">PMC7134609</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12859968.0" target="_blank">12859968.0</a></td>
      <td><a href="https://doi.org/10.1016/s0006-291x(03)01192-6" target="_blank">10.1016/s0006-291x(03)01192-6</a></td>
      <td>f3nbz3o1</td>
      <td>0.425037</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_21.html#i738t9lf">Xu_J_2016</a>, <a href="Topic_17.html#wj7yonjw">Gardner_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_21.html#itsr18vb">Kumar_2007</a>, <a href="Topic_37.html#1zoizzdp">Wang_2009</a></td>
    </tr>
    <tr>
      <th id="55ge7kmr";>340</th>
      <td>Routh_2013</td>
      <td>Discovery of functional genomic motifs in viruses with ViReMa–a Virus Recombination Mapper–for analysis of next-generation sequencing data</td>
      <td>Routh, Andrew; Johnson, John E.</td>
      <td>2013</td>
      <td>2013-10-09</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3902915" target="_blank">PMC3902915</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24137010.0" target="_blank">24137010.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkt916" target="_blank">10.1093/nar/gkt916</a></td>
      <td>55ge7kmr</td>
      <td>0.424571</td>
      <td><a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#ojy6s49w">Warren_2007</a></td>
      <td><a href="Topic_21.html#ks3nryfh">Routh_2012</a></td>
    </tr>
    <tr>
      <th id="91fhb66m";>341</th>
      <td>Hashemian_2010</td>
      <td>Advanced Querying Features for Disease Surveillance Systems</td>
      <td>Hashemian, Mohammad R.</td>
      <td>2010</td>
      <td>2010-04-09</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3615752" target="_blank">PMC3615752</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23569575.0" target="_blank">23569575.0</a></td>
      <td><a href="https://doi.org/10.5210/ojphi.v2i1.2847" target="_blank">10.5210/ojphi.v2i1.2847</a></td>
      <td>91fhb66m</td>
      <td>0.424288</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_39.html#1ecdj314">Glennon_2020</a>, <a href="Topic_04.html#j2ug18id">May_L_2009</a>, <a href="Topic_09.html#vkjz1orh">Chapman_2004</a></td>
    </tr>
    <tr>
      <th id="ovuofl9q";>342</th>
      <td>Chen_2018</td>
      <td>Probability Sampling by Connecting Space with Households Using GIS/GPS Technologies</td>
      <td>Chen, Xinguang; Hu, Hui; Xu, Xiaohui; Gong, Jie; Yan, Yaqiong; Li, Fang</td>
      <td>2018</td>
      <td>2018-01-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7107540" target="_blank">PMC7107540</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1093/jssam/smx032" target="_blank">10.1093/jssam/smx032</a></td>
      <td>ovuofl9q</td>
      <td>0.424277</td>
      <td><a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_39.html#01tncjq0">STRESMAN_2014</a>, <a href="Topic_06.html#fhwgdfry">Kiti_2019</a>, <a href="Topic_32.html#gvhjwqe7">Nöremark_2013</a>, <a href="Topic_02.html#22s9umrj">Lai_P_2009</a></td>
    </tr>
    <tr>
      <th id="o4y8ci0n";>343</th>
      <td>Mock_2019</td>
      <td>VIDHOP, viral host prediction with Deep Learning</td>
      <td>Florian Mock; Adrian Viehweger; Emanuel Barth; Manja Marz</td>
      <td>2019</td>
      <td>2019-03-12</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/575571" target="_blank">10.1101/575571</a></td>
      <td>o4y8ci0n</td>
      <td>0.422909</td>
      <td><a href="Topic_30.html#tnvlafa2">Lu_P_2016</a>, <a href="Topic_30.html#k9skma4k">Karlsson_2012</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_10.html#y12cq60a">Grewelle_2020</a>, <a href="Topic_10.html#sab90rov">Lumby_2018</a></td>
      <td><a href="Topic_17.html#c2lljdi7">Lopez-Rincon_2020</a>, <a href="Topic_36.html#tulne0v3">Rabaa_2015</a>, <a href="Topic_18.html#wgzpd6te">Wu_H_2016</a>, <a href="Topic_40.html#9ib77yws">Xie_X_2015</a></td>
    </tr>
    <tr>
      <th id="bqpsihm6";>344</th>
      <td>González-Díaz_2009</td>
      <td>Generalized lattice graphs for 2D-visualization of biological information</td>
      <td>González-Díaz, H.; Pérez-Montoto, L.G.; Duardo-Sanchez, A.; Paniagua, E.; Vázquez-Prieto, S.; Vilas, R.; Dea-Ayuela, M.A.; Bolas-Fernández, F.; Munteanu, C.R.; Dorado, J.; Costas, J.; Ubeira, F.M.</td>
      <td>2009</td>
      <td>2009-11-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094121" target="_blank">PMC7094121</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19646452.0" target="_blank">19646452.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2009.07.029" target="_blank">10.1016/j.jtbi.2009.07.029</a></td>
      <td>bqpsihm6</td>
      <td>0.422850</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_17.html#g4k6pc82">Munteanu_2008</a>, <a href="Topic_07.html#m8f5fsmc">Qi_Z_2011</a>, <a href="Topic_16.html#twmrilk8">Liu_X_2009</a>, <a href="Topic_17.html#gjild96e">Song_2005</a></td>
    </tr>
    <tr>
      <th id="x5bucsrd";>345</th>
      <td>Ueno_2014</td>
      <td>ELM: enhanced lowest common ancestor based method for detecting a pathogenic virus from a large sequence dataset</td>
      <td>Ueno, Keisuke; Ishii, Akihiro; Ito, Kimihito</td>
      <td>2014</td>
      <td>2014-07-28</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4124145" target="_blank">PMC4124145</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25069839.0" target="_blank">25069839.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-254" target="_blank">10.1186/1471-2105-15-254</a></td>
      <td>x5bucsrd</td>
      <td>0.421462</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_45.html#gccmkxly">Phan_2018</a>, <a href="Topic_07.html#zm38v6do">Kim_W_2016</a>, <a href="Topic_29.html#k4ra1job">Yadav_2013</a>, <a href="Topic_13.html#k0f9fser">Shi_W_2017</a></td>
    </tr>
    <tr>
      <th id="55lkfivh";>346</th>
      <td>Cho_M_2019</td>
      <td>Prediction of cross-species infection propensities of viruses with receptor similarity</td>
      <td>Cho, Myeongji; Son, Hyeon Seok</td>
      <td>2019</td>
      <td>2019-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7106226" target="_blank">PMC7106226</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31026604.0" target="_blank">31026604.0</a></td>
      <td><a href="https://doi.org/10.1016/j.meegid.2019.04.016" target="_blank">10.1016/j.meegid.2019.04.016</a></td>
      <td>55lkfivh</td>
      <td>0.420413</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#7qc1lqyp">Cheng_2012</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_36.html#e0plqpgb">Brierley_2019</a>, <a href="Topic_02.html#aj352do0">Tibayrenc_2015</a>, <a href="Topic_36.html#wa5il5go">Brierley_2019</a></td>
    </tr>
    <tr>
      <th id="ueglfhux";>347</th>
      <td>Dube_2008</td>
      <td>Mathematical Analysis of Copy Number Variation in a DNA Sample Using Digital PCR on a Nanofluidic Device</td>
      <td>Dube, Simant; Qin, Jian; Ramakrishnan, Ramesh</td>
      <td>2008</td>
      <td>2008-08-06</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2483940" target="_blank">PMC2483940</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18682853.0" target="_blank">18682853.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0002876" target="_blank">10.1371/journal.pone.0002876</a></td>
      <td>ueglfhux</td>
      <td>0.420097</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_29.html#prsvv6l9">Qin_J_2008</a>, <a href="Topic_29.html#2dwjb964">Wu_L_2010</a>, <a href="Topic_14.html#xo1cyzrg">Giles_2015</a></td>
    </tr>
    <tr>
      <th id="lwla5ugt";>348</th>
      <td>Brodie_2004</td>
      <td>Base-By-Base: Single nucleotide-level analysis of whole viral genome alignments</td>
      <td>Brodie, Ryan; Smith, Alex J; Roper, Rachel L; Tcherepanov, Vasily; Upton, Chris</td>
      <td>2004</td>
      <td>2004-07-14</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC481056" target="_blank">PMC481056</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15253776.0" target="_blank">15253776.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-5-96" target="_blank">10.1186/1471-2105-5-96</a></td>
      <td>lwla5ugt</td>
      <td>0.420032</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a></td>
      <td><a href="Topic_17.html#0lbxvudt">Wang_2010</a>, <a href="Topic_21.html#3g3apbon">Lauber_2013</a></td>
    </tr>
    <tr>
      <th id="om82nx6q";>349</th>
      <td>Matzke_2011</td>
      <td>Improved quality control processing of peptide-centric LC-MS proteomics data</td>
      <td>Matzke, Melissa M.; Waters, Katrina M.; Metz, Thomas O.; Jacobs, Jon M.; Sims, Amy C.; Baric, Ralph S.; Pounds, Joel G.; Webb-Robertson, Bobbie-Jo M.</td>
      <td>2011</td>
      <td>2011-10-15</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3187650" target="_blank">PMC3187650</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21852304.0" target="_blank">21852304.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btr479" target="_blank">10.1093/bioinformatics/btr479</a></td>
      <td>om82nx6q</td>
      <td>0.419884</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_69.html#udqu5jub">Ng_E_2013</a>, <a href="Topic_58.html#ngbcdkz3">Chen_2005</a>, <a href="Topic_69.html#8pvli8ai">Li_J_2012</a></td>
    </tr>
    <tr>
      <th id="3uz42vx8";>350</th>
      <td>Soontornpipit_2016</td>
      <td>Development of the Electronic Surveillance Monitoring System on Web Applications</td>
      <td>Soontornpipit, Pichitpong; Viwatwongkasem, Chukiat; Taratep, Chanvit; Teerawat, Watcharawan; Vanitchatchavan, Prachuab</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2016.05.110" target="_blank">10.1016/j.procs.2016.05.110</a></td>
      <td>3uz42vx8</td>
      <td>0.418985</td>
      <td><a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_04.html#zoai4kvn">Yang_2011</a>, <a href="Topic_17.html#jg13scgo">Tsui_2003</a>, <a href="Topic_22.html#8v0k41sn">Rajatonirina_2014</a></td>
    </tr>
    <tr>
      <th id="xjmicuki";>351</th>
      <td>Sun_J_2010</td>
      <td>Whole-genome based Archaea phylogeny and taxonomy: A composition vector approach</td>
      <td>Sun, JianDong; Xu, Zhao; Hao, BaiLin</td>
      <td>2010</td>
      <td>2010-08-11</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089326" target="_blank">PMC7089326</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11434-010-3008-8" target="_blank">10.1007/s11434-010-3008-8</a></td>
      <td>xjmicuki</td>
      <td>0.418985</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_50.html#eifztaov">Selvin_2019</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a>, <a href="Topic_61.html#kg53pbyw">Li_S_2014</a>, <a href="Topic_07.html#44h6rdwg">Zhou_2017</a></td>
    </tr>
    <tr>
      <th id="8y8ho2x5";>352</th>
      <td>Bekaert_2009</td>
      <td>Recode-2: new design, new search tools, and many more genes</td>
      <td>Bekaert, Michaël; Firth, Andrew E.; Zhang, Yan; Gladyshev, Vadim N.; Atkins, John F.; Baranov, Pavel V.</td>
      <td>2009</td>
      <td>2009-09-25</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808893" target="_blank">PMC2808893</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19783826.0" target="_blank">19783826.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkp788" target="_blank">10.1093/nar/gkp788</a></td>
      <td>8y8ho2x5</td>
      <td>0.417524</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_61.html#69gftii4">Cobucci-Ponzano_2006</a>, <a href="Topic_61.html#6z7rt5dd">Wang_2006</a></td>
    </tr>
    <tr>
      <th id="b2m3p3uh";>353</th>
      <td>Chen_2010</td>
      <td>Forecasting tourist arrivals by using the adaptive network-based fuzzy inference system</td>
      <td>Chen, Miao-Sheng; Ying, Li-Chih; Pan, Mei-Chiu</td>
      <td>2010</td>
      <td>2010-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126321" target="_blank">PMC7126321</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2009.06.032" target="_blank">10.1016/j.eswa.2009.06.032</a></td>
      <td>b2m3p3uh</td>
      <td>0.416727</td>
      <td><a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_71.html#vioymdp0">Xie_G_2020</a>, <a href="Topic_71.html#l21a1hsg">Balli_2019</a>, <a href="Topic_71.html#1pkuttkb">Jang_2008</a>, <a href="Topic_71.html#g247rjh9">Guirao_2016</a></td>
    </tr>
    <tr>
      <th id="j22q89l9";>354</th>
      <td>Vanderwende_2007</td>
      <td>Beyond SumBasic: Task-focused summarization with sentence simplification and lexical expansion</td>
      <td>Vanderwende, Lucy; Suzuki, Hisami; Brockett, Chris; Nenkova, Ani</td>
      <td>2007</td>
      <td>2007-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115785" target="_blank">PMC7115785</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ipm.2007.01.023" target="_blank">10.1016/j.ipm.2007.01.023</a></td>
      <td>j22q89l9</td>
      <td>0.416719</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_02.html#w45h121w">Tikhomirov_2018</a>, <a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_17.html#ak33vta6">Du_X_2019</a></td>
    </tr>
    <tr>
      <th id="k1lg8c7q";>355</th>
      <td>Jin_C_2020</td>
      <td>Development and Evaluation of an AI System for COVID-19</td>
      <td>Cheng Jin; Weixiang Chen; Yukun Cao; Zhanwei Xu; Xin Zhang; Lei Deng; Chuansheng Zheng; Jie Zhou; Heshui Shi; Jianjiang Feng</td>
      <td>2020</td>
      <td>2020-03-23</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.20.20039834" target="_blank">10.1101/2020.03.20.20039834</a></td>
      <td>k1lg8c7q</td>
      <td>0.416709</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_17.html#ll4rxd9p">Zheng_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="r7tg0s45";>356</th>
      <td>John_2019</td>
      <td>Shiny Framework Based Visualization and Analytics Tool for Middle East Respiratory Syndrome</td>
      <td>John, Maya; Shaiba, Hadil</td>
      <td>2019</td>
      <td>2019-12-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122755" target="_blank">PMC7122755</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-030-36365-9_16" target="_blank">10.1007/978-3-030-36365-9_16</a></td>
      <td>r7tg0s45</td>
      <td>0.415975</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a></td>
      <td><a href="Topic_55.html#4hk736ev">Alfaraj_2018</a>, <a href="Topic_55.html#vyoodpon">Aleanizy_2017</a>, <a href="Topic_55.html#41isp0wj">Al-Tawfiq_2016</a></td>
    </tr>
    <tr>
      <th id="7zgmiwbd";>357</th>
      <td>Lam_B_2016</td>
      <td>Jump point detection using empirical mode decomposition</td>
      <td>Lam, Benson S.Y.; Yu, Carisa K.W.; Choy, Siu-Kai; Leung, Jacky K.T.</td>
      <td>2016</td>
      <td>2016-12-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115718" target="_blank">PMC7115718</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.landusepol.2016.07.006" target="_blank">10.1016/j.landusepol.2016.07.006</a></td>
      <td>7zgmiwbd</td>
      <td>0.415651</td>
      <td><a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a></td>
      <td><a href="Topic_71.html#4ezebg45">Hui_E_2016</a>, <a href="Topic_71.html#vioymdp0">Xie_G_2020</a>, <a href="Topic_71.html#ak6zgyh1">Lin_T_2011</a>, <a href="Topic_71.html#tjnt01ld">Argyroudis_2019</a></td>
    </tr>
    <tr>
      <th id="jy22na69";>358</th>
      <td>Vinga_2013</td>
      <td>Information theory applications for biological sequence analysis</td>
      <td>Vinga, Susana</td>
      <td>2013</td>
      <td>2013-09-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109941" target="_blank">PMC7109941</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24058049.0" target="_blank">24058049.0</a></td>
      <td><a href="https://doi.org/10.1093/bib/bbt068" target="_blank">10.1093/bib/bbt068</a></td>
      <td>jy22na69</td>
      <td>0.415298</td>
      <td><a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a>, <a href="Topic_17.html#flznxurb">Coutinho_2015</a>, <a href="Topic_17.html#8k7d1riv">Muppirala_2011</a>, <a href="Topic_61.html#1u96c10x">Gupta_2019</a></td>
    </tr>
    <tr>
      <th id="81wig9xh";>359</th>
      <td>Chan_2010</td>
      <td>Surveillance and Epidemiology of Infectious Diseases using Spatial and Temporal Lustering Methods</td>
      <td>Chan, Ta-Chien; King, Chwan-Chuen</td>
      <td>2010</td>
      <td>2010-07-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120538" target="_blank">PMC7120538</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-1-4419-6892-0_10" target="_blank">10.1007/978-1-4419-6892-0_10</a></td>
      <td>81wig9xh</td>
      <td>0.413404</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_39.html#sxwwi4zq">Cao_Z_2010</a>, <a href="Topic_06.html#q5cmwzr8">Wen_T_2007</a>, <a href="Topic_06.html#01yc7lzk">Neri_2014</a>, <a href="Topic_06.html#28mhhwol">Clements_2009</a></td>
    </tr>
    <tr>
      <th id="sqyhzxty";>360</th>
      <td>Song_2020</td>
      <td>Prospect and application of Internet of Things technology for prevention of SARIs</td>
      <td>Song, Yuanlin; Jiang, Jinjun; Wang, Xun; Yang, Dawei; Bai, Chunxue</td>
      <td>2020</td>
      <td>2020-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ceh.2020.02.001" target="_blank">10.1016/j.ceh.2020.02.001</a></td>
      <td>sqyhzxty</td>
      <td>0.411871</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a></td>
      <td><a href="Topic_17.html#3l2dbnjb">Sood_2017</a>, <a href="Topic_17.html#2se42vec">Sareen_2016</a>, <a href="Topic_17.html#ax09x6gk">Wu_J_2016</a>, <a href="Topic_14.html#ityhcak7">Zhu_H_2020</a></td>
    </tr>
    <tr>
      <th id="9rx4jlw8";>361</th>
      <td>Kim_K_2007</td>
      <td>Selecting Genotyping Oligo Probes Via Logical Analysis of Data</td>
      <td>Kim, Kwangsoo; Ryoo, Hong Seo</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122177" target="_blank">PMC7122177</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72665-4_8" target="_blank">10.1007/978-3-540-72665-4_8</a></td>
      <td>9rx4jlw8</td>
      <td>0.411844</td>
      <td><a href="Topic_17.html#nfxcqsc2">Kim_K_2007</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#nfxcqsc2">Kim_K_2007</a>, <a href="Topic_60.html#kt9pmfdr">Kutyavin_2006</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a>, <a href="Topic_34.html#hlvfg1b2">Tan_J_2009</a></td>
    </tr>
    <tr>
      <th id="rx0os4ke";>362</th>
      <td>Wu_G_2007</td>
      <td>Prediction of mutations engineered by randomness in H5N1 hemagglutinins of influenza A virus</td>
      <td>Wu, G.; Yan, S.</td>
      <td>2007</td>
      <td>2007-11-02</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088403" target="_blank">PMC7088403</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17973072.0" target="_blank">17973072.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0602-4" target="_blank">10.1007/s00726-007-0602-4</a></td>
      <td>rx0os4ke</td>
      <td>0.411266</td>
      <td><a href="Topic_61.html#wv0f4hp4">Wu_G_2005</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#7qc1lqyp">Cheng_2012</a></td>
      <td><a href="Topic_17.html#wocuomb1">Wu_G_2007</a>, <a href="Topic_02.html#x0ygzz4u">Wu_G_2006</a>, <a href="Topic_30.html#kvfffe0b">Gutiérrez_2013</a>, <a href="Topic_31.html#l3qp0n9f">Deng_2019</a></td>
    </tr>
    <tr>
      <th id="04t18oqp";>363</th>
      <td>Kingsley_2016</td>
      <td>Ranking Protein-Protein Docking Results using Steered Molecular Dynamics and Potential of Mean Force Calculations</td>
      <td>Kingsley, Laura J.; Esquivel-Rodríguez, Juan; Yang, Ying; Kihara, Daisuke; Lill, Markus A.</td>
      <td>2016</td>
      <td>2016-05-27</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5015890" target="_blank">PMC5015890</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27232548.0" target="_blank">27232548.0</a></td>
      <td><a href="https://doi.org/10.1002/jcc.24412" target="_blank">10.1002/jcc.24412</a></td>
      <td>04t18oqp</td>
      <td>0.411034</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_16.html#gbnns8np">RAMAKRISHNA_2015</a>, <a href="Topic_16.html#40pe21lm">Zhang_2018</a>, <a href="Topic_18.html#mw7athol">Plewczynski_2007</a></td>
    </tr>
    <tr>
      <th id="fbl3szsu";>364</th>
      <td>Pfeiffer_2015</td>
      <td>Spatial and temporal epidemiological analysis in the Big Data era</td>
      <td>Pfeiffer, Dirk U.; Stevens, Kim B.</td>
      <td>2015</td>
      <td>2015-11-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114113" target="_blank">PMC7114113</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26092722.0" target="_blank">26092722.0</a></td>
      <td><a href="https://doi.org/10.1016/j.prevetmed.2015.05.012" target="_blank">10.1016/j.prevetmed.2015.05.012</a></td>
      <td>fbl3szsu</td>
      <td>0.410984</td>
      <td><a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#vfhlph80">Alves_2007</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_17.html#397hp1yt">Coelho_2008</a>, <a href="Topic_06.html#28mhhwol">Clements_2009</a>, <a href="Topic_13.html#ap92lgv7">dos_Santos_2018</a></td>
    </tr>
    <tr>
      <th id="7qc1lqyp";>365</th>
      <td>Cheng_2012</td>
      <td>Segmentation of DNA using simple recurrent neural network</td>
      <td>Cheng, Wei-Chen; Huang, Jau-Chi; Liou, Cheng-Yuan</td>
      <td>2012</td>
      <td>2012-02-29</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126336" target="_blank">PMC7126336</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2011.09.001" target="_blank">10.1016/j.knosys.2011.09.001</a></td>
      <td>7qc1lqyp</td>
      <td>0.409176</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a></td>
      <td><a href="Topic_21.html#x2w73spd">Goldberg_1991</a></td>
    </tr>
    <tr>
      <th id="fk8vz0g0";>366</th>
      <td>Zeng_2007</td>
      <td>Mass spectral profiling: An effective tool for quality control of herbal medicines</td>
      <td>Zeng, Zhong-Da; Liang, Yi-Zeng; Chau, Foo-Tim; Chen, Shuo; Daniel, Mok Kam-Wah; Chan, Chi-On</td>
      <td>2007</td>
      <td>2007-12-05</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7119444" target="_blank">PMC7119444</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17996528.0" target="_blank">17996528.0</a></td>
      <td><a href="https://doi.org/10.1016/j.aca.2007.09.057" target="_blank">10.1016/j.aca.2007.09.057</a></td>
      <td>fk8vz0g0</td>
      <td>0.409123</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_49.html#tjydn8k2">Lee_C_2020</a>, <a href="Topic_49.html#gtcr19qr">Fang_2019</a>, <a href="Topic_49.html#97elsy10">Nie_L_2016</a>, <a href="Topic_49.html#cr3j356z">YAO_R_2013</a></td>
    </tr>
    <tr>
      <th id="ohthcjxc";>367</th>
      <td>Yoon_2016</td>
      <td>Visualization of Data Regarding Infections Using Eye Tracking Techniques</td>
      <td>Yoon, Sunmoo; Cohen, Bevin; Cato, Kenrick D.; Liu, Jianfang; Larson, Elaine L.</td>
      <td>2016</td>
      <td>2016-04-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4939395" target="_blank">PMC4939395</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27061619.0" target="_blank">27061619.0</a></td>
      <td><a href="https://doi.org/10.1111/jnu.12204" target="_blank">10.1111/jnu.12204</a></td>
      <td>ohthcjxc</td>
      <td>0.408407</td>
      <td><a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_28.html#p5macofk">Biezen_2019</a>, <a href="Topic_28.html#6isk8jgj">Fix_G_2019</a>, <a href="Topic_28.html#0ztyvq50">Lam_K_2013</a></td>
    </tr>
    <tr>
      <th id="b265mbwp";>368</th>
      <td>Kao_H_2005</td>
      <td>A diagnostic reasoning and optimal treatment model for bacterial infections with fuzzy information</td>
      <td>Kao, Han-Ying; Li, Han-Lin</td>
      <td>2005</td>
      <td>2005-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125802" target="_blank">PMC7125802</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15639707.0" target="_blank">15639707.0</a></td>
      <td><a href="https://doi.org/10.1016/j.cmpb.2004.08.003" target="_blank">10.1016/j.cmpb.2004.08.003</a></td>
      <td>b265mbwp</td>
      <td>0.406657</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_06.html#kry1sejo">Lloyd_2009</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#mxdlii77">Arji_2019</a>, <a href="Topic_12.html#ey3sbrxx">El-Azeem_2013</a>, <a href="Topic_13.html#j9qmw4zy">UNKNOWN_1985</a></td>
    </tr>
    <tr>
      <th id="wocuomb1";>369</th>
      <td>Wu_G_2007</td>
      <td>Prediction of mutations engineered by randomness in H5N1 neuraminidases from influenza A virus</td>
      <td>Wu, G.; Yan, S.</td>
      <td>2007</td>
      <td>2007-08-28</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088166" target="_blank">PMC7088166</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17721674.0" target="_blank">17721674.0</a></td>
      <td><a href="https://doi.org/10.1007/s00726-007-0579-z" target="_blank">10.1007/s00726-007-0579-z</a></td>
      <td>wocuomb1</td>
      <td>0.406657</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_61.html#wv0f4hp4">Wu_G_2005</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#7qc1lqyp">Cheng_2012</a></td>
      <td><a href="Topic_17.html#rx0os4ke">Wu_G_2007</a>, <a href="Topic_31.html#l3qp0n9f">Deng_2019</a>, <a href="Topic_02.html#x0ygzz4u">Wu_G_2006</a>, <a href="Topic_30.html#kvfffe0b">Gutiérrez_2013</a></td>
    </tr>
    <tr>
      <th id="gl65i3s0";>370</th>
      <td>Tang_2015</td>
      <td>Inferring the hosts of coronavirus using dual statistical models based on nucleotide composition</td>
      <td>Tang, Qin; Song, Yulong; Shi, Mijuan; Cheng, Yingyin; Zhang, Wanting; Xia, Xiao-Qin</td>
      <td>2015</td>
      <td>2015-11-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4660426" target="_blank">PMC4660426</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26607834.0" target="_blank">26607834.0</a></td>
      <td><a href="https://doi.org/10.1038/srep17155" target="_blank">10.1038/srep17155</a></td>
      <td>gl65i3s0</td>
      <td>0.406657</td>
      <td><a href="Topic_07.html#d0imcmcl">Wang_2007</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#8ui0bpfd">Wang_2015</a>, <a href="Topic_30.html#tnvlafa2">Lu_P_2016</a>, <a href="Topic_21.html#59emvigh">Ryu_W_2017</a></td>
      <td><a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_45.html#gccmkxly">Phan_2018</a>, <a href="Topic_07.html#9aegg5sd">Gu_H_2020</a></td>
    </tr>
    <tr>
      <th id="q3vrzlz1";>371</th>
      <td>Weber_2017</td>
      <td>Novel algorithm for accelerated electroanatomic mapping and prediction of earliest activation of focal cardiac arrhythmias using mathematical optimization</td>
      <td>Weber, Tobias; Katus, Hugo A.; Sager, Sebastian; Scholz, Eberhard P.</td>
      <td>2017</td>
      <td>2017-06-10</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5446561" target="_blank">PMC5446561</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28279745.0" target="_blank">28279745.0</a></td>
      <td><a href="https://doi.org/10.1016/j.hrthm.2017.03.001" target="_blank">10.1016/j.hrthm.2017.03.001</a></td>
      <td>q3vrzlz1</td>
      <td>0.406489</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_17.html#fogzjrk2">Chorba_2020</a>, <a href="Topic_78.html#hr70o8r6">Alexander_1999</a>, <a href="Topic_67.html#7h33bode">Santilli_2019</a></td>
    </tr>
    <tr>
      <th id="hef1tfjx";>372</th>
      <td>Spencer_2004</td>
      <td>Do Newton’s rules of reasoning guarantee truth … must they?</td>
      <td>Spencer, Quayshawn</td>
      <td>2004</td>
      <td>2004-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134605" target="_blank">PMC7134605</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.shpsa.2004.02.001" target="_blank">10.1016/j.shpsa.2004.02.001</a></td>
      <td>hef1tfjx</td>
      <td>0.406341</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_02.html#8plec8t4">Looker_2006</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a></td>
      <td><a href="Topic_02.html#puzmxu65">Roush_2015</a>, <a href="Topic_02.html#nouieh2s">Fox_J_2003</a>, <a href="Topic_02.html#jho5as5k">Niaz_2017</a>, <a href="Topic_02.html#lsgyubdj">Siekmann_2018</a></td>
    </tr>
    <tr>
      <th id="j3jskkmw";>373</th>
      <td>Shao_2017</td>
      <td>Long-term prediction of dynamic distribution of passive contaminant in complex recirculating ventilation system</td>
      <td>Shao, Xiaoliang; Li, Xianting; Ma, Xiaojun; Zhu, Fenfei</td>
      <td>2017</td>
      <td>2017-08-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126526" target="_blank">PMC7126526</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2017.05.012" target="_blank">10.1016/j.buildenv.2017.05.012</a></td>
      <td>j3jskkmw</td>
      <td>0.404298</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_06.html#kry1sejo">Lloyd_2009</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_64.html#073t95p8">Li_X_2011</a>, <a href="Topic_64.html#kv4s5pld">Mu_D_2016</a>, <a href="Topic_64.html#1xtmwnu9">Mu_D_2017</a>, <a href="Topic_64.html#scwhwurr">Zhu_S_2010</a></td>
    </tr>
    <tr>
      <th id="uxmqlx4y";>374</th>
      <td>Yang_2019</td>
      <td>An implementation of real-time air quality and influenza-like illness data storage and processing platform</td>
      <td>Yang, Chao-Tung; Chen, Cai-Jin; Tsan, Yu-Tse; Liu, Po-Yu; Chan, Yu-Wei; Chan, Wei-Chen</td>
      <td>2019</td>
      <td>2019-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126110" target="_blank">PMC7126110</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.chb.2018.10.009" target="_blank">10.1016/j.chb.2018.10.009</a></td>
      <td>uxmqlx4y</td>
      <td>0.403725</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a></td>
      <td><a href="Topic_22.html#iwtevhmq">Abdalla_2020</a>, <a href="Topic_55.html#kaip0d1i">John_2019</a></td>
    </tr>
    <tr>
      <th id="k5u6tdkg";>375</th>
      <td>Sachdeva_2005</td>
      <td>SPAAN: a software program for prediction of adhesins and adhesin-like proteins using neural networks</td>
      <td>Sachdeva, Gaurav; Kumar, Kaushal; Jain, Preti; Ramachandran, Srinivasan</td>
      <td>2005</td>
      <td>2005-02-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7109999" target="_blank">PMC7109999</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15374866.0" target="_blank">15374866.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/bti028" target="_blank">10.1093/bioinformatics/bti028</a></td>
      <td>k5u6tdkg</td>
      <td>0.403561</td>
      <td><a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a>, <a href="Topic_10.html#tg1310br">Haasnoot_2010</a></td>
      <td><a href="Topic_41.html#b8czrq29">Ielasi_2016</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a>, <a href="Topic_69.html#jbb4qlab">Zhang_2011</a></td>
    </tr>
    <tr>
      <th id="5m269nzi";>376</th>
      <td>Lundegaard_2007</td>
      <td>Modeling the adaptive immune system: predictions and simulations</td>
      <td>Lundegaard, Claus; Lund, Ole; Keşmir, Can; Brunak, Søren; Nielsen, Morten</td>
      <td>2007</td>
      <td>2007-12-15</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110254" target="_blank">PMC7110254</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18045832.0" target="_blank">18045832.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btm471" target="_blank">10.1093/bioinformatics/btm471</a></td>
      <td>5m269nzi</td>
      <td>0.403561</td>
      <td><a href="Topic_13.html#4lou2mb6">Kapoor_2020</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_69.html#71940dh7">Kumar_2019</a>, <a href="Topic_69.html#n184795x">Zamanzadeh_2017</a></td>
    </tr>
    <tr>
      <th id="fewy8y4a";>377</th>
      <td>Wang_2018</td>
      <td>A Comprehensive In Silico Method to Study the QSTR of the Aconitine Alkaloids for Designing Novel Drugs</td>
      <td>Wang, Ming-Yang; Liang, Jing-Wei; Mohamed Olounfeh, Kamara; Sun, Qi; Zhao, Nan; Meng, Fan-Hao</td>
      <td>2018</td>
      <td>2018-09-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6225272" target="_blank">PMC6225272</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30231506.0" target="_blank">30231506.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules23092385" target="_blank">10.3390/molecules23092385</a></td>
      <td>fewy8y4a</td>
      <td>0.403412</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_47.html#y14atmnh">Su_J_2014</a>, <a href="Topic_18.html#4atavuzj">Tran_2011</a></td>
    </tr>
    <tr>
      <th id="okp8ofl0";>378</th>
      <td>Pereira_2015</td>
      <td>QSAR-Assisted Virtual Screening of Lead-Like Molecules from Marine and Microbial Natural Sources for Antitumor and Antibiotic Drug Discovery</td>
      <td>Pereira, Florbela; Latino, Diogo A. R. S.; Gaudêncio, Susana P.</td>
      <td>2015</td>
      <td>2015-03-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6272462" target="_blank">PMC6272462</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25789820.0" target="_blank">25789820.0</a></td>
      <td><a href="https://doi.org/10.3390/molecules20034848" target="_blank">10.3390/molecules20034848</a></td>
      <td>okp8ofl0</td>
      <td>0.402125</td>
      <td><a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a></td>
      <td><a href="Topic_17.html#ud3x78u3">Pereira_2014</a>, <a href="Topic_49.html#gl55zhgw">Alvin_2014</a>, <a href="Topic_18.html#bs17igjs">Zhou_2010</a></td>
    </tr>
    <tr>
      <th id="b25mi0sp";>379</th>
      <td>Zhao_2013</td>
      <td>Estimation with Right-Censored Observations Under A Semi-Markov Model</td>
      <td>Zhao, Lihui; Hu, X. Joan</td>
      <td>2013</td>
      <td>2013-03-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3713855" target="_blank">PMC3713855</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23874060.0" target="_blank">23874060.0</a></td>
      <td><a href="https://doi.org/10.1002/cjs.11176" target="_blank">10.1002/cjs.11176</a></td>
      <td>b25mi0sp</td>
      <td>0.400074</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_06.html#cok7vbmg">Soubeyrand_2007</a>, <a href="Topic_06.html#ecxcw96m">Black_2013</a>, <a href="Topic_06.html#88xk1e6h">Hirose_2007</a>, <a href="Topic_06.html#dqg8fkca">Roques_2020</a></td>
    </tr>
    <tr>
      <th id="1grzm2lq";>380</th>
      <td>Torres-Urquidy_2009</td>
      <td>Detection of Disease Outbreaks by the Use of Oral Manifestations</td>
      <td>Torres-Urquidy, M.H.; Wallstrom, G.; Schleyer, T.K.L.</td>
      <td>2009</td>
      <td>2009-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3144048" target="_blank">PMC3144048</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19131324.0" target="_blank">19131324.0</a></td>
      <td><a href="https://doi.org/10.1177/0022034508327546" target="_blank">10.1177/0022034508327546</a></td>
      <td>1grzm2lq</td>
      <td>0.400066</td>
      <td><a href="Topic_13.html#c22mytwr">Brucker_2020</a>, <a href="Topic_30.html#v4cf95qh">Walker_2010</a>, <a href="Topic_06.html#lakwurn0">Mondor_2012</a>, <a href="Topic_04.html#e0sqppeq">Guery_2014</a>, <a href="Topic_13.html#vubszdp2">Li_L_2014</a></td>
      <td><a href="Topic_09.html#89pf7c5i">Pennisi_2015</a>, <a href="Topic_09.html#r1ndy1ux">Pusterla_2013</a></td>
    </tr>
    <tr>
      <th id="cw2opick";>381</th>
      <td>Chou_2011</td>
      <td>Applications of analytical ultracentrifugation to protein size-and-shape distribution and structure-and-function analyses</td>
      <td>Chou, Chi-Yuan; Hsieh, Yi-Hui; Chang, Gu-Gang</td>
      <td>2011</td>
      <td>2011-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128498" target="_blank">PMC7128498</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21087667.0" target="_blank">21087667.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ymeth.2010.11.002" target="_blank">10.1016/j.ymeth.2010.11.002</a></td>
      <td>cw2opick</td>
      <td>0.400063</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a></td>
      <td><a href="Topic_16.html#g0h0itzs">Buller_2012</a>, <a href="Topic_16.html#y9ww3o1e">Ramírez-Sarmiento_2015</a></td>
    </tr>
    <tr>
      <th id="rxvnn7p5";>382</th>
      <td>Ugwu_2006</td>
      <td>Sustainability appraisal in infrastructure projects (SUSAIP) Part 1. Development of indicators and computational methods</td>
      <td>Ugwu, O.O.; Kumaraswamy, M.M.; Wong, A.; Ng, S.T.</td>
      <td>2006</td>
      <td>2006-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.autcon.2005.05.006" target="_blank">10.1016/j.autcon.2005.05.006</a></td>
      <td>rxvnn7p5</td>
      <td>0.400057</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#cvvyf3cb">Kelley_2011</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a></td>
      <td><a href="Topic_04.html#1b94201o">Ramachandran_2011</a>, <a href="Topic_04.html#ruc7qdg0">Pernetta_2013</a>, <a href="Topic_71.html#t727v0zt">Lee_P_2013</a>, <a href="Topic_04.html#z24ba1b6">Yip_Robin_2009</a></td>
    </tr>
    <tr>
      <th id="ff6kor0c";>383</th>
      <td>Holmes_2017</td>
      <td>Solving the master equation for Indels</td>
      <td>Holmes, Ian H.</td>
      <td>2017</td>
      <td>2017-05-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5427538" target="_blank">PMC5427538</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28494756.0" target="_blank">28494756.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-017-1665-1" target="_blank">10.1186/s12859-017-1665-1</a></td>
      <td>ff6kor0c</td>
      <td>0.397918</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a>, <a href="Topic_02.html#bkfkmtld">Redelings_2007</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_02.html#bkfkmtld">Redelings_2007</a>, <a href="Topic_17.html#zuh78iul">Pessia_2015</a>, <a href="Topic_06.html#zw5kt090">Hilton_2018</a></td>
    </tr>
    <tr>
      <th id="do0gb0g6";>384</th>
      <td>van_Dijk_2008</td>
      <td>Predicting sub-Golgi localization of type II membrane proteins</td>
      <td>van Dijk, A. D. J.; Bosch, D.; ter Braak, C. J. F.; van der Krol, A. R.; van Ham, R. C. H. J.</td>
      <td>2008</td>
      <td>2008-08-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7110242" target="_blank">PMC7110242</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18562268.0" target="_blank">18562268.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btn309" target="_blank">10.1093/bioinformatics/btn309</a></td>
      <td>do0gb0g6</td>
      <td>0.397800</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_26.html#een3sgpt">Fu_J_2000</a>, <a href="Topic_26.html#qbmcw4ck">Chen_1991</a></td>
    </tr>
    <tr>
      <th id="u4ybz8ds";>385</th>
      <td>Yu_C_2018</td>
      <td>Acral melanoma detection using a convolutional neural network for dermoscopy images</td>
      <td>Yu, Chanki; Yang, Sejung; Kim, Wonoh; Jung, Jinwoong; Chung, Kee-Yang; Lee, Sang Wook; Oh, Byungho</td>
      <td>2018</td>
      <td>2018-03-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5841780" target="_blank">PMC5841780</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29513718.0" target="_blank">29513718.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0193321" target="_blank">10.1371/journal.pone.0193321</a></td>
      <td>u4ybz8ds</td>
      <td>0.397304</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#oq64d1mb">Goradia_1988</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_09.html#hmd3quxa">Zeeneldin_2018</a>, <a href="Topic_22.html#s5w3azsd">Tzen_2007</a>, <a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="3l2dbnjb";>386</th>
      <td>Sood_2017</td>
      <td>Wearable IoT sensor based healthcare system for identifying and controlling chikungunya virus</td>
      <td>Sood, Sandeep K.; Mahajan, Isha</td>
      <td>2017</td>
      <td>2017-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114341" target="_blank">PMC7114341</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.compind.2017.05.006" target="_blank">10.1016/j.compind.2017.05.006</a></td>
      <td>3l2dbnjb</td>
      <td>0.396860</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#2se42vec">Sareen_2016</a>, <a href="Topic_14.html#ityhcak7">Zhu_H_2020</a>, <a href="Topic_04.html#ckpq0pvo">Lwin_2017</a>, <a href="Topic_17.html#c6ttke66">Sandhu_2016</a></td>
    </tr>
    <tr>
      <th id="1ef1cxc8";>387</th>
      <td>Chang_2011</td>
      <td>Genome analysis with the conditional multinomial distribution profile</td>
      <td>Chang, Guisong; Wang, Tianming</td>
      <td>2011</td>
      <td>2011-02-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094119" target="_blank">PMC7094119</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21129383.0" target="_blank">21129383.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.11.034" target="_blank">10.1016/j.jtbi.2010.11.034</a></td>
      <td>1ef1cxc8</td>
      <td>0.396664</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#lz0gy9el">Liu_L_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_07.html#z9p1ylsr">Sheikh_2020</a>, <a href="Topic_07.html#wt8zfqk0">Pavlović-Lažetić_2004</a></td>
    </tr>
    <tr>
      <th id="te049lxh";>388</th>
      <td>Jenkins_2020</td>
      <td>In silico approach to accelerate the development of mass spectrometry-based proteomics methods for detection of viral proteins: Application to COVID-19</td>
      <td>Conor Jenkins; Ben Orsburn</td>
      <td>2020</td>
      <td>2020-03-10</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.08.980383" target="_blank">10.1101/2020.03.08.980383</a></td>
      <td>te049lxh</td>
      <td>0.395902</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_23.html#pobxgffs">Jiang_2017</a>, <a href="Topic_37.html#1zoizzdp">Wang_2009</a>, <a href="Topic_16.html#o9kc3x6z">Saraswat_2020</a>, <a href="Topic_34.html#zbur2kdm">Yang_2017</a></td>
    </tr>
    <tr>
      <th id="yxd32zwk";>389</th>
      <td>Daly_2015</td>
      <td>Host Subtraction, Filtering and Assembly Validations for Novel Viral Discovery Using Next Generation Sequencing Data</td>
      <td>Daly, Gordon M.; Leggett, Richard M.; Rowe, William; Stubbs, Samuel; Wilkinson, Maxim; Ramirez-Gonzalez, Ricardo H.; Caccamo, Mario; Bernal, William; Heeney, Jonathan L.</td>
      <td>2015</td>
      <td>2015-06-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4476701" target="_blank">PMC4476701</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26098299.0" target="_blank">26098299.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0129059" target="_blank">10.1371/journal.pone.0129059</a></td>
      <td>yxd32zwk</td>
      <td>0.395902</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_14.html#16tii0ha">Pandya_2007</a></td>
      <td><a href="Topic_45.html#1a9u53za">Shabman_2016</a>, <a href="Topic_17.html#xyru2q9o">Shepard_2016</a>, <a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_07.html#og2vsgjn">Knutson_2017</a></td>
    </tr>
    <tr>
      <th id="lnch3qsq";>390</th>
      <td>Zhang_2019</td>
      <td>Identification of viruses with the potential to infect human</td>
      <td>Zheng Zhang; Zena Cai; Zhiying Tan; Congyu Lu; Gaihua Zhang; Yousong Peng</td>
      <td>2019</td>
      <td>2019-04-05</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/597963" target="_blank">10.1101/597963</a></td>
      <td>lnch3qsq</td>
      <td>0.395425</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_45.html#n8zg6rsj">Wang_2018</a>, <a href="Topic_10.html#jw4n2j6y">Smits_2014</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a></td>
      <td><a href="Topic_40.html#mwakww13">Van_Hoeven_2009</a>, <a href="Topic_30.html#1sy0vxdz">Paulson_2013</a></td>
    </tr>
    <tr>
      <th id="wd3ir3wg";>391</th>
      <td>Wang_2012</td>
      <td>VIGOR extended to annotate genomes for additional 12 different viruses</td>
      <td>Wang, Shiliang; Sundaram, Jaideep P.; Stockwell, Timothy B.</td>
      <td>2012</td>
      <td>2012-06-04</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3394299" target="_blank">PMC3394299</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22669909.0" target="_blank">22669909.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gks528" target="_blank">10.1093/nar/gks528</a></td>
      <td>wd3ir3wg</td>
      <td>0.395103</td>
      <td><a href="Topic_66.html#uee3lqrx">Davaro_2013</a>, <a href="Topic_05.html#adzvn69b">Garcia–Moliner_2013</a>, <a href="Topic_05.html#d9zmdm23">Crowe_2014</a>, <a href="Topic_21.html#59emvigh">Ryu_W_2017</a></td>
      <td><a href="Topic_21.html#sik7m86l">Tzeng_2006</a>, <a href="Topic_17.html#0lbxvudt">Wang_2010</a></td>
    </tr>
    <tr>
      <th id="wcpc32w4";>392</th>
      <td>Johnson_2019</td>
      <td>An Ecological Framework for Modeling the Geography of Disease Transmission</td>
      <td>Johnson, Erica E.; Escobar, Luis E.; Zambrana-Torrelio, Carlos</td>
      <td>2019</td>
      <td>2019-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114676" target="_blank">PMC7114676</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31078330.0" target="_blank">31078330.0</a></td>
      <td><a href="https://doi.org/10.1016/j.tree.2019.03.004" target="_blank">10.1016/j.tree.2019.03.004</a></td>
      <td>wcpc32w4</td>
      <td>0.394824</td>
      <td><a href="Topic_01.html#qxkmngyk">Kozakiewicz_2018</a>, <a href="Topic_06.html#iwwgfh9y">Grassly_2008</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a></td>
      <td><a href="Topic_02.html#v3spbo5u">Peterson_2008</a>, <a href="Topic_02.html#a2wwdeab">Vander_Wal_2014</a>, <a href="Topic_36.html#ts91ivwf">Becker_2019</a>, <a href="Topic_36.html#t5gt399y">Hayman_2013</a></td>
    </tr>
    <tr>
      <th id="cjcvm3x7";>393</th>
      <td>Chung_2007</td>
      <td>RE-MuSiC: a tool for multiple sequence alignment with regular expression constraints</td>
      <td>Chung, Yun-Sheng; Lee, Wei-Hsun; Tang, Chuan Yi; Lu, Chin Lung</td>
      <td>2007</td>
      <td>2007-05-08</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1933182" target="_blank">PMC1933182</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17488842.0" target="_blank">17488842.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm275" target="_blank">10.1093/nar/gkm275</a></td>
      <td>cjcvm3x7</td>
      <td>0.394295</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a></td>
      <td><a href="Topic_17.html#e9app2cv">Lu_C_2005</a>, <a href="Topic_17.html#gjild96e">Song_2005</a>, <a href="Topic_17.html#s7kzp795">Zemla_2011</a>, <a href="Topic_17.html#9zfct1yx">Altschul_1991</a></td>
    </tr>
    <tr>
      <th id="qah877pk";>394</th>
      <td>Ekins_2007</td>
      <td>In silico pharmacology for drug discovery: applications to targets and beyond</td>
      <td>Ekins, S; Mestres, J; Testa, B</td>
      <td>2007</td>
      <td>2007-09-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1978280" target="_blank">PMC1978280</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17549046.0" target="_blank">17549046.0</a></td>
      <td><a href="https://doi.org/10.1038/sj.bjp.0707306" target="_blank">10.1038/sj.bjp.0707306</a></td>
      <td>qah877pk</td>
      <td>0.393498</td>
      <td><a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a></td>
      <td><a href="Topic_18.html#s50g754b">Chiba_2012</a>, <a href="Topic_18.html#rhg5pbsq">van_der_Kooy_2013</a>, <a href="Topic_17.html#fewy8y4a">Wang_2018</a>, <a href="Topic_18.html#cyp148on">Li_J_2006</a></td>
    </tr>
    <tr>
      <th id="hstomkk5";>395</th>
      <td>Oreper_2017</td>
      <td>Inbred Strain Variant Database (ISVdb): A Repository for Probabilistically Informed Sequence Differences Among the Collaborative Cross Strains and Their Founders</td>
      <td>Oreper, Daniel; Cai, Yanwei; Tarantino, Lisa M.; de Villena, Fernando Pardo-Manuel; Valdar, William</td>
      <td>2017</td>
      <td>2017-06-05</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5473744" target="_blank">PMC5473744</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28592645.0" target="_blank">28592645.0</a></td>
      <td><a href="https://doi.org/10.1534/g3.117.041491" target="_blank">10.1534/g3.117.041491</a></td>
      <td>hstomkk5</td>
      <td>0.393378</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#41st827n">Srivastava_2017</a>, <a href="Topic_07.html#oudbcvx5">Shorter_2019</a>, <a href="Topic_34.html#ciqlspu9">Aylor_2011</a>, <a href="Topic_34.html#7zuufs7k">Graham_2017</a></td>
    </tr>
    <tr>
      <th id="qnvgg2e9";>396</th>
      <td>Langille_2010</td>
      <td>BioTorrents: A File Sharing Service for Scientific Data</td>
      <td>Langille, Morgan G. I.; Eisen, Jonathan A.</td>
      <td>2010</td>
      <td>2010-04-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2854681" target="_blank">PMC2854681</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20418944.0" target="_blank">20418944.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0010071" target="_blank">10.1371/journal.pone.0010071</a></td>
      <td>qnvgg2e9</td>
      <td>0.392491</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_17.html#3ulketgy">Snyder_2006</a>, <a href="Topic_17.html#0kkm0tgj">Zhang_2007</a>, <a href="Topic_17.html#2gc2dc46">McGarvey_2009</a></td>
    </tr>
    <tr>
      <th id="w88hdlmr";>397</th>
      <td>Qi_X_2020</td>
      <td>A network-based method with privacy-preserving for identifying influential providers in large healthcare service systems</td>
      <td>Qi, Xiaoyu; Mei, Gang; Cuomo, Salvatore; Xiao, Lei</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.future.2020.04.004" target="_blank">10.1016/j.future.2020.04.004</a></td>
      <td>w88hdlmr</td>
      <td>0.392360</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a></td>
      <td><a href="Topic_06.html#og0b2l6b">Chen_2007</a>, <a href="Topic_06.html#zejxb953">Chen_2010</a>, <a href="Topic_01.html#284rthek">Razaque_2019</a></td>
    </tr>
    <tr>
      <th id="w962fdvq";>398</th>
      <td>Links_2015</td>
      <td>Big Data is changing the battle against infectious diseases</td>
      <td>Links, MG</td>
      <td>2015</td>
      <td>2015-09-03</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5864264" target="_blank">PMC5864264</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29769955.0" target="_blank">29769955.0</a></td>
      <td><a href="https://doi.org/10.14745/ccdr.v41i09a03" target="_blank">10.14745/ccdr.v41i09a03</a></td>
      <td>w962fdvq</td>
      <td>0.390733</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a></td>
      <td><a href="Topic_17.html#fbl3szsu">Pfeiffer_2015</a>, <a href="Topic_19.html#tohbzenc">Bhola_2020</a>, <a href="Topic_04.html#jrvl0ykn">Nelson_2019</a>, <a href="Topic_04.html#03tx0rni">VanderWaal_2017</a></td>
    </tr>
    <tr>
      <th id="4qgjglx3";>399</th>
      <td>Tian_2007</td>
      <td>Epidemic Alert & Response Framework and Technology Based on Spreading Dynamics Simulation</td>
      <td>Tian, Chunhua; Zhang, Xun; Ding, Wei; Cao, Rongzeng</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120482" target="_blank">PMC7120482</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-540-72588-6_166" target="_blank">10.1007/978-3-540-72588-6_166</a></td>
      <td>4qgjglx3</td>
      <td>0.389701</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_71.html#22wnp1sv">von_der_Gracht_2010</a></td>
      <td><a href="Topic_06.html#bcv1wpfb">Chen_2018</a>, <a href="Topic_06.html#ccmd0dw8">Park_2016</a>, <a href="Topic_06.html#obzi61sa">Bucini_2019</a></td>
    </tr>
    <tr>
      <th id="yt302jyx";>400</th>
      <td>Nguyen_2019</td>
      <td>A non-contact infection screening system using medical radar and Linux-embedded FPGA: Implementation and preliminary validation</td>
      <td>Nguyen, Cuong V.; Le Quang, Truong; Vu, Trung Nguyen; Le Thi, Hoi; Van, Kinh Nguyen; Trong, Thanh Han; Trong, Tuan Do; Sun, Guanghao; Ishibashi, Koichiro</td>
      <td>2019</td>
      <td>2019-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103934" target="_blank">PMC7103934</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.imu.2019.100225" target="_blank">10.1016/j.imu.2019.100225</a></td>
      <td>yt302jyx</td>
      <td>0.389309</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_14.html#kjovtgua">Wu_J_2020</a>, <a href="Topic_32.html#4rulshop">Huang_2013</a>, <a href="Topic_17.html#2l1zw19o">Ozturk_2020</a></td>
    </tr>
    <tr>
      <th id="4ylv92td";>401</th>
      <td>Kim_D_2018</td>
      <td>Mapping Oil Spills from Dual-Polarized SAR Images Using an Artificial Neural Network: Application to Oil Spill in the Kerch Strait in November 2007</td>
      <td>Kim, Daeseong; Jung, Hyung-Sup</td>
      <td>2018</td>
      <td>2018-07-11</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6069476" target="_blank">PMC6069476</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29997367.0" target="_blank">29997367.0</a></td>
      <td><a href="https://doi.org/10.3390/s18072237" target="_blank">10.3390/s18072237</a></td>
      <td>4ylv92td</td>
      <td>0.388945</td>
      <td><a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#3u81ntle">Acar_2016</a>, <a href="Topic_64.html#alecmrj4">Ng_E_2011</a>, <a href="Topic_11.html#5kohk0c3">You_H_2016</a>, <a href="Topic_71.html#24qe0yn7">Shu_Z_2015</a></td>
    </tr>
    <tr>
      <th id="2upc2spn";>402</th>
      <td>Qiu_T_2018</td>
      <td>CE-BLAST makes it possible to compute antigenic similarity for newly emerging pathogens</td>
      <td>Qiu, Tianyi; Yang, Yiyan; Qiu, Jingxuan; Huang, Yang; Xu, Tianlei; Xiao, Han; Wu, Dingfeng; Zhang, Qingchen; Zhou, Chen; Zhang, Xiaoyan; Tang, Kailin; Xu, Jianqing; Cao, Zhiwei</td>
      <td>2018</td>
      <td>2018-05-02</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5932059" target="_blank">PMC5932059</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29720583.0" target="_blank">29720583.0</a></td>
      <td><a href="https://doi.org/10.1038/s41467-018-04171-2" target="_blank">10.1038/s41467-018-04171-2</a></td>
      <td>2upc2spn</td>
      <td>0.387992</td>
      <td><a href="Topic_15.html#z2u5frvq">Montefiori_2007</a>, <a href="Topic_17.html#61cl4k8k">Qiu_X_2019</a>, <a href="Topic_13.html#fqqnkjed">Fauci_2006</a>, <a href="Topic_15.html#neclg3gb">Tripp_2014</a>, <a href="Topic_13.html#vubszdp2">Li_L_2014</a></td>
      <td><a href="Topic_23.html#pobxgffs">Jiang_2017</a>, <a href="Topic_69.html#gmoq1i5x">Shi_J_2015</a>, <a href="Topic_14.html#u0rjntlc">Davidson_2019</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a></td>
    </tr>
    <tr>
      <th id="4grjr0h3";>403</th>
      <td>McClure_2019</td>
      <td>Unified feature association networks through integration of transcriptomic and proteomic data</td>
      <td>McClure, Ryan S.; Wendler, Jason P.; Adkins, Joshua N.; Swanstrom, Jesica; Baric, Ralph; Kaiser, Brooke L. Deatherage; Oxford, Kristie L.; Waters, Katrina M.; McDermott, Jason E.</td>
      <td>2019</td>
      <td>2019-09-17</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748406" target="_blank">PMC6748406</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31527878.0" target="_blank">31527878.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1007241" target="_blank">10.1371/journal.pcbi.1007241</a></td>
      <td>4grjr0h3</td>
      <td>0.387135</td>
      <td><a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_34.html#zlqh436d">Chasman_2016</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_17.html#hs30wwiq">Gibbs_2014</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_01.html#gmw88elf">Vella_2017</a>, <a href="Topic_17.html#g55witln">Gibbs_2013</a></td>
    </tr>
    <tr>
      <th id="wqa98m4v";>404</th>
      <td>Al-Turaiki_2016</td>
      <td>Building predictive models for MERS-CoV infections using data mining techniques</td>
      <td>Al-Turaiki, Isra; Alshahrani, Mona; Almutairi, Tahani</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102847" target="_blank">PMC7102847</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27641481.0" target="_blank">27641481.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jiph.2016.09.007" target="_blank">10.1016/j.jiph.2016.09.007</a></td>
      <td>wqa98m4v</td>
      <td>0.386780</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_06.html#sk2n2gxw">Eifan_2017</a>, <a href="Topic_55.html#dl194hu8">Faridi_2016</a>, <a href="Topic_07.html#idu4j78a">Sohrab_2019</a></td>
    </tr>
    <tr>
      <th id="ukcxh8qa";>405</th>
      <td>Lim_C_2018</td>
      <td>Know Your Enemy: Successful Bioinformatic Approaches to Predict Functional RNA Structures in Viral RNAs</td>
      <td>Lim, Chun Shen; Brown, Chris M.</td>
      <td>2018</td>
      <td>2018-01-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758548" target="_blank">PMC5758548</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29354101.0" target="_blank">29354101.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2017.02582" target="_blank">10.3389/fmicb.2017.02582</a></td>
      <td>ukcxh8qa</td>
      <td>0.386418</td>
      <td><a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_01.html#vnynbijl">Conzelmann_1996</a>, <a href="Topic_45.html#n8zg6rsj">Wang_2018</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a></td>
      <td><a href="Topic_21.html#d9urjb9e">Masante_2015</a>, <a href="Topic_21.html#yo9libo0">Zhou_2011</a></td>
    </tr>
    <tr>
      <th id="e89d3r95";>406</th>
      <td>Yin_W_2015</td>
      <td>From genome-scale data to models of infectious disease: a Bayesian network-based strategy to drive model development</td>
      <td>Yin, Weiwei; Kissinger, Jessica C.; Moreno, Alberto; Galinski, Mary R.; Styczynski, Mark P.</td>
      <td>2015</td>
      <td>2015-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4679518" target="_blank">PMC4679518</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26093035.0" target="_blank">26093035.0</a></td>
      <td><a href="https://doi.org/10.1016/j.mbs.2015.06.006" target="_blank">10.1016/j.mbs.2015.06.006</a></td>
      <td>e89d3r95</td>
      <td>0.385884</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a></td>
      <td><a href="Topic_06.html#ijw5bl20">Campbell_2019</a>, <a href="Topic_02.html#qv5zsleh">Baele_2016</a>, <a href="Topic_06.html#gctd950h">Cottam_2008</a>, <a href="Topic_06.html#c1a957ef">Jombart_2014</a></td>
    </tr>
    <tr>
      <th id="d5ghy39g";>407</th>
      <td>Castro_2019</td>
      <td>The effect of variant interference on de novo assembly for viral deep sequencing</td>
      <td>Christina J. Castro; Rachel L. Marine; Edward Ramos; Terry Fei Fan Ng</td>
      <td>2019</td>
      <td>2019-10-22</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/815480" target="_blank">10.1101/815480</a></td>
      <td>d5ghy39g</td>
      <td>0.385388</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_61.html#2fcl6f4i">Price_2019</a></td>
      <td><a href="Topic_21.html#d9urjb9e">Masante_2015</a>, <a href="Topic_37.html#ndjbrtrv">Satyanarayana_2003</a></td>
    </tr>
    <tr>
      <th id="ajzk86rq";>408</th>
      <td>van_Weezep_2019</td>
      <td>PCR diagnostics: In silico validation by an automated tool using freely available software programs</td>
      <td>van Weezep, Erik; Kooi, Engbert A.; van Rijn, Piet A.</td>
      <td>2019</td>
      <td>2019-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7113775" target="_blank">PMC7113775</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31095975.0" target="_blank">31095975.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jviromet.2019.05.002" target="_blank">10.1016/j.jviromet.2019.05.002</a></td>
      <td>ajzk86rq</td>
      <td>0.385066</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_14.html#06vkoev5">Rajagopala_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a></td>
      <td><a href="Topic_14.html#nn4rum8j">Anis_2018</a>, <a href="Topic_14.html#3jxyipbn">Lewandowska_2017</a></td>
    </tr>
    <tr>
      <th id="3m1zet9t";>409</th>
      <td>Cheng_2010</td>
      <td>Manifold construction based on local distance invariance</td>
      <td>Cheng, Wei-Chen; Liou, Cheng-Yuan</td>
      <td>2010</td>
      <td>2010-02-11</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7091362" target="_blank">PMC7091362</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s12293-009-0030-y" target="_blank">10.1007/s12293-009-0030-y</a></td>
      <td>3m1zet9t</td>
      <td>0.384804</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#m8f5fsmc">Qi_Z_2011</a></td>
      <td><a href="Topic_17.html#ear9cyri">Bakker_2018</a>, <a href="Topic_17.html#pd9a1em0">Liu_K_2007</a>, <a href="Topic_17.html#89xnnvuv">Saad_2005</a>, <a href="Topic_17.html#eydkfqi2">Feng_2019</a></td>
    </tr>
    <tr>
      <th id="odnlt6fr";>410</th>
      <td>Wu_J_2016</td>
      <td>Reducing Energy Consumption and Overhead Based on Mobile Health in Big Data Opportunistic Networks</td>
      <td>Wu, Jia; Chen, Zhigang</td>
      <td>2016</td>
      <td>2016-08-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088852" target="_blank">PMC7088852</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11277-016-3610-4" target="_blank">10.1007/s11277-016-3610-4</a></td>
      <td>odnlt6fr</td>
      <td>0.384686</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a></td>
      <td><a href="Topic_17.html#ax09x6gk">Wu_J_2016</a>, <a href="Topic_06.html#lwyeqqvy">Hu_J_2016</a>, <a href="Topic_17.html#3l2dbnjb">Sood_2017</a>, <a href="Topic_17.html#2se42vec">Sareen_2016</a></td>
    </tr>
    <tr>
      <th id="7pdh7alh";>411</th>
      <td>Hong_2013</td>
      <td>Viral IRES Prediction System - a Web Server for Prediction of the IRES Secondary Structure In Silico</td>
      <td>Hong, Jun-Jie; Wu, Tzong-Yuan; Chang, Tsair-Yuan; Chen, Chung-Yung</td>
      <td>2013</td>
      <td>2013-11-05</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818432" target="_blank">PMC3818432</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24223923.0" target="_blank">24223923.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0079288" target="_blank">10.1371/journal.pone.0079288</a></td>
      <td>7pdh7alh</td>
      <td>0.384667</td>
      <td><a href="Topic_61.html#7fx1ghd3">Morikawa_1992</a>, <a href="Topic_17.html#wt0lvhfl">Wu_T_2009</a>, <a href="Topic_61.html#ug4gjst2">Landry_2009</a>, <a href="Topic_07.html#kixzxs68">Naito_2007</a></td>
      <td><a href="Topic_17.html#wt0lvhfl">Wu_T_2009</a>, <a href="Topic_61.html#ywkxxye5">Fajardo_2012</a></td>
    </tr>
    <tr>
      <th id="uiqk356j";>412</th>
      <td>Kamvar_2019</td>
      <td>Epidemic curves made easy using the R package incidence</td>
      <td>Kamvar, Zhian N.; Cai, Jun; Pulliam, Juliet R.C.; Schumacher, Jakob; Jombart, Thibaut</td>
      <td>2019</td>
      <td>2019-01-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6509961" target="_blank">PMC6509961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31119031.0" target="_blank">31119031.0</a></td>
      <td><a href="https://doi.org/10.12688/f1000research.18002.1" target="_blank">10.12688/f1000research.18002.1</a></td>
      <td>uiqk356j</td>
      <td>0.383663</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a></td>
      <td><a href="Topic_06.html#m9icky9z">Song_2020</a>, <a href="Topic_06.html#c1a957ef">Jombart_2014</a>, <a href="Topic_06.html#2z32ln7g">Daughton_2017</a>, <a href="Topic_06.html#ge871ypq">Safi_2011</a></td>
    </tr>
    <tr>
      <th id="7ou1uqis";>413</th>
      <td>Bonham_2009</td>
      <td>Modeling tourism: A fully identified VECM approach</td>
      <td>Bonham, Carl; Gangnes, Byron; Zhou, Ting</td>
      <td>2009</td>
      <td>2009-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114862" target="_blank">PMC7114862</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ijforecast.2008.11.014" target="_blank">10.1016/j.ijforecast.2008.11.014</a></td>
      <td>7ou1uqis</td>
      <td>0.383663</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_04.html#efxh5grs">Gostin_2016</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a></td>
      <td><a href="Topic_71.html#sh5nmyxc">Lee_C_2008</a>, <a href="Topic_71.html#ffk0ifwy">Lim_C_2009</a>, <a href="Topic_71.html#s6hwui6n">Andraz_2016</a>, <a href="Topic_71.html#g247rjh9">Guirao_2016</a></td>
    </tr>
    <tr>
      <th id="ywd88kmq";>414</th>
      <td>CAO_S_2012</td>
      <td>Statistical mechanical modeling of RNA folding: from free energy landscape to tertiary structural prediction</td>
      <td>CAO, Song; CHEN, Shi-Jie</td>
      <td>2012</td>
      <td>2012-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4902161" target="_blank">PMC4902161</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27293312.0" target="_blank">27293312.0</a></td>
      <td><a href="https://doi.org/10.1007/978-3-642-25740-7_10" target="_blank">10.1007/978-3-642-25740-7_10</a></td>
      <td>ywd88kmq</td>
      <td>0.383247</td>
      <td><a href="Topic_61.html#9zm4per4">Dawson_2007</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#1voq7zxz">DiMaio_2016</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a></td>
      <td><a href="Topic_61.html#tv8hi2wy">Low_J_2010</a>, <a href="Topic_61.html#xuxlw4c0">Kolchanov_1996</a></td>
    </tr>
    <tr>
      <th id="m7x9zc12";>415</th>
      <td>Torres_2011</td>
      <td>LAVA: An Open-Source Approach To Designing LAMP (Loop-Mediated Isothermal Amplification) DNA Signatures</td>
      <td>Torres, Clinton; Vitalis, Elizabeth A; Baker, Brian R; Gardner, Shea N; Torres, Marisa W; Dzenitis, John M</td>
      <td>2011</td>
      <td>2011-06-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3213686" target="_blank">PMC3213686</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21679460.0" target="_blank">21679460.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-12-240" target="_blank">10.1186/1471-2105-12-240</a></td>
      <td>m7x9zc12</td>
      <td>0.383058</td>
      <td><a href="Topic_29.html#p61mpc6t">Onodera_2007</a>, <a href="Topic_29.html#cxq9v218">Nitsche_2004</a>, <a href="Topic_17.html#zz1hkbpc">Gardner_2014</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a></td>
      <td><a href="Topic_29.html#b8jxz225">Fallahi_2015</a>, <a href="Topic_29.html#fa0mnbzg">Nemoto_2015</a>, <a href="Topic_29.html#98sqovwb">Li_H_2019</a>, <a href="Topic_29.html#eiicdxre">Li_Z_2018</a></td>
    </tr>
    <tr>
      <th id="c2lljdi7";>416</th>
      <td>Lopez-Rincon_2020</td>
      <td>Accurate Identification of SARS-CoV-2 from Viral Genome Sequences using Deep Learning</td>
      <td>Alejandro Lopez-Rincon; Alberto Tonda; Lucero Mendoza-Maldonado; Eric Claassen; Johan Garssen; Aletta D. Kraneveld</td>
      <td>2020</td>
      <td>2020-03-14</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.13.990242" target="_blank">10.1101/2020.03.13.990242</a></td>
      <td>c2lljdi7</td>
      <td>0.382390</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#7qc1lqyp">Cheng_2012</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a></td>
      <td><a href="Topic_14.html#9595vm0k">OKBA_2020</a>, <a href="Topic_29.html#0zjpqnqp">Hoteit_2016</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_18.html#4rhlkhji">Aouadi_2017</a></td>
    </tr>
    <tr>
      <th id="qkymwvwb";>417</th>
      <td>Wei_C_2011</td>
      <td>Identification and Typing of Human Enterovirus: A Genomic Barcode Approach</td>
      <td>Wei, Chengguo; Wang, Guoqing; Chen, Xin; Huang, Honglan; Liu, Bin; Xu, Ying; Li, Fan</td>
      <td>2011</td>
      <td>2011-10-14</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3194813" target="_blank">PMC3194813</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22022592.0" target="_blank">22022592.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0026296" target="_blank">10.1371/journal.pone.0026296</a></td>
      <td>qkymwvwb</td>
      <td>0.381887</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_07.html#xdxfi2yy">de_Souza_Luna_2008</a></td>
    </tr>
    <tr>
      <th id="gxstlzuk";>418</th>
      <td>Cleemput_2020</td>
      <td>Genome Detective Coronavirus Typing Tool for rapid identification and characterization of novel coronavirus genomes</td>
      <td>Sara Cleemput; Wim Dumon; Vagner Fonseca; Wasim Abdool Karim; Marta Giovanetti; Luiz Carlos Alcantara; Koen Deforche; Tulio de Oliveira</td>
      <td>2020</td>
      <td>2020-02-02</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.31.928796" target="_blank">10.1101/2020.01.31.928796</a></td>
      <td>gxstlzuk</td>
      <td>0.381887</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#02opdk0m">Xu_Z_2009</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_17.html#st5idleq">Cleemput_2020</a>, <a href="Topic_45.html#gccmkxly">Phan_2018</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_29.html#xucldmvs">Tanaka_2015</a></td>
    </tr>
    <tr>
      <th id="0lbxvudt";>419</th>
      <td>Wang_2010</td>
      <td>VIGOR, an annotation program for small viral genomes</td>
      <td>Wang, Shiliang; Sundaram, Jaideep P; Spiro, David</td>
      <td>2010</td>
      <td>2010-09-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2942859" target="_blank">PMC2942859</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20822531.0" target="_blank">20822531.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-11-451" target="_blank">10.1186/1471-2105-11-451</a></td>
      <td>0lbxvudt</td>
      <td>0.380038</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_37.html#kihn83ny">Armesto_2007</a></td>
      <td><a href="Topic_21.html#sik7m86l">Tzeng_2006</a></td>
    </tr>
    <tr>
      <th id="kusjwue0";>420</th>
      <td>Bos_L_2014</td>
      <td>How integration of global omics-data could help preparing for pandemics – a scent of influenza</td>
      <td>Bos, Lieuwe D. J.; de Jong, Menno D.; Sterk, Peter J.; Schultz, Marcus J.</td>
      <td>2014</td>
      <td>2014-04-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4000993" target="_blank">PMC4000993</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24795745.0" target="_blank">24795745.0</a></td>
      <td><a href="https://doi.org/10.3389/fgene.2014.00080" target="_blank">10.3389/fgene.2014.00080</a></td>
      <td>kusjwue0</td>
      <td>0.379701</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_02.html#47goscew">Vazquez_2020</a></td>
      <td><a href="Topic_36.html#tulne0v3">Rabaa_2015</a>, <a href="Topic_01.html#0dhzv75e">Hörster_2019</a>, <a href="Topic_13.html#k0f9fser">Shi_W_2017</a>, <a href="Topic_55.html#foolgmqj">Gao_H_2016</a></td>
    </tr>
    <tr>
      <th id="eg63bjl5";>421</th>
      <td>Jacques_2010</td>
      <td>Small-angle scattering for structural biology—Expanding the frontier while avoiding the pitfalls</td>
      <td>Jacques, David A; Trewhella, Jill</td>
      <td>2010</td>
      <td>2010-01-29</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2867006" target="_blank">PMC2867006</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20120026.0" target="_blank">20120026.0</a></td>
      <td><a href="https://doi.org/10.1002/pro.351" target="_blank">10.1002/pro.351</a></td>
      <td>eg63bjl5</td>
      <td>0.379399</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#cw2opick">Chou_2011</a>, <a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_01.html#xl9qhcm8">Giegé_2010</a>, <a href="Topic_16.html#g0h0itzs">Buller_2012</a></td>
    </tr>
    <tr>
      <th id="g6obo798";>422</th>
      <td>Zheng_2008</td>
      <td>A complexity-based measure and its application to phylogenetic analysis</td>
      <td>Zheng, Xiaoqi; Li, Chun; Wang, Jun</td>
      <td>2008</td>
      <td>2008-12-09</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088198" target="_blank">PMC7088198</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32214590.0" target="_blank">32214590.0</a></td>
      <td><a href="https://doi.org/10.1007/s10910-008-9511-3" target="_blank">10.1007/s10910-008-9511-3</a></td>
      <td>g6obo798</td>
      <td>0.378589</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#le7v89mp">Chang_2011</a></td>
      <td><a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#8qnq02gd">Zheng_2009</a>, <a href="Topic_17.html#ne9d0siv">Banks_2006</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a></td>
    </tr>
    <tr>
      <th id="dslwklo4";>423</th>
      <td>Tao_J_2004</td>
      <td>A probabilistic approach to incorporating domain knowledge for closed-room people monitoring</td>
      <td>Tao, Ji; Tan, Yap-Peng</td>
      <td>2004</td>
      <td>2004-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126007" target="_blank">PMC7126007</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.image.2004.07.001" target="_blank">10.1016/j.image.2004.07.001</a></td>
      <td>dslwklo4</td>
      <td>0.378144</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#ym9j0ei0">Tao_J_2005</a>, <a href="Topic_17.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_17.html#gm1e9j80">Thi_T_2014</a>, <a href="Topic_17.html#gi8c1omv">Capotorti_2006</a></td>
    </tr>
    <tr>
      <th id="v1p4ut66";>424</th>
      <td>Lockwood_2011</td>
      <td>Neighborhood Properties Are Important Determinants of Temperature Sensitive Mutations</td>
      <td>Lockwood, Svetlana; Krishnamoorthy, Bala; Ye, Ping</td>
      <td>2011</td>
      <td>2011-12-02</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3229608" target="_blank">PMC3229608</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22164302.0" target="_blank">22164302.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0028507" target="_blank">10.1371/journal.pone.0028507</a></td>
      <td>v1p4ut66</td>
      <td>0.375735</td>
      <td><a href="Topic_61.html#wv0f4hp4">Wu_G_2005</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="h7zxyzca";>425</th>
      <td>Isakov_2015</td>
      <td>Deep sequencing analysis of viral infection and evolution allows rapid and detailed characterization of viral mutant spectrum</td>
      <td>Isakov, Ofer; Bordería, Antonio V.; Golan, David; Hamenahem, Amir; Celniker, Gershon; Yoffe, Liron; Blanc, Hervé; Vignuzzi, Marco; Shomron, Noam</td>
      <td>2015</td>
      <td>2015-07-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481840" target="_blank">PMC4481840</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25701575.0" target="_blank">25701575.0</a></td>
      <td><a href="https://doi.org/10.1093/bioinformatics/btv101" target="_blank">10.1093/bioinformatics/btv101</a></td>
      <td>h7zxyzca</td>
      <td>0.375063</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_07.html#thsq6pnk">Lachheb_2019</a>, <a href="Topic_30.html#kvfffe0b">Gutiérrez_2013</a></td>
    </tr>
    <tr>
      <th id="g55witln";>426</th>
      <td>Gibbs_2013</td>
      <td>Protein co-expression network analysis (ProCoNA)</td>
      <td>Gibbs, David L; Baratt, Arie; Baric, Ralph S; Kawaoka, Yoshihiro; Smith, Richard D; Orwoll, Eric S; Katze, Michael G; McWeeney, Shannon K</td>
      <td>2013</td>
      <td>2013-06-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3695838" target="_blank">PMC3695838</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23724967.0" target="_blank">23724967.0</a></td>
      <td><a href="https://doi.org/10.1186/2043-9113-3-11" target="_blank">10.1186/2043-9113-3-11</a></td>
      <td>g55witln</td>
      <td>0.375053</td>
      <td><a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_01.html#gmw88elf">Vella_2017</a>, <a href="Topic_69.html#elb93kp6">Li_S_2012</a>, <a href="Topic_34.html#j80hnhpb">Noh_H_2018</a></td>
    </tr>
    <tr>
      <th id="abvpy8dg";>427</th>
      <td>Shaban-Nejad_2012</td>
      <td>HAIKU: A Semantic Framework for Surveillance of Healthcare-Associated Infections</td>
      <td>Shaban-Nejad, Arash; Riazanov, Alexandre; Charland, Katia M.; Rose, Gregory W.; Baker, Christopher J.O.; Tamblyn, Robyn; Forster, Alan J.; Buckeridge, David L.</td>
      <td>2012</td>
      <td>2012-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2012.06.151" target="_blank">10.1016/j.procs.2012.06.151</a></td>
      <td>abvpy8dg</td>
      <td>0.374108</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_04.html#j2ug18id">May_L_2009</a>, <a href="Topic_04.html#yybatog9">Paterson_2013</a>, <a href="Topic_39.html#1ecdj314">Glennon_2020</a></td>
    </tr>
    <tr>
      <th id="jaki2kyn";>428</th>
      <td>Cho_V_2010</td>
      <td>MISMIS – A comprehensive decision support system for stock market investment</td>
      <td>Cho, Vincent</td>
      <td>2010</td>
      <td>2010-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127451" target="_blank">PMC7127451</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.knosys.2010.04.009" target="_blank">10.1016/j.knosys.2010.04.009</a></td>
      <td>jaki2kyn</td>
      <td>0.372157</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a></td>
      <td><a href="Topic_71.html#oezkz55m">Tung_2011</a>, <a href="Topic_71.html#e7hkfbxu">Tuyon_2016</a>, <a href="Topic_71.html#9uc54web">Wu_P_2016</a>, <a href="Topic_71.html#vo58pzqt">Chen_2005</a></td>
    </tr>
    <tr>
      <th id="b13p3mca";>429</th>
      <td>Prosperi_2011</td>
      <td>A novel methodology for large-scale phylogeny partition</td>
      <td>Prosperi, Mattia C.F.; Ciccozzi, Massimo; Fanti, luri; Saladini, Francesco; Pecorari, Monica; Borghi, Vanni; Di Giambenedetto, Simona; Bruzzone, Bianca; Capetti, Amedeo; Vivarelli, Angela; Rusconi, Stefano; Carla Re, Maria; Gismondo, Maria Rita; Sighinolfi, Laura; Gray, Rebecca R.; Salemi, Marco; Zazzi, Maurizio; De Luca, Andrea</td>
      <td>2011</td>
      <td>2011-05-24</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6045912" target="_blank">PMC6045912</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21610724.0" target="_blank">21610724.0</a></td>
      <td><a href="https://doi.org/10.1038/ncomms1325" target="_blank">10.1038/ncomms1325</a></td>
      <td>b13p3mca</td>
      <td>0.371220</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_06.html#3gj754dq">Giardina_2017</a></td>
      <td><a href="Topic_02.html#aj352do0">Tibayrenc_2015</a>, <a href="Topic_06.html#wich35l7">Velsko_2014</a></td>
    </tr>
    <tr>
      <th id="635lbedk";>430</th>
      <td>Zhao_2020</td>
      <td>Spreading predictability in complex networks</td>
      <td>Na Zhao; Jian Wang; Yong Yu; Jun-Yan Zhao; Duan-Bing Chen</td>
      <td>2020</td>
      <td>2020-01-28</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.28.922757" target="_blank">10.1101/2020.01.28.922757</a></td>
      <td>635lbedk</td>
      <td>0.369978</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_02.html#8plec8t4">Looker_2006</a></td>
      <td><a href="Topic_02.html#4gksnpwa">Scarpino_2019</a>, <a href="Topic_06.html#ot7hztyk">Yuan_2015</a>, <a href="Topic_06.html#5w90r6wu">Wu_Q_2014</a>, <a href="Topic_06.html#av8ww9fd">Liu_Y_2016</a></td>
    </tr>
    <tr>
      <th id="m56agj4z";>431</th>
      <td>Reddy_2007</td>
      <td>Positional clustering improves computational binding site detection and identifies novel cis-regulatory sites in mammalian GABA(A) receptor subunit genes</td>
      <td>Reddy, Timothy E.; Shakhnovich, Boris E.; Roberts, Daniel S.; Russek, Shelley J.; DeLisi, Charles</td>
      <td>2007</td>
      <td>2007-01-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1807961" target="_blank">PMC1807961</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17204484.0" target="_blank">17204484.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkl1062" target="_blank">10.1093/nar/gkl1062</a></td>
      <td>m56agj4z</td>
      <td>0.369921</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_61.html#vbdyglf2">Hagiwara_2012</a>, <a href="Topic_61.html#l92nlegt">Sarnowska_2007</a></td>
    </tr>
    <tr>
      <th id="vv1wjen0";>432</th>
      <td>Xiaoxuan_2020</td>
      <td>Can Search Query Forecast successfully in China's 2019-nCov pneumonia?</td>
      <td>Li Xiaoxuan; Wu Qi; Lv Benfu</td>
      <td>2020</td>
      <td>2020-02-18</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.12.20022400" target="_blank">10.1101/2020.02.12.20022400</a></td>
      <td>vv1wjen0</td>
      <td>0.368533</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_19.html#bcde0u7u">He_Y_2020</a>, <a href="Topic_19.html#odl1nv4i">Wang_2020</a>, <a href="Topic_19.html#qpt5dj8p">Li_M_2020</a></td>
    </tr>
    <tr>
      <th id="gk8ely05";>433</th>
      <td>Campbell_2018</td>
      <td>outbreaker2: a modular platform for outbreak reconstruction</td>
      <td>Campbell, Finlay; Didelot, Xavier; Fitzjohn, Rich; Ferguson, Neil; Cori, Anne; Jombart, Thibaut</td>
      <td>2018</td>
      <td>2018-10-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6196407" target="_blank">PMC6196407</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30343663.0" target="_blank">30343663.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-018-2330-z" target="_blank">10.1186/s12859-018-2330-z</a></td>
      <td>gk8ely05</td>
      <td>0.368068</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_06.html#iwwgfh9y">Grassly_2008</a>, <a href="Topic_01.html#hr4smx24">van_Kampen_2015</a></td>
      <td><a href="Topic_06.html#c1a957ef">Jombart_2014</a>, <a href="Topic_39.html#1ecdj314">Glennon_2020</a>, <a href="Topic_06.html#wich35l7">Velsko_2014</a>, <a href="Topic_36.html#tulne0v3">Rabaa_2015</a></td>
    </tr>
    <tr>
      <th id="7imgztwe";>434</th>
      <td>Frishman_2009</td>
      <td>Protein-protein interactions: analysis and prediction</td>
      <td>Frishman, D.; Albrecht, M.; Blankenburg, H.; Bork, P.; Harrington, E. D.; Hermjakob, H.; Juhl Jensen, L.; Juan, D. A.; Lengauer, T.; Pagel, P.; Schachter, V.; Valencia, A.</td>
      <td>2009</td>
      <td>2009-10-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120725" target="_blank">PMC7120725</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-211-75123-7_17" target="_blank">10.1007/978-3-211-75123-7_17</a></td>
      <td>7imgztwe</td>
      <td>0.367773</td>
      <td><a href="Topic_10.html#qdy29oym">Denison_2008</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_10.html#rg45m42j">Tessier_2019</a></td>
      <td><a href="Topic_17.html#0ec2xunt">Plewczyński_2008</a>, <a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a>, <a href="Topic_16.html#vhsipxs7">Järvelin_2016</a></td>
    </tr>
    <tr>
      <th id="uukrqp7j";>435</th>
      <td>Haibin_2004</td>
      <td>Prokaryote phylogeny based on ribosomal proteins and aminoacyl tRNA synthetases by using the compositional distance approach</td>
      <td>Haibin, Wei; Ji, Qi; Bailin, Hao</td>
      <td>2004</td>
      <td>2004-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088628" target="_blank">PMC7088628</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15493472.0" target="_blank">15493472.0</a></td>
      <td><a href="https://doi.org/10.1360/03yc0137" target="_blank">10.1360/03yc0137</a></td>
      <td>uukrqp7j</td>
      <td>0.367773</td>
      <td><a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a></td>
      <td><a href="Topic_17.html#xjmicuki">Sun_J_2010</a></td>
    </tr>
    <tr>
      <th id="4y5mnok8";>436</th>
      <td>Ehrt_2018</td>
      <td>A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs)</td>
      <td>Ehrt, Christiane; Brinkjost, Tobias; Koch, Oliver</td>
      <td>2018</td>
      <td>2018-11-08</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224041" target="_blank">PMC6224041</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30408032.0" target="_blank">30408032.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pcbi.1006483" target="_blank">10.1371/journal.pcbi.1006483</a></td>
      <td>4y5mnok8</td>
      <td>0.367595</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_18.html#ihnl8ajr">Lamberto_2017</a>, <a href="Topic_16.html#40pe21lm">Zhang_2018</a>, <a href="Topic_18.html#gqnzwyn3">Mukherjee_2011</a></td>
    </tr>
    <tr>
      <th id="rxhygr8x";>437</th>
      <td>Li_H_2018</td>
      <td>Comparative studies of alignment, alignment-free and SVM based approaches for predicting the hosts of viruses based on viral sequences</td>
      <td>Li, Han; Sun, Fengzhu</td>
      <td>2018</td>
      <td>2018-07-03</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030160" target="_blank">PMC6030160</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29968780.0" target="_blank">29968780.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-018-28308-x" target="_blank">10.1038/s41598-018-28308-x</a></td>
      <td>rxhygr8x</td>
      <td>0.367435</td>
      <td><a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_10.html#629kl04a">Bahir_2009</a></td>
    </tr>
    <tr>
      <th id="wczw4vn1";>438</th>
      <td>He_Y_2010</td>
      <td>Vaxign: The First Web-Based Vaccine Design Program for Reverse Vaccinology and Applications for Vaccine Development</td>
      <td>He, Yongqun; Xiang, Zuoshuang; Mobley, Harry L. T.</td>
      <td>2010</td>
      <td>2010-07-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2910479" target="_blank">PMC2910479</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20671958.0" target="_blank">20671958.0</a></td>
      <td><a href="https://doi.org/10.1155/2010/297505" target="_blank">10.1155/2010/297505</a></td>
      <td>wczw4vn1</td>
      <td>0.366763</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a></td>
      <td><a href="Topic_69.html#71940dh7">Kumar_2019</a>, <a href="Topic_15.html#0czzjeop">Chen_2007</a></td>
    </tr>
    <tr>
      <th id="2gc2dc46";>439</th>
      <td>McGarvey_2009</td>
      <td>Systems Integration of Biodefense Omics Data for Analysis of Pathogen-Host Interactions and Identification of Potential Targets</td>
      <td>McGarvey, Peter B.; Huang, Hongzhan; Mazumder, Raja; Zhang, Jian; Chen, Yongxing; Zhang, Chengdong; Cammer, Stephen; Will, Rebecca; Odle, Margie; Sobral, Bruno; Moore, Margaret; Wu, Cathy H.</td>
      <td>2009</td>
      <td>2009-09-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2745575" target="_blank">PMC2745575</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19779614.0" target="_blank">19779614.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0007162" target="_blank">10.1371/journal.pone.0007162</a></td>
      <td>2gc2dc46</td>
      <td>0.365337</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#t94aufs3">Aurrecoechea_2017</a></td>
      <td><a href="Topic_17.html#0kkm0tgj">Zhang_2007</a>, <a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_17.html#3ulketgy">Snyder_2006</a></td>
    </tr>
    <tr>
      <th id="8nw2al5o";>440</th>
      <td>Perez_2011</td>
      <td>Global animal disease surveillance</td>
      <td>Perez, A.; AlKhamis, M.; Carlsson, U.; Brito, B.; Carrasco-Medanic, R.; Whedbee, Z.; Willeberg, P.</td>
      <td>2011</td>
      <td>2011-09-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22748173.0" target="_blank">22748173.0</a></td>
      <td><a href="https://doi.org/10.1016/j.sste.2011.07.006" target="_blank">10.1016/j.sste.2011.07.006</a></td>
      <td>8nw2al5o</td>
      <td>0.364809</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_01.html#vfhlph80">Alves_2007</a></td>
      <td><a href="Topic_04.html#tfp2idq2">Hale_2019</a>, <a href="Topic_06.html#28mhhwol">Clements_2009</a>, <a href="Topic_39.html#guz3i7af">Frössling_2012</a>, <a href="Topic_36.html#tulne0v3">Rabaa_2015</a></td>
    </tr>
    <tr>
      <th id="iar66keo";>441</th>
      <td>Schwartz_2008</td>
      <td>Cost effective strategies for completing the Interactome</td>
      <td>Schwartz, Ariel S.; Yu, Jingkai; Gardenour, Kyle R.; Finley, Russell L.; Ideker, Trey</td>
      <td>2008</td>
      <td>2008-12-14</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2613168" target="_blank">PMC2613168</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19079254.0" target="_blank">19079254.0</a></td>
      <td><a href="https://doi.org/10.1038/nmeth.1283" target="_blank">10.1038/nmeth.1283</a></td>
      <td>iar66keo</td>
      <td>0.364675</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a></td>
      <td><a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_10.html#iefu6hph">Gaggar_2007</a>, <a href="Topic_14.html#06vkoev5">Rajagopala_2015</a></td>
    </tr>
    <tr>
      <th id="e7xwb03g";>442</th>
      <td>Yamashita_2016</td>
      <td>DGV: Dengue Genographic Viewer</td>
      <td>Yamashita, Akifumi; Sakamoto, Tetsuya; Sekizuka, Tsuyoshi; Kato, Kengo; Takasaki, Tomohiko; Kuroda, Makoto</td>
      <td>2016</td>
      <td>2016-06-07</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4894901" target="_blank">PMC4894901</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27375595.0" target="_blank">27375595.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2016.00875" target="_blank">10.3389/fmicb.2016.00875</a></td>
      <td>e7xwb03g</td>
      <td>0.364409</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_30.html#tnvlafa2">Lu_P_2016</a>, <a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a></td>
      <td><a href="Topic_07.html#spxgox52">Yu_J_2019</a>, <a href="Topic_07.html#rah5p1y4">Mohamed_2018</a>, <a href="Topic_07.html#thsq6pnk">Lachheb_2019</a></td>
    </tr>
    <tr>
      <th id="kyq5n601";>443</th>
      <td>Chen_2015</td>
      <td>A Markov chain model for predicting transient particle transport in enclosed environments</td>
      <td>Chen, Chun; Liu, Wei; Lin, Chao-Hsin; Chen, Qingyan</td>
      <td>2015</td>
      <td>2015-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117050" target="_blank">PMC7117050</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2015.03.024" target="_blank">10.1016/j.buildenv.2015.03.024</a></td>
      <td>kyq5n601</td>
      <td>0.362832</td>
      <td><a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#63wqn2fg">Höhle_2007</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_64.html#0ro04s5e">Liu_W_2019</a>, <a href="Topic_64.html#21ee4wxk">Chen_2013</a>, <a href="Topic_17.html#raycnjbl">Fontanini_2015</a>, <a href="Topic_64.html#tep4h5l1">Ding_2013</a></td>
    </tr>
    <tr>
      <th id="486j76e1";>444</th>
      <td>Swindell_2008</td>
      <td>HOW LONG WILL MY MOUSE LIVE? MACHINE LEARNING APPROACHES FOR PREDICTION OF MOUSE LIFESPAN</td>
      <td>Swindell, William R.; Harper, James M.; Miller, Richard A.</td>
      <td>2008</td>
      <td>2008-09-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2693389" target="_blank">PMC2693389</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18840793.0" target="_blank">18840793.0</a></td>
      <td></td>
      <td>486j76e1</td>
      <td>0.362735</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_34.html#etx1iegm">Lorè_2020</a>, <a href="Topic_74.html#9e3z9xdd">Dessale_2017</a>, <a href="Topic_01.html#h919gyvw">Chung_2010</a></td>
    </tr>
    <tr>
      <th id="in8mbp7c";>445</th>
      <td>Li_W_2011</td>
      <td>Cross-Crystal Averaging with Search Models to Improve Molecular Replacement Phases</td>
      <td>Li, Weikai; Li, Fang</td>
      <td>2011</td>
      <td>2011-02-09</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037595" target="_blank">PMC3037595</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21300285.0" target="_blank">21300285.0</a></td>
      <td><a href="https://doi.org/10.1016/j.str.2010.12.007" target="_blank">10.1016/j.str.2010.12.007</a></td>
      <td>in8mbp7c</td>
      <td>0.362612</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_16.html#gbnns8np">RAMAKRISHNA_2015</a>, <a href="Topic_16.html#toglzyci">Jonić_2016</a>, <a href="Topic_16.html#qu4enz7m">Frenz_2019</a></td>
    </tr>
    <tr>
      <th id="oq64d1mb";>446</th>
      <td>Goradia_1988</td>
      <td>Applications of coding theory to the design of somatic cell hybrid panels</td>
      <td>Goradia, Tushar Madhu; Lange, Kenneth</td>
      <td>1988</td>
      <td>1988-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7131684" target="_blank">PMC7131684</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/0025-5564(88)90014-4" target="_blank">10.1016/0025-5564(88)90014-4</a></td>
      <td>oq64d1mb</td>
      <td>0.361554</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_37.html#n5rhhzc9">Vashee_2017</a>, <a href="Topic_21.html#ov92qh6h">Nowak_1984</a>, <a href="Topic_37.html#5c466xvz">Fehr_2019</a>, <a href="Topic_23.html#4lmsl1qv">Tachibana_2011</a></td>
    </tr>
    <tr>
      <th id="9zm4per4";>447</th>
      <td>Dawson_2007</td>
      <td>Prediction of RNA Pseudoknots Using Heuristic Modeling with Mapping and Sequential Folding</td>
      <td>Dawson, Wayne K.; Fujiwara, Kazuya; Kawai, Gota</td>
      <td>2007</td>
      <td>2007-09-19</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1975678" target="_blank">PMC1975678</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17878940.0" target="_blank">17878940.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0000905" target="_blank">10.1371/journal.pone.0000905</a></td>
      <td>9zm4per4</td>
      <td>0.360091</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_17.html#iuglkdcp">Sperschneider_2012</a></td>
    </tr>
    <tr>
      <th id="p7ga4h70";>448</th>
      <td>Jung_2017</td>
      <td>Ontology-Based Approach to Social Data Sentiment Analysis: Detection of Adolescent Depression Signals</td>
      <td>Jung, Hyesil; Park, Hyeoun-Ae; Song, Tae-Min</td>
      <td>2017</td>
      <td>2017-07-24</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5547245" target="_blank">PMC5547245</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28739560.0" target="_blank">28739560.0</a></td>
      <td><a href="https://doi.org/10.2196/jmir.7452" target="_blank">10.2196/jmir.7452</a></td>
      <td>p7ga4h70</td>
      <td>0.359871</td>
      <td><a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a></td>
      <td><a href="Topic_54.html#bp7sray2">Hu_G_2019</a>, <a href="Topic_54.html#smlumvqc">Fung_2015</a>, <a href="Topic_54.html#0anubfxv">Zyoud_2018</a></td>
    </tr>
    <tr>
      <th id="45dpoepu";>449</th>
      <td>Hassanien_2020</td>
      <td>Automatic X-ray COVID-19 Lung Image Classification System based on Multi-Level Thresholding and Support Vector Machine</td>
      <td>Aboul Ella Hassanien; Lamia Nabil Mahdy; Kadry Ali Ezzat; Haytham H. Elmousalami; Hassan Aboul Ella</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.30.20047787" target="_blank">10.1101/2020.03.30.20047787</a></td>
      <td>45dpoepu</td>
      <td>0.359729</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a>, <a href="Topic_17.html#2l1zw19o">Ozturk_2020</a>, <a href="Topic_17.html#ll4rxd9p">Zheng_2020</a></td>
    </tr>
    <tr>
      <th id="j11yaga2";>450</th>
      <td>Meyerholz_2018</td>
      <td>Fundamental Concepts for Semiquantitative Tissue Scoring in Translational Research</td>
      <td>Meyerholz, David K; Beck, Amanda P</td>
      <td>2018</td>
      <td>2018-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6927897" target="_blank">PMC6927897</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30715381.0" target="_blank">30715381.0</a></td>
      <td><a href="https://doi.org/10.1093/ilar/ily025" target="_blank">10.1093/ilar/ily025</a></td>
      <td>j11yaga2</td>
      <td>0.359729</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_60.html#e61vrp7h">Graham_2007</a>, <a href="Topic_20.html#x42w9ssi">Vertrees_2009</a>, <a href="Topic_01.html#mslm0auu">Merrill_2008</a>, <a href="Topic_10.html#lwzwzg15">Meyerholz_2019</a></td>
    </tr>
    <tr>
      <th id="t2y62bg3";>451</th>
      <td>Cinnamon_2016</td>
      <td>Evidence and future potential of mobile phone data for disease disaster management</td>
      <td>Cinnamon, Jonathan; Jones, Sarah K.; Adger, W. Neil</td>
      <td>2016</td>
      <td>2016-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127132" target="_blank">PMC7127132</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.geoforum.2016.07.019" target="_blank">10.1016/j.geoforum.2016.07.019</a></td>
      <td>t2y62bg3</td>
      <td>0.359602</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_04.html#hbpk850a">Xi_Y_2019</a>, <a href="Topic_24.html#fghu8ouc">Hawryluck_2005</a>, <a href="Topic_24.html#i8q5grdh">Leung_2008</a></td>
    </tr>
    <tr>
      <th id="le7v89mp";>452</th>
      <td>Chang_2011</td>
      <td>Phylogenetic Analysis of Protein Sequences Based on Distribution of Length About Common Substring</td>
      <td>Chang, Guisong; Wang, Tianming</td>
      <td>2011</td>
      <td>2011-03-10</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088358" target="_blank">PMC7088358</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21461804.0" target="_blank">21461804.0</a></td>
      <td><a href="https://doi.org/10.1007/s10930-011-9318-0" target="_blank">10.1007/s10930-011-9318-0</a></td>
      <td>le7v89mp</td>
      <td>0.357364</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_07.html#fz9wvup0">Stephensen_1999</a>, <a href="Topic_17.html#gf718lbl">Yu_Z_2010</a>, <a href="Topic_17.html#641pmd1d">Zhang_2017</a></td>
    </tr>
    <tr>
      <th id="y5j07feh";>453</th>
      <td>Li_B_2008</td>
      <td>Flow-injection chemiluminescence determination of chrysin and baicalein assisted by theoretical prediction of chemiluminescence behavior of chrysin and baicalein</td>
      <td>Li, Baoxin; Guo, Lili; Xu, Chunli; Ma, Limei</td>
      <td>2008</td>
      <td>2008-12-01</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.saa.2008.02.007" target="_blank">10.1016/j.saa.2008.02.007</a></td>
      <td>y5j07feh</td>
      <td>0.357309</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_29.html#1qyu5zoo">Zhao_2009</a>, <a href="Topic_58.html#8hyw5z99">Xie_J_2018</a>, <a href="Topic_16.html#83t5mlpr">Guo_L_2015</a>, <a href="Topic_49.html#14vlwsh5">Li_H_2004</a></td>
    </tr>
    <tr>
      <th id="sbnnh2mm";>454</th>
      <td>Pellet_2009</td>
      <td>ViralORFeome: an integrated database to generate a versatile collection of viral ORFs</td>
      <td>Pellet, J.; Tafforeau, L.; Lucas-Hourani, M.; Navratil, V.; Meyniel, L.; Achaz, G.; Guironnet-Paquet, A.; Aublin-Gex, A.; Caignard, G.; Cassonnet, P.; Chaboud, A.; Chantier, T.; Deloire, A.; Demeret, C.; Le Breton, M.; Neveu, G.; Jacotot, L.; Vaglio, P.; Delmotte, S.; Gautier, C.; Combet, C.; Deleage, G.; Favre, M.; Tangy, F.; Jacob, Y.; Andre, P.; Lotteau, V.; Rabourdin-Combe, C.; Vidalain, P. O.</td>
      <td>2009</td>
      <td>2009-12-08</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808970" target="_blank">PMC2808970</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20007148.0" target="_blank">20007148.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkp1000" target="_blank">10.1093/nar/gkp1000</a></td>
      <td>sbnnh2mm</td>
      <td>0.357217</td>
      <td><a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_14.html#06vkoev5">Rajagopala_2015</a>, <a href="Topic_37.html#kihn83ny">Armesto_2007</a></td>
      <td><a href="Topic_21.html#sik7m86l">Tzeng_2006</a>, <a href="Topic_37.html#ndjbrtrv">Satyanarayana_2003</a>, <a href="Topic_21.html#27fd59qe">Meulenberg_1995</a></td>
    </tr>
    <tr>
      <th id="71njvx7l";>455</th>
      <td>Soam_2010</td>
      <td>Improved Prediction of MHC Class I Binders/Non-Binders Peptides Through Artificial Neural Network Using Variable Learning Rate: SARS Corona Virus, a Case Study</td>
      <td>Soam, Sudhir Singh; Bhasker, Bharat; Mishra, Bhartendu Nath</td>
      <td>2010</td>
      <td>2010-11-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123181" target="_blank">PMC7123181</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21431562.0" target="_blank">21431562.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-4419-7046-6_22" target="_blank">10.1007/978-1-4419-7046-6_22</a></td>
      <td>71njvx7l</td>
      <td>0.356709</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_69.html#wuty839o">Wang_2012</a>, <a href="Topic_69.html#71940dh7">Kumar_2019</a>, <a href="Topic_69.html#g5sum116">Hou_Y_2012</a>, <a href="Topic_69.html#wxagjqbt">Prachar_2020</a></td>
    </tr>
    <tr>
      <th id="qzrhihoz";>456</th>
      <td>Brahmbhatt_2016</td>
      <td>Epidemiology-based Task Assignment Algorithm for Distributed Systems</td>
      <td>Brahmbhatt, Parth; Camorlinga, Sergio G.</td>
      <td>2016</td>
      <td>2016-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2016.09.356" target="_blank">10.1016/j.procs.2016.09.356</a></td>
      <td>qzrhihoz</td>
      <td>0.355860</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#52esrbby">Krause_2007</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a></td>
      <td><a href="Topic_06.html#vqagpbiu">Denphedtnong_2013</a>, <a href="Topic_06.html#q1imoe7k">Zhong_2009</a>, <a href="Topic_17.html#397hp1yt">Coelho_2008</a></td>
    </tr>
    <tr>
      <th id="x3u9i1vq";>457</th>
      <td>Rose_2016</td>
      <td>Challenges in the analysis of viral metagenomes</td>
      <td>Rose, Rebecca; Constantinides, Bede; Tapinos, Avraam; Robertson, David L; Prosperi, Mattia</td>
      <td>2016</td>
      <td>2016-08-03</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5822887" target="_blank">PMC5822887</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29492275.0" target="_blank">29492275.0</a></td>
      <td><a href="https://doi.org/10.1093/ve/vew022" target="_blank">10.1093/ve/vew022</a></td>
      <td>x3u9i1vq</td>
      <td>0.355768</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_45.html#8pr5ispb">Warwick-Dugdale_2019</a>, <a href="Topic_10.html#jw4n2j6y">Smits_2014</a></td>
    </tr>
    <tr>
      <th id="0kkm0tgj";>458</th>
      <td>Zhang_2007</td>
      <td>An emerging cyberinfrastructure for biodefense pathogen and pathogen–host data</td>
      <td>Zhang, C.; Crasta, O.; Cammer, S.; Will, R.; Kenyon, R.; Sullivan, D.; Yu, Q.; Sun, W.; Jha, R.; Liu, D.; Xue, T.; Zhang, Y.; Moore, M.; McGarvey, P.; Huang, H.; Chen, Y.; Zhang, J.; Mazumder, R.; Wu, C.; Sobral, B.</td>
      <td>2007</td>
      <td>2007-11-04</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2239001" target="_blank">PMC2239001</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17984082.0" target="_blank">17984082.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm903" target="_blank">10.1093/nar/gkm903</a></td>
      <td>0kkm0tgj</td>
      <td>0.355768</td>
      <td><a href="Topic_17.html#a564l6vs">UNKNOWN_2016</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a></td>
      <td><a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_49.html#bmdpbkni">Tabares_2011</a></td>
    </tr>
    <tr>
      <th id="ob37jivi";>459</th>
      <td>Milanetti_2020</td>
      <td>In-Silico evidence for two receptors based strategy of SARS-CoV-2</td>
      <td>Edoardo Milanetti; Mattia Miotto; Lorenzo Di Rienzo; Michele Monti; Giorgio Gosti; Giancarlo Ruocco</td>
      <td>2020</td>
      <td>2020-03-27</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.24.006197" target="_blank">10.1101/2020.03.24.006197</a></td>
      <td>ob37jivi</td>
      <td>0.355660</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_41.html#gfiqjgxg">Zhang_2009</a>, <a href="Topic_16.html#8c05w1ix">Yap_Y_2005</a>, <a href="Topic_16.html#32ju2h1g">Niu_C_2008</a>, <a href="Topic_18.html#bsbgjt1y">Lee_H_2013</a></td>
    </tr>
    <tr>
      <th id="t89235t3";>460</th>
      <td>Zheng_2005</td>
      <td>Coronavirus phylogeny based on a geometric approach</td>
      <td>Zheng, Wen-Xin; Chen, Ling-Ling; Ou, Hong-Yu; Gao, Feng; Zhang, Chun-Ting</td>
      <td>2005</td>
      <td>2005-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7111192" target="_blank">PMC7111192</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15890535.0" target="_blank">15890535.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ympev.2005.03.030" target="_blank">10.1016/j.ympev.2005.03.030</a></td>
      <td>t89235t3</td>
      <td>0.353632</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#v38tjof3">Fahmi_2020</a>, <a href="Topic_07.html#tvhbox59">Di_Martino_2009</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a></td>
    </tr>
    <tr>
      <th id="kl99afiu";>461</th>
      <td>Xiang_2020</td>
      <td>VPTMdb: a viral post-translational modification database</td>
      <td>Yujia Xiang; Quan Zou; Lilin Zhao</td>
      <td>2020</td>
      <td>2020-04-02</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.01.019562" target="_blank">10.1101/2020.04.01.019562</a></td>
      <td>kl99afiu</td>
      <td>0.353632</td>
      <td><a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_10.html#vy4hfd62">Wimmer_2015</a></td>
    </tr>
    <tr>
      <th id="j97ugs3y";>462</th>
      <td>Santosh_2020</td>
      <td>AI-Driven Tools for Coronavirus Outbreak: Need of Active Learning and Cross-Population Train/Test Models on Multitudinal/Multimodal Data</td>
      <td>Santosh, K. C.</td>
      <td>2020</td>
      <td>2020-03-18</td>
      <td>PMC</td>
      <td>Y</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087612" target="_blank">PMC7087612</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32189081.0" target="_blank">32189081.0</a></td>
      <td><a href="https://doi.org/10.1007/s10916-020-01562-1" target="_blank">10.1007/s10916-020-01562-1</a></td>
      <td>j97ugs3y</td>
      <td>0.352249</td>
      <td><a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_71.html#22wnp1sv">von_der_Gracht_2010</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_06.html#na4mrzi7">Skrip_2019</a></td>
      <td><a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_17.html#nvavj9gk">Andre_2020</a>, <a href="Topic_14.html#kjovtgua">Wu_J_2020</a>, <a href="Topic_18.html#m0q7rm6z">Mahapatra_2020</a></td>
    </tr>
    <tr>
      <th id="x8zdlml2";>463</th>
      <td>Fertil_2005</td>
      <td>GENSTYLE: exploration and analysis of DNA sequences with genomic signature</td>
      <td>Fertil, Bernard; Massin, Matthieu; Lespinats, Sylvain; Devic, Caroline; Dumee, Philippe; Giron, Alain</td>
      <td>2005</td>
      <td>2005-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1160249" target="_blank">PMC1160249</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15980524.0" target="_blank">15980524.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gki489" target="_blank">10.1093/nar/gki489</a></td>
      <td>x8zdlml2</td>
      <td>0.352249</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#wj7yonjw">Gardner_2008</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a></td>
      <td><a href="Topic_17.html#yba7mdtb">Dufraigne_2005</a>, <a href="Topic_45.html#1a9u53za">Shabman_2016</a>, <a href="Topic_21.html#xl4u7n6r">Jonassen_2007</a></td>
    </tr>
    <tr>
      <th id="mrbi0a7r";>464</th>
      <td>Izakian_2012</td>
      <td>A new PSO-optimized geometry of spatial and spatio-temporal scan statistics for disease outbreak detection</td>
      <td>Izakian, Hesam; Pedrycz, Witold</td>
      <td>2012</td>
      <td>2012-06-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7104009" target="_blank">PMC7104009</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.swevo.2012.02.001" target="_blank">10.1016/j.swevo.2012.02.001</a></td>
      <td>mrbi0a7r</td>
      <td>0.351755</td>
      <td><a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#vt6nh3vs">Kenah_2007</a>, <a href="Topic_06.html#tlluxd11">Welch_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_39.html#fpiew1jy">Takahashi_2008</a>, <a href="Topic_39.html#sxwwi4zq">Cao_Z_2010</a>, <a href="Topic_06.html#q5cmwzr8">Wen_T_2007</a>, <a href="Topic_39.html#g0dskjv5">Routledge_2019</a></td>
    </tr>
    <tr>
      <th id="u2mt3xzc";>465</th>
      <td>smith_2019</td>
      <td>Kin and group selection are both flawed but useful data analysis tools</td>
      <td>jeff smith; R. Fredrik Inglis</td>
      <td>2019</td>
      <td>2019-08-29</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/742122" target="_blank">10.1101/742122</a></td>
      <td>u2mt3xzc</td>
      <td>0.351439</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a>, <a href="Topic_01.html#qlypp5z6">Bauer_2009</a></td>
      <td><a href="Topic_02.html#qv5zsleh">Baele_2016</a>, <a href="Topic_02.html#5gzgfudh">Getz_2019</a>, <a href="Topic_02.html#05y53vbg">Hünemeier_2012</a>, <a href="Topic_06.html#l0uya9dh">Lemey_2009</a></td>
    </tr>
    <tr>
      <th id="azkamnpa";>466</th>
      <td>Ecker_2005</td>
      <td>The Microbial Rosetta Stone Database: A compilation of global and emerging infectious microorganisms and bioterrorist threat agents</td>
      <td>Ecker, David J; Sampath, Rangarajan; Willett, Paul; Wyatt, Jacqueline R; Samant, Vivek; Massire, Christian; Hall, Thomas A; Hari, Kumar; McNeil, John A; Büchen-Osmond, Cornelia; Budowle, Bruce</td>
      <td>2005</td>
      <td>2005-04-25</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1127111" target="_blank">PMC1127111</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15850481.0" target="_blank">15850481.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2180-5-19" target="_blank">10.1186/1471-2180-5-19</a></td>
      <td>azkamnpa</td>
      <td>0.351439</td>
      <td><a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_01.html#vfhlph80">Alves_2007</a>, <a href="Topic_01.html#s6x0igia">Dove_2004</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_36.html#tulne0v3">Rabaa_2015</a>, <a href="Topic_69.html#ebl8yc92">Hoppe_2014</a></td>
    </tr>
    <tr>
      <th id="ctpgylnr";>467</th>
      <td>Yuan_2013</td>
      <td>Monitoring Influenza Epidemics in China with Search Query from Baidu</td>
      <td>Yuan, Qingyu; Nsoesie, Elaine O.; Lv, Benfu; Peng, Geng; Chunara, Rumi; Brownstein, John S.</td>
      <td>2013</td>
      <td>2013-05-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667820" target="_blank">PMC3667820</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23750192.0" target="_blank">23750192.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0064323" target="_blank">10.1371/journal.pone.0064323</a></td>
      <td>ctpgylnr</td>
      <td>0.351167</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#yupm6smb">Liang_2018</a>, <a href="Topic_22.html#fp76ssuu">Chen_2019</a>, <a href="Topic_19.html#tc8eru1w">Zhan_2020</a></td>
    </tr>
    <tr>
      <th id="g2f151oc";>468</th>
      <td>Whiteford_2005</td>
      <td>An analysis of the feasibility of short read sequencing</td>
      <td>Whiteford, Nava; Haslam, Niall; Weber, Gerald; Prügel-Bennett, Adam; Essex, Jonathan W.; Roach, Peter L.; Bradley, Mark; Neylon, Cameron</td>
      <td>2005</td>
      <td>2005-11-07</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1278949" target="_blank">PMC1278949</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16275781.0" target="_blank">16275781.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gni170" target="_blank">10.1093/nar/gni170</a></td>
      <td>g2f151oc</td>
      <td>0.351048</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a>, <a href="Topic_17.html#ojy6s49w">Warren_2007</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_21.html#fcvddgvy">Viehweger_2019</a>, <a href="Topic_21.html#xr2bs4bk">Viehweger_2018</a></td>
    </tr>
    <tr>
      <th id="48jx3de2";>469</th>
      <td>Mohammadzadeh_2017</td>
      <td>A data mining approach for modeling churn behavior via RFM model in specialized clinics Case study: A public sector hospital in Tehran</td>
      <td>Mohammadzadeh, Mehdi; Hoseini, Zeinab Zare; Derafshi, Hamid</td>
      <td>2017</td>
      <td>2017-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.procs.2017.11.206" target="_blank">10.1016/j.procs.2017.11.206</a></td>
      <td>48jx3de2</td>
      <td>0.350963</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_55.html#kaip0d1i">John_2019</a>, <a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_65.html#tk3861u0">Zhao_2020</a></td>
    </tr>
    <tr>
      <th id="s9eez11q";>470</th>
      <td>TANG_2007</td>
      <td>Group Argumentation and Its Analysis on a Highlighted Social Event — Practice of Qualitative Meta-synthesis</td>
      <td>TANG, Xi-jin; LIU, Yi-jun</td>
      <td>2007</td>
      <td>2007-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/s1874-8651(08)60023-x" target="_blank">10.1016/s1874-8651(08)60023-x</a></td>
      <td>s9eez11q</td>
      <td>0.350164</td>
      <td><a href="Topic_28.html#8ajy3e55">Ropeik_2004</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_04.html#a2zxfl6m">Haga_2006</a>, <a href="Topic_04.html#g04lgor0">UNKNOWN_2019</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_17.html#e2coy5ms">Matsuzaw_2011</a>, <a href="Topic_04.html#w2mwl436">Cheng_2020</a>, <a href="Topic_04.html#9lgr2xyg">Chen_2011</a>, <a href="Topic_04.html#oaqqh21e">Georgalakis_2020</a></td>
    </tr>
    <tr>
      <th id="k23xlzj0";>471</th>
      <td>Delli Ponti_2017</td>
      <td>A high-throughput approach to profile RNA structure</td>
      <td>Delli Ponti, Riccardo; Marti, Stefanie; Armaos, Alexandros; Tartaglia, Gian Gaetano</td>
      <td>2017</td>
      <td>2017-03-17</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5389523" target="_blank">PMC5389523</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27899588.0" target="_blank">27899588.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1094" target="_blank">10.1093/nar/gkw1094</a></td>
      <td>k23xlzj0</td>
      <td>0.350098</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_34.html#qbldmef1">Yu_H_2007</a>, <a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a></td>
      <td><a href="Topic_21.html#4gjo72hx">Tremblay-Létourneau_2011</a>, <a href="Topic_61.html#xuxlw4c0">Kolchanov_1996</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a></td>
    </tr>
    <tr>
      <th id="qpuaokuz";>472</th>
      <td>Yeh_J_2008</td>
      <td>Sunscope: A Video-guided Intubation System Through a Detachable Imaging Probe</td>
      <td>Yeh, Jia-Rong; Shieh, Jiann-Shing; Lin, Chih-Peng; Sun, Wei-Zen</td>
      <td>2008</td>
      <td>2008-06-30</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18593652.0" target="_blank">18593652.0</a></td>
      <td><a href="https://doi.org/10.1016/s1875-4597(08)60029-6" target="_blank">10.1016/s1875-4597(08)60029-6</a></td>
      <td>qpuaokuz</td>
      <td>0.350061</td>
      <td><a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#43igjn6x">Kylberg_2009</a></td>
      <td><a href="Topic_24.html#eptit46j">Wang_2012</a>, <a href="Topic_24.html#xj2gln49">Amirav_2015</a>, <a href="Topic_79.html#d2c3tavy">Kang_2018</a></td>
    </tr>
    <tr>
      <th id="1lgi1e60";>473</th>
      <td>Freifeld_2008</td>
      <td>HealthMap: Global Infectious Disease Monitoring through Automated Classification and Visualization of Internet Media Reports</td>
      <td>Freifeld, Clark C.; Mandl, Kenneth D.; Reis, Ben Y.; Brownstein, John S.</td>
      <td>2008</td>
      <td>2008-04-30</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2274789" target="_blank">PMC2274789</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18096908.0" target="_blank">18096908.0</a></td>
      <td><a href="https://doi.org/10.1197/jamia.m2544" target="_blank">10.1197/jamia.m2544</a></td>
      <td>1lgi1e60</td>
      <td>0.350055</td>
      <td><a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a></td>
      <td><a href="Topic_04.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_04.html#6q8eg9z4">Keller_2009</a>, <a href="Topic_17.html#r0lduvs1">Zhang_2009</a></td>
    </tr>
    <tr>
      <th id="ckby80vf";>474</th>
      <td>Gardner_2005</td>
      <td>Software for optimization of SNP and PCR-RFLP genotyping to discriminate many genomes with the fewest assays</td>
      <td>Gardner, Shea N; Wagner, Mark C</td>
      <td>2005</td>
      <td>2005-05-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1156889" target="_blank">PMC1156889</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15904493.0" target="_blank">15904493.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2164-6-73" target="_blank">10.1186/1471-2164-6-73</a></td>
      <td>ckby80vf</td>
      <td>0.349802</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a></td>
      <td><a href="Topic_07.html#frluobkw">Wang_1996</a>, <a href="Topic_07.html#ahifxfhn">Jang_2007</a>, <a href="Topic_43.html#5iqkh85d">Dettogni_2013</a></td>
    </tr>
    <tr>
      <th id="6mb9w76f";>475</th>
      <td>Chen_2015</td>
      <td>Nuclear Magnetic Resonance-Assisted Prediction of Secondary Structure for RNA: Incorporation of Direction-Dependent Chemical Shift Constraints</td>
      <td>Chen, Jonathan\nL.; Bellaousov, Stanislav; Tubbs, Jason D.; Kennedy, Scott D.; Lopez, Michael J.; Mathews, David H.; Turner, Douglas H.</td>
      <td>2015</td>
      <td>2015-10-09</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4666457" target="_blank">PMC4666457</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26451676.0" target="_blank">26451676.0</a></td>
      <td><a href="https://doi.org/10.1021/acs.biochem.5b00833" target="_blank">10.1021/acs.biochem.5b00833</a></td>
      <td>6mb9w76f</td>
      <td>0.348715</td>
      <td><a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_61.html#9zm4per4">Dawson_2007</a></td>
      <td><a href="Topic_61.html#n7o7x9ta">Cornish_2006</a>, <a href="Topic_61.html#tv8hi2wy">Low_J_2010</a></td>
    </tr>
    <tr>
      <th id="eaglecq7";>476</th>
      <td>Ji_X_2020</td>
      <td>TWIRLS, an automated topic-wise inference method based on massive literature, suggests a possible mechanism via ACE2 for the pathological changes in the human host after coronavirus infection</td>
      <td>Xiaoyang Ji; Chunming Zhang; Yubo Zhai; Zhonghai Zhang; Chunli Zhang; Yiqing Xue; Guangming Tan; Gang Niu</td>
      <td>2020</td>
      <td>2020-03-02</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.27.967588" target="_blank">10.1101/2020.02.27.967588</a></td>
      <td>eaglecq7</td>
      <td>0.348560</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#cx3ex9ru">Leach_2010</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a></td>
      <td><a href="Topic_17.html#f21dknmb">Ji_X_2020</a>, <a href="Topic_17.html#aeyf0yu1">Joshi_2020</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_54.html#nm0i0rtz">Choi_2017</a></td>
    </tr>
    <tr>
      <th id="ltf82dh3";>477</th>
      <td>Liu_W_2006</td>
      <td>Quantitative prediction of mouse class I MHC peptide binding affinity using support vector machine regression (SVR) models</td>
      <td>Liu, Wen; Meng, Xiangshan; Xu, Qiqi; Flower, Darren R; Li, Tongbin</td>
      <td>2006</td>
      <td>2006-03-31</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1513606" target="_blank">PMC1513606</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16579851.0" target="_blank">16579851.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-7-182" target="_blank">10.1186/1471-2105-7-182</a></td>
      <td>ltf82dh3</td>
      <td>0.347292</td>
      <td><a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_69.html#wuty839o">Wang_2012</a>, <a href="Topic_17.html#71njvx7l">Soam_2010</a></td>
    </tr>
    <tr>
      <th id="x1zq2i9h";>478</th>
      <td>Al-Qaness_2020</td>
      <td>Optimization Method for Forecasting Confirmed Cases of COVID-19 in China</td>
      <td>Al-Qaness, Mohammed A. A.; Ewees, Ahmed A.; Fan, Hong; Abd El Aziz, Mohamed</td>
      <td>2020</td>
      <td>2020-01-01</td>
      <td>COMM-USE</td>
      <td>Y</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32131537.0" target="_blank">32131537.0</a></td>
      <td><a href="https://doi.org/10.3390/jcm9030674" target="_blank">10.3390/jcm9030674</a></td>
      <td>x1zq2i9h</td>
      <td>0.347007</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_19.html#5dd89gnm">Liu_Q_2020</a>, <a href="Topic_06.html#p42cgpf0">Zareie_2020</a>, <a href="Topic_19.html#8b1esll4">Huang_2020</a>, <a href="Topic_19.html#s12vwz8m">youbin_2020</a></td>
    </tr>
    <tr>
      <th id="52nw9gxq";>479</th>
      <td>Martin_2020</td>
      <td>An artificial intelligence-based first-line defence against COVID-19: digitally screening citizens for risks via a chatbot</td>
      <td>Alistair Martin; Jama Nateqi; Stefanie Gruarin; Nicolas Munsch; Isselmou Abdarahmane; Bernhard Knapp</td>
      <td>2020</td>
      <td>2020-03-26</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.25.008805" target="_blank">10.1101/2020.03.25.008805</a></td>
      <td>52nw9gxq</td>
      <td>0.346855</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a></td>
      <td><a href="Topic_14.html#kjovtgua">Wu_J_2020</a>, <a href="Topic_17.html#j97ugs3y">Santosh_2020</a>, <a href="Topic_09.html#vde6vm2a">Marchese-Ragona_2020</a></td>
    </tr>
    <tr>
      <th id="4524vg4x";>480</th>
      <td>Collier_2007</td>
      <td>A multilingual ontology for infectious disease surveillance: rationale, design and challenges</td>
      <td>Collier, Nigel; Kawazoe, Ai; Jin, Lihua; Shigematsu, Mika; Dien, Dinh; Barrero, Roberto A.; Takeuchi, Koichi; Kawtrakul, Asanee</td>
      <td>2007</td>
      <td>2007-06-26</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087677" target="_blank">PMC7087677</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10579-007-9019-7" target="_blank">10.1007/s10579-007-9019-7</a></td>
      <td>4524vg4x</td>
      <td>0.346634</td>
      <td><a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_04.html#f4q1r3vu">Keller_2009</a>, <a href="Topic_54.html#t02jngq0">Ramshaw_2019</a>, <a href="Topic_36.html#tulne0v3">Rabaa_2015</a></td>
    </tr>
    <tr>
      <th id="dk7eij1l";>481</th>
      <td>Araújo_2011</td>
      <td>A rigorous approach to facilitate and guarantee the correctness of the genetic testing management in human genome information systems</td>
      <td>Araújo, Luciano V; Malkowski, Simon; Braghetto, Kelly R; Passos-Bueno, Maria R; Zatz, Mayana; Pu, Calton; Ferreira, João E</td>
      <td>2011</td>
      <td>2011-12-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3287582" target="_blank">PMC3287582</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22369688.0" target="_blank">22369688.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2164-12-s4-s13" target="_blank">10.1186/1471-2164-12-s4-s13</a></td>
      <td>dk7eij1l</td>
      <td>0.346209</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a></td>
      <td><a href="Topic_14.html#hcbvfpg0">Aarestrup_2012</a>, <a href="Topic_01.html#q6h1f69g">Dwivedi_2017</a>, <a href="Topic_14.html#nhb4o6ty">Johnson_2004</a></td>
    </tr>
    <tr>
      <th id="gmw88elf";>482</th>
      <td>Vella_2017</td>
      <td>From protein-protein interactions to protein co-expression networks: a new perspective to evaluate large-scale proteomic data</td>
      <td>Vella, Danila; Zoppis, Italo; Mauri, Giancarlo; Mauri, Pierluigi; Di Silvestre, Dario</td>
      <td>2017</td>
      <td>2017-03-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5359264" target="_blank">PMC5359264</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28477207.0" target="_blank">28477207.0</a></td>
      <td><a href="https://doi.org/10.1186/s13637-017-0059-z" target="_blank">10.1186/s13637-017-0059-z</a></td>
      <td>gmw88elf</td>
      <td>0.343141</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#hr4smx24">van_Kampen_2015</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_69.html#elb93kp6">Li_S_2012</a>, <a href="Topic_17.html#qyd6qyjf">Xiao_2017</a>, <a href="Topic_34.html#s8u5mfhv">Abraham_2015</a>, <a href="Topic_17.html#g55witln">Gibbs_2013</a></td>
    </tr>
    <tr>
      <th id="ne9d0siv";>483</th>
      <td>Banks_2006</td>
      <td>Dissecting the ancient rapid radiation of microgastrine wasp genera using additional nuclear genes</td>
      <td>Banks, Jonathan C.; Whitfield, James B.</td>
      <td>2006</td>
      <td>2006-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7129091" target="_blank">PMC7129091</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16854601.0" target="_blank">16854601.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ympev.2006.06.001" target="_blank">10.1016/j.ympev.2006.06.001</a></td>
      <td>ne9d0siv</td>
      <td>0.342956</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#3atcqejc">Kinene_2016</a>, <a href="Topic_02.html#tiyjmih7">Francis_2016</a></td>
      <td><a href="Topic_02.html#jl3uah1i">Harper_2013</a>, <a href="Topic_10.html#goidlh08">Walker_2011</a></td>
    </tr>
    <tr>
      <th id="r707jl16";>484</th>
      <td>Bhuvaneshwar_2018</td>
      <td>viGEN: An Open Source Pipeline for the Detection and Quantification of Viral RNA in Human Tumors</td>
      <td>Bhuvaneshwar, Krithika; Song, Lei; Madhavan, Subha; Gusev, Yuriy</td>
      <td>2018</td>
      <td>2018-06-05</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5996193" target="_blank">PMC5996193</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29922260.0" target="_blank">29922260.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2018.01172" target="_blank">10.3389/fmicb.2018.01172</a></td>
      <td>r707jl16</td>
      <td>0.342036</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a></td>
      <td><a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a>, <a href="Topic_21.html#d9urjb9e">Masante_2015</a></td>
    </tr>
    <tr>
      <th id="jsyuh368";>485</th>
      <td>Chu_H_2008</td>
      <td>A Delphi-based approach to developing expert systems with the cooperation of multiple experts</td>
      <td>Chu, Hui-Chun; Hwang, Gwo-Jen</td>
      <td>2008</td>
      <td>2008-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127119" target="_blank">PMC7127119</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.eswa.2007.05.034" target="_blank">10.1016/j.eswa.2007.05.034</a></td>
      <td>jsyuh368</td>
      <td>0.341299</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_04.html#g04lgor0">UNKNOWN_2019</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a></td>
      <td><a href="Topic_17.html#wr64fzjf">Serrano-Guerrero_2020</a>, <a href="Topic_01.html#8bws4xpp">Dorasamy_2013</a>, <a href="Topic_17.html#e2coy5ms">Matsuzaw_2011</a>, <a href="Topic_04.html#rxvnn7p5">Ugwu_2006</a></td>
    </tr>
    <tr>
      <th id="wslu5fof";>486</th>
      <td>Weir_2017</td>
      <td>Development of a microbial dose response visualization and modelling application for QMRA modelers and educators</td>
      <td>Weir, Mark H.; Mitchell, Jade; Flynn, William; Pope, Joanna M.</td>
      <td>2017</td>
      <td>2017-02-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5665384" target="_blank">PMC5665384</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29104445.0" target="_blank">29104445.0</a></td>
      <td><a href="https://doi.org/10.1016/j.envsoft.2016.11.011" target="_blank">10.1016/j.envsoft.2016.11.011</a></td>
      <td>wslu5fof</td>
      <td>0.339722</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_04.html#h0wyqnkh">Grant_2016</a>, <a href="Topic_64.html#95hx0nd4">Van_Leuken_2016</a>, <a href="Topic_06.html#vmnqlcyh">Alexander_2009</a></td>
    </tr>
    <tr>
      <th id="7t4lizde";>487</th>
      <td>Li_Z_2007</td>
      <td>Structural parameterization and functional prediction of antigenic polypeptome sequences with biological activity through quantitative sequence-activity models (QSAM) by molecular electronegativity edge-distance vector (VMED)</td>
      <td>Li, ZhiLiang; Wu, ShiRong; Chen, ZeCong; Ye, Nancy; Yang, ShengXi; Liao, ChunYang; Zhang, MengJun; Yang, Li; Mei, Hu; Yang, Yan; Zhao, Na; Zhou, Yuan; Zhou, Ping; Xiong, Qing; Xu, Hong; Liu, ShuShen; Ling, ZiHua; Chen, Gang; Li, GenRong</td>
      <td>2007</td>
      <td>2007-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089106" target="_blank">PMC7089106</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17879071.0" target="_blank">17879071.0</a></td>
      <td><a href="https://doi.org/10.1007/s11427-007-0080-7" target="_blank">10.1007/s11427-007-0080-7</a></td>
      <td>7t4lizde</td>
      <td>0.339455</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_26.html#5afv6ki2">Gordon_1991</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a></td>
    </tr>
    <tr>
      <th id="c674xxbo";>488</th>
      <td>Farina_2014</td>
      <td>Chapter 22 Integrated Sensor Systems and Data Fusion for Homeland Protection</td>
      <td>Farina, Alfonso; Ortenzi, Luciana; Ristic, Branko; Skvortsov, Alex</td>
      <td>2014</td>
      <td>2014-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-396500-4.00022-3" target="_blank">10.1016/b978-0-12-396500-4.00022-3</a></td>
      <td>c674xxbo</td>
      <td>0.339073</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_17.html#vgq2nt3q">Fontanini_2016</a>, <a href="Topic_04.html#et23099z">Quwaider_2016</a></td>
    </tr>
    <tr>
      <th id="y0wf456f";>489</th>
      <td>Zhang_2005</td>
      <td>PRED(BALB/c): a system for the prediction of peptide binding to H2(d) molecules, a haplotype of the BALB/c mouse</td>
      <td>Zhang, Guang Lan; Srinivasan, Kellathur N.; Veeramani, Anitha; August, J. Thomas; Brusic, Vladimir</td>
      <td>2005</td>
      <td>2005-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1160239" target="_blank">PMC1160239</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15980450.0" target="_blank">15980450.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gki479" target="_blank">10.1093/nar/gki479</a></td>
      <td>y0wf456f</td>
      <td>0.337982</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_69.html#g5sum116">Hou_Y_2012</a></td>
    </tr>
    <tr>
      <th id="o13ti3xx";>490</th>
      <td>Le_Mercier_2019</td>
      <td>A Bioinformatics View of Glycan–Virus Interactions</td>
      <td>Le Mercier, Philippe; Mariethoz, Julien; Lascano-Maillard, Josefina; Bonnardel, François; Imberty, Anne; Ricard-Blum, Sylvie; Lisacek, Frédérique</td>
      <td>2019</td>
      <td>2019-04-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6521074" target="_blank">PMC6521074</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31018588.0" target="_blank">31018588.0</a></td>
      <td><a href="https://doi.org/10.3390/v11040374" target="_blank">10.3390/v11040374</a></td>
      <td>o13ti3xx</td>
      <td>0.337634</td>
      <td><a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_01.html#hr4smx24">van_Kampen_2015</a></td>
      <td><a href="Topic_10.html#zdr0f2uq">Watanabe_2019</a>, <a href="Topic_10.html#1bxoxy30">Vigerust_2007</a>, <a href="Topic_10.html#lpbma80s">Vigerust_2011</a>, <a href="Topic_10.html#wrztpeb8">Zhang_2015</a></td>
    </tr>
    <tr>
      <th id="bmcku2yb";>491</th>
      <td>Wilm_2012</td>
      <td>LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets</td>
      <td>Wilm, Andreas; Aw, Pauline Poh Kim; Bertrand, Denis; Yeo, Grace Hui Ting; Ong, Swee Hoe; Wong, Chang Hua; Khor, Chiea Chuen; Petric, Rosemary; Hibberd, Martin Lloyd; Nagarajan, Niranjan</td>
      <td>2012</td>
      <td>2012-10-12</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" target="_blank">PMC3526318</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23066108.0" target="_blank">23066108.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gks918" target="_blank">10.1093/nar/gks918</a></td>
      <td>bmcku2yb</td>
      <td>0.336549</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_14.html#16tii0ha">Pandya_2007</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_14.html#604gho1u">Chen-Harris_2013</a>, <a href="Topic_02.html#rloey02j">Harel_2019</a>, <a href="Topic_14.html#3jxyipbn">Lewandowska_2017</a>, <a href="Topic_21.html#fcvddgvy">Viehweger_2019</a></td>
    </tr>
    <tr>
      <th id="2scgsmsc";>492</th>
      <td>Chou_2011</td>
      <td>Some remarks on protein attribute prediction and pseudo amino acid composition</td>
      <td>Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7125570" target="_blank">PMC7125570</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21168420.0" target="_blank">21168420.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jtbi.2010.12.024" target="_blank">10.1016/j.jtbi.2010.12.024</a></td>
      <td>2scgsmsc</td>
      <td>0.336300</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_17.html#2dbpm0nt">Choo_2009</a>, <a href="Topic_17.html#c3ewuebx">Jia_J_2016</a>, <a href="Topic_17.html#fsh3gsvd">Xiao_2005</a>, <a href="Topic_17.html#yjrs7c38">De_Ferrari_2014</a></td>
    </tr>
    <tr>
      <th id="y9y7d96c";>493</th>
      <td>Besançon_2020</td>
      <td>Umbrella Visualization: A method of analysis dedicated to glycan flexibility with UnityMol</td>
      <td>Besançon, Camille; Guillot, Alexandre; Blaise, Sébastien; Dauchez, Manuel; Belloy, Nicolas; Prévoteau-Jonquet, Jessica; Baud, Stéphanie</td>
      <td>2020</td>
      <td>2020-02-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7128144" target="_blank">PMC7128144</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31302178.0" target="_blank">31302178.0</a></td>
      <td><a href="https://doi.org/10.1016/j.ymeth.2019.07.010" target="_blank">10.1016/j.ymeth.2019.07.010</a></td>
      <td>y9y7d96c</td>
      <td>0.336087</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_41.html#3mujx7j4">Cao_L_2018</a>, <a href="Topic_26.html#jljpb7rv">Voß_D_2009</a>, <a href="Topic_16.html#jatetuim">Satoh_2015</a>, <a href="Topic_17.html#do0gb0g6">van_Dijk_2008</a></td>
    </tr>
    <tr>
      <th id="24qizp9k";>494</th>
      <td>Robinson_2006</td>
      <td>Validation of the Provincial Transfer Authorization Centre database: a comprehensive database containing records of all inter-facility patient transfers in the province of Ontario</td>
      <td>Robinson, Victoria A; MacDonald, Russell D; Manuel, Doug; Goel, Vivek</td>
      <td>2006</td>
      <td>2006-10-06</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1609112" target="_blank">PMC1609112</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17026763.0" target="_blank">17026763.0</a></td>
      <td><a href="https://doi.org/10.1186/1472-6963-6-129" target="_blank">10.1186/1472-6963-6-129</a></td>
      <td>24qizp9k</td>
      <td>0.334957</td>
      <td><a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a></td>
      <td><a href="Topic_24.html#o6tuulli">Damji_2005</a>, <a href="Topic_24.html#u7eppma9">MacDonald_2004</a>, <a href="Topic_55.html#lgbyaczs">AlRuthia_2019</a></td>
    </tr>
    <tr>
      <th id="oqxh7egj";>495</th>
      <td>Eswar_2007</td>
      <td>4.10 Comparative Modeling of Drug Target Proteins</td>
      <td>Eswar, N.; Sali, A.</td>
      <td>2007</td>
      <td>2007-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b0-08-045044-x/00251-0" target="_blank">10.1016/b0-08-045044-x/00251-0</a></td>
      <td>oqxh7egj</td>
      <td>0.333666</td>
      <td><a href="Topic_01.html#d66rz6dc">Webb_2014</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a></td>
      <td><a href="Topic_01.html#d66rz6dc">Webb_2014</a>, <a href="Topic_18.html#4g5dur1n">Ramajayam_2010</a>, <a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_16.html#olfpcndb">Thanigaimalai_2013</a></td>
    </tr>
    <tr>
      <th id="xrxttrh2";>496</th>
      <td>Yadav_2013</td>
      <td>Sequencing and Computational Approaches to Identification and Characterization of Microbial Organisms</td>
      <td>Yadav, Brijesh Singh; Ronda, Venkateswarlu; Vashista, Dinesh P; Sharma, Bhaskar</td>
      <td>2013</td>
      <td>2013-05-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147756" target="_blank">PMC4147756</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25288901.0" target="_blank">25288901.0</a></td>
      <td><a href="https://doi.org/10.4137/becb.s10886" target="_blank">10.4137/becb.s10886</a></td>
      <td>xrxttrh2</td>
      <td>0.333537</td>
      <td><a href="Topic_01.html#rzx9hy0o">Vernikos_2008</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a>, <a href="Topic_10.html#dkfrk71a">Smith_2013</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a></td>
      <td><a href="Topic_14.html#ag3n4ecn">Pokhriyal_2019</a>, <a href="Topic_07.html#15cxc32n">Shifman_2019</a>, <a href="Topic_50.html#eifztaov">Selvin_2019</a>, <a href="Topic_34.html#e0wmnfyn">McGuire_2012</a></td>
    </tr>
    <tr>
      <th id="p8n8va3y";>497</th>
      <td>Kim_M_2013</td>
      <td>A mass weighted chemical elastic network model elucidates closed form domain motions in proteins</td>
      <td>Kim, Min Hyeok; Seo, Sangjae; Jeong, Jay Il; Kim, Bum Joon; Liu, Wing Kam; Lim, Byeong Soo; Choi, Jae Boong; Kim, Moon Ki</td>
      <td>2013</td>
      <td>2013-03-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3649262" target="_blank">PMC3649262</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23456820.0" target="_blank">23456820.0</a></td>
      <td><a href="https://doi.org/10.1002/pro.2244" target="_blank">10.1002/pro.2244</a></td>
      <td>p8n8va3y</td>
      <td>0.333445</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_17.html#qrnhp1ek">Kirillova_2009</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_16.html#2paaybd0">Mascarenhas_2017</a>, <a href="Topic_16.html#gbnns8np">RAMAKRISHNA_2015</a>, <a href="Topic_17.html#ywd88kmq">CAO_S_2012</a>, <a href="Topic_16.html#38pb1bti">Eddy_2018</a></td>
    </tr>
    <tr>
      <th id="6zjf6qzz";>498</th>
      <td>Li_J_2015</td>
      <td>Concept learning via granular computing: A cognitive viewpoint</td>
      <td>Li, Jinhai; Mei, Changlin; Xu, Weihua; Qian, Yuhua</td>
      <td>2015</td>
      <td>2015-03-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094283" target="_blank">PMC7094283</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.ins.2014.12.010" target="_blank">10.1016/j.ins.2014.12.010</a></td>
      <td>6zjf6qzz</td>
      <td>0.333438</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a></td>
      <td><a href="Topic_17.html#jzpt9ldx">Ren_R_2016</a>, <a href="Topic_17.html#awe681e9">Amadieu_2009</a>, <a href="Topic_17.html#e2coy5ms">Matsuzaw_2011</a>, <a href="Topic_17.html#6dcghdxf">Jiang_2009</a></td>
    </tr>
    <tr>
      <th id="a0bbw3er";>499</th>
      <td>Waagmeester_2020</td>
      <td>A protocol for adding knowledge to Wikidata, a case report</td>
      <td>Andra Waagmeester; Egon L. Willighagen; Andrew I. Su; Martina Kutmon; Jose Emilio Labra Gayo; Daniel Fernández-Álvarez; Peter J. Schaap; Lisa M. Verhagen; Jasper J. Koehorst</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.05.026336" target="_blank">10.1101/2020.04.05.026336</a></td>
      <td>a0bbw3er</td>
      <td>0.333419</td>
      <td><a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_17.html#hgxhyg46">Greenbaum_2009</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_17.html#ycbyx6vu">McCrae_2008</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a></td>
    </tr>
    <tr>
      <th id="nvavj9gk";>500</th>
      <td>Andre_2020</td>
      <td>COVID-19 diagnosis prediction in emergency care patients: a machine learning approach</td>
      <td>Andre Filipe de Moraes Batista; Joao Luiz Miraglia; Thiago Henrique Rizzi Donato; Alexandre Dias Porto Chiavegatto Filho</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.04.20052092" target="_blank">10.1101/2020.04.04.20052092</a></td>
      <td>nvavj9gk</td>
      <td>0.333411</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a></td>
      <td><a href="Topic_43.html#ashjw2xs">Guo_L_2019</a>, <a href="Topic_17.html#fridlqhv">AlMoammar_2018</a>, <a href="Topic_12.html#s9j21zsy">Yan_L_2020</a></td>
    </tr>
    <tr>
      <th id="xf8160dx";>501</th>
      <td>Liu_W_2012</td>
      <td>State-of-the-art methods for studying air distributions in commercial airliner cabins</td>
      <td>Liu, Wei; Mazumdar, Sagnik; Zhang, Zhao; Poussou, Stephane B.; Liu, Junjie; Lin, Chao-Hsin; Chen, Qingyan</td>
      <td>2012</td>
      <td>2012-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126834" target="_blank">PMC7126834</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.buildenv.2011.07.005" target="_blank">10.1016/j.buildenv.2011.07.005</a></td>
      <td>xf8160dx</td>
      <td>0.330112</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_13.html#4lou2mb6">Kapoor_2020</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_64.html#scwhwurr">Zhu_S_2010</a>, <a href="Topic_64.html#ugzdyvof">You_R_2018</a>, <a href="Topic_64.html#21ee4wxk">Chen_2013</a>, <a href="Topic_64.html#0yg36ms1">You_R_2017</a></td>
    </tr>
    <tr>
      <th id="0y80k5kw";>502</th>
      <td>Jin_S_2014</td>
      <td>Characterizing and controlling the inflammatory network during influenza A virus infection</td>
      <td>Jin, Suoqin; Li, Yuanyuan; Pan, Ruangang; Zou, Xiufen</td>
      <td>2014</td>
      <td>2014-01-21</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3896911" target="_blank">PMC3896911</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24445954.0" target="_blank">24445954.0</a></td>
      <td><a href="https://doi.org/10.1038/srep03799" target="_blank">10.1038/srep03799</a></td>
      <td>0y80k5kw</td>
      <td>0.329907</td>
      <td><a href="Topic_34.html#zlqh436d">Chasman_2016</a>, <a href="Topic_10.html#tg1310br">Haasnoot_2010</a>, <a href="Topic_10.html#sblrsi6y">Trobaugh_2017</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a>, <a href="Topic_01.html#3veki0pr">Menachery_2013</a></td>
      <td><a href="Topic_08.html#d79twl34">McDermott_2016</a>, <a href="Topic_40.html#0bwf8f1i">Song_2018</a>, <a href="Topic_30.html#9rpoll7y">Mitchell_2019</a>, <a href="Topic_08.html#y2fkkpst">Maelfait_2012</a></td>
    </tr>
    <tr>
      <th id="mcr12mwb";>503</th>
      <td>Dhanda_2016</td>
      <td>A web-based resource for designing therapeutics against Ebola Virus</td>
      <td>Dhanda, Sandeep Kumar; Chaudhary, Kumardeep; Gupta, Sudheer; Brahmachari, Samir Kumar; Raghava, Gajendra P. S.</td>
      <td>2016</td>
      <td>2016-04-26</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4845023" target="_blank">PMC4845023</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27113850.0" target="_blank">27113850.0</a></td>
      <td><a href="https://doi.org/10.1038/srep24782" target="_blank">10.1038/srep24782</a></td>
      <td>mcr12mwb</td>
      <td>0.329907</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a>, <a href="Topic_26.html#dwjijmwz">Volmer_2010</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_69.html#464qi674">Xing_2009</a>, <a href="Topic_23.html#9ne5iwny">Li_S_2016</a>, <a href="Topic_69.html#k0ugjb6f">Liu_S_2006</a>, <a href="Topic_23.html#pobxgffs">Jiang_2017</a></td>
    </tr>
    <tr>
      <th id="jc5c87b9";>504</th>
      <td>Fountain-Jones_2019</td>
      <td>How to make more from exposure data? An integrated machine learning pipeline to predict pathogen exposure</td>
      <td>Nicholas Fountain-Jones; Gustavo Machado; Scott Carver; Craig Packer; Mariana Mendoza; Meggan E Craft</td>
      <td>2019</td>
      <td>2019-03-06</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/569012" target="_blank">10.1101/569012</a></td>
      <td>jc5c87b9</td>
      <td>0.329732</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#cx3ex9ru">Leach_2010</a>, <a href="Topic_06.html#evpj67t1">Bifolchi_2013</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_01.html#ivu4erpq">Castrignano_2015</a></td>
      <td><a href="Topic_17.html#e89d3r95">Yin_W_2015</a>, <a href="Topic_36.html#graro54x">Wardeh_2020</a>, <a href="Topic_06.html#obzi61sa">Bucini_2019</a>, <a href="Topic_36.html#wa5il5go">Brierley_2019</a></td>
    </tr>
    <tr>
      <th id="t54zftal";>505</th>
      <td>Kumar_2017</td>
      <td>Prediction of endoplasmic reticulum resident proteins using fragmented amino acid composition and support vector machine</td>
      <td>Kumar, Ravindra; Kumari, Bandana; Kumar, Manish</td>
      <td>2017</td>
      <td>2017-09-04</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588793" target="_blank">PMC5588793</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28890846.0" target="_blank">28890846.0</a></td>
      <td><a href="https://doi.org/10.7717/peerj.3561" target="_blank">10.7717/peerj.3561</a></td>
      <td>t54zftal</td>
      <td>0.329501</td>
      <td><a href="Topic_26.html#8nyoyrlj">UNKNOWN_1995</a>, <a href="Topic_26.html#fhl0ly3x">Wang_2013</a>, <a href="Topic_26.html#3dvoppu2">Saraste_1992</a>, <a href="Topic_26.html#1s6andwf">Hobman_1993</a></td>
      <td><a href="Topic_26.html#een3sgpt">Fu_J_2000</a>, <a href="Topic_25.html#lli45kok">Fox_R_2014</a>, <a href="Topic_26.html#tuf1ih4g">UNKNOWN_1984</a></td>
    </tr>
    <tr>
      <th id="lxnrikmu";>506</th>
      <td>Chen_2018</td>
      <td>The utility of LASSO-based models for real time forecasts of endemic infectious diseases: A cross country comparison</td>
      <td>Chen, Yirong; Chu, Collins Wenhan; Chen, Mark I.C.; Cook, Alex R.</td>
      <td>2018</td>
      <td>2018-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29496631.0" target="_blank">29496631.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jbi.2018.02.014" target="_blank">10.1016/j.jbi.2018.02.014</a></td>
      <td>lxnrikmu</td>
      <td>0.328751</td>
      <td><a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#ktzcwd52">Stenseth_2008</a></td>
      <td><a href="Topic_06.html#obzi61sa">Bucini_2019</a>, <a href="Topic_22.html#dafll0s6">Ak_Ç_2020</a>, <a href="Topic_06.html#fiz00uj3">Park_2017</a>, <a href="Topic_06.html#m9icky9z">Song_2020</a></td>
    </tr>
    <tr>
      <th id="uw76y16g";>507</th>
      <td>Tithi_2017</td>
      <td>Fast Virome Explorer: A Pipeline for Virus and Phage Identification and Abundance Profiling in Metagenomics Data</td>
      <td>Saima Sultana Tithi; Roderick V. Jensen; Liqing Zhang</td>
      <td>2017</td>
      <td>2017-10-02</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/196998" target="_blank">10.1101/196998</a></td>
      <td>uw76y16g</td>
      <td>0.328734</td>
      <td><a href="Topic_17.html#sjl8x1jt">Tithi_2018</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_17.html#sjl8x1jt">Tithi_2018</a>, <a href="Topic_35.html#aa4s7b21">Li_C_2020</a>, <a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a></td>
    </tr>
    <tr>
      <th id="k3ujatua";>508</th>
      <td>Yang_2013</td>
      <td>Mining Social Media and Web Searches For Disease Detection</td>
      <td>Yang, Y. Tony; Horneffer, Michael; DiLisio, Nicole</td>
      <td>2013</td>
      <td>2013-05-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4140326" target="_blank">PMC4140326</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25170475.0" target="_blank">25170475.0</a></td>
      <td><a href="https://doi.org/10.4081/jphr.2013.e4" target="_blank">10.4081/jphr.2013.e4</a></td>
      <td>k3ujatua</td>
      <td>0.328331</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a></td>
      <td><a href="Topic_17.html#jx8wulcg">Seo_D_2017</a>, <a href="Topic_04.html#u20p22ch">VELASCO_2014</a>, <a href="Topic_17.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a></td>
    </tr>
    <tr>
      <th id="50oy9qqy";>509</th>
      <td>Bai_X_2020</td>
      <td>Predicting COVID-19 malignant progression with AI techniques</td>
      <td>Xiang Bai; Cong Fang; Yu Zhou; Song Bai; Zaiyi Liu; Qianlan Chen; Yongchao Xu; Tian Xia; Shi Gong; Xudong Xie; Dejia Song; Ronghui Du; Chunhua Zhou; Chengyang Chen; Dianer Nie; Dandan Tu; Changzheng Zhang; Xiaowu Liu; Lixin Qin; Weiwei Chen</td>
      <td>2020</td>
      <td>2020-03-23</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.20.20037325" target="_blank">10.1101/2020.03.20.20037325</a></td>
      <td>50oy9qqy</td>
      <td>0.328263</td>
      <td><a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a></td>
      <td><a href="Topic_12.html#hi38itc9">Wu_J_2020</a>, <a href="Topic_12.html#gag50cmf">Chen_2020</a>, <a href="Topic_67.html#6lngz7y1">Zhao_2020</a>, <a href="Topic_32.html#29cgwivt">Baier_2018</a></td>
    </tr>
    <tr>
      <th id="t30dg6oj";>510</th>
      <td>Parker_2016</td>
      <td>An Ecological Framework of the Human Virome Provides Classification of Current Knowledge and Identifies Areas of Forthcoming Discovery</td>
      <td>Parker, Michael T.</td>
      <td>2016</td>
      <td>2016-09-30</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5045143" target="_blank">PMC5045143</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27698618.0" target="_blank">27698618.0</a></td>
      <td></td>
      <td>t30dg6oj</td>
      <td>0.328056</td>
      <td><a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_01.html#ivu4erpq">Castrignano_2015</a></td>
      <td><a href="Topic_45.html#g3rfwhwr">Dacheux_2014</a>, <a href="Topic_33.html#k6g7u6b9">Russell_2018</a>, <a href="Topic_36.html#2ho8sr1a">Barrette_2011</a></td>
    </tr>
    <tr>
      <th id="86srlles";>511</th>
      <td>Li_W_2016</td>
      <td>Determination of total organic carbon and soluble solids contents in Tanreqing injection intermediates with NIR spectroscopy and chemometrics</td>
      <td>Li, Wenlong; Qu, Haibin</td>
      <td>2016</td>
      <td>2016-03-15</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114577" target="_blank">PMC7114577</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.chemolab.2015.12.018" target="_blank">10.1016/j.chemolab.2015.12.018</a></td>
      <td>86srlles</td>
      <td>0.327962</td>
      <td><a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#qaqvg453">Saw_A_2019</a>, <a href="Topic_17.html#nw3tybw9">Hoang_2015</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a></td>
      <td><a href="Topic_58.html#3wxdiok1">Sun_Y_2017</a>, <a href="Topic_49.html#dkkrorwm">Navarro_Escamilla_2013</a>, <a href="Topic_49.html#gtcr19qr">Fang_2019</a>, <a href="Topic_58.html#3sqyll6t">Wang_2013</a></td>
    </tr>
    <tr>
      <th id="5dx3lscp";>512</th>
      <td>Boulos_2005</td>
      <td>Web GIS in practice III: creating a simple interactive map of England's Strategic Health Authorities using Google Maps API, Google Earth KML, and MSN Virtual Earth Map Control</td>
      <td>Boulos, Maged N Kamel</td>
      <td>2005</td>
      <td>2005-09-21</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1242244" target="_blank">PMC1242244</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16176577.0" target="_blank">16176577.0</a></td>
      <td><a href="https://doi.org/10.1186/1476-072x-4-22" target="_blank">10.1186/1476-072x-4-22</a></td>
      <td>5dx3lscp</td>
      <td>0.327685</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_54.html#zehnqsp7">MacIntyre_2020</a>, <a href="Topic_04.html#ge5iri3v">Rodier_2007</a></td>
      <td><a href="Topic_04.html#ri5v6u4x">Boulos_2004</a>, <a href="Topic_24.html#yyp4a0ei">Kost_2019</a>, <a href="Topic_04.html#drhgcek5">Stevens_2015</a>, <a href="Topic_04.html#1b94201o">Ramachandran_2011</a></td>
    </tr>
    <tr>
      <th id="0mn1gbll";>513</th>
      <td>Hu_L_2011</td>
      <td>Predicting Biological Functions of Compounds Based on Chemical-Chemical Interactions</td>
      <td>Hu, Le-Le; Chen, Chen; Huang, Tao; Cai, Yu-Dong; Chou, Kuo-Chen</td>
      <td>2011</td>
      <td>2011-12-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3248422" target="_blank">PMC3248422</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22220213.0" target="_blank">22220213.0</a></td>
      <td><a href="https://doi.org/10.1371/journal.pone.0029491" target="_blank">10.1371/journal.pone.0029491</a></td>
      <td>0mn1gbll</td>
      <td>0.327652</td>
      <td><a href="Topic_13.html#4lou2mb6">Kapoor_2020</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_17.html#fewy8y4a">Wang_2018</a>, <a href="Topic_47.html#y14atmnh">Su_J_2014</a>, <a href="Topic_49.html#yhnk7iyq">Xing_2014</a></td>
    </tr>
    <tr>
      <th id="uh5rkjwx";>514</th>
      <td>González-Díaz_2014</td>
      <td>Model for Vaccine Design by Prediction of B-Epitopes of IEDB Given Perturbations in Peptide Sequence, In Vivo Process, Experimental Techniques, and Source or Host Organisms</td>
      <td>González-Díaz, Humberto; Pérez-Montoto, Lázaro G.; Ubeira, Florencio M.</td>
      <td>2014</td>
      <td>2014-01-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3987976" target="_blank">PMC3987976</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24741624.0" target="_blank">24741624.0</a></td>
      <td><a href="https://doi.org/10.1155/2014/768515" target="_blank">10.1155/2014/768515</a></td>
      <td>uh5rkjwx</td>
      <td>0.327535</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#qlwc8g2s">Gamermann_2014</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a></td>
      <td><a href="Topic_69.html#71940dh7">Kumar_2019</a>, <a href="Topic_16.html#x8jgcqts">Uslupehlivan_2020</a>, <a href="Topic_23.html#pobxgffs">Jiang_2017</a>, <a href="Topic_69.html#jlfipzvt">Huang_2012</a></td>
    </tr>
    <tr>
      <th id="ps3mnpzq";>515</th>
      <td>Zhu_X_2006</td>
      <td>ProCAT: a data analysis approach for protein microarrays</td>
      <td>Zhu, Xiaowei; Gerstein, Mark; Snyder, Michael</td>
      <td>2006</td>
      <td>2006-11-16</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1794587" target="_blank">PMC1794587</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17109749.0" target="_blank">17109749.0</a></td>
      <td><a href="https://doi.org/10.1186/gb-2006-7-11-r110" target="_blank">10.1186/gb-2006-7-11-r110</a></td>
      <td>ps3mnpzq</td>
      <td>0.327481</td>
      <td><a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_01.html#g0jiaenw">Stoevesandt_2014</a>, <a href="Topic_01.html#buebxb54">Reymond_Sutandy_2013</a>, <a href="Topic_14.html#7t44ufjf">Wingren_2008</a>, <a href="Topic_01.html#acg688jm">Hu_S_2010</a></td>
    </tr>
    <tr>
      <th id="6fthj1op";>516</th>
      <td>Moon_2019</td>
      <td>Development of a Single Leg Knee Exoskeleton and Sensing Knee Center of Rotation Change for Intention Detection</td>
      <td>Moon, Dae-Hoon; Kim, Donghan; Hong, Young-Dae</td>
      <td>2019</td>
      <td>2019-09-13</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6767670" target="_blank">PMC6767670</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31540298.0" target="_blank">31540298.0</a></td>
      <td><a href="https://doi.org/10.3390/s19183960" target="_blank">10.3390/s19183960</a></td>
      <td>6fthj1op</td>
      <td>0.326648</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_79.html#2gk7yepn">Huang_2011</a>, <a href="Topic_79.html#k09j0t98">Chun_2016</a>, <a href="Topic_79.html#ebzlt1rb">Kennedy_2006</a></td>
    </tr>
    <tr>
      <th id="66qoykl1";>517</th>
      <td>Shulman-Peleg_2004</td>
      <td>Recognition of Functional Sites in Protein Structures</td>
      <td>Shulman-Peleg, Alexandra; Nussinov, Ruth; Wolfson, Haim J.</td>
      <td>2004</td>
      <td>2004-06-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126412" target="_blank">PMC7126412</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15147845.0" target="_blank">15147845.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmb.2004.04.012" target="_blank">10.1016/j.jmb.2004.04.012</a></td>
      <td>66qoykl1</td>
      <td>0.326609</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_16.html#h7leow0q">Kuhn_2006</a>, <a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_16.html#p2iazl7z">Ziółkowska_2006</a></td>
    </tr>
    <tr>
      <th id="x53utltq";>518</th>
      <td>Lai_P_2010</td>
      <td>Spatial Analysis of the 2008 Influenza Outbreak of Hong Kong</td>
      <td>Lai, Poh-chin; Kwong, Kim-hung</td>
      <td>2010</td>
      <td>2010-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7122061" target="_blank">PMC7122061</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-642-12156-2_29" target="_blank">10.1007/978-3-642-12156-2_29</a></td>
      <td>x53utltq</td>
      <td>0.326212</td>
      <td><a href="Topic_04.html#ktzcwd52">Stenseth_2008</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a></td>
      <td><a href="Topic_13.html#ap92lgv7">dos_Santos_2018</a>, <a href="Topic_39.html#05u4t67u">Mao_Y_2019</a>, <a href="Topic_06.html#vok1dni9">Chin_2019</a>, <a href="Topic_22.html#dafll0s6">Ak_Ç_2020</a></td>
    </tr>
    <tr>
      <th id="tfw4zqiy";>519</th>
      <td>Scotch_2009</td>
      <td>Linkages between animal and human health sentinel data</td>
      <td>Scotch, Matthew; Odofin, Lynda; Rabinowitz, Peter</td>
      <td>2009</td>
      <td>2009-04-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2679002" target="_blank">PMC2679002</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19389228.0" target="_blank">19389228.0</a></td>
      <td><a href="https://doi.org/10.1186/1746-6148-5-15" target="_blank">10.1186/1746-6148-5-15</a></td>
      <td>tfw4zqiy</td>
      <td>0.326150</td>
      <td><a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_39.html#fan1rta9">Childs_2007</a>, <a href="Topic_36.html#vtkhwj57">Klous_2016</a>, <a href="Topic_36.html#3egio7z7">Greatorex_2016</a>, <a href="Topic_04.html#mmiffmve">Standley_2019</a></td>
    </tr>
    <tr>
      <th id="kauk7ze0";>520</th>
      <td>Durrant_2013</td>
      <td>Comparing Neural-Network Scoring Functions and the State of the Art: Applications to Common Library Screening</td>
      <td>Durrant, Jacob D.; Friedman, Aaron\nJ.; Rogers, Kathleen\nE.; McCammon, J. Andrew</td>
      <td>2013</td>
      <td>2013-06-04</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3735370" target="_blank">PMC3735370</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23734946.0" target="_blank">23734946.0</a></td>
      <td><a href="https://doi.org/10.1021/ci400042y" target="_blank">10.1021/ci400042y</a></td>
      <td>kauk7ze0</td>
      <td>0.325145</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#udzz15bk">Li_H_2009</a>, <a href="Topic_18.html#gqnzwyn3">Mukherjee_2011</a>, <a href="Topic_18.html#8fwxa8um">Lee_H_2014</a>, <a href="Topic_16.html#oyxaa93g">Wang_2006</a></td>
    </tr>
    <tr>
      <th id="l8tx7tyy";>521</th>
      <td>Thakur_2011</td>
      <td>VIRsiRNAdb: a curated database of experimentally validated viral siRNA/shRNA</td>
      <td>Thakur, Nishant; Qureshi, Abid; Kumar, Manoj</td>
      <td>2011</td>
      <td>2011-12-01</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245049" target="_blank">PMC3245049</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22139916.0" target="_blank">22139916.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkr1147" target="_blank">10.1093/nar/gkr1147</a></td>
      <td>l8tx7tyy</td>
      <td>0.323616</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a></td>
      <td><a href="Topic_37.html#tex6bgab">Sui_H_2009</a>, <a href="Topic_37.html#3mi2aop5">Haddad_2011</a>, <a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_37.html#pmeovccv">Zhang_2010</a></td>
    </tr>
    <tr>
      <th id="ytf6svyv";>522</th>
      <td>Najera_2011</td>
      <td>Real-time location and inpatient care systems based on passive RFID</td>
      <td>Najera, Pablo; Lopez, Javier; Roman, Rodrigo</td>
      <td>2011</td>
      <td>2011-05-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.jnca.2010.04.011" target="_blank">10.1016/j.jnca.2010.04.011</a></td>
      <td>ytf6svyv</td>
      <td>0.323396</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a></td>
      <td><a href="Topic_17.html#j72jbohi">Cheng_2015</a>, <a href="Topic_24.html#dvsni1be">Haddara_2018</a>, <a href="Topic_32.html#4rulshop">Huang_2013</a>, <a href="Topic_32.html#532myy40">Kumar_2014</a></td>
    </tr>
    <tr>
      <th id="w4z5avld";>523</th>
      <td>Bailer_2009</td>
      <td>Connecting viral with cellular interactomes</td>
      <td>Bailer, SM; Haas, J</td>
      <td>2009</td>
      <td>2009-08-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7108267" target="_blank">PMC7108267</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19632888.0" target="_blank">19632888.0</a></td>
      <td><a href="https://doi.org/10.1016/j.mib.2009.06.004" target="_blank">10.1016/j.mib.2009.06.004</a></td>
      <td>w4z5avld</td>
      <td>0.322770</td>
      <td><a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#acg688jm">Hu_S_2010</a></td>
      <td><a href="Topic_10.html#wrztpeb8">Zhang_2015</a>, <a href="Topic_10.html#ebr8jy1e">Yang_2014</a></td>
    </tr>
    <tr>
      <th id="2eycqf8o";>524</th>
      <td>Robertson_2010</td>
      <td>Review of methods for space–time disease surveillance</td>
      <td>Robertson, Colin; Nelson, Trisalyn A.; MacNab, Ying C.; Lawson, Andrew B.</td>
      <td>2010</td>
      <td>2010-07-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.sste.2009.12.001" target="_blank">10.1016/j.sste.2009.12.001</a></td>
      <td>2eycqf8o</td>
      <td>0.322707</td>
      <td><a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_04.html#j2ug18id">May_L_2009</a>, <a href="Topic_22.html#z9f8cval">Taniguchi_2008</a>, <a href="Topic_04.html#4e5ekhyp">Arita_2004</a></td>
    </tr>
    <tr>
      <th id="7kiahjyy";>525</th>
      <td>Nandy_2015</td>
      <td>Chapter 5 The GRANCH Techniques for Analysis of DNA, RNA and Protein Sequences</td>
      <td>Nandy, Ashesh</td>
      <td>2015</td>
      <td>2015-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-1-68108-053-6.50005-3" target="_blank">10.1016/b978-1-68108-053-6.50005-3</a></td>
      <td>7kiahjyy</td>
      <td>0.322234</td>
      <td><a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_21.html#ov92qh6h">Nowak_1984</a>, <a href="Topic_01.html#m3k8o573">Ghosh_2011</a>, <a href="Topic_21.html#4z5jxxf2">Otsuka_1999</a></td>
    </tr>
    <tr>
      <th id="3b3hv53b";>526</th>
      <td>Hoffmann_2020</td>
      <td>PriSeT: Efficient De Novo Primer Discovery</td>
      <td>Marie Hoffmann; Michael T. Monaghan; Knut Reinert</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.06.027961" target="_blank">10.1101/2020.04.06.027961</a></td>
      <td>3b3hv53b</td>
      <td>0.321939</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_07.html#g4sjyytr">Phillippy_2009</a></td>
      <td><a href="Topic_17.html#641pmd1d">Zhang_2017</a>, <a href="Topic_29.html#vrd89yk0">Severgnini_2009</a>, <a href="Topic_14.html#6w8qcv1c">Ayginin_2018</a>, <a href="Topic_29.html#ln92w6gm">Mukai_2006</a></td>
    </tr>
    <tr>
      <th id="jx8wulcg";>527</th>
      <td>Seo_D_2017</td>
      <td>Methods Using Social Media and Search Queries to Predict Infectious Disease Outbreaks</td>
      <td>Seo, Dong-Woo; Shin, Soo-Yong</td>
      <td>2017</td>
      <td>2017-10-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5688036" target="_blank">PMC5688036</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29181246.0" target="_blank">29181246.0</a></td>
      <td><a href="https://doi.org/10.4258/hir.2017.23.4.343" target="_blank">10.4258/hir.2017.23.4.343</a></td>
      <td>jx8wulcg</td>
      <td>0.321255</td>
      <td><a href="Topic_17.html#71tnv4lp">Santillana_2014</a>, <a href="Topic_54.html#yvgzkt8a">UNKNOWN_2010</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_17.html#ihh521ur">Lu_Q_2006</a></td>
      <td><a href="Topic_54.html#oypk94fc">Fung_2013</a>, <a href="Topic_06.html#h51zr96k">Kim_L_2019</a>, <a href="Topic_04.html#u20p22ch">VELASCO_2014</a></td>
    </tr>
    <tr>
      <th id="vioymdp0";>528</th>
      <td>Xie_G_2020</td>
      <td>A decomposition-ensemble approach for tourism forecasting</td>
      <td>Xie, Gang; Qian, Yatong; Wang, Shouyang</td>
      <td>2020</td>
      <td>2020-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.annals.2020.102891" target="_blank">10.1016/j.annals.2020.102891</a></td>
      <td>vioymdp0</td>
      <td>0.321134</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a>, <a href="Topic_71.html#22wnp1sv">von_der_Gracht_2010</a>, <a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_06.html#3aa8wgr0">Chowell_2017</a></td>
      <td><a href="Topic_71.html#g247rjh9">Guirao_2016</a>, <a href="Topic_71.html#b2m3p3uh">Chen_2010</a>, <a href="Topic_71.html#hse115gy">Wang_2009</a>, <a href="Topic_71.html#cl60qmu3">Campa_2016</a></td>
    </tr>
    <tr>
      <th id="06kffdm5";>529</th>
      <td>Garg_2017</td>
      <td>miPepBase: A Database of Experimentally Verified Peptides Involved in Molecular Mimicry</td>
      <td>Garg, Anjali; Kumari, Bandana; Kumar, Ravindra; Kumar, Manish</td>
      <td>2017</td>
      <td>2017-10-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5660332" target="_blank">PMC5660332</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29109711.0" target="_blank">29109711.0</a></td>
      <td><a href="https://doi.org/10.3389/fmicb.2017.02053" target="_blank">10.3389/fmicb.2017.02053</a></td>
      <td>06kffdm5</td>
      <td>0.319277</td>
      <td><a href="Topic_17.html#6qpsxmgi">Wang_2006</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a></td>
      <td><a href="Topic_69.html#n184795x">Zamanzadeh_2017</a>, <a href="Topic_37.html#clwkj4wa">Carmo_2014</a>, <a href="Topic_26.html#ikni2acz">Li_Z_2019</a></td>
    </tr>
    <tr>
      <th id="tg52towp";>530</th>
      <td>Naccache_2014</td>
      <td>A cloud-compatible bioinformatics pipeline for ultrarapid pathogen identification from next-generation sequencing of clinical samples</td>
      <td>Naccache, Samia N.; Federman, Scot; Veeraraghavan, Narayanan; Zaharia, Matei; Lee, Deanna; Samayoa, Erik; Bouquet, Jerome; Greninger, Alexander L.; Luk, Ka-Cheung; Enge, Barryett; Wadford, Debra A.; Messenger, Sharon L.; Genrich, Gillian L.; Pellegrino, Kristen; Grard, Gilda; Leroy, Eric; Schneider, Bradley S.; Fair, Joseph N.; Martínez, Miguel A.; Isa, Pavel; Crump, John A.; DeRisi, Joseph L.; Sittler, Taylor; Hackett, John; Miller, Steve; Chiu, Charles Y.</td>
      <td>2014</td>
      <td>2014-06-04</td>
      <td>None</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4079973" target="_blank">PMC4079973</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24899342.0" target="_blank">24899342.0</a></td>
      <td><a href="https://doi.org/10.1101/gr.171934.113" target="_blank">10.1101/gr.171934.113</a></td>
      <td>tg52towp</td>
      <td>0.317650</td>
      <td><a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_01.html#uvm72cpi">Djikeng_2009</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_45.html#fu3cl7lq">Manning_2020</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_14.html#nn4rum8j">Anis_2018</a>, <a href="Topic_35.html#aa4s7b21">Li_C_2020</a>, <a href="Topic_14.html#75j8jydo">Lewandowski_2019</a>, <a href="Topic_07.html#t4215imq">Alquezar-Planas_2013</a></td>
    </tr>
    <tr>
      <th id="7bkf7wrs";>531</th>
      <td>Valdivia-Granda_2009</td>
      <td>ORION-VIRCAT: a tool for mapping ICTV and NCBI taxonomies</td>
      <td>Valdivia-Granda, Willy; Larson, Francis</td>
      <td>2009</td>
      <td>2009-10-12</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2790308" target="_blank">PMC2790308</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20157487.0" target="_blank">20157487.0</a></td>
      <td><a href="https://doi.org/10.1093/database/bap014" target="_blank">10.1093/database/bap014</a></td>
      <td>7bkf7wrs</td>
      <td>0.317438</td>
      <td><a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_21.html#sik7m86l">Tzeng_2006</a>, <a href="Topic_21.html#r68l7g2e">Gorbalenya_2006</a></td>
    </tr>
    <tr>
      <th id="r0lduvs1";>532</th>
      <td>Zhang_2009</td>
      <td>Automatic online news monitoring and classification for syndromic surveillance</td>
      <td>Zhang, Yulei; Dang, Yan; Chen, Hsinchun; Thurmond, Mark; Larson, Cathy</td>
      <td>2009</td>
      <td>2009-11-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7114309" target="_blank">PMC7114309</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.dss.2009.04.016" target="_blank">10.1016/j.dss.2009.04.016</a></td>
      <td>r0lduvs1</td>
      <td>0.316768</td>
      <td><a href="Topic_04.html#ge5iri3v">Rodier_2007</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#5gy4nkhb">Fielding_2014</a></td>
      <td><a href="Topic_39.html#1ecdj314">Glennon_2020</a>, <a href="Topic_04.html#yybatog9">Paterson_2013</a>, <a href="Topic_17.html#ngow9w7p">Lu_H_2009</a>, <a href="Topic_04.html#vr65eg6g">Fast_2017</a></td>
    </tr>
    <tr>
      <th id="fl5tybzb";>533</th>
      <td>van_Hemert_2018</td>
      <td>Euclidean Distance Analysis Enables Nucleotide Skew Analysis in Viral Genomes</td>
      <td>van Hemert, Formijn; Jebbink, Maarten; van der Ark, Andries; Scholer, Frits; Berkhout, Ben</td>
      <td>2018</td>
      <td>2018-10-30</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6232797" target="_blank">PMC6232797</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30510593.0" target="_blank">30510593.0</a></td>
      <td><a href="https://doi.org/10.1155/2018/6490647" target="_blank">10.1155/2018/6490647</a></td>
      <td>fl5tybzb</td>
      <td>0.315882</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#zocunore">Liao_2006</a></td>
      <td><a href="Topic_61.html#48p9dkgp">van_Hemert_2016</a>, <a href="Topic_21.html#3g3apbon">Lauber_2013</a></td>
    </tr>
    <tr>
      <th id="h7leow0q";>534</th>
      <td>Kuhn_2006</td>
      <td>From the Similarity Analysis of Protein Cavities to the Functional Classification of Protein Families Using Cavbase</td>
      <td>Kuhn, Daniel; Weskamp, Nils; Schmitt, Stefan; Hüllermeier, Eyke; Klebe, Gerhard</td>
      <td>2006</td>
      <td>2006-06-16</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7094329" target="_blank">PMC7094329</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16697007.0" target="_blank">16697007.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jmb.2006.04.024" target="_blank">10.1016/j.jmb.2006.04.024</a></td>
      <td>h7leow0q</td>
      <td>0.314772</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_16.html#ov1f1uyt">Ivarsson_2012</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_16.html#hsh2yynq">Kinjo_2012</a></td>
      <td><a href="Topic_16.html#h11kclwx">Mönttinen_2019</a>, <a href="Topic_16.html#0a3okta0">Myllykoski_2017</a>, <a href="Topic_16.html#p2iazl7z">Ziółkowska_2006</a></td>
    </tr>
    <tr>
      <th id="61t686w7";>535</th>
      <td>Ma_L_2018</td>
      <td>Differential network as an indicator of osteoporosis with network entropy</td>
      <td>Ma, Lili; Du, Hongmei; Chen, Guangdong</td>
      <td>2018</td>
      <td>2018-05-16</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5995033" target="_blank">PMC5995033</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29896257.0" target="_blank">29896257.0</a></td>
      <td><a href="https://doi.org/10.3892/etm.2018.6169" target="_blank">10.3892/etm.2018.6169</a></td>
      <td>61t686w7</td>
      <td>0.314616</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_06.html#tlluxd11">Welch_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_69.html#7tpnbr8q">CHEN_2014</a>, <a href="Topic_69.html#bfyfcc7a">Chen_2015</a>, <a href="Topic_34.html#zvu9ww8j">Li_P_2017</a>, <a href="Topic_34.html#rsslmlaa">Wei_B_2015</a></td>
    </tr>
    <tr>
      <th id="zwbc7nnn";>536</th>
      <td>Sathiamurthy_2005</td>
      <td>An ontology for immune epitopes: application to the design of a broad scope database of immune reactivities</td>
      <td>Sathiamurthy, Muthuraman; Peters, Bjoern; Bui, Huynh-Hoa; Sidney, John; Mokili, John; Wilson, Stephen S; Fleri, Ward; McGuinness, Deborah L; Bourne, Philip E; Sette, Alessandro</td>
      <td>2005</td>
      <td>2005-09-20</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1287064" target="_blank">PMC1287064</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/16305755.0" target="_blank">16305755.0</a></td>
      <td><a href="https://doi.org/10.1186/1745-7580-1-2" target="_blank">10.1186/1745-7580-1-2</a></td>
      <td>zwbc7nnn</td>
      <td>0.313959</td>
      <td><a href="Topic_08.html#hntb1734">Lester_2014</a>, <a href="Topic_15.html#z2u5frvq">Montefiori_2007</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_33.html#742r5m9s">Kolls_2012</a>, <a href="Topic_34.html#zlqh436d">Chasman_2016</a></td>
      <td><a href="Topic_69.html#b6r9zdwa">Lollier_2011</a>, <a href="Topic_23.html#9q3f4w6u">Xue_G_2013</a>, <a href="Topic_69.html#60xkxdhe">Hua_R_2013</a>, <a href="Topic_69.html#awtfagku">Ignjatovic_2005</a></td>
    </tr>
    <tr>
      <th id="5s30ihv3";>537</th>
      <td>Salg_2020</td>
      <td>A globally available COVID-19 - Template for clinical imaging studies</td>
      <td>Gabriel Alexander Salg; Maria Katharina Ganten; Matthias Baumhauer; Claus Peter Heussel; Jens Kleesiek</td>
      <td>2020</td>
      <td>2020-04-07</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.02.20048793" target="_blank">10.1101/2020.04.02.20048793</a></td>
      <td>5s30ihv3</td>
      <td>0.313416</td>
      <td><a href="Topic_01.html#jyq88sdp">Curcin_2005</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a></td>
      <td><a href="Topic_67.html#g93dacvk">Miao_2020</a>, <a href="Topic_12.html#0wbu1yap">Zhu_Y_2020</a>, <a href="Topic_67.html#5skk3nj4">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="awe681e9";>538</th>
      <td>Amadieu_2009</td>
      <td>Prior knowledge in learning from a non-linear electronic document: Disorientation and coherence of the reading sequences</td>
      <td>Amadieu, Franck; Tricot, André; Mariné, Claudette</td>
      <td>2009</td>
      <td>2009-03-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7126386" target="_blank">PMC7126386</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/j.chb.2008.12.017" target="_blank">10.1016/j.chb.2008.12.017</a></td>
      <td>awe681e9</td>
      <td>0.313257</td>
      <td><a href="Topic_02.html#6zjf6qzz">Li_J_2015</a>, <a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_06.html#52esrbby">Krause_2007</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_17.html#e2coy5ms">Matsuzaw_2011</a>, <a href="Topic_54.html#q0e9qw5k">Ojima_2006</a>, <a href="Topic_02.html#xr067v2n">van_Aalst_2009</a>, <a href="Topic_28.html#0k4tsgus">Pearson_2006</a></td>
    </tr>
    <tr>
      <th id="qd9ps3zb";>539</th>
      <td>Kizza_2017</td>
      <td>Introduction to Computer Network Vulnerabilities</td>
      <td>Kizza, Joseph Migga</td>
      <td>2017</td>
      <td>2017-02-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123680" target="_blank">PMC7123680</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/978-3-319-55606-2_4" target="_blank">10.1007/978-3-319-55606-2_4</a></td>
      <td>qd9ps3zb</td>
      <td>0.312890</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_02.html#8plec8t4">Looker_2006</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#vdddhcuq">Kizza_2012</a></td>
      <td><a href="Topic_04.html#vdddhcuq">Kizza_2012</a>, <a href="Topic_04.html#mp9dlbgf">Kizza_2014</a>, <a href="Topic_04.html#08sdzn5l">Smith_2016</a>, <a href="Topic_17.html#62cc135r">Laurie_Lau_2015</a></td>
    </tr>
    <tr>
      <th id="n10ia3j3";>540</th>
      <td>Liu_H_2008</td>
      <td>A new approach to the prediction of transmembrane structures</td>
      <td>Liu, HongDe; Wang, Rui; Lu, XiaoQuan; Chen, Jing; Liu, Xiuhui; Ding, Lan</td>
      <td>2008</td>
      <td>2008-05-23</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088861" target="_blank">PMC7088861</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11434-008-0055-5" target="_blank">10.1007/s11434-008-0055-5</a></td>
      <td>n10ia3j3</td>
      <td>0.312695</td>
      <td><a href="Topic_26.html#eblj0qw6">UNKNOWN_1987</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_61.html#wv0f4hp4">Wu_G_2005</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_17.html#rktwtqtq">Cai_C_2006</a>, <a href="Topic_16.html#gbnns8np">RAMAKRISHNA_2015</a>, <a href="Topic_17.html#2dbpm0nt">Choo_2009</a></td>
    </tr>
    <tr>
      <th id="i1g9ikdb";>541</th>
      <td>Tobias_2020</td>
      <td>COVID19-Tracker: A shiny app to produce comprehensive data visualization for SARS-CoV-2 epidemic in Spain</td>
      <td>Aurelio Tobias; Joan Valls; Pau Satorra; Cristian Tebe</td>
      <td>2020</td>
      <td>2020-04-06</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.01.20049684" target="_blank">10.1101/2020.04.01.20049684</a></td>
      <td>i1g9ikdb</td>
      <td>0.312656</td>
      <td><a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a></td>
      <td><a href="Topic_17.html#r7tg0s45">John_2019</a>, <a href="Topic_19.html#17oac3bg">Silverman_2020</a>, <a href="Topic_04.html#5gbkrs73">Zhou_2020</a></td>
    </tr>
    <tr>
      <th id="dvfmpm54";>542</th>
      <td>Ai_H_2018</td>
      <td>GenomeLandscaper: Landscape analysis of genome-fingerprints maps assessing chromosome architecture</td>
      <td>Ai, Hannan; Ai, Yuncan; Meng, Fanmei</td>
      <td>2018</td>
      <td>2018-01-18</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5773709" target="_blank">PMC5773709</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29348569.0" target="_blank">29348569.0</a></td>
      <td><a href="https://doi.org/10.1038/s41598-018-19366-2" target="_blank">10.1038/s41598-018-19366-2</a></td>
      <td>dvfmpm54</td>
      <td>0.312100</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_17.html#jvfjf7aw">Feng_2010</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td><a href="Topic_17.html#b8jlvkg7">Ai_Y_2013</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_34.html#t25skqxk">Evans_2020</a>, <a href="Topic_07.html#x7e6e77d">Pontius_2007</a></td>
    </tr>
    <tr>
      <th id="5cemz8d4";>543</th>
      <td>Liu_Z_2020</td>
      <td>Second-Generation Sequencing with Deep Reinforcement Learning for Lung Infection Detection</td>
      <td>Liu, Zhuo; Zhang, Gerui; Jingyuan, Zhao; Yu, Liyan; Sheng, Junxiu; Zhang, Na; Yuan, Hong</td>
      <td>2020</td>
      <td>2020-02-22</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7060411" target="_blank">PMC7060411</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32184978.0" target="_blank">32184978.0</a></td>
      <td><a href="https://doi.org/10.1155/2020/3264801" target="_blank">10.1155/2020/3264801</a></td>
      <td>5cemz8d4</td>
      <td>0.312042</td>
      <td><a href="Topic_71.html#22wnp1sv">von_der_Gracht_2010</a>, <a href="Topic_04.html#e0sqppeq">Guery_2014</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a></td>
      <td><a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_01.html#0dhzv75e">Hörster_2019</a>, <a href="Topic_67.html#ilc2bzkx">Zhou_2020</a></td>
    </tr>
    <tr>
      <th id="pj330wn4";>544</th>
      <td>Smith_2012</td>
      <td>ciliaFA: a research tool for automated, high-throughput measurement of ciliary beat frequency using freely available software</td>
      <td>Smith, Claire M; Djakow, Jana; Free, Robert C; Djakow, Petr; Lonnen, Rana; Williams, Gwyneth; Pohunek, Petr; Hirst, Robert A; Easton, Andrew J; Andrew, Peter W; O’Callaghan, Christopher</td>
      <td>2012</td>
      <td>2012-08-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3607980" target="_blank">PMC3607980</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23351276.0" target="_blank">23351276.0</a></td>
      <td><a href="https://doi.org/10.1186/2046-2530-1-14" target="_blank">10.1186/2046-2530-1-14</a></td>
      <td>pj330wn4</td>
      <td>0.311686</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_60.html#9dj07sac">Liu_L_2016</a>, <a href="Topic_17.html#noydd8mw">Mao_H_2019</a></td>
      <td><a href="Topic_75.html#majlef5h">Papon_2012</a>, <a href="Topic_75.html#r7w6fxs1">Chilvers_2003</a></td>
    </tr>
    <tr>
      <th id="xr0uhccv";>545</th>
      <td>Carreño-Durán_2013</td>
      <td>Design of a set of probes with high potential for influenza virus epidemiological surveillance</td>
      <td>Carreño-Durán, Luis R; Larios-Serrato, V; Jaimes-Díaz, Hueman; Pérez-Cervantes, Hilda; Zepeda-López, Héctor; Sánchez-Vallejo, Carlos Javier; Olguín-Ruiz, Gabriela Edith; Maldonado-Rodríguez, Rogelio; Méndez-Tenorio, Alfonso</td>
      <td>2013</td>
      <td>2013-04-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3670124" target="_blank">PMC3670124</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23750091.0" target="_blank">23750091.0</a></td>
      <td><a href="https://doi.org/10.6026/97320630009414" target="_blank">10.6026/97320630009414</a></td>
      <td>xr0uhccv</td>
      <td>0.311578</td>
      <td><a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#g6obo798">Zheng_2008</a>, <a href="Topic_10.html#sd81bz0m">Gardner_2004</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a></td>
      <td><a href="Topic_07.html#v71436np">Ren_X_2008</a>, <a href="Topic_07.html#frluobkw">Wang_1996</a></td>
    </tr>
    <tr>
      <th id="xc61osdx";>546</th>
      <td>Qureshi_2014</td>
      <td>AVPdb: a database of experimentally validated antiviral peptides targeting medically important viruses</td>
      <td>Qureshi, Abid; Thakur, Nishant; Tandon, Himani; Kumar, Manoj</td>
      <td>2014</td>
      <td>2014-01-01</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3964995" target="_blank">PMC3964995</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24285301.0" target="_blank">24285301.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkt1191" target="_blank">10.1093/nar/gkt1191</a></td>
      <td>xc61osdx</td>
      <td>0.310702</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_01.html#vnynbijl">Conzelmann_1996</a>, <a href="Topic_17.html#l8tx7tyy">Thakur_2011</a></td>
      <td><a href="Topic_18.html#gxbrgy6g">Lee_C_2019</a>, <a href="Topic_18.html#qhhdjgeq">Zhu_X_2015</a>, <a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_18.html#x2kp0wpp">Lu_Y_2019</a></td>
    </tr>
    <tr>
      <th id="xrn6jv4z";>547</th>
      <td>Naghavi_2012</td>
      <td>Design and validation of a questionnaire to measure the attitudes of hospital staff concerning pandemic influenza</td>
      <td>Naghavi, Seyed Hamid Reza; Shabestari, Omid; Roudsari, Abdul V.; Harrison, John</td>
      <td>2012</td>
      <td>2012-02-29</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102719" target="_blank">PMC7102719</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/22341848.0" target="_blank">22341848.0</a></td>
      <td><a href="https://doi.org/10.1016/j.jiph.2011.11.002" target="_blank">10.1016/j.jiph.2011.11.002</a></td>
      <td>xrn6jv4z</td>
      <td>0.310593</td>
      <td><a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_01.html#qbct63p4">Lipsitch_2014</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_24.html#atg93t3p">Tzeng_2016</a>, <a href="Topic_28.html#0ztyvq50">Lam_K_2013</a></td>
    </tr>
    <tr>
      <th id="zmoayelz";>548</th>
      <td>Chen_2011</td>
      <td>Advances of bioinformatics tools applied in virus epitopes prediction</td>
      <td>Chen, Ping; Rayner, Simon; Hu, Kang-hong</td>
      <td>2011</td>
      <td>2011-02-18</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7090880" target="_blank">PMC7090880</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/21331885.0" target="_blank">21331885.0</a></td>
      <td><a href="https://doi.org/10.1007/s12250-011-3159-4" target="_blank">10.1007/s12250-011-3159-4</a></td>
      <td>zmoayelz</td>
      <td>0.309826</td>
      <td><a href="Topic_01.html#6vja5e5o">Johnson_2018</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a></td>
      <td><a href="Topic_69.html#71940dh7">Kumar_2019</a>, <a href="Topic_69.html#p37j3olh">Khan_2006</a>, <a href="Topic_69.html#fmumym1x">Lon_J_2020</a>, <a href="Topic_69.html#9lqh0evn">Zhang_2013</a></td>
    </tr>
    <tr>
      <th id="ax09x6gk";>549</th>
      <td>Wu_J_2016</td>
      <td>Data Decision and Transmission Based on Mobile Data Health Records on Sensor Devices in Wireless Networks</td>
      <td>Wu, Jia; Chen, Zhigang</td>
      <td>2016</td>
      <td>2016-06-20</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7088664" target="_blank">PMC7088664</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s11277-016-3438-y" target="_blank">10.1007/s11277-016-3438-y</a></td>
      <td>ax09x6gk</td>
      <td>0.309675</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_17.html#odnlt6fr">Wu_J_2016</a>, <a href="Topic_17.html#3l2dbnjb">Sood_2017</a>, <a href="Topic_24.html#yd29gk7q">Li_K_2012</a>, <a href="Topic_32.html#4rulshop">Huang_2013</a></td>
    </tr>
    <tr>
      <th id="f4q1r3vu";>550</th>
      <td>Keller_2009</td>
      <td>Automated vocabulary discovery for geo-parsing online epidemic intelligence</td>
      <td>Keller, Mikaela; Freifeld, Clark C; Brownstein, John S</td>
      <td>2009</td>
      <td>2009-11-24</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2787530" target="_blank">PMC2787530</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19930702.0" target="_blank">19930702.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-10-385" target="_blank">10.1186/1471-2105-10-385</a></td>
      <td>f4q1r3vu</td>
      <td>0.309625</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#hdqa12es">Wei_B_2011</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#lgcmamfb">Dentico_2005</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a></td>
      <td><a href="Topic_17.html#1lgi1e60">Freifeld_2008</a>, <a href="Topic_04.html#4524vg4x">Collier_2007</a>, <a href="Topic_17.html#r0lduvs1">Zhang_2009</a>, <a href="Topic_04.html#6q8eg9z4">Keller_2009</a></td>
    </tr>
    <tr>
      <th id="d8onhd04";>551</th>
      <td>Mei_H_2004</td>
      <td>Identification of encoding proteins related to SARS-CoV</td>
      <td>Mei, Hu; Sun, Lili; Zhou, Yuan; Xiong, Qing; Li, Zhiliang</td>
      <td>2004</td>
      <td>2004-01-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089014" target="_blank">PMC7089014</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1360/03wb0198" target="_blank">10.1360/03wb0198</a></td>
      <td>d8onhd04</td>
      <td>0.309322</td>
      <td><a href="Topic_17.html#7my2q928">Largo_2016</a>, <a href="Topic_06.html#vs12l0uv">Choudhury_2011</a>, <a href="Topic_60.html#xdihvg6k">David_2013</a>, <a href="Topic_19.html#tgyxiwuo">Abou-Hamdan_2018</a></td>
      <td><a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_41.html#2d7l26tl">Liu_M_2006</a>, <a href="Topic_26.html#me5ff1pb">Zeng_2004</a>, <a href="Topic_21.html#02q9y011">Frick_2020</a></td>
    </tr>
    <tr>
      <th id="j5nmifhj";>552</th>
      <td>Bai_J_2016</td>
      <td>Pharmacodynamics and Systems Pharmacology Approaches to Repurposing Drugs in the Wake of Global Health Burden</td>
      <td>Bai, Jane P.F.</td>
      <td>2016</td>
      <td>2016-10-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7127093" target="_blank">PMC7127093</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27522921.0" target="_blank">27522921.0</a></td>
      <td><a href="https://doi.org/10.1016/j.xphs.2016.07.004" target="_blank">10.1016/j.xphs.2016.07.004</a></td>
      <td>j5nmifhj</td>
      <td>0.308440</td>
      <td><a href="Topic_01.html#s6x0igia">Dove_2004</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a></td>
      <td><a href="Topic_01.html#icyut3xa">Pillaiyar_2020</a>, <a href="Topic_18.html#rhg5pbsq">van_der_Kooy_2013</a>, <a href="Topic_01.html#4h73lukj">Peyclit_2019</a>, <a href="Topic_47.html#p6d0nx01">Subbian_2016</a></td>
    </tr>
    <tr>
      <th id="egd1gxby";>553</th>
      <td>Barh_2014</td>
      <td>Chapter 21 In Silico Models From Simple Networks to Complex Diseases</td>
      <td>Barh, Debmalya; Chaitankar, Vijender; Yiannakopoulou, Eugenia Ch; Salawu, Emmanuel O.; Chowbina, Sudhir; Ghosh, Preetam; Azevedo, Vasco</td>
      <td>2014</td>
      <td>2014-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-416002-6.00021-3" target="_blank">10.1016/b978-0-12-416002-6.00021-3</a></td>
      <td>egd1gxby</td>
      <td>0.308185</td>
      <td><a href="Topic_01.html#mh30ocvo">Duan_2015</a>, <a href="Topic_13.html#648c2ox3">Tong_2011</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_01.html#qhgvf35j">UNKNOWN_2004</a></td>
      <td><a href="Topic_13.html#648c2ox3">Tong_2011</a>, <a href="Topic_01.html#so5yzryi">Janjić_2012</a>, <a href="Topic_01.html#9xz28ys8">Moghadas_2006</a>, <a href="Topic_01.html#d1o5smtv">Daun_2007</a></td>
    </tr>
    <tr>
      <th id="gm1e9j80";>554</th>
      <td>Thi_T_2014</td>
      <td>Recognizing flu-like symptoms from videos</td>
      <td>Thi, Tuan Hue; Wang, Li; Ye, Ning; Zhang, Jian; Maurer-Stroh, Sebastian; Cheng, Li</td>
      <td>2014</td>
      <td>2014-09-12</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4180141" target="_blank">PMC4180141</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25217118.0" target="_blank">25217118.0</a></td>
      <td><a href="https://doi.org/10.1186/1471-2105-15-300" target="_blank">10.1186/1471-2105-15-300</a></td>
      <td>gm1e9j80</td>
      <td>0.307763</td>
      <td><a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_02.html#8plec8t4">Looker_2006</a>, <a href="Topic_28.html#8ajy3e55">Ropeik_2004</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a></td>
      <td><a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_17.html#ki0nnptv">Guo_S_2020</a>, <a href="Topic_67.html#ilc2bzkx">Zhou_2020</a>, <a href="Topic_55.html#4hk736ev">Alfaraj_2018</a></td>
    </tr>
    <tr>
      <th id="9d6pqx19";>555</th>
      <td>Kilianski_2015</td>
      <td>Pathosphere.org: pathogen detection and characterization through a web-based, open source informatics platform</td>
      <td>Kilianski, Andy; Carcel, Patrick; Yao, Shijie; Roth, Pierce; Schulte, Josh; Donarum, Greg B.; Fochler, Ed T.; Hill, Jessica M.; Liem, Alvin T.; Wiley, Michael R.; Ladner, Jason T.; Pfeffer, Bradley P.; Elliot, Oliver; Petrosov, Alexandra; Jima, Dereje D.; Vallard, Tyghe G.; Melendrez, Melanie C.; Skowronski, Evan; Quan, Phenix-Lan; Lipkin, W. Ian; Gibbons, Henry S.; Hirschberg, David L.; Palacios, Gustavo F.; Rosenzweig, C. Nicole</td>
      <td>2015</td>
      <td>2015-12-29</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4696252" target="_blank">PMC4696252</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/26714571.0" target="_blank">26714571.0</a></td>
      <td><a href="https://doi.org/10.1186/s12859-015-0840-5" target="_blank">10.1186/s12859-015-0840-5</a></td>
      <td>9d6pqx19</td>
      <td>0.307750</td>
      <td><a href="Topic_17.html#ca6pff0p">UNKNOWN_2018</a>, <a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_54.html#z5n9ys86">Murray_2017</a>, <a href="Topic_04.html#kduisp14">Cui_H_2015</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a></td>
      <td><a href="Topic_17.html#2gc2dc46">McGarvey_2009</a>, <a href="Topic_70.html#3l196l2f">Sazmand_2019</a>, <a href="Topic_17.html#0kkm0tgj">Zhang_2007</a>, <a href="Topic_17.html#3ulketgy">Snyder_2006</a></td>
    </tr>
    <tr>
      <th id="9ahn4q88";>556</th>
      <td>He_M_2019</td>
      <td>Structural Analysis of Biomolecules through a Combination of Mobility Capillary Electrophoresis and Mass Spectrometry</td>
      <td>He, Muyi; Luo, Pan; Hong, Jie; Wang, Xiaofeng; Wu, Haimei; Zhang, Rongkai; Qu, Feng; Xiang, Ye; Xu, Wei</td>
      <td>2019</td>
      <td>2019-01-31</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6648644" target="_blank">PMC6648644</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/31459477.0" target="_blank">31459477.0</a></td>
      <td><a href="https://doi.org/10.1021/acsomega.8b03224" target="_blank">10.1021/acsomega.8b03224</a></td>
      <td>9ahn4q88</td>
      <td>0.307131</td>
      <td><a href="Topic_17.html#zocunore">Liao_2006</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#qy4kirx1">Mendizabal-Ruiz_2018</a></td>
      <td><a href="Topic_58.html#ngbcdkz3">Chen_2005</a>, <a href="Topic_29.html#1qyu5zoo">Zhao_2009</a>, <a href="Topic_01.html#x5g7g2xu">Munusamy_2020</a></td>
    </tr>
    <tr>
      <th id="fn7l93wh";>557</th>
      <td>Magar_2020</td>
      <td>Potential Neutralizing Antibodies Discovered for Novel Corona Virus Using Machine Learning</td>
      <td>Rishikesh Magar; Prakarsh Yadav; Amir Barati Farimani</td>
      <td>2020</td>
      <td>2020-03-20</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.03.14.992156" target="_blank">10.1101/2020.03.14.992156</a></td>
      <td>fn7l93wh</td>
      <td>0.307058</td>
      <td><a href="Topic_10.html#4ih3d2d1">Rice_1992</a>, <a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_10.html#qdy29oym">Denison_2008</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_10.html#9et0iwj4">Berkhout_2006</a></td>
      <td><a href="Topic_15.html#i8xi0h4g">Akahata_2010</a>, <a href="Topic_23.html#8arzvh2l">Traggiai_2004</a>, <a href="Topic_69.html#60xkxdhe">Hua_R_2013</a></td>
    </tr>
    <tr>
      <th id="40pe21lm";>558</th>
      <td>Zhang_2018</td>
      <td>Exploring the Binding Mechanism between Human Profilin (PFN1) and Polyproline-10 through Binding Mode Screening</td>
      <td>Leili Zhang; David R. Bell; Binquan Luan; Ruhong Zhou</td>
      <td>2018</td>
      <td>2018-09-16</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/418830" target="_blank">10.1101/418830</a></td>
      <td>40pe21lm</td>
      <td>0.306331</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#sgw66xhn">Saw_A_2019</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#0ec2xunt">Plewczyński_2008</a></td>
      <td><a href="Topic_16.html#zwdmd7uf">Wang_2013</a>, <a href="Topic_16.html#8c05w1ix">Yap_Y_2005</a>, <a href="Topic_16.html#32ju2h1g">Niu_C_2008</a>, <a href="Topic_16.html#4jrro0gd">Du_Q_2004</a></td>
    </tr>
    <tr>
      <th id="llv3cvdr";>559</th>
      <td>Qiang_2020</td>
      <td>Using the spike protein feature to predict infection risk and monitor the evolutionary dynamic of coronavirus</td>
      <td>Qiang, Xiao-Li; Xu, Peng; Fang, Gang; Liu, Wen-Bin; Kou, Zheng</td>
      <td>2020</td>
      <td>2020-03-25</td>
      <td>COMM-USE</td>
      <td>Y</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7093988" target="_blank">PMC7093988</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32209118.0" target="_blank">32209118.0</a></td>
      <td><a href="https://doi.org/10.1186/s40249-020-00649-8" target="_blank">10.1186/s40249-020-00649-8</a></td>
      <td>llv3cvdr</td>
      <td>0.305888</td>
      <td><a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_07.html#d0imcmcl">Wang_2007</a>, <a href="Topic_07.html#jquxicdr">Wang_2013</a>, <a href="Topic_41.html#o9jdt86j">Callison_1999</a></td>
      <td><a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a>, <a href="Topic_07.html#idu4j78a">Sohrab_2019</a>, <a href="Topic_07.html#vbadg8ki">Jeong_2005</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="t9bt70f2";>560</th>
      <td>Guo_Q_2020</td>
      <td>Host and infectivity prediction of Wuhan 2019 novel coronavirus using deep learning algorithm</td>
      <td>Qian Guo; Mo Li; Chunhui Wang; Peihong Wang; Zhencheng Fang; Jie tan; Shufang Wu; Yonghong Xiao; Huaiqiu Zhu</td>
      <td>2020</td>
      <td>2020-01-24</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.01.21.914044" target="_blank">10.1101/2020.01.21.914044</a></td>
      <td>t9bt70f2</td>
      <td>0.305815</td>
      <td><a href="Topic_30.html#tnvlafa2">Lu_P_2016</a>, <a href="Topic_21.html#59emvigh">Ryu_W_2017</a>, <a href="Topic_30.html#k9skma4k">Karlsson_2012</a>, <a href="Topic_72.html#tq0f6d01">Weaver_2004</a>, <a href="Topic_53.html#fqqbyqnh">Tallmadge_2008</a></td>
      <td><a href="Topic_17.html#o4y8ci0n">Mock_2019</a>, <a href="Topic_07.html#v38tjof3">Fahmi_2020</a></td>
    </tr>
    <tr>
      <th id="hi7z3nob";>561</th>
      <td>Huang_2010</td>
      <td>SAROTUP: Scanner and Reporter of Target-Unrelated Peptides</td>
      <td>Huang, Jian; Ru, Beibei; Li, Shiyong; Lin, Hao; Guo, Feng-Biao</td>
      <td>2010</td>
      <td>2010-03-21</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842971" target="_blank">PMC2842971</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20339521.0" target="_blank">20339521.0</a></td>
      <td><a href="https://doi.org/10.1155/2010/101932" target="_blank">10.1155/2010/101932</a></td>
      <td>hi7z3nob</td>
      <td>0.305736</td>
      <td><a href="Topic_07.html#kixzxs68">Naito_2007</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_18.html#l36rzqnx">Park_2012</a>, <a href="Topic_15.html#6y0xvp37">Welsh_2007</a></td>
      <td><a href="Topic_16.html#ul0k95x2">van_der_Linden_2014</a>, <a href="Topic_16.html#njlqgih3">Robinson_2013</a>, <a href="Topic_01.html#92esdjy9">Delhalle_2012</a>, <a href="Topic_19.html#e8qubwha">Rut_W_2020</a></td>
    </tr>
    <tr>
      <th id="k26orfrt";>562</th>
      <td>Jiang_2017</td>
      <td>Identifying Personal Health Experience Tweets with Deep Neural Networks</td>
      <td>Jiang, Keyuan; Gupta, Ravish; Gupta, Matrika; Calix, Ricardo A.; Bernard, Gordon R.</td>
      <td>2017</td>
      <td>2017-07-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5702551" target="_blank">PMC5702551</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29060084.0" target="_blank">29060084.0</a></td>
      <td><a href="https://doi.org/10.1109/embc.2017.8037039" target="_blank">10.1109/embc.2017.8037039</a></td>
      <td>k26orfrt</td>
      <td>0.305676</td>
      <td><a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#qbemfymp">Zhou_2020</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_04.html#1witi1rg">Griffiths_2010</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a></td>
      <td><a href="Topic_17.html#zqmdcj1z">Dandekar_2017</a>, <a href="Topic_54.html#bp7sray2">Hu_G_2019</a></td>
    </tr>
    <tr>
      <th id="a9q0tcgz";>563</th>
      <td>Roy_K_2015</td>
      <td>Chapter 10 Other Related Techniques</td>
      <td>Roy, Kunal; Kar, Supratik; Das, Rudra Narayan</td>
      <td>2015</td>
      <td>2015-12-31</td>
      <td>PMC</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1016/b978-0-12-801505-6.00010-7" target="_blank">10.1016/b978-0-12-801505-6.00010-7</a></td>
      <td>a9q0tcgz</td>
      <td>0.305467</td>
      <td><a href="Topic_18.html#l36rzqnx">Park_2012</a>, <a href="Topic_01.html#d66rz6dc">Webb_2014</a>, <a href="Topic_16.html#0oq2n0af">Liu_Z_2008</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_17.html#w8luu86h">Liou_2013</a></td>
      <td><a href="Topic_17.html#qah877pk">Ekins_2007</a>, <a href="Topic_18.html#mw7athol">Plewczynski_2007</a>, <a href="Topic_18.html#gqnzwyn3">Mukherjee_2011</a>, <a href="Topic_16.html#40pe21lm">Zhang_2018</a></td>
    </tr>
    <tr>
      <th id="emr0eh0i";>564</th>
      <td>Wahba_2020</td>
      <td>Identification of a Pangolin Niche for a 2019-nCoV-like Coronavirus via an Extensive Meta-metagenomic Search</td>
      <td>Lamia Wahba; Nimit Jain; Andrew Z. Fire; Massa J. Shoura; Karen L. Artiles; Matthew J. McCoy; Dae-Eun Jeong</td>
      <td>2020</td>
      <td>2020-02-14</td>
      <td>BioRxiv</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.02.08.939660" target="_blank">10.1101/2020.02.08.939660</a></td>
      <td>emr0eh0i</td>
      <td>0.304789</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a></td>
      <td><a href="Topic_45.html#0l33i6s4">Anderson_2020</a>, <a href="Topic_07.html#v38tjof3">Fahmi_2020</a>, <a href="Topic_45.html#lq12vg4z">Niemeyer_2018</a></td>
    </tr>
    <tr>
      <th id="hmgmsviu";>565</th>
      <td>Paez-Espino_2017</td>
      <td>IMG/VR: a database of cultured and uncultured DNA Viruses and retroviruses</td>
      <td>Paez-Espino, David; Chen, I.-Min A.; Palaniappan, Krishna; Ratner, Anna; Chu, Ken; Szeto, Ernest; Pillay, Manoj; Huang, Jinghua; Markowitz, Victor M.; Nielsen, Torben; Huntemann, Marcel; K. Reddy, T. B.; Pavlopoulos, Georgios A.; Sullivan, Matthew B.; Campbell, Barbara J.; Chen, Feng; McMahon, Katherine; Hallam, Steve J.; Denef, Vincent; Cavicchioli, Ricardo; Caffrey, Sean M.; Streit, Wolfgang R.; Webster, John; Handley, Kim M.; Salekdeh, Ghasem H.; Tsesmetzis, Nicolas; Setubal, Joao C.; Pope, Phillip B.; Liu, Wen-Tso; Rivers, Adam R.; Ivanova, Natalia N.; Kyrpides, Nikos C.</td>
      <td>2017</td>
      <td>2017-01-04</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210529" target="_blank">PMC5210529</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27799466.0" target="_blank">27799466.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkw1030" target="_blank">10.1093/nar/gkw1030</a></td>
      <td>hmgmsviu</td>
      <td>0.304789</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_10.html#jw4n2j6y">Smits_2014</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_10.html#4ih3d2d1">Rice_1992</a></td>
      <td><a href="Topic_45.html#1a9u53za">Shabman_2016</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="7plt4s6b";>566</th>
      <td>Noto_2015</td>
      <td>CSAX: Characterizing Systematic Anomalies in eXpression Data</td>
      <td>Noto, Keith; Majidi, Saeed; Edlow, Andrea G.; Wick, Heather C.; Bianchi, Diana W.; Slonim, Donna K.</td>
      <td>2015</td>
      <td>2015-05-01</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424968" target="_blank">PMC4424968</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25651392.0" target="_blank">25651392.0</a></td>
      <td><a href="https://doi.org/10.1089/cmb.2014.0155" target="_blank">10.1089/cmb.2014.0155</a></td>
      <td>7plt4s6b</td>
      <td>0.304722</td>
      <td><a href="Topic_01.html#zjnlibu4">Bocharov_2018</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a></td>
      <td><a href="Topic_01.html#x4hrwhkz">Prokop_2020</a>, <a href="Topic_14.html#wheblwm3">Narrandes_2018</a>, <a href="Topic_17.html#hs30wwiq">Gibbs_2014</a>, <a href="Topic_12.html#qz2tpmq4">Wang_2008</a></td>
    </tr>
    <tr>
      <th id="46w17i8r";>567</th>
      <td>Mendes_2019</td>
      <td>DEN-IM: Dengue Virus identification from shotgun and targeted metagenomics</td>
      <td>C I Mendes; E Lizarazo; M P Machado; D N Silva; A Tami; M Ramirez; N Couto; J W A Rossen; J A Carriço</td>
      <td>2019</td>
      <td>2019-05-06</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/628073" target="_blank">10.1101/628073</a></td>
      <td>46w17i8r</td>
      <td>0.303110</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#ts1eb8fk">Montgomery_2004</a>, <a href="Topic_17.html#gr108zuo">Deng_2015</a>, <a href="Topic_17.html#wjf8t8vp">Brister_2015</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a></td>
      <td><a href="Topic_07.html#thsq6pnk">Lachheb_2019</a>, <a href="Topic_70.html#0gi6qzw0">Takhampunya_2019</a>, <a href="Topic_07.html#v71436np">Ren_X_2008</a></td>
    </tr>
    <tr>
      <th id="fogzjrk2";>568</th>
      <td>Chorba_2020</td>
      <td>A Deep Learning Algorithm for Automated Cardiac Murmur Detection Via a Digital Stethoscope Platform</td>
      <td>John S Chorba; Avi M Shapiro; Le Le; John Maidens; John Prince; Steve Pham; Mia M Kanzawa; Daniel N Barbosa; Brent E White; Jason Paek; Sophie G Fuller; Grant W Stalker; Sara A Bravo; Dina Jean; Subramaniam Venkatraman; Patrick M McCarthy; James D Thomas</td>
      <td>2020</td>
      <td>2020-04-03</td>
      <td>BioRxiv</td>
      <td>N</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1101/2020.04.01.20050518" target="_blank">10.1101/2020.04.01.20050518</a></td>
      <td>fogzjrk2</td>
      <td>0.302551</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_24.html#j1rrv1eq">Lo_W_2014</a></td>
      <td><a href="Topic_14.html#1r4gm2d4">Liu_Y_2020</a>, <a href="Topic_17.html#96r8l6vq">Fu_M_2020</a>, <a href="Topic_17.html#ll4rxd9p">Zheng_2020</a>, <a href="Topic_14.html#u2uswgy3">Wang_2020</a></td>
    </tr>
    <tr>
      <th id="47sgz7l3";>569</th>
      <td>Charlton_Hume_2017</td>
      <td>Platform technologies for modern vaccine manufacturing</td>
      <td>Charlton Hume, Hayley K.; Lua, Linda H.L.</td>
      <td>2017</td>
      <td>2017-08-16</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115529" target="_blank">PMC7115529</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28347504.0" target="_blank">28347504.0</a></td>
      <td><a href="https://doi.org/10.1016/j.vaccine.2017.02.069" target="_blank">10.1016/j.vaccine.2017.02.069</a></td>
      <td>47sgz7l3</td>
      <td>0.301798</td>
      <td><a href="Topic_01.html#s6x0igia">Dove_2004</a>, <a href="Topic_01.html#nlnugpty">Kanekiyo_2019</a>, <a href="Topic_13.html#v7xeubps">Gupta_2009</a>, <a href="Topic_01.html#lzdjfvh7">May_E_2016</a></td>
      <td><a href="Topic_15.html#unjz4kzi">Fujita_2011</a>, <a href="Topic_01.html#vc9ytkio">Brun_2008</a></td>
    </tr>
    <tr>
      <th id="snqdma0s";>570</th>
      <td>Binkowski_2005</td>
      <td>Correcting errors in synthetic DNA through consensus shuffling</td>
      <td>Binkowski, Brock F.; Richmond, Kathryn E.; Kaysen, James; Sussman, Michael R.; Belshaw, Peter J.</td>
      <td>2005</td>
      <td>2005-03-30</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1072806" target="_blank">PMC1072806</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15800206.0" target="_blank">15800206.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gni053" target="_blank">10.1093/nar/gni053</a></td>
      <td>snqdma0s</td>
      <td>0.301003</td>
      <td><a href="Topic_07.html#bbvxu8op">Karaman_2005</a>, <a href="Topic_07.html#g2f151oc">Whiteford_2005</a>, <a href="Topic_61.html#7fx1ghd3">Morikawa_1992</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_21.html#hpv6916w">Krause_1987</a></td>
      <td><a href="Topic_37.html#ndjbrtrv">Satyanarayana_2003</a>, <a href="Topic_21.html#ov92qh6h">Nowak_1984</a></td>
    </tr>
    <tr>
      <th id="ujhgb3b0";>571</th>
      <td>Huang_2007</td>
      <td>CoVDB: a comprehensive database for comparative analysis of coronavirus genes and genomes</td>
      <td>Huang, Yi; Lau, Susanna K. P.; Woo, Patrick C. Y.; Yuen, Kwok-yung</td>
      <td>2007</td>
      <td>2007-10-02</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238867" target="_blank">PMC2238867</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17913743.0" target="_blank">17913743.0</a></td>
      <td><a href="https://doi.org/10.1093/nar/gkm754" target="_blank">10.1093/nar/gkm754</a></td>
      <td>ujhgb3b0</td>
      <td>0.300852</td>
      <td><a href="Topic_17.html#tg06vxza">Cheng_2013</a>, <a href="Topic_17.html#t1e0tr4w">Lee_C_2017</a>, <a href="Topic_17.html#jsg0yc2t">O'Leary_2016</a>, <a href="Topic_17.html#wj7yonjw">Gardner_2008</a></td>
      <td><a href="Topic_45.html#bkm486jd">Tao_Y_2012</a>, <a href="Topic_37.html#1zoizzdp">Wang_2009</a></td>
    </tr>
    <tr>
      <th id="47goscew";>572</th>
      <td>Vazquez_2020</td>
      <td>Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine</td>
      <td>Vazquez, Arni S.R. Srinivasa Rao; Jose A.</td>
      <td>2020</td>
      <td>2020-01-01</td>
      <td>COMM-USE</td>
      <td>Y</td>
      <td></td>
      <td></td>
      <td><a href="https://doi.org/10.1017/ice.2020.61" target="_blank">10.1017/ice.2020.61</a></td>
      <td>47goscew</td>
      <td>0.300667</td>
      <td><a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a>, <a href="Topic_04.html#9ud7y3mq">Mack_2007</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a></td>
      <td><a href="Topic_17.html#52nw9gxq">Martin_2020</a>, <a href="Topic_13.html#ap92lgv7">dos_Santos_2018</a>, <a href="Topic_22.html#ong6f9al">Kuan_2012</a></td>
    </tr>
    <tr>
      <th id="t6j2qwdi";>573</th>
      <td>Ó’Fágáin_2010</td>
      <td>Gel-Filtration Chromatography</td>
      <td>Ó’Fágáin, Ciarán; Cummins, Philip M.; O’Connor, Brendan F.</td>
      <td>2010</td>
      <td>2010-06-24</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7120005" target="_blank">PMC7120005</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20978958.0" target="_blank">20978958.0</a></td>
      <td><a href="https://doi.org/10.1007/978-1-60761-913-0_2" target="_blank">10.1007/978-1-60761-913-0_2</a></td>
      <td>t6j2qwdi</td>
      <td>0.300445</td>
      <td><a href="Topic_17.html#fvuwick5">Qi_Z_2018</a>, <a href="Topic_17.html#2lg8i9r1">Dai_Q_2012</a>, <a href="Topic_17.html#clagd6rh">Sen_D_2018</a>, <a href="Topic_58.html#ejub0f3e">Ó’Fágáin_2016</a></td>
      <td><a href="Topic_58.html#ejub0f3e">Ó’Fágáin_2016</a>, <a href="Topic_58.html#9pkg7mbh">Regnier_1987</a>, <a href="Topic_58.html#gcyzg4ss">Mant_2007</a></td>
    </tr>
    <tr>
      <th id="1lojd0xa";>574</th>
      <td>Gao_J_2010</td>
      <td>Inverse folding of RNA pseudoknot structures</td>
      <td>Gao, James ZM; Li, Linda YM; Reidys, Christian M</td>
      <td>2010</td>
      <td>2010-06-23</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2909241" target="_blank">PMC2909241</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20573197.0" target="_blank">20573197.0</a></td>
      <td><a href="https://doi.org/10.1186/1748-7188-5-27" target="_blank">10.1186/1748-7188-5-27</a></td>
      <td>1lojd0xa</td>
      <td>0.300096</td>
      <td><a href="Topic_61.html#9zm4per4">Dawson_2007</a>, <a href="Topic_61.html#0brhn8oc">Staple_2005</a>, <a href="Topic_07.html#1ef1cxc8">Chang_2011</a>, <a href="Topic_61.html#a5htfx76">Dost_2006</a>, <a href="Topic_17.html#fvuwick5">Qi_Z_2018</a></td>
      <td></td>
    </tr>
    <tr>
      <th id="aebmrvea";>575</th>
      <td>Davis_2007</td>
      <td>Assessing the Certainty of Locations Produced by an Address Geocoding System</td>
      <td>Davis, Clodoveu A.; Fonseca, Frederico T.</td>
      <td>2007</td>
      <td>2007-01-13</td>
      <td>PMC</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7087595" target="_blank">PMC7087595</a></td>
      <td></td>
      <td><a href="https://doi.org/10.1007/s10707-006-0015-7" target="_blank">10.1007/s10707-006-0015-7</a></td>
      <td>aebmrvea</td>
      <td>0.300070</td>
      <td><a href="Topic_02.html#hhl47b25">Mooney_2013</a>, <a href="Topic_04.html#udqsus2a">Coker_2007</a>, <a href="Topic_54.html#vgg306kx">Nelson_2012</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a>, <a href="Topic_17.html#5ge7ozpd">Zheng_2020</a></td>
      <td><a href="Topic_17.html#eydkfqi2">Feng_2019</a>, <a href="Topic_39.html#01tncjq0">STRESMAN_2014</a>, <a href="Topic_17.html#ovuofl9q">Chen_2018</a>, <a href="Topic_04.html#f4q1r3vu">Keller_2009</a></td>
    </tr>
    <tr>
      <th id="1go3jjeu";>576</th>
      <td>Park_2018</td>
      <td>Digital Epidemiology: Use of Digital Data Collected for Non-epidemiological Purposes in Epidemiological Studies</td>
      <td>Park, Hyeoun-Ae; Jung, Hyesil; On, Jeongah; Park, Seul Ki; Kang, Hannah</td>
      <td>2018</td>
      <td>2018-10-31</td>
      <td>NONCOMM</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6230537" target="_blank">PMC6230537</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30443413.0" target="_blank">30443413.0</a></td>
      <td><a href="https://doi.org/10.4258/hir.2018.24.4.253" target="_blank">10.4258/hir.2018.24.4.253</a></td>
      <td>1go3jjeu</td>
      <td>0.300067</td>
      <td><a href="Topic_17.html#7my2q928">Largo_2016</a>, <a href="Topic_06.html#vs12l0uv">Choudhury_2011</a>, <a href="Topic_19.html#tgyxiwuo">Abou-Hamdan_2018</a>, <a href="Topic_60.html#xdihvg6k">David_2013</a></td>
      <td><a href="Topic_54.html#gap5rs40">Hoang_2019</a>, <a href="Topic_28.html#upl3ny5a">Sridhar_2016</a>, <a href="Topic_04.html#1kyrptgx">Samaras_2020</a>, <a href="Topic_28.html#0vecbxny">Cabello_2020</a></td>
    </tr>
    <tr>
      <th id="7izahzqy";>577</th>
      <td>Rochefort_2015</td>
      <td>Accuracy of using automated methods for detecting adverse events from electronic health record data: a research protocol</td>
      <td>Rochefort, Christian M; Buckeridge, David L; Forster, Alan J</td>
      <td>2015</td>
      <td>2015-01-08</td>
      <td>COMM-USE</td>
      <td>N</td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4296680" target="_blank">PMC4296680</a></td>
      <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25567422.0" target="_blank">25567422.0</a></td>
      <td><a href="https://doi.org/10.1186/s13012-014-0197-6" target="_blank">10.1186/s13012-014-0197-6</a></td>
      <td>7izahzqy</td>
      <td>0.300058</td>
      <td><a href="Topic_11.html#h6r6qdna">Polverino_2017</a>, <a href="Topic_04.html#baby6742">Knapp_2008</a>, <a href="Topic_04.html#f4x85a3g">Mwabu_2008</a>, <a href="Topic_24.html#evz3gwac">Amirov_2007</a>, <a href="Topic_54.html#2ubh9u8r">Nelson_2012</a></td>
      <td><a href="Topic_22.html#twn07poq">Nikolay_2017</a>, <a href="Topic_04.html#j7kb4kvu">Alroy_2020</a></td>
    </tr>
  </tbody>
</table>
</body>
</html>
